{
 "cells": [
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "<img src=\"https://www.ikiam.edu.ec/img/logo-ikiam-grey.png\" width=300 height=240 />\n",
    "\n",
    "# EXAMEN PARCIAL PYTHON\n",
    "    \n",
    "\n",
    "## GBI6-2021II: BIOINFORMÁTICA\n",
    "\n",
    "**MAMALLACTA GREFA, NAYELI GLADYS**\n",
    "\n",
    "**18-02-2021**"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "<font color='red'>NO AGREGUE NUEVAS CELDAS, UTILICE ÚNICAMENTE LAS QUE ESTÁN CREADAS</font>\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### REQUERIMIENTOS PARA EL EXAMEN\n",
    "Para este examen se requiere dos documentos: \n",
    "\n",
    "1. Archivo ```miningscience.py``` donde tendrá dos funciones:\n",
    "\n",
    "2. Archivo ```2021II_GBI6_ExamenParcialPython``` donde se llamará las funciones y se obtendrá resultados."
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Ejercicio 1 [2 puntos]\n",
    "Cree el archivo ```miningscience.py``` con las siguientes dos funciones:\n",
    "    \n",
    "i. ```download_pubmed```: para descargar la data de PubMed utilizando el **ENTREZ** de Biopython. El parámetro de entrada para la función es el ```keyword```. \n",
    "    \n",
    "ii. ```mining_pubs```: esta función debe utilizar el módulo **re** y utilizar el párametro ```tipo``` para realizar lo siguiente:\n",
    "        \n",
    "* Si el tipo es \"DP\" recupera el año de publicación del artículo. El retorno es un *dataframe* con el **PMID** y el **DP_year**.\n",
    "      \n",
    "* Si el tipo es \"AU\" recupera el número de autores por **PMID**. El retorno es un *dataframe* con el **PMID** y el **num_auth**. \n",
    "      \n",
    "* Si el tipo es \"AD\" recupera el conteo de autores por país. El retorno es un *dataframe* con el **country** y el **num_auth**. \n",
    "\n",
    "* *Puede utilizar un ```if``` dentro de la función.* \n",
    "\n",
    "* *Cree un docstring para cada función.* \n",
    "\n",
    "Luego de crear las funciones, cargue el módulo ```miningscience``` como ```msc``` e imprima docstring de cada función.\n",
    "\n",
    "\n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 1,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "55\n"
     ]
    },
    {
     "data": {
      "text/plain": [
       "'\\nPMID- 35120121\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20220224\\nLR  - 20220224\\nIS  - 1553-7404 (Electronic)\\nIS  - 1553-7390 (Linking)\\nVI  - 18\\nIP  - 2\\nDP  - 2022 Feb\\nTI  - Population genomics and geographic dispersal in Chagas disease vectors: Landscape\\n      drivers and evidence of possible adaptation to the domestic setting.\\nPG  - e1010019\\nLID - 10.1371/journal.pgen.1010019 [doi]\\nAB  - Accurate prediction of vectors dispersal, as well as identification of\\n      adaptations that allow blood-feeding vectors to thrive in built environments, are\\n      a basis for effective disease control. Here we adopted a landscape genomics\\n      approach to assay gene flow, possible local adaptation, and drivers of population\\n      structure in Rhodnius ecuadoriensis, an important vector of Chagas disease. We\\n      used a reduced-representation sequencing technique (2b-RADseq) to obtain 2,552\\n      SNP markers across 272 R. ecuadoriensis samples from 25 collection sites in\\n      southern Ecuador. Evidence of high and directional gene flow between seven wild\\n      and domestic population pairs across our study site indicates insecticide-based\\n      control will be hindered by repeated re-infestation of houses from the forest.\\n      Preliminary genome scans across multiple population pairs revealed shared outlier\\n      loci potentially consistent with local adaptation to the domestic setting, which \\n      we mapped to genes involved with embryogenesis and saliva production. Landscape\\n      genomic models showed elevation is a key barrier to R. ecuadoriensis dispersal.\\n      Together our results shed early light on the genomic adaptation in triatomine\\n      vectors and facilitate vector control by predicting that spatially-targeted,\\n      proactive interventions would be more efficacious than current, reactive\\n      approaches.\\nFAU - Hernandez-Castro, Luis E\\nAU  - Hernandez-Castro LE\\nAUID- ORCID: 0000-0002-2342-1655\\nAD  - Institute of Biodiversity, Animal Health and Comparative Medicine, University of \\n      Glasgow, Glasgow, United Kingdom.\\nAD  - The Epidemiology, Economics and Risk Assessment Group, The Roslin Institute,\\n      Easter Bush Campus, The University of Edinburgh, Midlothian, United Kingdom.\\nFAU - Villacis, Anita G\\nAU  - Villacis AG\\nAD  - Centro de Investigacion para la Salud en America Latina, Facultad de Ciencias\\n      Exactas y Naturales, Pontificia Universidad Catolica del Ecuador, Quito, Ecuador.\\nFAU - Jacobs, Arne\\nAU  - Jacobs A\\nAUID- ORCID: 0000-0001-7635-5447\\nAD  - Institute of Biodiversity, Animal Health and Comparative Medicine, University of \\n      Glasgow, Glasgow, United Kingdom.\\nAD  - Department of Natural Resources and the Environment, Cornell University, Ithaca, \\n      New York, United States of America.\\nFAU - Cheaib, Bachar\\nAU  - Cheaib B\\nAUID- ORCID: 0000-0002-0325-823X\\nAD  - Institute of Biodiversity, Animal Health and Comparative Medicine, University of \\n      Glasgow, Glasgow, United Kingdom.\\nFAU - Day, Casey C\\nAU  - Day CC\\nAUID- ORCID: 0000-0001-6096-3448\\nAD  - Computational Ecology Lab, School of Public and Community Health Sciences,\\n      University of Montana, Missoula, Montana, United States of America.\\nFAU - Ocana-Mayorga, Sofia\\nAU  - Ocana-Mayorga S\\nAUID- ORCID: 0000-0003-3861-9784\\nAD  - Centro de Investigacion para la Salud en America Latina, Facultad de Ciencias\\n      Exactas y Naturales, Pontificia Universidad Catolica del Ecuador, Quito, Ecuador.\\nFAU - Yumiseva, Cesar A\\nAU  - Yumiseva CA\\nAD  - Centro de Investigacion para la Salud en America Latina, Facultad de Ciencias\\n      Exactas y Naturales, Pontificia Universidad Catolica del Ecuador, Quito, Ecuador.\\nFAU - Bacigalupo, Antonella\\nAU  - Bacigalupo A\\nAUID- ORCID: 0000-0003-1661-8916\\nAD  - Institute of Biodiversity, Animal Health and Comparative Medicine, University of \\n      Glasgow, Glasgow, United Kingdom.\\nFAU - Andersson, Bjorn\\nAU  - Andersson B\\nAUID- ORCID: 0000-0002-4624-0259\\nAD  - Department of Cell and Molecular Biology, Karolinska Institutet, Stockholm,\\n      Sweden.\\nFAU - Matthews, Louise\\nAU  - Matthews L\\nAUID- ORCID: 0000-0003-4420-8367\\nAD  - Institute of Biodiversity, Animal Health and Comparative Medicine, University of \\n      Glasgow, Glasgow, United Kingdom.\\nFAU - Landguth, Erin L\\nAU  - Landguth EL\\nAUID- ORCID: 0000-0002-7044-346X\\nAD  - Computational Ecology Lab, School of Public and Community Health Sciences,\\n      University of Montana, Missoula, Montana, United States of America.\\nAD  - Center for Population Health Research, School of Public and Community Health\\n      Sciences, University of Montana, Missoula, Montana, United States of America.\\nFAU - Costales, Jaime A\\nAU  - Costales JA\\nAUID- ORCID: 0000-0001-9418-6640\\nAD  - Centro de Investigacion para la Salud en America Latina, Facultad de Ciencias\\n      Exactas y Naturales, Pontificia Universidad Catolica del Ecuador, Quito, Ecuador.\\nFAU - Llewellyn, Martin S\\nAU  - Llewellyn MS\\nAUID- ORCID: 0000-0001-9856-1591\\nAD  - Institute of Biodiversity, Animal Health and Comparative Medicine, University of \\n      Glasgow, Glasgow, United Kingdom.\\nFAU - Grijalva, Mario J\\nAU  - Grijalva MJ\\nAUID- ORCID: 0000-0003-1964-1425\\nAD  - Centro de Investigacion para la Salud en America Latina, Facultad de Ciencias\\n      Exactas y Naturales, Pontificia Universidad Catolica del Ecuador, Quito, Ecuador.\\nAD  - Infectious and Tropical Disease Institute, Department of Biomedical Sciences,\\n      Heritage College of Osteopathic Medicine, Ohio University, Athens, Ohio, United\\n      States of America.\\nLA  - eng\\nGR  - P20 GM130418/GM/NIGMS NIH HHS/United States\\nGR  - R15 AI105749/AI/NIAID NIH HHS/United States\\nPT  - Journal Article\\nPT  - Research Support, N.I.H., Extramural\\nPT  - Research Support, Non-U.S. Gov\\'t\\nDEP - 20220204\\nPL  - United States\\nTA  - PLoS Genet\\nJT  - PLoS genetics\\nJID - 101239074\\nSB  - IM\\nMH  - Adaptation, Biological/genetics\\nMH  - Animals\\nMH  - Chagas Disease/*epidemiology/*genetics\\nMH  - Disease Vectors\\nMH  - Ecosystem\\nMH  - Ecuador/epidemiology\\nMH  - Gene Expression/genetics\\nMH  - Gene Expression Profiling/methods\\nMH  - Gene Flow\\nMH  - Insect Vectors/genetics\\nMH  - Metagenomics/methods\\nMH  - Polymorphism, Single Nucleotide/genetics\\nMH  - Population Density\\nMH  - Rhodnius/*genetics/pathogenicity\\nMH  - Transcriptome/genetics\\nMH  - Trypanosoma cruzi/genetics\\nPMC - PMC8849464\\nCOIS- The authors have declared that no competing interests exist.\\nEDAT- 2022/02/05 06:00\\nMHDA- 2022/02/25 06:00\\nCRDT- 2022/02/04 17:11\\nPHST- 2021/07/12 00:00 [received]\\nPHST- 2022/01/06 00:00 [accepted]\\nPHST- 2022/02/16 00:00 [revised]\\nPHST- 2022/02/05 06:00 [pubmed]\\nPHST- 2022/02/25 06:00 [medline]\\nPHST- 2022/02/04 17:11 [entrez]\\nAID - 10.1371/journal.pgen.1010019 [doi]\\nAID - PGENETICS-D-21-00937 [pii]\\nPST - epublish\\nSO  - PLoS Genet. 2022 Feb 4;18(2):e1010019. doi: 10.1371/journal.pgen.1010019.\\n      eCollection 2022 Feb.\\n\\nPMID- 35064679\\nOWN - NLM\\nSTAT- Publisher\\nLR  - 20220208\\nIS  - 1469-8137 (Electronic)\\nIS  - 0028-646X (Linking)\\nDP  - 2022 Jan 22\\nTI  - Discovery and characterization of sweetpotato\\'s closest tetraploid relative.\\nLID - 10.1111/nph.17991 [doi]\\nAB  - The origin of sweetpotato, a hexaploid species, is poorly understood, partly\\n      because the identity of its tetraploid progenitor remains unknown. In this study,\\n      we identify, describe and characterize a new species of Ipomoea that is\\n      sweetpotato\\'s closest tetraploid relative known to date and probably a direct\\n      descendant of its tetraploid progenitor. We integrate morphological,\\n      phylogenetic, and genomic analyses of herbarium and germplasm accessions of the\\n      hexaploid sweetpotato, its closest known diploid relative Ipomoea trifida, and\\n      various tetraploid plants closely related to them from across the American\\n      continent. We identify wild autotetraploid plants from Ecuador that are\\n      morphologically distinct from Ipomoea batatas and I. trifida, but monophyletic\\n      and sister to I. batatas in phylogenetic analysis of nuclear data. We describe\\n      this new species as Ipomoea aequatoriensis T. Wells & P. Munoz sp. nov.,\\n      distinguish it from hybrid tetraploid material collected in Mexico; and show that\\n      it likely played a direct role in the origin of sweetpotato\\'s hexaploid genome.\\n      This discovery transforms our understanding of sweetpotato\\'s origin.\\nCI  - (c) 2022 The Authors. New Phytologist (c) 2022 New Phytologist Foundation.\\nFAU - Munoz-Rodriguez, Pablo\\nAU  - Munoz-Rodriguez P\\nAUID- ORCID: https://orcid.org/0000-0002-3580-8136\\nAD  - Department of Plant Sciences, University of Oxford, South Parks Road, Oxford, OX1\\n      3RB, UK.\\nFAU - Wells, Tom\\nAU  - Wells T\\nAUID- ORCID: https://orcid.org/0000-0002-4664-7868\\nAD  - Department of Plant Sciences, University of Oxford, South Parks Road, Oxford, OX1\\n      3RB, UK.\\nFAU - Wood, John R I\\nAU  - Wood JRI\\nAUID- ORCID: https://orcid.org/0000-0001-5102-3729\\nAD  - Department of Plant Sciences, University of Oxford, South Parks Road, Oxford, OX1\\n      3RB, UK.\\nAD  - Royal Botanic Gardens, Kew, Richmond, Surrey, TW9 3AB, UK.\\nFAU - Carruthers, Tom\\nAU  - Carruthers T\\nAUID- ORCID: https://orcid.org/0000-0003-1586-3557\\nAD  - Royal Botanic Gardens, Kew, Richmond, Surrey, TW9 3AB, UK.\\nFAU - Anglin, Noelle L\\nAU  - Anglin NL\\nAUID- ORCID: https://orcid.org/0000-0002-3454-1142\\nAD  - International Potato Center, Avenida La Molina 1895, Distrito de La Molina, Lima,\\n      15023, Peru.\\nAD  - United States Department of Agriculture, 1109 Experiment Street, Griffin, GA,\\n      3022, USA.\\nFAU - Jarret, Robert L\\nAU  - Jarret RL\\nAUID- ORCID: https://orcid.org/0000-0002-0426-6186\\nAD  - United States Department of Agriculture, 1109 Experiment Street, Griffin, GA,\\n      3022, USA.\\nFAU - Scotland, Robert W\\nAU  - Scotland RW\\nAUID- ORCID: https://orcid.org/0000-0002-6371-2238\\nAD  - Department of Plant Sciences, University of Oxford, South Parks Road, Oxford, OX1\\n      3RB, UK.\\nLA  - eng\\nGR  - T001445/1/BB_/Biotechnology and Biological Sciences Research Council/United\\n      Kingdom\\nPT  - Journal Article\\nDEP - 20220122\\nPL  - England\\nTA  - New Phytol\\nJT  - The New phytologist\\nJID - 9882884\\nSB  - IM\\nOTO - NOTNLM\\nOT  - Ipomoea aequatoriensis\\nOT  - Ecuador\\nOT  - crop wild relatives\\nOT  - genomics\\nOT  - herbarium specimens\\nOT  - new species\\nOT  - tetraploid\\nEDAT- 2022/01/23 06:00\\nMHDA- 2022/01/23 06:00\\nCRDT- 2022/01/22 08:36\\nPHST- 2021/11/23 00:00 [received]\\nPHST- 2022/01/16 00:00 [accepted]\\nPHST- 2022/01/23 06:00 [pubmed]\\nPHST- 2022/01/23 06:00 [medline]\\nPHST- 2022/01/22 08:36 [entrez]\\nAID - 10.1111/nph.17991 [doi]\\nPST - aheadofprint\\nSO  - New Phytol. 2022 Jan 22. doi: 10.1111/nph.17991.\\n\\nPMID- 35047841\\nOWN - NLM\\nSTAT- PubMed-not-MEDLINE\\nLR  - 20220121\\nIS  - 2666-2477 (Electronic)\\nIS  - 2666-2477 (Linking)\\nVI  - 2\\nIP  - 4\\nDP  - 2021 Oct 14\\nTI  - Genetic ancestry and ethnic identity in Ecuador.\\nPG  - 100050\\nLID - 10.1016/j.xhgg.2021.100050 [doi]\\nAB  - We investigated the ancestral origins of four Ecuadorian ethnic\\n      groups-Afro-Ecuadorian, Mestizo, Montubio, and the Indigenous Tsachila-in an\\n      effort to gain insight on the relationship between ancestry, culture, and the\\n      formation of ethnic identities in Latin America. The observed patterns of genetic\\n      ancestry are largely concordant with ethnic identities and historical records of \\n      conquest and colonization in Ecuador. Nevertheless, a number of exceptional\\n      findings highlight the complex relationship between genetic ancestry and\\n      ethnicity in Ecuador. Afro-Ecuadorians show far less African ancestry, and the\\n      highest levels of Native American ancestry, seen for any Afro-descendant\\n      population in the Americas. Mestizos in Ecuador show high levels of Native\\n      American ancestry, with substantially less European ancestry, despite the\\n      relatively low Indigenous population in the country. The recently recognized\\n      Montubio ethnic group is highly admixed, with substantial contributions from all \\n      three continental ancestries. The Tsachila show two distinct ancestry subgroups, \\n      with most individuals showing almost exclusively Native American ancestry and a\\n      smaller group showing a Mestizo characteristic pattern. Considered together with \\n      historical data and sociological studies, our results indicate the extent to\\n      which ancestry and culture interact, often in unexpected ways, to shape ethnic\\n      identity in Ecuador.\\nCI  - (c) 2021 The Authors.\\nFAU - Nagar, Shashwat Deepali\\nAU  - Nagar SD\\nAD  - School of Biological Sciences, Georgia Institute of Technology, Atlanta, GA, USA.\\nAD  - PanAmerican Bioinformatics Institute, Cali, Valle del Cauca, Colombia.\\nFAU - Conley, Andrew B\\nAU  - Conley AB\\nAD  - PanAmerican Bioinformatics Institute, Cali, Valle del Cauca, Colombia.\\nAD  - IHRC-Georgia Tech Applied Bioinformatics Laboratory, Atlanta, GA, USA.\\nAD  - National Institute on Minority Health and Health Disparities, National Institutes\\n      of Health, Bethesda, MD, USA.\\nFAU - Chande, Aroon T\\nAU  - Chande AT\\nAD  - School of Biological Sciences, Georgia Institute of Technology, Atlanta, GA, USA.\\nAD  - PanAmerican Bioinformatics Institute, Cali, Valle del Cauca, Colombia.\\nAD  - IHRC-Georgia Tech Applied Bioinformatics Laboratory, Atlanta, GA, USA.\\nFAU - Rishishwar, Lavanya\\nAU  - Rishishwar L\\nAD  - PanAmerican Bioinformatics Institute, Cali, Valle del Cauca, Colombia.\\nAD  - IHRC-Georgia Tech Applied Bioinformatics Laboratory, Atlanta, GA, USA.\\nAD  - National Institute on Minority Health and Health Disparities, National Institutes\\n      of Health, Bethesda, MD, USA.\\nFAU - Sharma, Shivam\\nAU  - Sharma S\\nAD  - National Institute on Minority Health and Health Disparities, National Institutes\\n      of Health, Bethesda, MD, USA.\\nFAU - Marino-Ramirez, Leonardo\\nAU  - Marino-Ramirez L\\nAD  - PanAmerican Bioinformatics Institute, Cali, Valle del Cauca, Colombia.\\nAD  - National Institute on Minority Health and Health Disparities, National Institutes\\n      of Health, Bethesda, MD, USA.\\nFAU - Aguinaga-Romero, Gabriela\\nAU  - Aguinaga-Romero G\\nAD  - Faculty of Medical Sciences, Central University of Ecuador, Quito, Ecuador.\\nFAU - Gonzalez-Andrade, Fabricio\\nAU  - Gonzalez-Andrade F\\nAD  - Faculty of Medical Sciences, Central University of Ecuador, Quito, Ecuador.\\nFAU - Jordan, I King\\nAU  - Jordan IK\\nAD  - School of Biological Sciences, Georgia Institute of Technology, Atlanta, GA, USA.\\nAD  - PanAmerican Bioinformatics Institute, Cali, Valle del Cauca, Colombia.\\nAD  - IHRC-Georgia Tech Applied Bioinformatics Laboratory, Atlanta, GA, USA.\\nLA  - eng\\nPT  - Journal Article\\nDEP - 20210820\\nPL  - United States\\nTA  - HGG Adv\\nJT  - HGG advances\\nJID - 101772885\\nPMC - PMC8756502\\nOTO - NOTNLM\\nOT  - Afro-Ecuadorian\\nOT  - Ecuador\\nOT  - Indigenous\\nOT  - Latin America\\nOT  - Mestizo\\nOT  - Montubio\\nOT  - ethnicity\\nOT  - genetic ancestry\\nOT  - genomics\\nOT  - population genetics\\nCOIS- The authors declare no competing interests.\\nEDAT- 2022/01/21 06:00\\nMHDA- 2022/01/21 06:01\\nCRDT- 2022/01/20 06:09\\nPHST- 2021/01/19 00:00 [received]\\nPHST- 2021/08/09 00:00 [accepted]\\nPHST- 2022/01/20 06:09 [entrez]\\nPHST- 2022/01/21 06:00 [pubmed]\\nPHST- 2022/01/21 06:01 [medline]\\nAID - 10.1016/j.xhgg.2021.100050 [doi]\\nAID - S2666-2477(21)00031-2 [pii]\\nPST - epublish\\nSO  - HGG Adv. 2021 Aug 20;2(4):100050. doi: 10.1016/j.xhgg.2021.100050. eCollection\\n      2021 Oct 14.\\n\\nPMID- 35018222\\nOWN - NLM\\nSTAT- PubMed-not-MEDLINE\\nLR  - 20220224\\nIS  - 2052-2975 (Print)\\nIS  - 2052-2975 (Linking)\\nVI  - 45\\nDP  - 2022 Jan\\nTI  - First detection of SARS-CoV-2 variant B.1.1.529 (Omicron) in Ecuador.\\nPG  - 100951\\nLID - 10.1016/j.nmni.2022.100951 [doi]\\nAB  - The National Institute of Research and Public Health reported the first local\\n      record of the Omicron variant detected in Ecuador. A fully vaccinated subject\\n      returned from South Africa with a negative RT-PCR. We present the cumulative\\n      frequency of the variants in Ecuador and a phylogenetic analysis of this new\\n      Omicron.\\nCI  - (c) 2022 The Authors.\\nFAU - Carrazco-Montalvo, A\\nAU  - Carrazco-Montalvo A\\nAD  - Direccion de Investigacion, Desarrollo e Innovacion, Instituto Nacional de\\n      Investigacion en Salud Publica \"Leopoldo Izquieta Perez\", Guayaquil, Ecuador.\\nFAU - Armendariz-Castillo, I\\nAU  - Armendariz-Castillo I\\nAD  - Coordinacion Zonal 9, Instituto Nacional de Investigacion en Salud Publica\\n      \"Leopoldo Izquieta Perez\", Quito, Ecuador.\\nFAU - Tello, C L\\nAU  - Tello CL\\nAD  - Direccion de Investigacion, Desarrollo e Innovacion, Instituto Nacional de\\n      Investigacion en Salud Publica \"Leopoldo Izquieta Perez\", Guayaquil, Ecuador.\\nFAU - Morales, D\\nAU  - Morales D\\nAD  - Centro de Referencia Nacional de Influenza y Otros Virus Respiratorios, Instituto\\n      Nacional de Investigacion en Salud Publica \"Leopoldo Izquieta Perez\", Guayaquil, \\n      Ecuador.\\nFAU - Armas-Gonzalez, R\\nAU  - Armas-Gonzalez R\\nAD  - Centro de Referencia Nacional de Influenza y Otros Virus Respiratorios, Instituto\\n      Nacional de Investigacion en Salud Publica \"Leopoldo Izquieta Perez\", Guayaquil, \\n      Ecuador.\\nFAU - Guizado-Herrera, D\\nAU  - Guizado-Herrera D\\nAD  - Direccion de Investigacion, Desarrollo e Innovacion, Instituto Nacional de\\n      Investigacion en Salud Publica \"Leopoldo Izquieta Perez\", Guayaquil, Ecuador.\\nFAU - Leon-Sosa, A\\nAU  - Leon-Sosa A\\nAD  - Direccion de Investigacion, Desarrollo e Innovacion, Instituto Nacional de\\n      Investigacion en Salud Publica \"Leopoldo Izquieta Perez\", Guayaquil, Ecuador.\\nFAU - Ramos-Sarmiento, D\\nAU  - Ramos-Sarmiento D\\nAD  - Centro de Referencia Nacional de Influenza y Otros Virus Respiratorios, Instituto\\n      Nacional de Investigacion en Salud Publica \"Leopoldo Izquieta Perez\", Guayaquil, \\n      Ecuador.\\nFAU - Fuertes, B\\nAU  - Fuertes B\\nAD  - Centro de Referencia Nacional de Influenza y Otros Virus Respiratorios, Instituto\\n      Nacional de Investigacion en Salud Publica \"Leopoldo Izquieta Perez\", Guayaquil, \\n      Ecuador.\\nCN  - USFQ-Consortium\\nFAU - Patino, L\\nAU  - Patino L\\nAD  - Direccion de Investigacion, Desarrollo e Innovacion, Instituto Nacional de\\n      Investigacion en Salud Publica \"Leopoldo Izquieta Perez\", Guayaquil, Ecuador.\\nLA  - eng\\nPT  - Case Reports\\nDEP - 20220107\\nPL  - England\\nTA  - New Microbes New Infect\\nJT  - New microbes and new infections\\nJID - 101624750\\nPMC - PMC8739794\\nOTO - NOTNLM\\nOT  - COVID-19\\nOT  - Omicron\\nOT  - SARS-CoV-2\\nOT  - genomics\\nOT  - variant of concern\\nIR  - Cardenas P\\nFIR - Cardenas, P\\nIR  - Marquez S\\nFIR - Marquez, S\\nIR  - Prado-Vivar B\\nFIR - Prado-Vivar, B\\nIR  - Guadalupe JJ\\nFIR - Guadalupe, J J\\nIR  - Gutierrez B\\nFIR - Gutierrez, B\\nIR  - Wong MB\\nFIR - Wong, M B\\nIR  - Grunauer M\\nFIR - Grunauer, M\\nIR  - Trueba G\\nFIR - Trueba, G\\nIR  - Rojas-Silva P\\nFIR - Rojas-Silva, P\\nIR  - Barragan V\\nFIR - Barragan, V\\nEDAT- 2022/01/13 06:00\\nMHDA- 2022/01/13 06:01\\nCRDT- 2022/01/12 06:32\\nPHST- 2021/12/29 00:00 [received]\\nPHST- 2022/01/04 00:00 [accepted]\\nPHST- 2022/01/13 06:00 [pubmed]\\nPHST- 2022/01/13 06:01 [medline]\\nPHST- 2022/01/12 06:32 [entrez]\\nAID - 10.1016/j.nmni.2022.100951 [doi]\\nAID - S2052-2975(22)00003-8 [pii]\\nPST - ppublish\\nSO  - New Microbes New Infect. 2022 Jan;45:100951. doi: 10.1016/j.nmni.2022.100951.\\n      Epub 2022 Jan 7.\\n\\nPMID- 34962985\\nOWN - NLM\\nSTAT- In-Process\\nLR  - 20220216\\nIS  - 1759-6653 (Electronic)\\nIS  - 1759-6653 (Linking)\\nVI  - 14\\nIP  - 1\\nDP  - 2022 Jan 4\\nTI  - De Novo Genome Assembly and Annotation of an Andean Caddisfly, Atopsyche\\n      davidsoni Sykora, 1991, a Model for Genome Research of High-Elevation\\n      Adaptations.\\nLID - evab286 [pii]\\nLID - 10.1093/gbe/evab286 [doi]\\nAB  - We sequence, assemble, and annotate the genome of Atopsyche davidsoni Sykora,\\n      1991, the first whole-genome assembly for the caddisfly family Hydrobiosidae.\\n      This free-living and predatory caddisfly inhabits streams in the high-elevation\\n      Andes and is separated by more than 200 Myr of evolutionary history from the most\\n      closely related caddisfly species with genome assemblies available. We\\n      demonstrate the promise of PacBio HiFi reads by assembling the most contiguous\\n      caddisfly genome assembly to date with a contig N50 of 14 Mb, which is more than \\n      6x more contiguous than the current most contiguous assembly for a caddisfly\\n      (Hydropsyche tenuis). We recover 98.8% of insect BUSCO genes indicating a high\\n      level of gene completeness. We also provide a genome annotation of 12,232\\n      annotated proteins. This new genome assembly provides an important new resource\\n      for studying genomic adaptation of aquatic insects to harsh, high-altitude\\n      environments.\\nCI  - (c) The Author(s) 2021. Published by Oxford University Press on behalf of the\\n      Society for Molecular Biology and Evolution.\\nFAU - Rios-Touma, Blanca\\nAU  - Rios-Touma B\\nAD  - Facultad de Ingenierias y Ciencias Aplicadas, Ingenieria Ambiental, Grupo de\\n      Investigacion en Biodiversidad, Medio Ambiente y Salud (BIOMAS), Universidad de\\n      las Americas, Quito, Ecuador.\\nFAU - Holzenthal, Ralph W\\nAU  - Holzenthal RW\\nAD  - Department of Entomology, University of Minnesota, St. Paul, Minnesota, USA.\\nFAU - Razuri-Gonzales, Ernesto\\nAU  - Razuri-Gonzales E\\nAD  - Department of Entomology, University of Minnesota, St. Paul, Minnesota, USA.\\nAD  - Department of Terrestrial Zoology, Entomology III, Senckenberg Research Institute\\n      and Natural History Museum Frankfurt, Germany.\\nFAU - Heckenhauer, Jacqueline\\nAU  - Heckenhauer J\\nAD  - Department of Terrestrial Zoology, Entomology III, Senckenberg Research Institute\\n      and Natural History Museum Frankfurt, Germany.\\nAD  - LOEWE Centre for Translational Biodiversity Genomics (LOEWE-TBG), Frankfurt,\\n      Germany.\\nFAU - Pauls, Steffen U\\nAU  - Pauls SU\\nAD  - Department of Terrestrial Zoology, Entomology III, Senckenberg Research Institute\\n      and Natural History Museum Frankfurt, Germany.\\nAD  - LOEWE Centre for Translational Biodiversity Genomics (LOEWE-TBG), Frankfurt,\\n      Germany.\\nAD  - Institute of Insect Biotechnology, Justus-Liebig University, Giessen, Germany.\\nFAU - Storer, Caroline G\\nAU  - Storer CG\\nAD  - McGuire Center for Lepidoptera and Biodiversity, Florida Museum of Natural\\n      History, University of Florida, Gainesville, Florida, USA.\\nFAU - Frandsen, Paul B\\nAU  - Frandsen PB\\nAUID- ORCID: 0000-0002-4801-7579\\nAD  - LOEWE Centre for Translational Biodiversity Genomics (LOEWE-TBG), Frankfurt,\\n      Germany.\\nAD  - Department of Plant and Wildlife Sciences, Brigham Young University, Provo, Utah,\\n      USA.\\nLA  - eng\\nSI  - figshare/10.6084/m9.figshare.16836871.v1\\nSI  - figshare/10.6084/m9.figshare.16837963.v1\\nPT  - Journal Article\\nPT  - Research Support, Non-U.S. Gov\\'t\\nPL  - England\\nTA  - Genome Biol Evol\\nJT  - Genome biology and evolution\\nJID - 101509707\\nSB  - IM\\nPMC - PMC8767365\\nOTO - NOTNLM\\nOT  - *Insecta\\nOT  - *PacBio HiFi\\nOT  - *Trichoptera\\nOT  - *caddisfly genomics\\nOT  - *extreme environments\\nOT  - *high altitude\\nEDAT- 2021/12/29 06:00\\nMHDA- 2021/12/29 06:00\\nCRDT- 2021/12/28 17:18\\nPHST- 2021/12/20 00:00 [accepted]\\nPHST- 2021/12/29 06:00 [pubmed]\\nPHST- 2021/12/29 06:00 [medline]\\nPHST- 2021/12/28 17:18 [entrez]\\nAID - 6486422 [pii]\\nAID - 10.1093/gbe/evab286 [doi]\\nPST - ppublish\\nSO  - Genome Biol Evol. 2022 Jan 4;14(1). pii: 6486422. doi: 10.1093/gbe/evab286.\\n\\nPMID- 34878103\\nOWN - NLM\\nSTAT- In-Data-Review\\nLR  - 20220209\\nIS  - 2160-1836 (Electronic)\\nIS  - 2160-1836 (Linking)\\nVI  - 12\\nIP  - 2\\nDP  - 2022 Feb 4\\nTI  - The genome sequence of the avian vampire fly (Philornis downsi), an invasive nest\\n      parasite of Darwin\\'s finches in Galapagos.\\nLID - jkab414 [pii]\\nLID - 10.1093/g3journal/jkab414 [doi]\\nAB  - The invasive avian vampire fly (Philornis downsi, Diptera: Muscidae) is\\n      considered one of the greatest threats to the endemic avifauna of the Galapagos\\n      Islands. The fly larvae parasitize nearly every passerine species, including\\n      Darwin\\'s finches. Most P. downsi research to date has focused on the effects of\\n      the fly on avian host fitness and mitigation methods. A lag in research related\\n      to the genetics of this invasion demonstrates, in part, the need to develop\\n      full-scale genomic resources with which to address further questions within this \\n      system. In this study, an adult female P. downsi was sequenced to generate a\\n      high-quality genome assembly. We examined various features of the genome (e.g.,\\n      coding regions and noncoding transposable elements) and carried out comparative\\n      genomics analysis against other dipteran genomes. We identified lists of gene\\n      families that are significantly expanding or contracting in P. downsi that are\\n      related to insecticide resistance, detoxification, and counter defense against\\n      host immune responses. The P. downsi genome assembly provides an important\\n      resource for studying the molecular basis of successful invasion in the Galapagos\\n      and the dynamics of its population across multiple islands. The findings of\\n      significantly changing gene families associated with insecticide resistance and\\n      immune responses highlight the need for further investigations into the role of\\n      different gene families in aiding the fly\\'s successful invasion. Furthermore,\\n      this genomic resource provides a necessary tool to better inform future research \\n      studies and mitigation strategies aimed at minimizing the fly\\'s impact on\\n      Galapagos birds.\\nCI  - (c) The Author(s) 2021. Published by Oxford University Press on behalf of\\n      Genetics Society of America.\\nFAU - Romine, Melia G\\nAU  - Romine MG\\nAD  - School of Biomedical Sciences, Kent State University, Kent, OH 44240, USA.\\nFAU - Knutie, Sarah A\\nAU  - Knutie SA\\nAD  - Department of Ecology and Evolutionary Biology, University of Connecticut,\\n      Storrs, CT 06269, USA.\\nAD  - Institute for Systems Genomics, University of Connecticut, Storrs, CT 06269, USA.\\nFAU - Crow, Carly M\\nAU  - Crow CM\\nAD  - Department of Biological Sciences, Northern Illinois University, DeKalb, IL\\n      60115, USA.\\nFAU - Vaziri, Grace J\\nAU  - Vaziri GJ\\nAD  - Department of Ecology and Evolutionary Biology, University of Connecticut,\\n      Storrs, CT 06269, USA.\\nFAU - Chaves, Jaime A\\nAU  - Chaves JA\\nAD  - Department of Biology, San Francisco State University, San Francisco, CA 94132,\\n      USA.\\nAD  - Colegio de Ciencias Biologicas y Ambientales, Universidad San Francisco de Quito,\\n      Quito 170901, Ecuador.\\nFAU - Koop, Jennifer A H\\nAU  - Koop JAH\\nAD  - Department of Biological Sciences, Northern Illinois University, DeKalb, IL\\n      60115, USA.\\nFAU - Lamichhaney, Sangeet\\nAU  - Lamichhaney S\\nAUID- ORCID: 0000-0003-4826-0349\\nAD  - School of Biomedical Sciences, Kent State University, Kent, OH 44240, USA.\\nAD  - Department of Biological Sciences, Kent State University, Kent, OH 44240, USA.\\nLA  - eng\\nGR  - Department of Biological Sciences, Kent State University\\nGR  - Northern Illinois University Division of Research and Innovation Partnerships\\nGR  - University Library\\nGR  - University of Connecticut\\nPT  - Journal Article\\nPL  - England\\nTA  - G3 (Bethesda)\\nJT  - G3 (Bethesda, Md.)\\nJID - 101566598\\nSB  - IM\\nOTO - NOTNLM\\nOT  - Darwin\\'s finches\\nOT  - Galapagos\\nOT  - avian vampire fly genome\\nOT  - insecticide resistance\\nOT  - parasitic invasion\\nEDAT- 2021/12/09 06:00\\nMHDA- 2021/12/09 06:00\\nCRDT- 2021/12/08 09:20\\nPHST- 2021/10/06 00:00 [received]\\nPHST- 2021/11/15 00:00 [accepted]\\nPHST- 2021/12/09 06:00 [pubmed]\\nPHST- 2021/12/09 06:00 [medline]\\nPHST- 2021/12/08 09:20 [entrez]\\nAID - 6456303 [pii]\\nAID - 10.1093/g3journal/jkab414 [doi]\\nPST - ppublish\\nSO  - G3 (Bethesda). 2022 Feb 4;12(2). pii: 6456303. doi: 10.1093/g3journal/jkab414.\\n\\nPMID- 34233044\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20220128\\nLR  - 20220128\\nIS  - 1365-294X (Electronic)\\nIS  - 0962-1083 (Linking)\\nVI  - 30\\nIP  - 23\\nDP  - 2021 Dec\\nTI  - Genomics of altitude-associated wing shape in two tropical butterflies.\\nPG  - 6387-6402\\nLID - 10.1111/mec.16067 [doi]\\nAB  - Understanding how organisms adapt to their local environment is central to\\n      evolution. With new whole-genome sequencing technologies and the explosion of\\n      data, deciphering the genomic basis of complex traits that are ecologically\\n      relevant is becoming increasingly feasible. Here, we studied the genomic basis of\\n      wing shape in two Neotropical butterflies that inhabit large geographical ranges.\\n      Heliconius butterflies at high elevations have been shown to generally have\\n      rounder wings than those in the lowlands. We reared over 1,100 butterflies from\\n      71 broods of H. erato and H. melpomene in common-garden conditions and showed\\n      that wing aspect ratio, that is, elongatedness, is highly heritable in both\\n      species and that elevation-associated wing aspect ratio differences are\\n      maintained. Genome-wide associations with a published data set of 666 whole\\n      genomes from across a hybrid zone, uncovered a highly polygenic basis to wing\\n      aspect ratio variation in the wild. We identified several genes that have roles\\n      in wing morphogenesis or wing aspect ratio variation in Drosophila flies, making \\n      them promising candidates for future studies. There was little evidence for\\n      molecular parallelism in the two species, with only one shared candidate gene,\\n      nor for a role of the four known colour pattern loci, except for optix in H.\\n      erato. Thus, we present the first insights into the heritability and genomic\\n      basis of within-species wing aspect ratio in two Heliconius species, adding to a \\n      growing body of evidence that polygenic adaptation may underlie many ecologically\\n      relevant traits.\\nCI  - (c) 2021 The Authors. Molecular Ecology published by John Wiley & Sons Ltd.\\nFAU - Montejo-Kovacevich, Gabriela\\nAU  - Montejo-Kovacevich G\\nAUID- ORCID: 0000-0003-3716-9929\\nAD  - Department of Zoology, University of Cambridge, Cambridge, UK.\\nFAU - Salazar, Patricio A\\nAU  - Salazar PA\\nAUID- ORCID: 0000-0001-8988-0769\\nAD  - Animal and Plant Sciences, University of Sheffield, Sheffield, UK.\\nFAU - Smith, Sophie H\\nAU  - Smith SH\\nAUID- ORCID: 0000-0001-9530-3896\\nAD  - Animal and Plant Sciences, University of Sheffield, Sheffield, UK.\\nFAU - Gavilanes, Kimberly\\nAU  - Gavilanes K\\nAD  - Universidad Regional Amazonica de Ikiam, Tena, Ecuador.\\nFAU - Bacquet, Caroline N\\nAU  - Bacquet CN\\nAUID- ORCID: 0000-0002-1954-1806\\nAD  - Universidad Regional Amazonica de Ikiam, Tena, Ecuador.\\nFAU - Chan, Yingguang Frank\\nAU  - Chan YF\\nAUID- ORCID: 0000-0001-6292-9681\\nAD  - Friedrich Miescher Laboratory of the Max Planck Society, Tubingen, Germany.\\nFAU - Jiggins, Chris D\\nAU  - Jiggins CD\\nAUID- ORCID: 0000-0002-7809-062X\\nAD  - Department of Zoology, University of Cambridge, Cambridge, UK.\\nFAU - Meier, Joana I\\nAU  - Meier JI\\nAUID- ORCID: 0000-0001-7726-2875\\nAD  - Department of Zoology, University of Cambridge, Cambridge, UK.\\nAD  - St John\\'s College, University of Cambridge, Cambridge, UK.\\nFAU - Nadeau, Nicola J\\nAU  - Nadeau NJ\\nAUID- ORCID: 0000-0002-9319-921X\\nAD  - Animal and Plant Sciences, University of Sheffield, Sheffield, UK.\\nLA  - eng\\nGR  - 639096 \"HybridMiX\" and 339873 \"SpeciationGenetics\"/European Research Council\\n      Starting Grant\\nGR  - 2018SPE0000400194/Agencia Espanola de Cooperacion Internacional para el\\n      Desarrollo\\nGR  - NE/L002507/1/Natural Environment Research Council\\nGR  - NE/R010331/1/Natural Environment Research Council\\nPT  - Journal Article\\nPT  - Research Support, Non-U.S. Gov\\'t\\nDEP - 20210728\\nPL  - England\\nTA  - Mol Ecol\\nJT  - Molecular ecology\\nJID - 9214478\\nSB  - IM\\nMH  - Altitude\\nMH  - Animals\\nMH  - *Butterflies/genetics\\nMH  - Female\\nMH  - Genomics\\nMH  - Phenotype\\nMH  - Pigmentation\\nMH  - Pregnancy\\nMH  - Wings, Animal\\nOTO - NOTNLM\\nOT  - * Heliconius\\nOT  - *GWAS\\nOT  - *altitude\\nOT  - *local adaptation\\nOT  - *polygenic\\nOT  - *wing shape\\nEDAT- 2021/07/08 06:00\\nMHDA- 2022/01/29 06:00\\nCRDT- 2021/07/07 17:25\\nPHST- 2020/12/04 00:00 [received]\\nPHST- 2021/07/01 00:00 [accepted]\\nPHST- 2021/07/08 06:00 [pubmed]\\nPHST- 2022/01/29 06:00 [medline]\\nPHST- 2021/07/07 17:25 [entrez]\\nAID - 10.1111/mec.16067 [doi]\\nPST - ppublish\\nSO  - Mol Ecol. 2021 Dec;30(23):6387-6402. doi: 10.1111/mec.16067. Epub 2021 Jul 28.\\n\\nPMID- 34229490\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20210803\\nLR  - 20210830\\nIS  - 1471-2954 (Electronic)\\nIS  - 0962-8452 (Linking)\\nVI  - 288\\nIP  - 1954\\nDP  - 2021 Jul 14\\nTI  - Green, yellow or black? Genetic differentiation and adaptation signatures in a\\n      highly migratory marine turtle.\\nPG  - 20210754\\nLID - 10.1098/rspb.2021.0754 [doi]\\nAB  - Marine species may exhibit genetic structure accompanied by phenotypic\\n      differentiation related to adaptation despite their high mobility. Two\\n      shape-based morphotypes have been identified for the green turtle (Chelonia\\n      mydas) in the Pacific Ocean: the south-central/western or yellow turtle and\\n      north-central/eastern or black turtle. The genetic differentiation between these \\n      morphotypes and the adaptation of the black turtle to environmentally contrasting\\n      conditions of the eastern Pacific region has remained a mystery for decades. Here\\n      we addressed both questions using a reduced-representation genome approach\\n      (Dartseq; 9473 neutral SNPs) and identifying candidate outlier loci (67 outlier\\n      SNPs) of biological relevance between shape-based morphotypes from eight Pacific \\n      foraging grounds (n = 158). Our results support genetic divergence between\\n      morphotypes, probably arising from strong natal homing behaviour. Genes and\\n      enriched biological functions linked to thermoregulation, hypoxia, melanism,\\n      morphogenesis, osmoregulation, diet and reproduction were found to be outliers\\n      for differentiation, providing evidence for adaptation of C. mydas to the eastern\\n      Pacific region and suggesting independent evolutionary trajectories of the\\n      shape-based morphotypes. Our findings support the evolutionary distinctness of\\n      the enigmatic black turtle and contribute to the adaptive research and\\n      conservation genomics of a long-lived and highly mobile vertebrate.\\nFAU - Alvarez-Varas, Rocio\\nAU  - Alvarez-Varas R\\nAD  - Departamento de Ciencias Ecologicas, Facultad de Ciencias, Universidad de Chile, \\n      Santiago, Chile.\\nAD  - Nucleo Milenio de Ecologia y Manejo Sustentable de Islas Oceanicas (ESMOI),\\n      Departamento de Biologia Marina, Universidad Catolica del Norte, Coquimbo, Chile.\\nAD  - Qarapara Tortugas Marinas Chile NGO, Santiago, Chile.\\nFAU - Rojas-Hernandez, Noemi\\nAU  - Rojas-Hernandez N\\nAD  - Departamento de Ciencias Ecologicas, Facultad de Ciencias, Universidad de Chile, \\n      Santiago, Chile.\\nFAU - Heidemeyer, Maike\\nAU  - Heidemeyer M\\nAD  - Centro de Investigacion en Biologia Celular y Molecular (CIBCM), Universidad de\\n      Costa Rica, San Jose, Costa Rica.\\nFAU - Riginos, Cynthia\\nAU  - Riginos C\\nAD  - School of Biological Sciences, The University of Queensland, Brisbane, Australia.\\nFAU - Benitez, Hugo A\\nAU  - Benitez HA\\nAD  - Laboratorio de Ecologia y Morfometria Evolutiva, Centro de Investigacion de\\n      Estudios Avanzados del Maule, Universidad Catolica del Maule, Talca, Chile.\\nFAU - Araya-Donoso, Raul\\nAU  - Araya-Donoso R\\nAD  - School of Life Sciences, Arizona State University, Tempe, AZ, USA.\\nFAU - Resendiz, Eduardo\\nAU  - Resendiz E\\nAD  - Departamento Academico de Ciencias Marinas y Costeras, Universidad Autonoma de\\n      Baja California Sur, Mexico.\\nFAU - Lara-Uc, Monica\\nAU  - Lara-Uc M\\nAD  - Departamento Academico de Ciencias Marinas y Costeras, Universidad Autonoma de\\n      Baja California Sur, Mexico.\\nFAU - Godoy, Daniel A\\nAU  - Godoy DA\\nAD  - Coastal-Marine Research Group, Institute of Natural and Mathematical Sciences,\\n      Massey University, Auckland, New Zealand.\\nFAU - Munoz-Perez, Juan Pablo\\nAU  - Munoz-Perez JP\\nAD  - Galapagos Science Center GSC (Universidad San Francisco de Quito USFQ-University \\n      of North Carolina at Chapel Hill UNC), Isla San Cristobal, Galapagos, Ecuador.\\nAD  - University of the Sunshine Coast USC, 90 Sippy Downs Dr, Sippy Downs, Queensland \\n      4556, Australia.\\nFAU - Alarcon-Ruales, Daniela E\\nAU  - Alarcon-Ruales DE\\nAD  - Galapagos Science Center GSC (Universidad San Francisco de Quito USFQ-University \\n      of North Carolina at Chapel Hill UNC), Isla San Cristobal, Galapagos, Ecuador.\\nFAU - Alfaro-Shigueto, Joanna\\nAU  - Alfaro-Shigueto J\\nAD  - ProDelphinus, Lima, Peru.\\nAD  - Facultad de Biologia Marina, Universidad Cientifica del Peru, Lima, Peru.\\nFAU - Ortiz-Alvarez, Clara\\nAU  - Ortiz-Alvarez C\\nAD  - ProDelphinus, Lima, Peru.\\nFAU - Mangel, Jeffrey C\\nAU  - Mangel JC\\nAD  - ProDelphinus, Lima, Peru.\\nFAU - Vianna, Juliana A\\nAU  - Vianna JA\\nAD  - Departamento de Ecosistemas y Medio Ambiente, Facultad de Agronomia e Ingenieria \\n      Forestal, Pontificia Universidad Catolica de Chile, Santiago, Chile.\\nFAU - Veliz, David\\nAU  - Veliz D\\nAD  - Departamento de Ciencias Ecologicas, Facultad de Ciencias, Universidad de Chile, \\n      Santiago, Chile.\\nAD  - Nucleo Milenio de Ecologia y Manejo Sustentable de Islas Oceanicas (ESMOI),\\n      Departamento de Biologia Marina, Universidad Catolica del Norte, Coquimbo, Chile.\\nLA  - eng\\nSI  - figshare/10.6084/m9.figshare.c.5477735\\nPT  - Journal Article\\nPT  - Research Support, Non-U.S. Gov\\'t\\nDEP - 20210707\\nPL  - England\\nTA  - Proc Biol Sci\\nJT  - Proceedings. Biological sciences\\nJID - 101245157\\nSB  - IM\\nMH  - Adaptation, Physiological/genetics\\nMH  - Animals\\nMH  - Genetic Drift\\nMH  - Pacific Ocean\\nMH  - *Turtles/genetics\\nPMC - PMC8261206\\nOTO - NOTNLM\\nOT  - *Chelonia mydas\\nOT  - *candidate genes\\nOT  - *conservation genomics\\nOT  - *eastern Pacific\\nOT  - *genetic structure\\nOT  - *melanism\\nEDAT- 2021/07/08 06:00\\nMHDA- 2021/08/04 06:00\\nCRDT- 2021/07/07 05:33\\nPMCR- 2022/07/14 00:00\\nPHST- 2022/07/14 00:00 [pmc-release]\\nPHST- 2021/07/07 05:33 [entrez]\\nPHST- 2021/07/08 06:00 [pubmed]\\nPHST- 2021/08/04 06:00 [medline]\\nAID - 10.1098/rspb.2021.0754 [doi]\\nPST - ppublish\\nSO  - Proc Biol Sci. 2021 Jul 14;288(1954):20210754. doi: 10.1098/rspb.2021.0754. Epub \\n      2021 Jul 7.\\n\\nPMID- 34165082\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20211020\\nLR  - 20211020\\nIS  - 2050-084X (Electronic)\\nIS  - 2050-084X (Linking)\\nVI  - 10\\nDP  - 2021 Jun 24\\nTI  - Introgression shapes fruit color convergence in invasive Galapagos tomato.\\nLID - 10.7554/eLife.64165 [doi]\\nLID - e64165 [pii]\\nAB  - Invasive species represent one of the foremost risks to global biodiversity.\\n      Here, we use population genomics to evaluate the history and consequences of an\\n      invasion of wild tomato-Solanum pimpinellifolium-onto the Galapagos Islands from \\n      continental South America. Using >300 archipelago and mainland collections, we\\n      infer this invasion was recent and largely the result of a single event from\\n      central Ecuador. Patterns of ancestry within the genomes of invasive plants also \\n      reveal post-colonization hybridization and introgression between S.\\n      pimpinellifolium and the closely related Galapagos endemic Solanum cheesmaniae.\\n      Of admixed invasive individuals, those that carry endemic alleles at one of two\\n      different carotenoid biosynthesis loci also have orange fruits-characteristic of \\n      the endemic species-instead of typical red S. pimpinellifolium fruits. We infer\\n      that introgression of two independent fruit color loci explains this observed\\n      trait convergence, suggesting that selection has favored repeated transitions of \\n      red to orange fruits on the Galapagos.\\nCI  - (c) 2021, Gibson et al.\\nFAU - Gibson, Matthew Js\\nAU  - Gibson MJ\\nAUID- ORCID: 0000-0001-7855-1628\\nAD  - Department of Biology, Indiana University, Bloomington, United States.\\nFAU - Torres, Maria de Lourdes\\nAU  - Torres ML\\nAUID- ORCID: 0000-0001-7207-4568\\nAD  - Universidad San Francisco de Quito (USFQ). Colegio de Ciencias Biologicas y\\n      Ambientales, Laboratorio de Biotecnologia Vegetal. Campus Cumbaya, Quito,\\n      Ecuador.\\nAD  - Galapagos Science Center, Universidad San Francisco de Quito and University of\\n      North Carolina at Chapel Hill, Galapagos, Ecuador.\\nFAU - Brandvain, Yaniv\\nAU  - Brandvain Y\\nAD  - Department of Plant Biology, University of Minnesota-Twin Cities, St. Paul,\\n      United States.\\nFAU - Moyle, Leonie C\\nAU  - Moyle LC\\nAUID- ORCID: 0000-0003-4960-8001\\nAD  - Department of Biology, Indiana University, Bloomington, United States.\\nLA  - eng\\nSI  - Dryad/10.5061/dryad.2v6wwpzkm\\nPT  - Journal Article\\nPT  - Research Support, Non-U.S. Gov\\'t\\nPT  - Research Support, U.S. Gov\\'t, Non-P.H.S.\\nDEP - 20210624\\nPL  - England\\nTA  - Elife\\nJT  - eLife\\nJID - 101579614\\nRN  - 0 (Pigments, Biological)\\nSB  - IM\\nMH  - Ecuador\\nMH  - Fruit/*genetics/*physiology\\nMH  - *Genetic Introgression\\nMH  - Genetic Variation\\nMH  - *Introduced Species\\nMH  - Pigments, Biological\\nMH  - Solanum/*genetics/*physiology\\nMH  - Species Specificity\\nPMC - PMC8294854\\nOTO - NOTNLM\\nOT  - *Galapagos\\nOT  - *convergence\\nOT  - *evolutionary biology\\nOT  - *genetics\\nOT  - *genomics\\nOT  - *in situ hybridization\\nOT  - *introgression\\nOT  - *invasion\\nOT  - *tomato\\nCOIS- MG, MT, YB, LM No competing interests declared\\nEDAT- 2021/06/25 06:00\\nMHDA- 2021/10/21 06:00\\nCRDT- 2021/06/24 08:50\\nPHST- 2020/10/20 00:00 [received]\\nPHST- 2021/06/23 00:00 [accepted]\\nPHST- 2021/06/25 06:00 [pubmed]\\nPHST- 2021/10/21 06:00 [medline]\\nPHST- 2021/06/24 08:50 [entrez]\\nAID - 10.7554/eLife.64165 [doi]\\nAID - 64165 [pii]\\nPST - epublish\\nSO  - Elife. 2021 Jun 24;10. pii: 64165. doi: 10.7554/eLife.64165.\\n\\nPMID- 34025776\\nOWN - NLM\\nSTAT- PubMed-not-MEDLINE\\nLR  - 20210526\\nIS  - 1752-4571 (Print)\\nIS  - 1752-4571 (Linking)\\nVI  - 14\\nIP  - 5\\nDP  - 2021 May\\nTI  - Population genomics of invasive rodents on islands: Genetic consequences of\\n      colonization and prospects for localized synthetic gene drive.\\nPG  - 1421-1435\\nLID - 10.1111/eva.13210 [doi]\\nAB  - Introduced rodent populations pose significant threats worldwide, with\\n      particularly severe impacts on islands. Advancements in genome editing have\\n      motivated interest in synthetic gene drives that could potentially provide\\n      efficient and localized suppression of invasive rodent populations. Application\\n      of such technologies will require rigorous population genomic surveys to evaluate\\n      population connectivity, taxonomic identification, and to inform design of gene\\n      drive localization mechanisms. One proposed approach leverages the predicted\\n      shifts in genetic variation that accompany island colonization, wherein founder\\n      effects, genetic drift, and island-specific selection are expected to result in\\n      locally fixed alleles (LFA) that are variable in neighboring nontarget\\n      populations. Engineering of guide RNAs that target LFA may thus yield gene drives\\n      that spread within invasive island populations, but would have limited impacts on\\n      nontarget populations in the event of an escape. Here we used pooled whole-genome\\n      sequencing of invasive mouse (Mus musculus) populations on four islands along\\n      with paired putative source populations to test genetic predictions of island\\n      colonization and characterize locally fixed Cas9 genomic targets. Patterns of\\n      variation across the genome reflected marked reductions in allelic diversity in\\n      island populations and moderate to high degrees of differentiation from nearby\\n      source populations despite relatively recent colonization. Locally fixed Cas9\\n      sites in female fertility genes were observed in all island populations,\\n      including a small number with multiplexing potential. In practice, rigorous\\n      sampling of presumptive LFA will be essential to fully assess risk of resistance \\n      alleles. These results should serve to guide development of improved, spatially\\n      limited gene drive design in future applications.\\nCI  - (c) 2021 The Authors. Evolutionary Applications published by John Wiley & Sons\\n      Ltd. This article has been contributed to by US Government employees and their\\n      work is in the public domain in the USA.\\nFAU - Oh, Kevin P\\nAU  - Oh KP\\nAUID- ORCID: https://orcid.org/0000-0001-5094-5510\\nAD  - National Wildlife Research Center USDA APHIS Wildlife Services Fort Collins\\n      Colorado USA.\\nAD  - Department of Microbiology, Immunology and Pathology Colorado State University\\n      Fort Collins Colorado USA.\\nFAU - Shiels, Aaron B\\nAU  - Shiels AB\\nAD  - National Wildlife Research Center USDA APHIS Wildlife Services Fort Collins\\n      Colorado USA.\\nFAU - Shiels, Laura\\nAU  - Shiels L\\nAD  - National Wildlife Research Center USDA APHIS Wildlife Services Fort Collins\\n      Colorado USA.\\nFAU - Blondel, Dimitri V\\nAU  - Blondel DV\\nAD  - Department of Biological Sciences North Carolina State University Raleigh North\\n      Carolina USA.\\nFAU - Campbell, Karl J\\nAU  - Campbell KJ\\nAD  - Island Conservation Puerto Ayora Ecuador.\\nAD  - School of Agriculture and Food Sciences The University of Queensland Gatton\\n      Queensland Australia.\\nFAU - Saah, J Royden\\nAU  - Saah JR\\nAD  - Island Conservation Puerto Ayora Ecuador.\\nAD  - Genetic Engineering and Society Center North Carolina State University Raleigh\\n      North Carolina USA.\\nFAU - Lloyd, Alun L\\nAU  - Lloyd AL\\nAD  - Genetic Engineering and Society Center North Carolina State University Raleigh\\n      North Carolina USA.\\nAD  - Biomathematics Graduate Program and Department of Mathematics North Carolina\\n      State University Raleigh North Carolina USA.\\nFAU - Thomas, Paul Q\\nAU  - Thomas PQ\\nAD  - The Robinson Research Institute and School of Medicine The University of Adelaide\\n      Adelaide South Australia Australia.\\nFAU - Gould, Fred\\nAU  - Gould F\\nAD  - Genetic Engineering and Society Center North Carolina State University Raleigh\\n      North Carolina USA.\\nAD  - Department of Entomology and Plant Pathology North Carolina State University\\n      Raleigh North Carolina USA.\\nFAU - Abdo, Zaid\\nAU  - Abdo Z\\nAD  - Department of Microbiology, Immunology and Pathology Colorado State University\\n      Fort Collins Colorado USA.\\nFAU - Godwin, John R\\nAU  - Godwin JR\\nAD  - Department of Biological Sciences North Carolina State University Raleigh North\\n      Carolina USA.\\nAD  - Genetic Engineering and Society Center North Carolina State University Raleigh\\n      North Carolina USA.\\nFAU - Piaggio, Antoinette J\\nAU  - Piaggio AJ\\nAD  - National Wildlife Research Center USDA APHIS Wildlife Services Fort Collins\\n      Colorado USA.\\nLA  - eng\\nPT  - Journal Article\\nDEP - 20210310\\nPL  - England\\nTA  - Evol Appl\\nJT  - Evolutionary applications\\nJID - 101461828\\nPMC - PMC8127709\\nOTO - NOTNLM\\nOT  - CRISPR\\nOT  - Cas9\\nOT  - Mus musculus\\nOT  - genetic biocontrol\\nOT  - genome editing\\nOT  - synthetic biology\\nEDAT- 2021/05/25 06:00\\nMHDA- 2021/05/25 06:01\\nCRDT- 2021/05/24 08:05\\nPHST- 2020/07/16 00:00 [received]\\nPHST- 2021/02/15 00:00 [revised]\\nPHST- 2021/02/16 00:00 [accepted]\\nPHST- 2021/05/24 08:05 [entrez]\\nPHST- 2021/05/25 06:00 [pubmed]\\nPHST- 2021/05/25 06:01 [medline]\\nAID - 10.1111/eva.13210 [doi]\\nAID - EVA13210 [pii]\\nPST - epublish\\nSO  - Evol Appl. 2021 Mar 10;14(5):1421-1435. doi: 10.1111/eva.13210. eCollection 2021 \\n      May.\\n\\nPMID- 33846529\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20210804\\nLR  - 20210804\\nIS  - 2399-3642 (Electronic)\\nIS  - 2399-3642 (Linking)\\nVI  - 4\\nIP  - 1\\nDP  - 2021 Apr 12\\nTI  - Population genomics and antimicrobial resistance dynamics of Escherichia coli in \\n      wastewater and river environments.\\nPG  - 457\\nLID - 10.1038/s42003-021-01949-x [doi]\\nAB  - Aquatic environments are key niches for the emergence, evolution and\\n      dissemination of antimicrobial resistance. However, the population diversity and \\n      the genetic elements that drive the dynamics of resistant bacteria in different\\n      aquatic environments are still largely unknown. The aim of this study was to\\n      understand the population genomics and evolutionary events of Escherichia coli\\n      resistant to clinically important antibiotics including aminoglycosides, in\\n      anthropogenic and natural water ecosystems. Here we show that less different E.\\n      coli sequence types (STs) are identified in wastewater than in rivers, albeit\\n      more resistant to antibiotics, and with significantly more plasmids/cell (6.36 vs\\n      3.72). However, the genomic diversity within E. coli STs in both aquatic\\n      environments is similar. Wastewater environments favor the selection of conserved\\n      chromosomal structures associated with diverse flexible plasmids, unraveling\\n      promiscuous interplasmidic resistance genes flux. On the contrary, the key driver\\n      for river E. coli adaptation is a mutable chromosome along with few plasmid types\\n      shared between diverse STs harboring a limited resistance gene content.\\nFAU - Delgado-Blas, Jose F\\nAU  - Delgado-Blas JF\\nAUID- ORCID: 0000-0001-6447-7113\\nAD  - Antimicrobial Resistance Unit (ARU), Animal Health Department, Faculty of\\n      Veterinary Medicine and VISAVET Health Surveillance Centre, Complutense\\n      University of Madrid, Madrid, Spain.\\nFAU - Ovejero, Cristina M\\nAU  - Ovejero CM\\nAD  - Antimicrobial Resistance Unit (ARU), Animal Health Department, Faculty of\\n      Veterinary Medicine and VISAVET Health Surveillance Centre, Complutense\\n      University of Madrid, Madrid, Spain.\\nFAU - David, Sophia\\nAU  - David S\\nAD  - Centre for Genomic Pathogen Surveillance (CGPS), Wellcome Sanger Institute,\\n      Hinxton, UK.\\nFAU - Montero, Natalia\\nAU  - Montero N\\nAD  - Antimicrobial Resistance Unit (ARU), Animal Health Department, Faculty of\\n      Veterinary Medicine and VISAVET Health Surveillance Centre, Complutense\\n      University of Madrid, Madrid, Spain.\\nFAU - Calero-Caceres, William\\nAU  - Calero-Caceres W\\nAUID- ORCID: 0000-0003-3819-9206\\nAD  - Department of Genetics, Microbiology and Statistics, Faculty of Biology,\\n      University of Barcelona, Barcelona, Spain.\\nAD  - UTA RAM One Health, Faculty of Food Science, Engineering and Biotechnology,\\n      Technical University of Ambato, Ambato, Ecuador.\\nFAU - Garcillan-Barcia, M Pilar\\nAU  - Garcillan-Barcia MP\\nAUID- ORCID: 0000-0001-7058-5428\\nAD  - Institute of Biomedicine and Biotechnology (IBBTEC), CSIC, University of\\n      Cantabria, Santander, Spain.\\nFAU - de la Cruz, Fernando\\nAU  - de la Cruz F\\nAUID- ORCID: 0000-0003-4758-6857\\nAD  - Institute of Biomedicine and Biotechnology (IBBTEC), CSIC, University of\\n      Cantabria, Santander, Spain.\\nFAU - Muniesa, Maite\\nAU  - Muniesa M\\nAUID- ORCID: 0000-0001-5549-1394\\nAD  - Department of Genetics, Microbiology and Statistics, Faculty of Biology,\\n      University of Barcelona, Barcelona, Spain.\\nFAU - Aanensen, David M\\nAU  - Aanensen DM\\nAD  - Centre for Genomic Pathogen Surveillance (CGPS), Wellcome Sanger Institute,\\n      Hinxton, UK.\\nFAU - Gonzalez-Zorn, Bruno\\nAU  - Gonzalez-Zorn B\\nAUID- ORCID: 0000-0002-6958-4716\\nAD  - Antimicrobial Resistance Unit (ARU), Animal Health Department, Faculty of\\n      Veterinary Medicine and VISAVET Health Surveillance Centre, Complutense\\n      University of Madrid, Madrid, Spain. bgzorn@ucm.es.\\nLA  - eng\\nPT  - Journal Article\\nPT  - Research Support, Non-U.S. Gov\\'t\\nDEP - 20210412\\nPL  - England\\nTA  - Commun Biol\\nJT  - Communications biology\\nJID - 101719179\\nRN  - 0 (Anti-Bacterial Agents)\\nRN  - 0 (Waste Water)\\nSB  - IM\\nMH  - Anti-Bacterial Agents/*pharmacology\\nMH  - *Drug Resistance, Bacterial\\nMH  - Escherichia coli/*genetics\\nMH  - *Genetic Variation\\nMH  - *Genome, Bacterial\\nMH  - Metagenomics\\nMH  - Plasmids/physiology\\nMH  - Rivers/*microbiology\\nMH  - Spain\\nMH  - Waste Water/*microbiology\\nPMC - PMC8041779\\nEDAT- 2021/04/14 06:00\\nMHDA- 2021/08/05 06:00\\nCRDT- 2021/04/13 06:22\\nPHST- 2020/08/05 00:00 [received]\\nPHST- 2021/03/02 00:00 [accepted]\\nPHST- 2021/04/13 06:22 [entrez]\\nPHST- 2021/04/14 06:00 [pubmed]\\nPHST- 2021/08/05 06:00 [medline]\\nAID - 10.1038/s42003-021-01949-x [doi]\\nAID - 10.1038/s42003-021-01949-x [pii]\\nPST - epublish\\nSO  - Commun Biol. 2021 Apr 12;4(1):457. doi: 10.1038/s42003-021-01949-x.\\n\\nPMID- 33845910\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20210506\\nLR  - 20210506\\nIS  - 2049-2618 (Electronic)\\nIS  - 2049-2618 (Linking)\\nVI  - 9\\nIP  - 1\\nDP  - 2021 Apr 12\\nTI  - Microbial community characterization of shrimp survivors to AHPND challenge test \\n      treated with an effective shrimp probiotic (Vibrio diabolicus).\\nPG  - 88\\nLID - 10.1186/s40168-021-01043-8 [doi]\\nAB  - BACKGROUND: Acute hepatopancreatic necrosis disease (AHPND) is an important\\n      shrimp bacterial disease caused by some Vibrio species. The severity of the\\n      impact of this disease on aquaculture worldwide has made it necessary to develop \\n      alternatives to prophylactic antibiotics use, such as the application of\\n      probiotics. To assess the potential to use probiotics in order to limit the\\n      detrimental effects of AHNPD, we evaluated the effect of the ILI strain, a Vibrio\\n      sp. bacterium and efficient shrimp probiotic, using metabarcoding (16S rRNA gene)\\n      on the gastrointestinal microbiota of shrimp after being challenged with\\n      AHPND-causing V. parahaemolyticus. RESULTS: We showed how the gastrointestinal\\n      microbiome of shrimp varied between healthy and infected organisms. Nevertheless,\\n      a challenge of working with AHPND-causing Vibrio pathogens and Vibrio-related\\n      bacteria as probiotics is the potential risk of the probiotic strain becoming\\n      pathogenic. Consequently, we evaluated whether ILI strain can acquire the plasmid\\n      pV-AHPND via horizontal transfer and further cause the disease in shrimp.\\n      Conjugation assays were performed resulting in a high frequency (70%) of colonies\\n      harboring the pv-AHPND. However, no shrimp mortality was observed when\\n      transconjugant colonies of the ILI strain were used in a challenge test using\\n      healthy shrimp. We sequenced the genome of the ILI strain and performed\\n      comparative genomics analyses using AHPND and non-AHPND Vibrio isolates. Using\\n      available phylogenetic and phylogenomics analyses, we reclassified the ILI strain\\n      as Vibrio diabolicus. In summary, this work represents an effort to study the\\n      role that probiotics play in the normal gastrointestinal shrimp microbiome and in\\n      AHPND-infected shrimp, showing that the ILI probiotic was able to control\\n      pathogenic bacterial populations in the host\\'s gastrointestinal tract and\\n      stimulate the shrimp\\'s survival. The identification of probiotic bacterial\\n      species that are effective in the host\\'s colonization is important to promote\\n      animal health and prevent disease. CONCLUSIONS: This study describes probiotic\\n      bacteria capable of controlling pathogenic populations of bacteria in the shrimp \\n      gastrointestinal tract. Our work provides new insights into the complex dynamics \\n      between shrimp and the changes in the microbiota. It also addresses the practical\\n      application of probiotics to solve problems with pathogens that cause high\\n      mortality-rate in shrimp farming around the world. Video Abstract.\\nFAU - Restrepo, Leda\\nAU  - Restrepo L\\nAD  - Department of Biological Sciences, Universidad de los Andes, Bogota, Colombia.\\nAD  - Max Planck Tandem Group in Computational Biology, Universidad de los Andes,\\n      Bogota, Colombia.\\nAD  - Escuela Superior Politecnica del Litoral, ESPOL, Centro Nacional de Acuicultura e\\n      Investigaciones Marinas, CENAIM, Campus Gustavo Galindo Km 30.5 Via Perimetral,\\n      P.O. Box 09-01-5863, Guayaquil, Ecuador.\\nFAU - Dominguez-Borbor, Cristobal\\nAU  - Dominguez-Borbor C\\nAD  - Escuela Superior Politecnica del Litoral, ESPOL, Centro Nacional de Acuicultura e\\n      Investigaciones Marinas, CENAIM, Campus Gustavo Galindo Km 30.5 Via Perimetral,\\n      P.O. Box 09-01-5863, Guayaquil, Ecuador.\\nFAU - Bajana, Leandro\\nAU  - Bajana L\\nAD  - Escuela Superior Politecnica del Litoral, ESPOL, Centro Nacional de Acuicultura e\\n      Investigaciones Marinas, CENAIM, Campus Gustavo Galindo Km 30.5 Via Perimetral,\\n      P.O. Box 09-01-5863, Guayaquil, Ecuador.\\nFAU - Betancourt, Irma\\nAU  - Betancourt I\\nAD  - Escuela Superior Politecnica del Litoral, ESPOL, Centro Nacional de Acuicultura e\\n      Investigaciones Marinas, CENAIM, Campus Gustavo Galindo Km 30.5 Via Perimetral,\\n      P.O. Box 09-01-5863, Guayaquil, Ecuador.\\nFAU - Rodriguez, Jenny\\nAU  - Rodriguez J\\nAD  - Escuela Superior Politecnica del Litoral, ESPOL, Centro Nacional de Acuicultura e\\n      Investigaciones Marinas, CENAIM, Campus Gustavo Galindo Km 30.5 Via Perimetral,\\n      P.O. Box 09-01-5863, Guayaquil, Ecuador.\\nAD  - Escuela Superior Politecnica del Litoral, ESPOL, Facultad de Ciencias de la Vida,\\n      FCV, Campus Gustavo Galindo Km 30.5 Via Perimetral, P.O. Box 09-01-5863,\\n      Guayaquil, Ecuador.\\nFAU - Bayot, Bonny\\nAU  - Bayot B\\nAD  - Escuela Superior Politecnica del Litoral, ESPOL, Centro Nacional de Acuicultura e\\n      Investigaciones Marinas, CENAIM, Campus Gustavo Galindo Km 30.5 Via Perimetral,\\n      P.O. Box 09-01-5863, Guayaquil, Ecuador. bbayot@espol.edu.ec.\\nAD  - Escuela Superior Politecnica del Litoral, ESPOL, Facultad de Ingenieria Maritima \\n      y Ciencias del Mar, FIMCM, Campus Gustavo Galindo Km 30.5 Via Perimetral, P.O.\\n      Box 09-01-5863, Guayaquil, Ecuador. bbayot@espol.edu.ec.\\nFAU - Reyes, Alejandro\\nAU  - Reyes A\\nAUID- ORCID: 0000-0003-2907-3265\\nAD  - Department of Biological Sciences, Universidad de los Andes, Bogota, Colombia.\\n      a.reyes@uniandes.edu.co.\\nAD  - Max Planck Tandem Group in Computational Biology, Universidad de los Andes,\\n      Bogota, Colombia. a.reyes@uniandes.edu.co.\\nAD  - Center for Genome Sciences and Systems Biology, Department of Pathology and\\n      Immunology, Washington University in Saint Louis, Saint Louis, MO, USA.\\n      a.reyes@uniandes.edu.co.\\nLA  - eng\\nPT  - Journal Article\\nPT  - Research Support, Non-U.S. Gov\\'t\\nPT  - Video-Audio Media\\nDEP - 20210412\\nPL  - England\\nTA  - Microbiome\\nJT  - Microbiome\\nJID - 101615147\\nRN  - 0 (RNA, Ribosomal, 16S)\\nRN  - Vibrio diabolicus\\nSB  - IM\\nMH  - Animals\\nMH  - Humans\\nMH  - *Microbiota\\nMH  - Necrosis\\nMH  - *Penaeidae\\nMH  - Phylogeny\\nMH  - *Probiotics\\nMH  - RNA, Ribosomal, 16S/genetics\\nMH  - Survivors\\nMH  - Vibrio\\nMH  - *Vibrio parahaemolyticus/genetics\\nPMC - PMC8042889\\nOTO - NOTNLM\\nOT  - *Acute hepatopancreatic necrosis disease (AHPND)\\nOT  - *Comparative genomics\\nOT  - *Horizontal gene transfer (HGT)\\nOT  - *Metagenomics\\nOT  - *Microbiome\\nOT  - *Probiotics\\nOT  - *Vibrio diabolicus\\nEDAT- 2021/04/14 06:00\\nMHDA- 2021/05/07 06:00\\nCRDT- 2021/04/13 05:38\\nPHST- 2020/12/07 00:00 [received]\\nPHST- 2021/03/05 00:00 [accepted]\\nPHST- 2021/04/13 05:38 [entrez]\\nPHST- 2021/04/14 06:00 [pubmed]\\nPHST- 2021/05/07 06:00 [medline]\\nAID - 10.1186/s40168-021-01043-8 [doi]\\nAID - 10.1186/s40168-021-01043-8 [pii]\\nPST - epublish\\nSO  - Microbiome. 2021 Apr 12;9(1):88. doi: 10.1186/s40168-021-01043-8.\\n\\nPMID- 33822137\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20210920\\nLR  - 20210920\\nIS  - 1537-1719 (Electronic)\\nIS  - 0737-4038 (Linking)\\nVI  - 38\\nIP  - 8\\nDP  - 2021 Jul 29\\nTI  - Migration through a Major Andean Ecogeographic Disruption as a Driver of Genetic \\n      and Phenotypic Diversity in a Wild Tomato Species.\\nPG  - 3202-3219\\nLID - 10.1093/molbev/msab092 [doi]\\nAB  - Evolutionary dynamics at the population level play a central role in creating the\\n      diversity of life on our planet. In this study, we sought to understand the\\n      origins of such population-level variation in mating systems and defensive\\n      acylsugar chemistry in Solanum habrochaites-a wild tomato species found in\\n      diverse Andean habitats in Ecuador and Peru. Using\\n      Restriction-site-Associated-DNA-Sequencing (RAD-seq) of 50 S. habrochaites\\n      accessions, we identified eight population clusters generated via isolation and\\n      hybridization dynamics of 4-6 ancestral populations. Detailed characterization of\\n      mating systems of these clusters revealed emergence of multiple self-compatible\\n      (SC) groups from progenitor self-incompatible populations in the northern part of\\n      the species range. Emergence of these SC groups was also associated with fixation\\n      of deleterious alleles inactivating acylsugar acetylation. The\\n      Amotape-Huancabamba Zone-a geographical landmark in the Andes with high endemism \\n      and isolated microhabitats-was identified as a major driver of differentiation in\\n      the northern species range, whereas large geographical distances contributed to\\n      population structure and evolution of a novel SC group in the central and\\n      southern parts of the range, where the species was also inferred to have\\n      originated. Findings presented here highlight the role of the diverse\\n      ecogeography of Peru and Ecuador in generating population differentiation, and\\n      enhance our understanding of the microevolutionary processes that create\\n      biological diversity.\\nCI  - (c) The Author(s) 2021. Published by Oxford University Press on behalf of the\\n      Society for Molecular Biology and Evolution.\\nFAU - Landis, Jacob B\\nAU  - Landis JB\\nAD  - Department of Botany and Plant Sciences, University of California, Riverside,\\n      Riverside, CA, USA.\\nAD  - Plant Biology Section, School of Integrative Plant Science, Cornell University,\\n      Ithaca, NY, USA.\\nFAU - Miller, Christopher M\\nAU  - Miller CM\\nAD  - Department of Biology, Colorado State University, Fort Collins, CO, USA.\\nFAU - Broz, Amanda K\\nAU  - Broz AK\\nAD  - Department of Biology, Colorado State University, Fort Collins, CO, USA.\\nFAU - Bennett, Alexandra A\\nAU  - Bennett AA\\nAD  - Plant Biology Section, School of Integrative Plant Science, Cornell University,\\n      Ithaca, NY, USA.\\nFAU - Carrasquilla-Garcia, Noelia\\nAU  - Carrasquilla-Garcia N\\nAD  - Department of Plant Pathology, University of California, Davis, Davis, CA, USA.\\nFAU - Cook, Douglas R\\nAU  - Cook DR\\nAD  - Department of Plant Pathology, University of California, Davis, Davis, CA, USA.\\nFAU - Last, Robert L\\nAU  - Last RL\\nAD  - Department of Biochemistry and Molecular Biology, Michigan State University, East\\n      Lansing, MI, USA.\\nAD  - Department of Plant Biology, Michigan State University, East Lansing, MI, USA.\\nFAU - Bedinger, Patricia A\\nAU  - Bedinger PA\\nAD  - Department of Biology, Colorado State University, Fort Collins, CO, USA.\\nFAU - Moghe, Gaurav D\\nAU  - Moghe GD\\nAUID- ORCID: 0000-0002-8761-064X\\nAD  - Plant Biology Section, School of Integrative Plant Science, Cornell University,\\n      Ithaca, NY, USA.\\nAD  - Department of Biochemistry and Molecular Biology, Michigan State University, East\\n      Lansing, MI, USA.\\nLA  - eng\\nPT  - Journal Article\\nPT  - Research Support, Non-U.S. Gov\\'t\\nPT  - Research Support, U.S. Gov\\'t, Non-P.H.S.\\nPL  - United States\\nTA  - Mol Biol Evol\\nJT  - Molecular biology and evolution\\nJID - 8501455\\nSB  - IM\\nMH  - Acetylation\\nMH  - Ecuador\\nMH  - *Gene Flow\\nMH  - Lycopersicon esculentum/*genetics/metabolism\\nMH  - Peru\\nMH  - Phylogeography\\nMH  - Self-Fertilization\\nMH  - Self-Incompatibility in Flowering Plants/*genetics\\nMH  - Solanum/*genetics/metabolism\\nPMC - PMC8321546\\nOTO - NOTNLM\\nOT  - *RAD-seq\\nOT  - *biodiversity\\nOT  - *mating systems\\nOT  - *metabolic evolution\\nOT  - *plant evolution\\nOT  - *population genomics\\nEDAT- 2021/04/07 06:00\\nMHDA- 2021/09/21 06:00\\nCRDT- 2021/04/06 12:37\\nPHST- 2021/04/07 06:00 [pubmed]\\nPHST- 2021/09/21 06:00 [medline]\\nPHST- 2021/04/06 12:37 [entrez]\\nAID - 6209742 [pii]\\nAID - 10.1093/molbev/msab092 [doi]\\nPST - ppublish\\nSO  - Mol Biol Evol. 2021 Jul 29;38(8):3202-3219. doi: 10.1093/molbev/msab092.\\n\\nPMID- 33747978\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20210616\\nLR  - 20210617\\nIS  - 2235-2988 (Electronic)\\nIS  - 2235-2988 (Linking)\\nVI  - 11\\nDP  - 2021\\nTI  - Repeat-Driven Generation of Antigenic Diversity in a Major Human Pathogen,\\n      Trypanosoma cruzi.\\nPG  - 614665\\nLID - 10.3389/fcimb.2021.614665 [doi]\\nAB  - Trypanosoma cruzi, a zoonotic kinetoplastid protozoan parasite, is the causative \\n      agent of American trypanosomiasis (Chagas disease). Having a very plastic,\\n      repetitive and complex genome, the parasite displays a highly diverse repertoire \\n      of surface molecules, with pivotal roles in cell invasion, immune evasion and\\n      pathogenesis. Before 2016, the complexity of the genomic regions containing these\\n      genes impaired the assembly of a genome at chromosomal level, making it\\n      impossible to study the structure and function of the several thousand repetitive\\n      genes encoding the surface molecules of the parasite. We here describe the genome\\n      assembly of the Sylvio X10/1 genome sequence, which since 2016 has been used as a\\n      reference genome sequence for T. cruzi clade I (TcI), produced using high\\n      coverage PacBio single-molecule sequencing. It was used to analyze deep Illumina \\n      sequence data from 34 T. cruzi TcI isolates and clones from different geographic \\n      locations, sample sources and clinical outcomes. Resolution of the surface\\n      molecule gene distribution showed the unusual duality in the organization of the \\n      parasite genome, a synteny of the core genomic region with related protozoa\\n      flanked by unique and highly plastic multigene family clusters encoding surface\\n      antigens. The presence of abundant interspersed retrotransposons in these\\n      multigene family clusters suggests that these elements are involved in a\\n      recombination mechanism for the generation of antigenic variation and evasion of \\n      the host immune response on these TcI strains. The comparative genomic analysis\\n      of the cohort of TcI strains revealed multiple cases of such recombination events\\n      involving surface molecule genes and has provided new insights into T. cruzi\\n      population structure.\\nCI  - Copyright (c) 2021 Talavera-Lopez, Messenger, Lewis, Yeo, Reis-Cunha, Matos,\\n      Bartholomeu, Calzada, Saldana, Ramirez, Guhl, Ocana-Mayorga, Costales, Gorchakov,\\n      Jones, Nolan, Teixeira, Carrasco, Bottazzi, Hotez, Murray, Grijalva, Burleigh,\\n      Grisard, Miles and Andersson.\\nFAU - Talavera-Lopez, Carlos\\nAU  - Talavera-Lopez C\\nAD  - Department of Cell and Molecular Biology, Karolinska Institutet, Stockholm,\\n      Sweden.\\nAD  - European Bioinformatics Institute, Wellcome Sanger Institute, Hinxton, United\\n      Kingdom.\\nFAU - Messenger, Louisa A\\nAU  - Messenger LA\\nAD  - Faculty of Infectious and Tropical Diseases, London School of Hygiene and\\n      Tropical Medicine, London, United Kingdom.\\nFAU - Lewis, Michael D\\nAU  - Lewis MD\\nAD  - Faculty of Infectious and Tropical Diseases, London School of Hygiene and\\n      Tropical Medicine, London, United Kingdom.\\nFAU - Yeo, Matthew\\nAU  - Yeo M\\nAD  - Faculty of Infectious and Tropical Diseases, London School of Hygiene and\\n      Tropical Medicine, London, United Kingdom.\\nFAU - Reis-Cunha, Joao Luis\\nAU  - Reis-Cunha JL\\nAD  - Departamento de Parasitologia, Universidade Federal de Minas Gerais, Belo\\n      Horizonte, Brazil.\\nFAU - Matos, Gabriel Machado\\nAU  - Matos GM\\nAD  - Departamento de Biologia Celular, Embriologia e Genetica, Universidade Federal\\n      Santa Catarina, Florianopolis, Brazil.\\nFAU - Bartholomeu, Daniella C\\nAU  - Bartholomeu DC\\nAD  - Departamento de Parasitologia, Universidade Federal de Minas Gerais, Belo\\n      Horizonte, Brazil.\\nFAU - Calzada, Jose E\\nAU  - Calzada JE\\nAD  - Departamento de Parasitologia, Instituto Conmemorativo Gorgas de Estudios de la\\n      Salud, Ciudad de Panama, Panama.\\nFAU - Saldana, Azael\\nAU  - Saldana A\\nAD  - Departamento de Parasitologia, Instituto Conmemorativo Gorgas de Estudios de la\\n      Salud, Ciudad de Panama, Panama.\\nFAU - Ramirez, Juan David\\nAU  - Ramirez JD\\nAD  - Grupo de Investigaciones Microbiologicas-UR (GIMUR), Departamento de Biologia,\\n      Facultad de Ciencias Naturales, Universidad del Rosario, Bogota, Colombia.\\nFAU - Guhl, Felipe\\nAU  - Guhl F\\nAD  - Grupo de Investigaciones en Microbiologia y Parasitologia Tropical (CIMPAT),\\n      Tropical Parasitology Research Center, Universidad de Los Andes, Bogota,\\n      Colombia.\\nFAU - Ocana-Mayorga, Sofia\\nAU  - Ocana-Mayorga S\\nAD  - Centro de Investigacion para la Salud en America Latina (CISeAL), Escuela de\\n      Ciencias Biologicas, Pontificia Universidad Catolica del Ecuador, Quito, Ecuador.\\nFAU - Costales, Jaime A\\nAU  - Costales JA\\nAD  - Centro de Investigacion para la Salud en America Latina (CISeAL), Escuela de\\n      Ciencias Biologicas, Pontificia Universidad Catolica del Ecuador, Quito, Ecuador.\\nFAU - Gorchakov, Rodion\\nAU  - Gorchakov R\\nAD  - Sabin Vaccine Institute and Texas Children\\'s Hospital Center for Vaccine\\n      Development, National School of Tropical Medicine, Department of Pediatrics -\\n      Tropical Medicine, Baylor College of Medicine, Houston, TX, United States.\\nFAU - Jones, Kathryn\\nAU  - Jones K\\nAD  - Sabin Vaccine Institute and Texas Children\\'s Hospital Center for Vaccine\\n      Development, National School of Tropical Medicine, Department of Pediatrics -\\n      Tropical Medicine, Baylor College of Medicine, Houston, TX, United States.\\nFAU - Nolan, Melissa S\\nAU  - Nolan MS\\nAD  - Sabin Vaccine Institute and Texas Children\\'s Hospital Center for Vaccine\\n      Development, National School of Tropical Medicine, Department of Pediatrics -\\n      Tropical Medicine, Baylor College of Medicine, Houston, TX, United States.\\nFAU - Teixeira, Santuza M R\\nAU  - Teixeira SMR\\nAD  - Departamento de Bioquimica e Imunologia, Universidade Federal de Minas Gerais,\\n      Belo Horizonte, Brazil.\\nFAU - Carrasco, Hernan Jose\\nAU  - Carrasco HJ\\nAD  - Laboratorio de Biologia Molecular de Protozoarios, Instituto de Medicina\\n      Tropical, Facultad de Medicina, Universidad Central de Venezuela, Caracas,\\n      Venezuela.\\nFAU - Bottazzi, Maria Elena\\nAU  - Bottazzi ME\\nAD  - Sabin Vaccine Institute and Texas Children\\'s Hospital Center for Vaccine\\n      Development, National School of Tropical Medicine, Department of Pediatrics -\\n      Tropical Medicine, Baylor College of Medicine, Houston, TX, United States.\\nFAU - Hotez, Peter J\\nAU  - Hotez PJ\\nAD  - Sabin Vaccine Institute and Texas Children\\'s Hospital Center for Vaccine\\n      Development, National School of Tropical Medicine, Department of Pediatrics -\\n      Tropical Medicine, Baylor College of Medicine, Houston, TX, United States.\\nFAU - Murray, Kristy O\\nAU  - Murray KO\\nAD  - Sabin Vaccine Institute and Texas Children\\'s Hospital Center for Vaccine\\n      Development, National School of Tropical Medicine, Department of Pediatrics -\\n      Tropical Medicine, Baylor College of Medicine, Houston, TX, United States.\\nFAU - Grijalva, Mario J\\nAU  - Grijalva MJ\\nAD  - Centro de Investigacion para la Salud en America Latina (CISeAL), Escuela de\\n      Ciencias Biologicas, Pontificia Universidad Catolica del Ecuador, Quito, Ecuador.\\nAD  - Department of Biomedical Sciences, Heritage College of Osteopathic Medicine,\\n      Infectious and Tropical Disease Institute, Ohio University, Athens, OH, United\\n      States.\\nFAU - Burleigh, Barbara\\nAU  - Burleigh B\\nAD  - Department of Immunology and Infectious Diseases, T.H. Chan School of Public\\n      Health, Harvard University, Boston, MA, United States.\\nFAU - Grisard, Edmundo C\\nAU  - Grisard EC\\nAD  - Departamento de Microbiologia, Imunologia e Parasitologia, Universidade Federal\\n      Santa Catarina, Florianopolis, Brazil.\\nFAU - Miles, Michael A\\nAU  - Miles MA\\nAD  - Faculty of Infectious and Tropical Diseases, London School of Hygiene and\\n      Tropical Medicine, London, United Kingdom.\\nFAU - Andersson, Bjorn\\nAU  - Andersson B\\nAD  - Department of Cell and Molecular Biology, Karolinska Institutet, Stockholm,\\n      Sweden.\\nLA  - eng\\nGR  - R01 AI107028/AI/NIAID NIH HHS/United States\\nPT  - Journal Article\\nPT  - Research Support, N.I.H., Extramural\\nPT  - Research Support, Non-U.S. Gov\\'t\\nDEP - 20210303\\nPL  - Switzerland\\nTA  - Front Cell Infect Microbiol\\nJT  - Frontiers in cellular and infection microbiology\\nJID - 101585359\\nSB  - IM\\nMH  - *Antigenic Variation\\nMH  - Multigene Family\\nMH  - Synteny\\nMH  - *Trypanosoma cruzi/genetics\\nPMC - PMC7966520\\nOTO - NOTNLM\\nOT  - *Trypanosoma cruzi\\nOT  - *antigenic variation\\nOT  - *genome sequence\\nOT  - *microbial genomics\\nOT  - *parasitology\\nOT  - *pathology of infectious diseases\\nOT  - *population genetics\\nOT  - *tropical medicine\\nCOIS- The authors declare that the research was conducted in the absence of any\\n      commercial or financial relationships that could be construed as a potential\\n      conflict of interest.\\nEDAT- 2021/03/23 06:00\\nMHDA- 2021/06/17 06:00\\nCRDT- 2021/03/22 08:20\\nPHST- 2020/10/06 00:00 [received]\\nPHST- 2021/01/22 00:00 [accepted]\\nPHST- 2021/03/22 08:20 [entrez]\\nPHST- 2021/03/23 06:00 [pubmed]\\nPHST- 2021/06/17 06:00 [medline]\\nAID - 10.3389/fcimb.2021.614665 [doi]\\nPST - epublish\\nSO  - Front Cell Infect Microbiol. 2021 Mar 3;11:614665. doi:\\n      10.3389/fcimb.2021.614665. eCollection 2021.\\n\\nPMID- 33668747\\nOWN - NLM\\nSTAT- PubMed-not-MEDLINE\\nLR  - 20210413\\nIS  - 2076-2615 (Print)\\nIS  - 2076-2615 (Linking)\\nVI  - 11\\nIP  - 3\\nDP  - 2021 Feb 25\\nTI  - Genomic Analysis, Progress and Future Perspectives in Dairy Cattle Selection: A\\n      Review.\\nLID - 599 [pii]\\nLID - 10.3390/ani11030599 [doi]\\nAB  - Genomics comprises a set of current and valuable technologies implemented as\\n      selection tools in dairy cattle commercial breeding programs. The intensive\\n      progeny testing for production and reproductive traits based on genomic breeding \\n      values (GEBVs) has been crucial to increasing dairy cattle productivity. The\\n      knowledge of key genes and haplotypes, including their regulation mechanisms, as \\n      markers for productivity traits, may improve the strategies on the present and\\n      future for dairy cattle selection. Genome-wide association studies (GWAS) such as\\n      quantitative trait loci (QTL), single nucleotide polymorphisms (SNPs), or\\n      single-step genomic best linear unbiased prediction (ssGBLUP) methods have\\n      already been included in global dairy programs for the estimation of\\n      marker-assisted selection-derived effects. The increase in genetic progress based\\n      on genomic predicting accuracy has also contributed to the understanding of\\n      genetic effects in dairy cattle offspring. However, the crossing within\\n      inbred-lines critically increased homozygosis with accumulated negative effects\\n      of inbreeding like a decline in reproductive performance. Thus, inaccurate-biased\\n      estimations based on empirical-conventional models of dairy production systems\\n      face an increased risk of providing suboptimal results derived from errors in the\\n      selection of candidates of high genetic merit-based just on low-heritability\\n      phenotypic traits. This extends the generation intervals and increases costs due \\n      to the significant reduction of genetic gains. The remarkable progress of genomic\\n      prediction increases the accurate selection of superior candidates. The scope of \\n      the present review is to summarize and discuss the advances and challenges of\\n      genomic tools for dairy cattle selection for optimizing breeding programs and\\n      controlling negative inbreeding depression effects on productivity and\\n      consequently, achieving economic-effective advances in food production\\n      efficiency. Particular attention is given to the potential genomic\\n      selection-derived results to facilitate precision management on modern dairy\\n      farms, including an overview of novel genome editing methodologies as\\n      perspectives toward the future.\\nFAU - Gutierrez-Reinoso, Miguel A\\nAU  - Gutierrez-Reinoso MA\\nAD  - Facultad de Ciencias Agropecuarias y Recursos Naturales, Carrera de Medicina\\n      Veterinaria, Universidad Tecnica de Cotopaxi (UTC), Latacunga 05-0150, Ecuador.\\nAD  - Laboratorio de Biotecnologia Animal, Departamento de Ciencia Animal, Facultad de \\n      Ciencias Veterinarias, Universidad de Concepcion (UdeC), Chillan 3780000, Chile.\\nFAU - Aponte, Pedro M\\nAU  - Aponte PM\\nAD  - Colegio de Ciencias Biologicas y Ambientales (COCIBA), Universidad San Francisco \\n      de Quito (USFQ), Quito 170157, Ecuador.\\nAD  - Campus Cumbaya, Instituto de Investigaciones en Biomedicina \"One-health\",\\n      Universidad San Francisco de Quito (USFQ), Quito 170157, Ecuador.\\nFAU - Garcia-Herreros, Manuel\\nAU  - Garcia-Herreros M\\nAUID- ORCID: 0000-0002-0911-6689\\nAD  - Instituto Nacional de Investigacao Agraria e Veterinaria (INIAV), 2005-048\\n      Santarem, Portugal.\\nLA  - eng\\nGR  - 20/10/UTC2020/Universidad Tecnica de Cotopaxi (UTC), Ecuador.\\nGR  - 2020 - 21201280/Universidad San Francisco de Quito (USFQ), Ecuador; Secretariat\\n      of Higher Education, Science, Technology and Innovation (SENESCYT) of the\\n      Government of Ecuador; Agencia Nacional de Investigacion y Desarrollo (ANID);\\n      Programa de Becas / Doctorado Nacional\\nPT  - Journal Article\\nPT  - Review\\nDEP - 20210225\\nPL  - Switzerland\\nTA  - Animals (Basel)\\nJT  - Animals : an open access journal from MDPI\\nJID - 101635614\\nPMC - PMC7996307\\nOTO - NOTNLM\\nOT  - dairy cattle\\nOT  - environment\\nOT  - gene edition\\nOT  - genomic analysis\\nOT  - health\\nOT  - linear types\\nOT  - nutrition\\nOT  - production\\nOT  - reproduction\\nOT  - welfare\\nEDAT- 2021/03/07 06:00\\nMHDA- 2021/03/07 06:01\\nCRDT- 2021/03/06 01:02\\nPHST- 2020/10/26 00:00 [received]\\nPHST- 2020/11/23 00:00 [revised]\\nPHST- 2020/11/24 00:00 [accepted]\\nPHST- 2021/03/06 01:02 [entrez]\\nPHST- 2021/03/07 06:00 [pubmed]\\nPHST- 2021/03/07 06:01 [medline]\\nAID - ani11030599 [pii]\\nAID - 10.3390/ani11030599 [doi]\\nPST - epublish\\nSO  - Animals (Basel). 2021 Feb 25;11(3). pii: ani11030599. doi: 10.3390/ani11030599.\\n\\nPMID- 33652915\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20210510\\nLR  - 20210510\\nIS  - 2072-6643 (Electronic)\\nIS  - 2072-6643 (Linking)\\nVI  - 13\\nIP  - 3\\nDP  - 2021 Feb 26\\nTI  - Nutrition and Rheumatoid Arthritis in the \\'Omics\\' Era.\\nLID - 763 [pii]\\nLID - 10.3390/nu13030763 [doi]\\nAB  - Modern high-throughput \\'omics\\' science tools (including genomics,\\n      transcriptomics, proteomics, metabolomics and microbiomics) are currently being\\n      applied to nutritional sciences to unravel the fundamental processes of health\\n      effects ascribed to particular nutrients in humans and to contribute to more\\n      precise nutritional advice. Diet and food components are key environmental\\n      factors that interact with the genome, transcriptome, proteome, metabolome and\\n      the microbiota, and this life-long interplay defines health and diseases state of\\n      the individual. Rheumatoid arthritis (RA) is a chronic autoimmune disease\\n      featured by a systemic immune-inflammatory response, in genetically susceptible\\n      individuals exposed to environmental triggers, including diet. In recent years\\n      increasing evidences suggested that nutritional factors and gut microbiome have a\\n      central role in RA risk and progression. The aim of this review is to summarize\\n      the main and most recent applications of \\'omics\\' technologies in human nutrition \\n      and in RA research, examining the possible influences of some nutrients and\\n      nutritional patterns on RA pathogenesis, following a nutrigenomics approach. The \\n      opportunities and challenges of novel \\'omics technologies\\' in the exploration of \\n      new avenues in RA and nutritional research to prevent and manage RA will be also \\n      discussed.\\nFAU - Cassotta, Manuela\\nAU  - Cassotta M\\nAD  - Research Group on Foods, Nutritional Biochemistry and Health, Universidad Europea\\n      del Atlantico, 39011 Santander, Spain.\\nFAU - Forbes-Hernandez, Tamara Y\\nAU  - Forbes-Hernandez TY\\nAD  - Nutrition and Food Science Group, Department of Analytical and Food Chemistry,\\n      CITACA, CACTI, University of Vigo, 36310 Vigo, Spain.\\nFAU - Cianciosi, Danila\\nAU  - Cianciosi D\\nAUID- ORCID: 0000-0002-8781-3535\\nAD  - Department of Clinical Sciences, Faculty of Medicine, Polytechnic University of\\n      Marche, 60131 Ancona, Italy.\\nFAU - Elexpuru Zabaleta, Maria\\nAU  - Elexpuru Zabaleta M\\nAUID- ORCID: 0000-0003-2997-1049\\nAD  - Research Group on Foods, Nutritional Biochemistry and Health, Universidad Europea\\n      del Atlantico, 39011 Santander, Spain.\\nFAU - Sumalla Cano, Sandra\\nAU  - Sumalla Cano S\\nAD  - Research Group on Foods, Nutritional Biochemistry and Health, Universidad Europea\\n      del Atlantico, 39011 Santander, Spain.\\nFAU - Dominguez, Irma\\nAU  - Dominguez I\\nAD  - Research Group on Foods, Nutritional Biochemistry and Health, Universidad Europea\\n      del Atlantico, 39011 Santander, Spain.\\nFAU - Bullon, Beatriz\\nAU  - Bullon B\\nAD  - Department of Periodontology, Dental School, University of Sevilla, 41004\\n      Sevilla, Spain.\\nFAU - Regolo, Lucia\\nAU  - Regolo L\\nAUID- ORCID: 0000-0001-9432-0408\\nAD  - Department of Clinical Sciences, Faculty of Medicine, Polytechnic University of\\n      Marche, 60131 Ancona, Italy.\\nFAU - Alvarez-Suarez, Jose Miguel\\nAU  - Alvarez-Suarez JM\\nAUID- ORCID: 0000-0001-8509-1498\\nAD  - AgroScience & Food Research Group, Universidad de Las Americas, Quito 170125,\\n      Ecuador.\\nAD  - King Fahd Medical Research Center, King Abdulaziz University, Jedda 21589, Saudi \\n      Arabia.\\nFAU - Giampieri, Francesca\\nAU  - Giampieri F\\nAD  - Department of Clinical Sciences, Faculty of Medicine, Polytechnic University of\\n      Marche, 60131 Ancona, Italy.\\nAD  - Department of Biochemistry, Faculty of Sciences, King Abdulaziz University,\\n      Jeddah 21589, Saudi Arabia.\\nFAU - Battino, Maurizio\\nAU  - Battino M\\nAD  - Department of Clinical Sciences, Faculty of Medicine, Polytechnic University of\\n      Marche, 60131 Ancona, Italy.\\nAD  - International Research Center for Food Nutrition and Safety, Jiangsu University, \\n      Zhenjiang 212013, China.\\nLA  - eng\\nPT  - Journal Article\\nPT  - Review\\nDEP - 20210226\\nPL  - Switzerland\\nTA  - Nutrients\\nJT  - Nutrients\\nJID - 101521595\\nSB  - IM\\nMH  - Arthritis, Rheumatoid/*etiology\\nMH  - Computational Biology/*trends\\nMH  - Gene Expression Profiling/trends\\nMH  - Genomics/trends\\nMH  - Humans\\nMH  - Metabolomics/trends\\nMH  - Microbiota\\nMH  - Nutrigenomics/*trends\\nMH  - Proteomics/trends\\nPMC - PMC7996781\\nOTO - NOTNLM\\nOT  - diet\\nOT  - metabolomics\\nOT  - microbiome\\nOT  - microbiomics\\nOT  - nutrigenomics\\nOT  - proteomics\\nOT  - rheumatoid arthritis\\nEDAT- 2021/03/04 06:00\\nMHDA- 2021/05/11 06:00\\nCRDT- 2021/03/03 01:01\\nPHST- 2021/01/07 00:00 [received]\\nPHST- 2021/02/16 00:00 [revised]\\nPHST- 2021/02/24 00:00 [accepted]\\nPHST- 2021/03/03 01:01 [entrez]\\nPHST- 2021/03/04 06:00 [pubmed]\\nPHST- 2021/05/11 06:00 [medline]\\nAID - nu13030763 [pii]\\nAID - 10.3390/nu13030763 [doi]\\nPST - epublish\\nSO  - Nutrients. 2021 Feb 26;13(3). pii: nu13030763. doi: 10.3390/nu13030763.\\n\\nPMID- 33574059\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20210721\\nLR  - 20210818\\nIS  - 1091-6490 (Electronic)\\nIS  - 0027-8424 (Linking)\\nVI  - 118\\nIP  - 7\\nDP  - 2021 Feb 16\\nTI  - The genomics of ecological flexibility, large brains, and long lives in capuchin \\n      monkeys revealed with fecalFACS.\\nLID - e2010632118 [pii]\\nLID - 10.1073/pnas.2010632118 [doi]\\nAB  - Ecological flexibility, extended lifespans, and large brains have long intrigued \\n      evolutionary biologists, and comparative genomics offers an efficient and\\n      effective tool for generating new insights into the evolution of such traits.\\n      Studies of capuchin monkeys are particularly well situated to shed light on the\\n      selective pressures and genetic underpinnings of local adaptation to diverse\\n      habitats, longevity, and brain development. Distributed widely across Central and\\n      South America, they are inventive and extractive foragers, known for their\\n      sensorimotor intelligence. Capuchins have among the largest relative brain size\\n      of any monkey and a lifespan that exceeds 50 y, despite their small (3 to 5 kg)\\n      body size. We assemble and annotate a de novo reference genome for Cebus imitator\\n      Through high-depth sequencing of DNA derived from blood, various tissues, and\\n      feces via fluorescence-activated cell sorting (fecalFACS) to isolate monkey\\n      epithelial cells, we compared genomes of capuchin populations from tropical dry\\n      forests and lowland rainforests and identified population divergence in genes\\n      involved in water balance, kidney function, and metabolism. Through a comparative\\n      genomics approach spanning a wide diversity of mammals, we identified genes under\\n      positive selection associated with longevity and brain development. Additionally,\\n      we provide a technological advancement in the use of noninvasive genomics for\\n      studies of free-ranging mammals. Our intra- and interspecific comparative study\\n      of capuchin genomics provides insights into processes underlying local adaptation\\n      to diverse and physiologically challenging environments, as well as the molecular\\n      basis of brain evolution and longevity.\\nCI  - Copyright (c) 2021 the Author(s). Published by PNAS.\\nFAU - Orkin, Joseph D\\nAU  - Orkin JD\\nAUID- ORCID: 0000-0001-6922-2072\\nAD  - Department of Anthropology and Archaeology, University of Calgary, Calgary, AB\\n      T2N 1N4, Canada; joseph.orkin@upf.edu amanda.melin@ucalgary.ca.\\nAD  - Institut de Biologia Evolutiva, Universitat Pompeu Fabra, Consejo Superior de\\n      Investigaciones Cientificas, 08003 Barcelona, Spain.\\nAD  - Alberta Children\\'s Hospital Research Institute, University of Calgary, Calgary,\\n      AB T38 6A8, Canada.\\nFAU - Montague, Michael J\\nAU  - Montague MJ\\nAUID- ORCID: 0000-0003-0253-4404\\nAD  - Department of Neuroscience, Perelman School of Medicine, University of\\n      Pennsylvania, Philadelphia, PA 19146.\\nFAU - Tejada-Martinez, Daniela\\nAU  - Tejada-Martinez D\\nAUID- ORCID: 0000-0001-9531-5280\\nAD  - Department of Biochemistry and Molecular Biology, Thomas Jefferson University,\\n      Philadelphia, PA 19107.\\nAD  - Doctorado en Ciencias mencion Ecologia y Evolucion, Instituto de Ciencias\\n      Ambientales y Evolutivas, Facultad de Ciencias, Universidad Austral de Chile,\\n      Valdivia 5090000, Chile.\\nAD  - Integrative Genomics of Ageing Group, Institute of Ageing and Chronic Disease,\\n      University of Liverpool, Liverpool L7 8TX, United Kingdom.\\nFAU - de Manuel, Marc\\nAU  - de Manuel M\\nAD  - Institut de Biologia Evolutiva, Universitat Pompeu Fabra, Consejo Superior de\\n      Investigaciones Cientificas, 08003 Barcelona, Spain.\\nFAU - Del Campo, Javier\\nAU  - Del Campo J\\nAUID- ORCID: 0000-0002-5292-1421\\nAD  - Institut de Biologia Evolutiva, Universitat Pompeu Fabra, Consejo Superior de\\n      Investigaciones Cientificas, 08003 Barcelona, Spain.\\nFAU - Cheves Hernandez, Saul\\nAU  - Cheves Hernandez S\\nAD  - Area de Concervacion Guanacaste, 169-5000 Guanacaste, Costa Rica.\\nFAU - Di Fiore, Anthony\\nAU  - Di Fiore A\\nAUID- ORCID: 0000-0001-8893-9052\\nAD  - Department of Anthropology and Primate Molecular Ecology and Evolution\\n      Laboratory, University of Texas at Austin, Austin, TX 78712.\\nAD  - College of Biological and Environmental Sciences, Universidad San Francisco de\\n      Quito, 170901 Cumbaya, Ecuador.\\nFAU - Fontsere, Claudia\\nAU  - Fontsere C\\nAUID- ORCID: 0000-0003-2233-6026\\nAD  - Institut de Biologia Evolutiva, Universitat Pompeu Fabra, Consejo Superior de\\n      Investigaciones Cientificas, 08003 Barcelona, Spain.\\nFAU - Hodgson, Jason A\\nAU  - Hodgson JA\\nAUID- ORCID: 0000-0002-3849-2245\\nAD  - Department of Anthropology, Pennsylvania State University, University Park, PA\\n      16802.\\nAD  - Department of Zoology, University of Cambridge, Cambridge CB2 3EJ, United\\n      Kingdom.\\nFAU - Janiak, Mareike C\\nAU  - Janiak MC\\nAUID- ORCID: 0000-0002-7759-2556\\nAD  - Department of Anthropology and Archaeology, University of Calgary, Calgary, AB\\n      T2N 1N4, Canada.\\nAD  - Alberta Children\\'s Hospital Research Institute, University of Calgary, Calgary,\\n      AB T38 6A8, Canada.\\nAD  - School of Science, Engineering and Environment, University of Salford, Salford M5\\n      4WT, United Kingdom.\\nFAU - Kuderna, Lukas F K\\nAU  - Kuderna LFK\\nAUID- ORCID: 0000-0002-9992-9295\\nAD  - Institut de Biologia Evolutiva, Universitat Pompeu Fabra, Consejo Superior de\\n      Investigaciones Cientificas, 08003 Barcelona, Spain.\\nFAU - Lizano, Esther\\nAU  - Lizano E\\nAUID- ORCID: 0000-0003-3304-9807\\nAD  - Institut de Biologia Evolutiva, Universitat Pompeu Fabra, Consejo Superior de\\n      Investigaciones Cientificas, 08003 Barcelona, Spain.\\nAD  - Institut Catala de Paleontologia Miquel Crusafont, Universitat Autonoma de\\n      Barcelona, 08193 Cerdanyola del Valles, Barcelona, Spain.\\nFAU - Martin, Maria Pia\\nAU  - Martin MP\\nAD  - Kids Saving the Rainforest Wildlife Rescue Center, 60601 Quepos, Costa Rica.\\nFAU - Niimura, Yoshihito\\nAU  - Niimura Y\\nAUID- ORCID: 0000-0001-7010-2537\\nAD  - Department of Applied Biological Chemistry, Graduate School of Agricultural and\\n      Life Sciences, The University of Tokyo, Tokyo 113-8657, Japan.\\nFAU - Perry, George H\\nAU  - Perry GH\\nAUID- ORCID: 0000-0003-4527-3806\\nAD  - Department of Anthropology, Pennsylvania State University, University Park, PA\\n      16802.\\nAD  - Department of Biology, Pennsylvania State University, University Park, PA 16802.\\nFAU - Valverde, Carmen Soto\\nAU  - Valverde CS\\nAUID- ORCID: 0000-0001-9506-5429\\nAD  - Kids Saving the Rainforest Wildlife Rescue Center, 60601 Quepos, Costa Rica.\\nFAU - Tang, Jia\\nAU  - Tang J\\nAUID- ORCID: 0000-0002-6374-0036\\nAD  - Department of Anthropology and Archaeology, University of Calgary, Calgary, AB\\n      T2N 1N4, Canada.\\nFAU - Warren, Wesley C\\nAU  - Warren WC\\nAUID- ORCID: 0000-0003-2255-2730\\nAD  - Division of Animal Sciences, School of Medicine, University of Missouri,\\n      Columbia, MO 65211.\\nFAU - de Magalhaes, Joao Pedro\\nAU  - de Magalhaes JP\\nAUID- ORCID: 0000-0002-6363-2465\\nAD  - Integrative Genomics of Ageing Group, Institute of Ageing and Chronic Disease,\\n      University of Liverpool, Liverpool L7 8TX, United Kingdom.\\nFAU - Kawamura, Shoji\\nAU  - Kawamura S\\nAUID- ORCID: 0000-0003-0350-6050\\nAD  - Department of Integrated Biosciences, Graduate School of Frontier Sciences, The\\n      University of Tokyo, 277-8562 Chiba, Japan.\\nFAU - Marques-Bonet, Tomas\\nAU  - Marques-Bonet T\\nAUID- ORCID: 0000-0002-5597-3075\\nAD  - Institut de Biologia Evolutiva, Universitat Pompeu Fabra, Consejo Superior de\\n      Investigaciones Cientificas, 08003 Barcelona, Spain.\\nAD  - Institut Catala de Paleontologia Miquel Crusafont, Universitat Autonoma de\\n      Barcelona, 08193 Cerdanyola del Valles, Barcelona, Spain.\\nAD  - Catalan Institution of Research and Advanced Studies, 08010 Barcelona, Spain.\\nAD  - Centro Nacional de Analisis Genomico-Centre for Genomic Regulation, Barcelona\\n      Institute of Science and Technology, 08028 Barcelona, Spain.\\nFAU - Krawetz, Roman\\nAU  - Krawetz R\\nAUID- ORCID: 0000-0002-2576-4504\\nAD  - Department of Cell Biology and Anatomy, Cumming School of Medicine, University of\\n      Calgary, Calgary, AB T38 6A8, Canada.\\nFAU - Melin, Amanda D\\nAU  - Melin AD\\nAUID- ORCID: 0000-0002-0612-2514\\nAD  - Department of Anthropology and Archaeology, University of Calgary, Calgary, AB\\n      T2N 1N4, Canada; joseph.orkin@upf.edu amanda.melin@ucalgary.ca.\\nAD  - Alberta Children\\'s Hospital Research Institute, University of Calgary, Calgary,\\n      AB T38 6A8, Canada.\\nAD  - Department of Medical Genetics, Cumming School of Medicine, University of\\n      Calgary, Calgary, AB T38 6A8, Canada.\\nLA  - eng\\nGR  - BB/R014949/1/BB_/Biotechnology and Biological Sciences Research Council/United\\n      Kingdom\\nPT  - Journal Article\\nPT  - Research Support, Non-U.S. Gov\\'t\\nPL  - United States\\nTA  - Proc Natl Acad Sci U S A\\nJT  - Proceedings of the National Academy of Sciences of the United States of America\\nJID - 7505876\\nSB  - IM\\nEIN - Proc Natl Acad Sci U S A. 2021 Aug 24;118(34):. PMID: 34400510\\nMH  - *Adaptation, Physiological\\nMH  - Animals\\nMH  - Brain/*growth & development\\nMH  - Cebus/*genetics\\nMH  - Evolution, Molecular\\nMH  - Flow Cytometry/methods\\nMH  - Forests\\nMH  - *Genome\\nMH  - Genomics/methods\\nMH  - Longevity/*genetics\\nPMC - PMC7896301\\nOTO - NOTNLM\\nOT  - *brain size\\nOT  - *flow cytometry\\nOT  - *longevity\\nOT  - *noninvasive genomics\\nOT  - *seasonality\\nCOIS- The authors declare no competing interest.\\nEDAT- 2021/02/13 06:00\\nMHDA- 2021/07/22 06:00\\nCRDT- 2021/02/12 05:52\\nPHST- 2021/02/12 05:52 [entrez]\\nPHST- 2021/02/13 06:00 [pubmed]\\nPHST- 2021/07/22 06:00 [medline]\\nAID - 2010632118 [pii]\\nAID - 10.1073/pnas.2010632118 [doi]\\nPST - ppublish\\nSO  - Proc Natl Acad Sci U S A. 2021 Feb 16;118(7). pii: 2010632118. doi:\\n      10.1073/pnas.2010632118.\\n\\nPMID- 33544737\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20210701\\nLR  - 20210701\\nIS  - 1932-6203 (Electronic)\\nIS  - 1932-6203 (Linking)\\nVI  - 16\\nIP  - 2\\nDP  - 2021\\nTI  - Genetic characterization of a mild isolate of papaya ringspot virus type-P\\n      (PRSV-P) and assessment of its cross-protection potential under greenhouse and\\n      field conditions.\\nPG  - e0241652\\nLID - 10.1371/journal.pone.0241652 [doi]\\nAB  - A mild isolate of Papaya ringspot virus type-P, abbreviated as PRSV-mild, from\\n      Ecuador was sequenced and characterized. The most distinguishing symptom induced \\n      by PRSV-mild was gray powder-like leaf patches radiating from secondary veins. In\\n      greenhouse experiments, PRSV-mild did not confer durable protection against a\\n      severe isolate of the virus (PRSV-sev), obtained from the same field.\\n      Furthermore, isolate specific detection in mixed-infected plants showed that\\n      PRSV-sev becomes dominant in infections, rendering PRSV-mild undetectable at\\n      90-120 days post superinfection. Virus testing using isolate-specific primers\\n      detected PRSV-mild in two out of five surveyed provinces, with 10% and 48% of\\n      incidence in Santo Domingo and Los Rios, respectively. Comparative genomics\\n      showed that PRSV-mild lacks two amino acids from the coat protein region, whereas\\n      amino acid determinants for asymptomatic phenotypes were not identified.\\n      Recombination events were not predicted in the genomes of the Ecuadorean\\n      isolates. Phylogenetic analyses placed both PRSV-mild and PRSV-sev in a clade\\n      that includes an additional PRSV isolate from Ecuador and others from South\\n      America.\\nFAU - Medina-Salguero, Andres X\\nAU  - Medina-Salguero AX\\nAD  - Centro de Investigaciones Biotecnologicas del Ecuador, Escuela Superior\\n      Politecnica del Litoral, ESPOL, Guayaquil, Guayas, Ecuador.\\nFAU - Cornejo-Franco, Juan F\\nAU  - Cornejo-Franco JF\\nAD  - Centro de Investigaciones Biotecnologicas del Ecuador, Escuela Superior\\n      Politecnica del Litoral, ESPOL, Guayaquil, Guayas, Ecuador.\\nFAU - Grinstead, Sam\\nAU  - Grinstead S\\nAUID- ORCID: 0000-0002-1708-8235\\nAD  - National Germplasm Resources Laboratory, USDA-ARS, Beltsville, MD, United States \\n      of America.\\nFAU - Mowery, Joseph\\nAU  - Mowery J\\nAUID- ORCID: 0000-0001-7128-1397\\nAD  - Electron and Confocal Microscopy Unit, USDA ARS, Beltsville, MD, United States of\\n      America.\\nFAU - Mollov, Dimitre\\nAU  - Mollov D\\nAD  - National Germplasm Resources Laboratory, USDA-ARS, Beltsville, MD, United States \\n      of America.\\nFAU - Quito-Avila, Diego F\\nAU  - Quito-Avila DF\\nAUID- ORCID: 0000-0002-6170-1113\\nAD  - Centro de Investigaciones Biotecnologicas del Ecuador, Escuela Superior\\n      Politecnica del Litoral, ESPOL, Guayaquil, Guayas, Ecuador.\\nAD  - Facultad de Ciencias de la Vida, Escuela Superior Politecnica del Litoral, ESPOL,\\n      Guayaquil, Guayas, Ecuador.\\nLA  - eng\\nPT  - Journal Article\\nPT  - Research Support, Non-U.S. Gov\\'t\\nDEP - 20210205\\nPL  - United States\\nTA  - PLoS One\\nJT  - PloS one\\nJID - 101285081\\nRN  - Papaya ringspot virus\\nSB  - IM\\nMH  - Carica/*virology\\nMH  - Genome, Viral\\nMH  - Phylogeny\\nMH  - Plant Diseases/*virology\\nMH  - Potyvirus/*genetics/isolation & purification\\nPMC - PMC7864462\\nCOIS- The authors have declared that no competing interests exist.\\nEDAT- 2021/02/06 06:00\\nMHDA- 2021/07/02 06:00\\nCRDT- 2021/02/05 17:12\\nPHST- 2020/10/15 00:00 [received]\\nPHST- 2021/01/20 00:00 [accepted]\\nPHST- 2021/02/05 17:12 [entrez]\\nPHST- 2021/02/06 06:00 [pubmed]\\nPHST- 2021/07/02 06:00 [medline]\\nAID - 10.1371/journal.pone.0241652 [doi]\\nAID - PONE-D-20-32443 [pii]\\nPST - epublish\\nSO  - PLoS One. 2021 Feb 5;16(2):e0241652. doi: 10.1371/journal.pone.0241652.\\n      eCollection 2021.\\n\\nPMID- 33503026\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20210514\\nLR  - 20210514\\nIS  - 1553-7358 (Electronic)\\nIS  - 1553-734X (Linking)\\nVI  - 17\\nIP  - 1\\nDP  - 2021 Jan\\nTI  - One is not enough: On the effects of reference genome for the mapping and\\n      subsequent analyses of short-reads.\\nPG  - e1008678\\nLID - 10.1371/journal.pcbi.1008678 [doi]\\nAB  - Mapping of high-throughput sequencing (HTS) reads to a single arbitrary reference\\n      genome is a frequently used approach in microbial genomics. However, the choice\\n      of a reference may represent a source of errors that may affect subsequent\\n      analyses such as the detection of single nucleotide polymorphisms (SNPs) and\\n      phylogenetic inference. In this work, we evaluated the effect of reference choice\\n      on short-read sequence data from five clinically and epidemiologically relevant\\n      bacteria (Klebsiella pneumoniae, Legionella pneumophila, Neisseria gonorrhoeae,\\n      Pseudomonas aeruginosa and Serratia marcescens). Publicly available whole-genome \\n      assemblies encompassing the genomic diversity of these species were selected as\\n      reference sequences, and read alignment statistics, SNP calling, recombination\\n      rates, dN/dS ratios, and phylogenetic trees were evaluated depending on the\\n      mapping reference. The choice of different reference genomes proved to have an\\n      impact on almost all the parameters considered in the five species. In addition, \\n      these biases had potential epidemiological implications such as\\n      including/excluding isolates of particular clades and the estimation of genetic\\n      distances. These findings suggest that the single reference approach might\\n      introduce systematic errors during mapping that affect subsequent analyses,\\n      particularly for data sets with isolates from genetically diverse backgrounds. In\\n      any case, exploring the effects of different references on the final conclusions \\n      is highly recommended.\\nFAU - Valiente-Mullor, Carlos\\nAU  - Valiente-Mullor C\\nAUID- ORCID: 0000-0003-4009-6759\\nAD  - Joint Research Unit \"Infection and Public Health\" FISABIO-University of Valencia,\\n      Institute for Integrative Systems Biology (I2SysBio), Valencia, Spain.\\nFAU - Beamud, Beatriz\\nAU  - Beamud B\\nAUID- ORCID: 0000-0003-0863-3390\\nAD  - Joint Research Unit \"Infection and Public Health\" FISABIO-University of Valencia,\\n      Institute for Integrative Systems Biology (I2SysBio), Valencia, Spain.\\nFAU - Ansari, Ivan\\nAU  - Ansari I\\nAUID- ORCID: 0000-0003-0545-3837\\nAD  - Joint Research Unit \"Infection and Public Health\" FISABIO-University of Valencia,\\n      Institute for Integrative Systems Biology (I2SysBio), Valencia, Spain.\\nFAU - Frances-Cuesta, Carlos\\nAU  - Frances-Cuesta C\\nAUID- ORCID: 0000-0003-4195-054X\\nAD  - Joint Research Unit \"Infection and Public Health\" FISABIO-University of Valencia,\\n      Institute for Integrative Systems Biology (I2SysBio), Valencia, Spain.\\nFAU - Garcia-Gonzalez, Neris\\nAU  - Garcia-Gonzalez N\\nAUID- ORCID: 0000-0001-9089-4033\\nAD  - Joint Research Unit \"Infection and Public Health\" FISABIO-University of Valencia,\\n      Institute for Integrative Systems Biology (I2SysBio), Valencia, Spain.\\nFAU - Mejia, Lorena\\nAU  - Mejia L\\nAUID- ORCID: 0000-0002-7600-6543\\nAD  - Joint Research Unit \"Infection and Public Health\" FISABIO-University of Valencia,\\n      Institute for Integrative Systems Biology (I2SysBio), Valencia, Spain.\\nAD  - Instituto de Microbiologia, Colegio de Ciencias Biologicas y Ambientales,\\n      Universidad San Francisco de Quito, Quito, Ecuador.\\nFAU - Ruiz-Hueso, Paula\\nAU  - Ruiz-Hueso P\\nAUID- ORCID: 0000-0003-0217-471X\\nAD  - Joint Research Unit \"Infection and Public Health\" FISABIO-University of Valencia,\\n      Institute for Integrative Systems Biology (I2SysBio), Valencia, Spain.\\nFAU - Gonzalez-Candelas, Fernando\\nAU  - Gonzalez-Candelas F\\nAUID- ORCID: 0000-0002-0879-5798\\nAD  - Joint Research Unit \"Infection and Public Health\" FISABIO-University of Valencia,\\n      Institute for Integrative Systems Biology (I2SysBio), Valencia, Spain.\\nAD  - CIBER in Epidemiology and Public Health, Valencia, Spain.\\nLA  - eng\\nPT  - Journal Article\\nPT  - Research Support, Non-U.S. Gov\\'t\\nDEP - 20210127\\nPL  - United States\\nTA  - PLoS Comput Biol\\nJT  - PLoS computational biology\\nJID - 101238922\\nSB  - IM\\nMH  - Bacteria/classification/genetics\\nMH  - Chromosome Mapping/*methods/*standards\\nMH  - Genome, Bacterial/genetics\\nMH  - Genomics/*methods\\nMH  - High-Throughput Nucleotide Sequencing/*methods\\nMH  - Phylogeny\\nMH  - Polymorphism, Single Nucleotide/genetics\\nMH  - Sequence Alignment\\nPMC - PMC7870062\\nCOIS- The authors have declared that no competing interests exist.\\nEDAT- 2021/01/28 06:00\\nMHDA- 2021/05/15 06:00\\nCRDT- 2021/01/27 17:12\\nPHST- 2020/04/27 00:00 [received]\\nPHST- 2021/01/05 00:00 [accepted]\\nPHST- 2021/02/08 00:00 [revised]\\nPHST- 2021/01/28 06:00 [pubmed]\\nPHST- 2021/05/15 06:00 [medline]\\nPHST- 2021/01/27 17:12 [entrez]\\nAID - 10.1371/journal.pcbi.1008678 [doi]\\nAID - PCOMPBIOL-D-20-00703 [pii]\\nPST - epublish\\nSO  - PLoS Comput Biol. 2021 Jan 27;17(1):e1008678. doi: 10.1371/journal.pcbi.1008678. \\n      eCollection 2021 Jan.\\n\\nPMID- 33294020\\nOWN - NLM\\nSTAT- PubMed-not-MEDLINE\\nLR  - 20201210\\nIS  - 1752-4571 (Print)\\nIS  - 1752-4571 (Linking)\\nVI  - 13\\nIP  - 10\\nDP  - 2020 Dec\\nTI  - Host plants and Wolbachia shape the population genetics of sympatric herbivore\\n      populations.\\nPG  - 2740-2753\\nLID - 10.1111/eva.13079 [doi]\\nAB  - Changing climate and land-use practices have the potential to bring previously\\n      isolated populations of pest insects into new sympatry. This heightens the need\\n      to better understand how differing patterns of host-plant association, and unique\\n      endosymbionts, serve to promote genetic isolation or integration. We addressed\\n      these factors in populations of potato psyllid, Bactericera cockerelli (Sulc), a \\n      generalist herbivore that vectors a bacterial pathogen (Candidatus Liberibacter\\n      solanacearum, causal pathogen of zebra chip disease) of potato (Solanum tuberosum\\n      L.). Genome-wide SNP data revealed two major genetic clusters-psyllids collected \\n      from potato crops were genetically similar to psyllids found on a common weed,\\n      Lycium spp., but dissimilar from those found on another common non-crop host,\\n      Solanum dulcamara L. Most psyllids found on Lycium spp. and potato represented a \\n      single mitochondrial cytochrome oxidase I (COI) haplotype that has been suggested\\n      to not be native to the region, and whose arrival may have been concurrent with\\n      zebra chip disease first emerging. The putatively introduced COI haplotype\\n      usually co-occurred with endosymbiotic Wolbachia, while the putatively resident\\n      COI haplotype generally did not. Genetic intermediates between the two genetic\\n      populations of insects were rare, consistent with recent sympatry or reproductive\\n      isolation, although admixture patterns of apparent hybrids were consistent with\\n      introgression of genes from introduced into resident populations. Our results\\n      suggest that both host-plant associations and endosymbionts are shaping the\\n      population genetic structure of sympatric psyllid populations associated with\\n      different non-crop hosts. It is of future interest to explicitly examine\\n      vectorial capacity of the two populations and their potential hybrids, as\\n      population structure and hybridization might alter regional vector capacity and\\n      disease outbreaks.\\nCI  - (c) 2020 The Authors. Evolutionary Applications published by John Wiley & Sons\\n      Ltd.\\nFAU - Fu, Zhen\\nAU  - Fu Z\\nAUID- ORCID: https://orcid.org/0000-0002-8120-590X\\nAD  - Department of Entomology Washington State University Pullman WA USA.\\nAD  - Present address: Department of Entomology Texas A&M University College Station TX\\n      USA.\\nFAU - Meier, Amanda R\\nAU  - Meier AR\\nAUID- ORCID: https://orcid.org/0000-0003-1085-4715\\nAD  - Department of Entomology University of Georgia Athens GA USA.\\nFAU - Epstein, Brendan\\nAU  - Epstein B\\nAUID- ORCID: https://orcid.org/0000-0001-7083-1588\\nAD  - Department of Plant and Microbial Biology University of Minnesota St. Paul MN\\n      USA.\\nFAU - Bergland, Alan O\\nAU  - Bergland AO\\nAUID- ORCID: https://orcid.org/0000-0001-7145-7575\\nAD  - Department of Biology University of Virginia Charlottesville VA USA.\\nFAU - Castillo Carrillo, Carmen I\\nAU  - Castillo Carrillo CI\\nAUID- ORCID: https://orcid.org/0000-0001-7893-5218\\nAD  - Departamento de Proteccion Vegetal Estacion Experimental Santa Catalina Instituto\\n      Nacional de Investigaciones Agropecuarias (INIAP) Quito Ecuador.\\nFAU - Cooper, William R\\nAU  - Cooper WR\\nAD  - Temperate Tree Fruit and Vegetable Research USDA-ARS Wapato WA USA.\\nFAU - Cruzado, Regina K\\nAU  - Cruzado RK\\nAD  - Department of Entomology, Plant Pathology, and Nematology University of Idaho\\n      Moscow ID USA.\\nFAU - Horton, David R\\nAU  - Horton DR\\nAD  - Temperate Tree Fruit and Vegetable Research USDA-ARS Wapato WA USA.\\nFAU - Jensen, Andrew S\\nAU  - Jensen AS\\nAUID- ORCID: https://orcid.org/0000-0002-6675-4185\\nAD  - Northwest Potato Research Consortium Lakeview OR USA.\\nFAU - Kelley, Joanna L\\nAU  - Kelley JL\\nAUID- ORCID: https://orcid.org/0000-0002-7731-605X\\nAD  - School of Biological Sciences Washington State University Pullman WA USA.\\nFAU - Rashed, Arash\\nAU  - Rashed A\\nAUID- ORCID: https://orcid.org/0000-0001-6640-2457\\nAD  - Department of Entomology, Plant Pathology, and Nematology University of Idaho\\n      Moscow ID USA.\\nFAU - Reitz, Stuart R\\nAU  - Reitz SR\\nAD  - Malheur Experiment Station Oregon State University Ontario OR USA.\\nFAU - Rondon, Silvia I\\nAU  - Rondon SI\\nAUID- ORCID: https://orcid.org/0000-0002-7819-6934\\nAD  - Department of Crop and Soil Science Hermiston Agricultural Research and Extension\\n      Center Hermiston OR USA.\\nFAU - Thinakaran, Jenita\\nAU  - Thinakaran J\\nAD  - Temperate Tree Fruit and Vegetable Research USDA-ARS Wapato WA USA.\\nFAU - Wenninger, Erik J\\nAU  - Wenninger EJ\\nAD  - Department of Entomology, Plant Pathology, and Nematology Kimberly Research and\\n      Extension Center University of Idaho Kimberly ID USA.\\nFAU - Wohleb, Carrie H\\nAU  - Wohleb CH\\nAD  - Washington State University Extension Moses Lake WA USA.\\nFAU - Crowder, David W\\nAU  - Crowder DW\\nAD  - Department of Entomology Washington State University Pullman WA USA.\\nFAU - Snyder, William E\\nAU  - Snyder WE\\nAUID- ORCID: https://orcid.org/0000-0002-2747-3102\\nAD  - Department of Entomology University of Georgia Athens GA USA.\\nLA  - eng\\nPT  - Journal Article\\nDEP - 20200912\\nPL  - England\\nTA  - Evol Appl\\nJT  - Evolutionary applications\\nJID - 101461828\\nPMC - PMC7691456\\nOTO - NOTNLM\\nOT  - Wolbachia\\nOT  - herbivore interactions\\nOT  - interbreeding\\nOT  - pest management\\nOT  - plant\\nOT  - population genomics\\nEDAT- 2020/12/10 06:00\\nMHDA- 2020/12/10 06:01\\nCRDT- 2020/12/09 06:13\\nPHST- 2020/01/16 00:00 [received]\\nPHST- 2020/07/17 00:00 [revised]\\nPHST- 2020/07/21 00:00 [accepted]\\nPHST- 2020/12/09 06:13 [entrez]\\nPHST- 2020/12/10 06:00 [pubmed]\\nPHST- 2020/12/10 06:01 [medline]\\nAID - 10.1111/eva.13079 [doi]\\nAID - EVA13079 [pii]\\nPST - epublish\\nSO  - Evol Appl. 2020 Sep 12;13(10):2740-2753. doi: 10.1111/eva.13079. eCollection 2020\\n      Dec.\\n\\nPMID- 33080926\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20210716\\nLR  - 20210716\\nIS  - 2073-4425 (Electronic)\\nIS  - 2073-4425 (Linking)\\nVI  - 11\\nIP  - 10\\nDP  - 2020 Oct 17\\nTI  - Landscape Genomics of a Widely Distributed Snake, Dolichophis caspius (Gmelin,\\n      1789) across Eastern Europe and Western Asia.\\nLID - E1218 [pii]\\nLID - 10.3390/genes11101218 [doi]\\nAB  - Across the distribution of the Caspian whipsnake (Dolichophis caspius),\\n      populations have become increasingly disconnected due to habitat alteration. To\\n      understand population dynamics and this widespread but locally endangered snake\\'s\\n      adaptive potential, we investigated population structure, admixture, and\\n      effective migration patterns. We took a landscape-genomic approach to identify\\n      selected genotypes associated with environmental variables relevant to D.\\n      caspius. With double-digest restriction-site associated DNA (ddRAD) sequencing of\\n      53 samples resulting in 17,518 single nucleotide polymorphisms (SNPs), we\\n      identified 8 clusters within D. caspius reflecting complex evolutionary patterns \\n      of the species. Estimated Effective Migration Surfaces (EEMS) revealed\\n      higher-than-average gene flow in most of the Balkan Peninsula and\\n      lower-than-average gene flow along the middle section of the Danube River.\\n      Landscape genomic analysis identified 751 selected genotypes correlated with 7\\n      climatic variables. Isothermality correlated with the highest number of selected \\n      genotypes (478) located in 41 genes, followed by annual range (127) and annual\\n      mean temperature (87). We conclude that environmental variables, especially the\\n      day-to-night temperature oscillation in comparison to the summer-to-winter\\n      oscillation, may have an important role in the distribution and adaptation of D. \\n      caspius.\\nFAU - Mahtani-Williams, Sarita\\nAU  - Mahtani-Williams S\\nAD  - Research Institute of Wildlife Ecology, Vetmeduni Vienna, Savoyenstrasse 1,\\n      A-1160 Vienna, Austria.\\nAD  - Cardiff School of Biosciences, Cardiff University, The Sir Martin Evans Building,\\n      Museum Ave, Cardiff CF103AX, UK.\\nAD  - Fundacion Charles Darwin, Avenida Charles Darwin s/n, Casilla 200144, Puerto\\n      Ayora EC-200350, Ecuador.\\nFAU - Fulton, William\\nAU  - Fulton W\\nAUID- ORCID: 0000-0002-2803-5008\\nAD  - Research Institute of Wildlife Ecology, Vetmeduni Vienna, Savoyenstrasse 1,\\n      A-1160 Vienna, Austria.\\nAD  - Cardiff School of Biosciences, Cardiff University, The Sir Martin Evans Building,\\n      Museum Ave, Cardiff CF103AX, UK.\\nFAU - Desvars-Larrive, Amelie\\nAU  - Desvars-Larrive A\\nAUID- ORCID: 0000-0001-7671-696X\\nAD  - Research Institute of Wildlife Ecology, Vetmeduni Vienna, Savoyenstrasse 1,\\n      A-1160 Vienna, Austria.\\nAD  - Institute of Food Safety, Food Technology and Veterinary Public Health, Vetmeduni\\n      Vienna, Veterinaerplatz 1, A-1210 Vienna, Austria.\\nAD  - Complexity Science Hub Vienna, Josefstadter Strasse 39, A-1080 Vienna, Austria.\\nFAU - Lado, Sara\\nAU  - Lado S\\nAD  - Research Institute of Wildlife Ecology, Vetmeduni Vienna, Savoyenstrasse 1,\\n      A-1160 Vienna, Austria.\\nFAU - Elbers, Jean Pierre\\nAU  - Elbers JP\\nAD  - Research Institute of Wildlife Ecology, Vetmeduni Vienna, Savoyenstrasse 1,\\n      A-1160 Vienna, Austria.\\nFAU - Halpern, Balint\\nAU  - Halpern B\\nAD  - MME Birdlife Hungary, Kolto utca 21., H-1121 Budapest, Hungary.\\nFAU - Herczeg, David\\nAU  - Herczeg D\\nAUID- ORCID: 0000-0001-5098-961X\\nAD  - Lendulet Evolutionary Ecology Research Group, Centre for Agricultural Research,\\n      Plant Protection Institute, Herman Otto ut 15., H-1022 Budapest, Hungary.\\nFAU - Babocsay, Gergely\\nAU  - Babocsay G\\nAD  - MME Birdlife Hungary, Kolto utca 21., H-1121 Budapest, Hungary.\\nAD  - Matra Museum of the Hungarian Natural History Museum, Kossuth Lajos utca 40.,\\n      H-3200 Gyongyos, Hungary.\\nFAU - Laus, Boris\\nAU  - Laus B\\nAD  - Association HYLA, Lipocac I., No. 7, C-10000 Zagreb, Croatia.\\nFAU - Nagy, Zoltan Tamas\\nAU  - Nagy ZT\\nAUID- ORCID: 0000-0002-8455-040X\\nAD  - Independent Researcher, Hielscherstrasse 25, D-13158 Berlin, Germany.\\nFAU - Jablonski, Daniel\\nAU  - Jablonski D\\nAUID- ORCID: 0000-0002-5394-0114\\nAD  - Department of Zoology, Comenius University in Bratislava, Ilkovicova 6, Mlynska\\n      Dolina, S-84215 Bratislava, Slovakia.\\nFAU - Kukushkin, Oleg\\nAU  - Kukushkin O\\nAD  - Department of Biodiversity Studies and Ecological Monitoring, T. I. Vyazemsky\\n      Karadag Scientific Station-Nature Reserve-Branch of Institute of Biology of the\\n      Southern Seas of the Russian Academy of Sciences, Nauki Street 24, R-298188\\n      Theodosia, Crimea.\\nAD  - Department of Herpetology, Zoological Institute of the Russian Academy of\\n      Sciences, Universitetskaya Embankment 1, R-199034 Saint Petersburg, Russia.\\nFAU - Orozco-terWengel, Pablo\\nAU  - Orozco-terWengel P\\nAUID- ORCID: 0000-0002-7951-4148\\nAD  - Cardiff School of Biosciences, Cardiff University, The Sir Martin Evans Building,\\n      Museum Ave, Cardiff CF103AX, UK.\\nFAU - Voros, Judit\\nAU  - Voros J\\nAD  - Department of Zoology, Hungarian Natural History Museum, Baross u. 13., H-1088\\n      Budapest, Hungary.\\nAD  - Molecular Taxonomy Laboratory, Hungarian Natural History Museum, Ludovika ter\\n      2-6., H-1083 Budapest, Hungary.\\nFAU - Burger, Pamela Anna\\nAU  - Burger PA\\nAUID- ORCID: 0000-0002-6941-0257\\nAD  - Research Institute of Wildlife Ecology, Vetmeduni Vienna, Savoyenstrasse 1,\\n      A-1160 Vienna, Austria.\\nLA  - eng\\nSI  - Dryad/10.5061/dryad.mgqnk98xm\\nPT  - Journal Article\\nPT  - Research Support, Non-U.S. Gov\\'t\\nDEP - 20201017\\nPL  - Switzerland\\nTA  - Genes (Basel)\\nJT  - Genes\\nJID - 101551097\\nRN  - 0 (Reptilian Proteins)\\nSB  - IM\\nMH  - Adaptation, Physiological\\nMH  - Animals\\nMH  - Asia, Western\\nMH  - Biological Evolution\\nMH  - Europe, Eastern\\nMH  - Gene Flow\\nMH  - *Genetic Variation\\nMH  - *Genetics, Population\\nMH  - *Genome\\nMH  - Genomics/*methods\\nMH  - Genotype\\nMH  - Reptilian Proteins/*genetics\\nMH  - Snakes/*genetics\\nPMC - PMC7603136\\nOTO - NOTNLM\\nOT  - *Caspian whipsnake\\nOT  - *adaptive evolution\\nOT  - *ddRAD\\nOT  - *environmental correlates\\nOT  - *genetic diversity\\nEDAT- 2020/10/22 06:00\\nMHDA- 2021/07/17 06:00\\nCRDT- 2020/10/21 01:01\\nPHST- 2020/08/31 00:00 [received]\\nPHST- 2020/10/02 00:00 [revised]\\nPHST- 2020/10/15 00:00 [accepted]\\nPHST- 2020/10/21 01:01 [entrez]\\nPHST- 2020/10/22 06:00 [pubmed]\\nPHST- 2021/07/17 06:00 [medline]\\nAID - genes11101218 [pii]\\nAID - 10.3390/genes11101218 [doi]\\nPST - epublish\\nSO  - Genes (Basel). 2020 Oct 17;11(10). pii: genes11101218. doi:\\n      10.3390/genes11101218.\\n\\nPMID- 32807182\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20210510\\nLR  - 20210510\\nIS  - 1755-8794 (Electronic)\\nIS  - 1755-8794 (Linking)\\nVI  - 13\\nIP  - 1\\nDP  - 2020 Aug 17\\nTI  - Clinical, genomics and networking analyses of a high-altitude native American\\n      Ecuadorian patient with congenital insensitivity to pain with anhidrosis: a case \\n      report.\\nPG  - 113\\nLID - 10.1186/s12920-020-00764-3 [doi]\\nAB  - BACKGROUND: Congenital insensitivity to pain with anhidrosis (CIPA) is an\\n      extremely rare autosomal recessive disorder characterized by insensitivity to\\n      pain, inability to sweat and intellectual disability. CIPA is caused by mutations\\n      in the neurotrophic tyrosine kinase receptor type 1 gene (NTRK1) that encodes the\\n      high-affinity receptor of nerve growth factor (NGF). CASE PRESENTATION: Here, we \\n      present clinical and molecular findings in a 9-year-old girl with CIPA. The\\n      high-altitude indigenous Ecuadorian patient presented several health problems\\n      such as anhidrosis, bone fractures, self-mutilation, osteochondroma, intellectual\\n      disability and Riga-Fede disease. After the mutational analysis of NTRK1, the\\n      patient showed a clearly autosomal recessive inheritance pattern with the\\n      pathogenic mutation rs763758904 (Arg602*) and the second missense mutation\\n      rs80356677 (Asp674Tyr). Additionally, the genomic analysis showed 69 pathogenic\\n      and/or likely pathogenic variants in 46 genes possibly related to phenotypic\\n      heterogeneity, including the rs324420 variant in the FAAH gene. The gene ontology\\n      enrichment analysis showed 28 mutated genes involved in several biological\\n      processes. As a novel contribution, the protein-protein interaction network\\n      analysis showed that NTRK1, SPTBN2 and GRM6 interact with several proteins of the\\n      pain matrix involved in the response to stimulus and nervous system development. \\n      CONCLUSIONS: This is the first study that associates clinical, genomics and\\n      networking analyses in a Native American patient with consanguinity background in\\n      order to better understand CIPA pathogenesis.\\nFAU - Lopez-Cortes, Andres\\nAU  - Lopez-Cortes A\\nAUID- ORCID: 0000-0003-1503-1929\\nAD  - Centro de Investigacion Genetica y Genomica. Facultad de Ciencias de la Salud\\n      Eugenio Espejo, Universidad UTE, Mariscal Sucre Avenue, 170129, Quito, Ecuador.\\n      aalc84@gmail.com.\\nAD  - Latin American Network for Implementation and Validation of Clinical\\n      Pharmacogenomics Guidelines (RELIVAF-CYTED), Madrid, Spain. aalc84@gmail.com.\\nFAU - Zambrano, Ana Karina\\nAU  - Zambrano AK\\nAD  - Centro de Investigacion Genetica y Genomica. Facultad de Ciencias de la Salud\\n      Eugenio Espejo, Universidad UTE, Mariscal Sucre Avenue, 170129, Quito, Ecuador.\\nFAU - Guevara-Ramirez, Patricia\\nAU  - Guevara-Ramirez P\\nAD  - Centro de Investigacion Genetica y Genomica. Facultad de Ciencias de la Salud\\n      Eugenio Espejo, Universidad UTE, Mariscal Sucre Avenue, 170129, Quito, Ecuador.\\nFAU - Echeverria, Byron Albuja\\nAU  - Echeverria BA\\nAD  - Hospital San Luis de Otavalo, Ministerio de Salud Publica, Antonio Jose de Sucre \\n      Avenue, 100201, Otavalo, Ecuador.\\nFAU - Guerrero, Santiago\\nAU  - Guerrero S\\nAD  - Centro de Investigacion Genetica y Genomica. Facultad de Ciencias de la Salud\\n      Eugenio Espejo, Universidad UTE, Mariscal Sucre Avenue, 170129, Quito, Ecuador.\\nFAU - Cabascango, Eliana\\nAU  - Cabascango E\\nAD  - Sistemas Medicos (SIME), Universidad San Francisco de Quito, Interoceanica Avenue\\n      and Chimborazo, 170902, Cumbaya, Ecuador.\\nFAU - Perez-Villa, Andy\\nAU  - Perez-Villa A\\nAD  - Centro de Investigacion Genetica y Genomica. Facultad de Ciencias de la Salud\\n      Eugenio Espejo, Universidad UTE, Mariscal Sucre Avenue, 170129, Quito, Ecuador.\\nFAU - Armendariz-Castillo, Isaac\\nAU  - Armendariz-Castillo I\\nAD  - Centro de Investigacion Genetica y Genomica. Facultad de Ciencias de la Salud\\n      Eugenio Espejo, Universidad UTE, Mariscal Sucre Avenue, 170129, Quito, Ecuador.\\nFAU - Garcia-Cardenas, Jennyfer M\\nAU  - Garcia-Cardenas JM\\nAD  - Centro de Investigacion Genetica y Genomica. Facultad de Ciencias de la Salud\\n      Eugenio Espejo, Universidad UTE, Mariscal Sucre Avenue, 170129, Quito, Ecuador.\\nFAU - Yumiceba, Veronica\\nAU  - Yumiceba V\\nAD  - Centro de Investigacion Genetica y Genomica. Facultad de Ciencias de la Salud\\n      Eugenio Espejo, Universidad UTE, Mariscal Sucre Avenue, 170129, Quito, Ecuador.\\nFAU - Perez-M, Gabriela\\nAU  - Perez-M G\\nAD  - Ministerio de Salud Publica, 100117, Ibarra, Ecuador.\\nFAU - Leone, Paola E\\nAU  - Leone PE\\nAD  - Centro de Investigacion Genetica y Genomica. Facultad de Ciencias de la Salud\\n      Eugenio Espejo, Universidad UTE, Mariscal Sucre Avenue, 170129, Quito, Ecuador.\\nFAU - Paz-Y-Mino, Cesar\\nAU  - Paz-Y-Mino C\\nAD  - Centro de Investigacion Genetica y Genomica. Facultad de Ciencias de la Salud\\n      Eugenio Espejo, Universidad UTE, Mariscal Sucre Avenue, 170129, Quito, Ecuador.\\n      cesar.pazymino@ute.edu.ec.\\nLA  - eng\\nPT  - Case Reports\\nPT  - Journal Article\\nPT  - Research Support, Non-U.S. Gov\\'t\\nDEP - 20200817\\nPL  - England\\nTA  - BMC Med Genomics\\nJT  - BMC medical genomics\\nJID - 101319628\\nRN  - 0 (Genetic Markers)\\nSB  - IM\\nMH  - *Altitude\\nMH  - Child\\nMH  - DNA Mutational Analysis\\nMH  - Female\\nMH  - *Genetic Markers\\nMH  - Genomics\\nMH  - Humans\\nMH  - Hypohidrosis/genetics/metabolism/*pathology\\nMH  - *Mutation\\nMH  - Pain/genetics/metabolism/*pathology\\nMH  - Pain Insensitivity, Congenital/genetics/metabolism/*pathology\\nMH  - Protein Interaction Maps\\nPMC - PMC7437939\\nOTO - NOTNLM\\nOT  - *CIPA\\nOT  - *Ecuadorian\\nOT  - *Genomics analysis\\nOT  - *NTRK1\\nOT  - *Native American\\nEDAT- 2020/08/19 06:00\\nMHDA- 2021/05/11 06:00\\nCRDT- 2020/08/19 06:00\\nPHST- 2020/01/02 00:00 [received]\\nPHST- 2020/08/10 00:00 [accepted]\\nPHST- 2020/08/19 06:00 [entrez]\\nPHST- 2020/08/19 06:00 [pubmed]\\nPHST- 2021/05/11 06:00 [medline]\\nAID - 10.1186/s12920-020-00764-3 [doi]\\nAID - 10.1186/s12920-020-00764-3 [pii]\\nPST - epublish\\nSO  - BMC Med Genomics. 2020 Aug 17;13(1):113. doi: 10.1186/s12920-020-00764-3.\\n\\nPMID- 32719690\\nOWN - NLM\\nSTAT- PubMed-not-MEDLINE\\nLR  - 20200928\\nIS  - 1664-462X (Print)\\nIS  - 1664-462X (Linking)\\nVI  - 11\\nDP  - 2020\\nTI  - Comparative Plastid Genomics of Neotropical Bulbophyllum (Orchidaceae;\\n      Epidendroideae).\\nPG  - 799\\nLID - 10.3389/fpls.2020.00799 [doi]\\nAB  - Pantropical Bulbophyllum, with approximately 2,200 species, is one of the largest\\n      genera in Orchidaceae. Although phylogenetics and taxonomy of the approximately\\n      60 American species in the genus are generally well understood, some species\\n      complexes need more study to clearly delimit their component species and provide \\n      information about their evolutionary history. Previous research has suggested\\n      that the plastid genome includes phylogenetic markers capable of providing\\n      resolution at low taxonomic levels, and thus it could be an effective tool if\\n      these divergent regions can be identified. In this study, we sequenced the\\n      complete plastid genome of eight Bulbophyllum species, representing five of six\\n      Neotropical taxonomic sections. All plastomes conserve the typical quadripartite \\n      structure, and, although the general structure of plastid genomes is conserved,\\n      differences in ndh-gene composition and total length were detected. Total length \\n      was determined by contraction and expansion of the small single-copy region, a\\n      result of an independent loss of the seven ndh genes. Selection analyses\\n      indicated that protein-coding genes were generally well conserved, but in four\\n      genes, we identified 95 putative sites under positive selection. Furthermore, a\\n      total of 54 polymorphic simple sequence repeats were identified, for which we\\n      developed amplification primers. In addition, we propose 10 regions with\\n      potential to improve phylogenetic analyses of Neotropical Bulbophyllum species.\\nCI  - Copyright (c) 2020 Zavala-Paez, Vieira, Baura, Balsanelli, Souza, Cevallos, Chase\\n      and Smidt.\\nFAU - Zavala-Paez, Michelle\\nAU  - Zavala-Paez M\\nAD  - Departamento de Botanica, Universidade Federal do Parana, Curitiba, Brazil.\\nFAU - Vieira, Leila do Nascimento\\nAU  - Vieira LDN\\nAD  - Departamento de Botanica, Universidade Federal do Parana, Curitiba, Brazil.\\nFAU - de Baura, Valter Antonio\\nAU  - de Baura VA\\nAD  - Departamento de Bioquimica e Biologia Molecular, Universidade Federal do Parana, \\n      Curitiba, Brazil.\\nFAU - Balsanelli, Eduardo\\nAU  - Balsanelli E\\nAD  - Departamento de Bioquimica e Biologia Molecular, Universidade Federal do Parana, \\n      Curitiba, Brazil.\\nFAU - de Souza, Emanuel Maltempi\\nAU  - de Souza EM\\nAD  - Departamento de Bioquimica e Biologia Molecular, Universidade Federal do Parana, \\n      Curitiba, Brazil.\\nFAU - Cevallos, Marco Cerna\\nAU  - Cevallos MC\\nAD  - Grupo de Investigacion Nunkui Wakan, Universidad Politecnica Salesiana, Quito,\\n      Ecuador.\\nFAU - Chase, Mark W\\nAU  - Chase MW\\nAD  - Royal Botanic Gardens, Kew, Richmond, United Kingdom.\\nAD  - Department of Environment and Agriculture, Curtin University, Perth, WA,\\n      Australia.\\nFAU - Smidt, Eric de Camargo\\nAU  - Smidt EC\\nAD  - Departamento de Botanica, Universidade Federal do Parana, Curitiba, Brazil.\\nLA  - eng\\nPT  - Journal Article\\nDEP - 20200703\\nPL  - Switzerland\\nTA  - Front Plant Sci\\nJT  - Frontiers in plant science\\nJID - 101568200\\nPMC - PMC7347972\\nOTO - NOTNLM\\nOT  - Neotropical orchids\\nOT  - molecular evolution\\nOT  - molecular markers\\nOT  - next-generation sequencing\\nOT  - plastid genome\\nEDAT- 2020/07/29 06:00\\nMHDA- 2020/07/29 06:01\\nCRDT- 2020/07/29 06:00\\nPHST- 2020/03/18 00:00 [received]\\nPHST- 2020/05/19 00:00 [accepted]\\nPHST- 2020/07/29 06:00 [entrez]\\nPHST- 2020/07/29 06:00 [pubmed]\\nPHST- 2020/07/29 06:01 [medline]\\nAID - 10.3389/fpls.2020.00799 [doi]\\nPST - epublish\\nSO  - Front Plant Sci. 2020 Jul 3;11:799. doi: 10.3389/fpls.2020.00799. eCollection\\n      2020.\\n\\nPMID- 32648524\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20210202\\nLR  - 20210202\\nIS  - 0031-949X (Print)\\nIS  - 0031-949X (Linking)\\nVI  - 111\\nIP  - 1\\nDP  - 2021 Jan\\nTI  - Natural Defect of a Plant Rhabdovirus Glycoprotein Gene: A Case Study of\\n      Virus-Plant Coevolution.\\nPG  - 227-236\\nLID - 10.1094/PHYTO-05-20-0191-FI [doi]\\nAB  - Seven isolates of a putative cytorhabdovirus (family Rhabdoviridae, order\\n      Mononegavirales) designated as citrus-associated rhabdovirus (CiaRV) were\\n      identified in citrus, passion fruit, and paper bush from the same geographical\\n      area in China. CiaRV, bean-associated cytorhabdovirus (Brazil), and papaya virus \\n      E (Ecuador) should be taxonomically classified in the species Papaya\\n      cytorhabdovirus. Due to natural mutations, the glycoprotein (G) and P4 genes were\\n      impaired in citrus-infecting isolates of CiaRV, resulting in an atypical\\n      rhabdovirus genome organization of 3\\' leader-N-P-P3-M-L-5\\' trailer. The P3\\n      protein of CiaRV shared a common origin with begomoviral movement proteins\\n      (family Geminiviridae). Secondary structure analysis and trans-complementation of\\n      movement-deficient tomato mosaic virus and potato virus X mutants by CiaRV P3\\n      supported its function in viral cell-to-cell trafficking. The wide geographical\\n      dispersal of CiaRV and related viruses suggests an efficient transmission\\n      mechanism, as well as an underlying risk to global agriculture. Both the natural \\n      phenomenon and experimental analyses demonstrated presence of the \"degraded\" type\\n      of CiaRV in citrus, in parallel to \"undegraded\" types in other host plant\\n      species. This case study shows a plant virus losing the function of an important \\n      but nonessential gene, likely due to host shift and adaption, which deepened our \\n      understanding of course of natural viral diversification.\\nFAU - Zhang, Song\\nAU  - Zhang S\\nAD  - National Citrus Engineering and Technology Research Center, Citrus Research\\n      Institute, Southwest University, Beibei, Chongqing 400712, China.\\nAD  - Academy of Agricultural Sciences, Southwest University, Beibei, Chongqing 400715,\\n      China.\\nFAU - Huang, Aijun\\nAU  - Huang A\\nAUID- ORCID: http://orcid.org/0000-0003-0733-2633\\nAD  - National Navel Orange Research Center, College of Life Science, Gannan Normal\\n      University, Ganzhou, China.\\nFAU - Zhou, Xin\\nAU  - Zhou X\\nAD  - State Key Laboratory of Rice Biology, Institute of Biotechnology, Zhejiang\\n      University, Hangzhou, Zhejiang, China.\\nFAU - Li, Zhenghe\\nAU  - Li Z\\nAUID- ORCID: http://orcid.org/0000-0001-5497-2575\\nAD  - State Key Laboratory of Rice Biology, Institute of Biotechnology, Zhejiang\\n      University, Hangzhou, Zhejiang, China.\\nFAU - Dietzgen, Ralf G\\nAU  - Dietzgen RG\\nAUID- ORCID: http://orcid.org/0000-0002-7772-2250\\nAD  - Queensland Alliance for Agriculture and Food Innovation, The University of\\n      Queensland, St. Lucia, Queensland 4072, Australia.\\nFAU - Zhou, Changyong\\nAU  - Zhou C\\nAD  - National Citrus Engineering and Technology Research Center, Citrus Research\\n      Institute, Southwest University, Beibei, Chongqing 400712, China.\\nAD  - Academy of Agricultural Sciences, Southwest University, Beibei, Chongqing 400715,\\n      China.\\nFAU - Cao, Mengji\\nAU  - Cao M\\nAUID- ORCID: http://orcid.org/0000-0003-0396-262X\\nAD  - National Citrus Engineering and Technology Research Center, Citrus Research\\n      Institute, Southwest University, Beibei, Chongqing 400712, China.\\nAD  - Academy of Agricultural Sciences, Southwest University, Beibei, Chongqing 400715,\\n      China.\\nLA  - eng\\nPT  - Journal Article\\nDEP - 20201110\\nPL  - United States\\nTA  - Phytopathology\\nJT  - Phytopathology\\nJID - 9427222\\nRN  - 0 (Glycoproteins)\\nSB  - IM\\nMH  - Brazil\\nMH  - China\\nMH  - Ecuador\\nMH  - Genome, Viral\\nMH  - Glycoproteins\\nMH  - Open Reading Frames\\nMH  - Phylogeny\\nMH  - Plant Diseases\\nMH  - *Plant Viruses/genetics\\nMH  - *Rhabdoviridae/genetics\\nOTO - NOTNLM\\nOT  - citrus\\nOT  - evolution\\nOT  - gene defect\\nOT  - genomics\\nOT  - movement protein complementation\\nOT  - plant rhabdovirus\\nOT  - viral quasi-species\\nOT  - virology\\nEDAT- 2020/07/11 06:00\\nMHDA- 2021/02/03 06:00\\nCRDT- 2020/07/11 06:00\\nPHST- 2020/07/11 06:00 [pubmed]\\nPHST- 2021/02/03 06:00 [medline]\\nPHST- 2020/07/11 06:00 [entrez]\\nAID - 10.1094/PHYTO-05-20-0191-FI [doi]\\nPST - ppublish\\nSO  - Phytopathology. 2021 Jan;111(1):227-236. doi: 10.1094/PHYTO-05-20-0191-FI. Epub\\n      2020 Nov 10.\\n\\nPMID- 32512194\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20201116\\nLR  - 20201116\\nIS  - 1095-9513 (Electronic)\\nIS  - 1055-7903 (Linking)\\nVI  - 150\\nDP  - 2020 Sep\\nTI  - Diversification history of clown tree frogs in Neotropical rainforests (Anura,\\n      Hylidae, Dendropsophus leucophyllatus group).\\nPG  - 106877\\nLID - S1055-7903(20)30149-4 [pii]\\nLID - 10.1016/j.ympev.2020.106877 [doi]\\nAB  - General consensus emphasizes that no single biological process can explain the\\n      patterns of species\\' distributions and diversification in the Neotropics.\\n      Instead, the interplay of several processes across space and time must be taken\\n      into account. Here we investigated the phylogenetic relationships and\\n      biogeographic history of tree frogs in the Dendropsophus leucophyllatus species\\n      group (Amphibia: Hylidae), which is distributed across Amazonia and the Atlantic \\n      rainforests. Using Next Generation Sequencing (NGS) and double digest\\n      restriction-site associated DNA (ddRADseq), we inferred phylogenetic\\n      relationships, species limits, and temporal and geographic patterns of\\n      diversification relative to the history of these biomes. Our results indicate\\n      that the D. leucophyllatus species group includes at least 14 independent\\n      lineages, which are currently arranged into ten described species. Therefore, a\\n      significant portion of species in the group are still unnamed. Different\\n      processes were associated to the group diversification history. For instance, the\\n      Andes uplift likely caused allopatric speciation for Cis-Andean species, whereas \\n      it may also be responsible for changes in the Amazonian landscape triggering\\n      parapatric speciation by local adaptation to ecological factors. Meanwhile,\\n      Atlantic Forest ancestors unable to cross the dry diagonal biomes after\\n      rainforest\\'s retraction, evolved in isolation into different species.\\n      Diversification in the group began in the early Miocene, when connections between\\n      Atlantic Forest and the Andes (Pacific Dominion) by way of a south corridor were \\n      possible. The historical scenario in Amazonia, characterized by several\\n      speciation events and habitat heterogeneity, helped promoting diversification,\\n      resulting in the highest species diversity for the group. This marked species\\n      diversification did not happen in Atlantic Forest, where speciation is very\\n      recent (late Pliocene and Pleistocene), despite its remarkable climatic\\n      heterogeneity.\\nCI  - Copyright (c) 2020 Elsevier Inc. All rights reserved.\\nFAU - Pirani, Renata M\\nAU  - Pirani RM\\nAD  - Programa de Pos-Graduacao em Genetica, Conservacao e Biologia Evolutiva,\\n      Instituto Nacional de Pesquisas da Amazonia, Av. Ephigenio Sales 2239, 69060-000,\\n      Manaus, Amazonas, Brazil. Electronic address: renatampirani@gmail.com.\\nFAU - Peloso, Pedro L V\\nAU  - Peloso PLV\\nAD  - Instituto de Ciencias Biologicas, Universidade Federal do Para, Rua Augusto\\n      Correa, 01, Guama, 66075-110, Belem, Para, Brazil.\\nFAU - Prado, Joyce R\\nAU  - Prado JR\\nAD  - Departamento de Ciencias Biologicas, Universidade de Sao Paulo, Piracicaba, Sao\\n      Paulo, Brazil.\\nFAU - Polo, Erico M\\nAU  - Polo EM\\nAD  - Departamento de Genetica, Instituto de Ciencias Biologicas, Universidade Federal \\n      do Amazonas, Av. General Rodrigo Octavio, 6200, Manaus, Amazonas, Brazil.\\nFAU - Knowles, L Lacey\\nAU  - Knowles LL\\nAD  - Department of Ecology and Evolutionary Biology, The University of Michigan, Ann\\n      Arbor, MI 41809-1079, USA.\\nFAU - Ron, Santiago R\\nAU  - Ron SR\\nAD  - Museo de Zoologia, Facultad de Ciencias Exactas y Naturales, Pontificia\\n      Universidad Catolica del Ecuador, Quito, Ecuador.\\nFAU - Rodrigues, Miguel T\\nAU  - Rodrigues MT\\nAD  - Departamento de Zoologia, Instituto de Biociencias, Universidade de Sao Paulo,\\n      CEP 05508-090 Sao Paulo, Sao Paulo, Brazil.\\nFAU - Sturaro, Marcelo J\\nAU  - Sturaro MJ\\nAD  - Departamento de Ecologia e Biologia Evolutiva, Universidade Federal de Sao Paulo,\\n      Av. Professor Artur Riedel, 275, Jardim Eldorado, Diadema, CEP 09972-270 Sao\\n      Paulo, Brazil.\\nFAU - Werneck, Fernanda P\\nAU  - Werneck FP\\nAD  - Programa de Pos-Graduacao em Genetica, Conservacao e Biologia Evolutiva,\\n      Instituto Nacional de Pesquisas da Amazonia, Av. Ephigenio Sales 2239, 69060-000,\\n      Manaus, Amazonas, Brazil; Coordenacao de Biodiversidade, Programa de Colecoes\\n      Cientificas Biologicas, Instituto Nacional de Pesquisas da Amazonia, Av. Andre\\n      Araujo 2936, CEP 69067-375 Manaus, Amazonas, Brazil.\\nLA  - eng\\nPT  - Journal Article\\nPT  - Research Support, Non-U.S. Gov\\'t\\nPT  - Research Support, U.S. Gov\\'t, Non-P.H.S.\\nDEP - 20200606\\nPL  - United States\\nTA  - Mol Phylogenet Evol\\nJT  - Molecular phylogenetics and evolution\\nJID - 9304400\\nSB  - IM\\nMH  - Animals\\nMH  - Anura/*classification/genetics\\nMH  - Biodiversity\\nMH  - Databases, Genetic\\nMH  - Phylogeny\\nMH  - Rainforest\\nMH  - South America\\nMH  - Species Specificity\\nOTO - NOTNLM\\nOT  - *Biogeography\\nOT  - *Cryptic species\\nOT  - *Genomics\\nOT  - *Parapatric speciation\\nOT  - *South America\\nOT  - *Species delimitation\\nEDAT- 2020/06/09 06:00\\nMHDA- 2020/11/18 06:00\\nCRDT- 2020/06/09 06:00\\nPHST- 2019/11/26 00:00 [received]\\nPHST- 2020/05/29 00:00 [revised]\\nPHST- 2020/06/01 00:00 [accepted]\\nPHST- 2020/06/09 06:00 [pubmed]\\nPHST- 2020/11/18 06:00 [medline]\\nPHST- 2020/06/09 06:00 [entrez]\\nAID - S1055-7903(20)30149-4 [pii]\\nAID - 10.1016/j.ympev.2020.106877 [doi]\\nPST - ppublish\\nSO  - Mol Phylogenet Evol. 2020 Sep;150:106877. doi: 10.1016/j.ympev.2020.106877. Epub \\n      2020 Jun 6.\\n\\nPMID- 32377357\\nOWN - NLM\\nSTAT- PubMed-not-MEDLINE\\nLR  - 20211230\\nIS  - 2662-6810 (Print)\\nIS  - 2052-7276 (Linking)\\nVI  - 7\\nDP  - 2020\\nTI  - Exploiting the diversity of tomato: the development of a phenotypically and\\n      genetically detailed germplasm collection.\\nPG  - 66\\nLID - 10.1038/s41438-020-0291-7 [doi]\\nAB  - A collection of 163 accessions, including Solanum pimpinellifolium, Solanum\\n      lycopersicum var. cerasiforme and Solanum lycopersicum var. lycopersicum, was\\n      selected to represent the genetic and morphological variability of tomato at its \\n      centers of origin and domestication: Andean regions of Peru and Ecuador and\\n      Mesoamerica. The collection is enriched with S. lycopersicum var. cerasiforme\\n      from the Amazonian region that has not been analyzed previously nor used\\n      extensively. The collection has been morphologically characterized showing\\n      diversity for fruit, flower and vegetative traits. Their genomes were sequenced\\n      in the Varitome project and are publicly available\\n      (solgenomics.net/projects/varitome). The identified SNPs have been annotated with\\n      respect to their impact and a total number of 37,974 out of 19,364,146 SNPs have \\n      been described as high impact by the SnpEeff analysis. GWAS has shown\\n      associations for different traits, demonstrating the potential of this collection\\n      for this kind of analysis. We have not only identified known QTLs and genes, but \\n      also new regions associated with traits such as fruit color, number of flowers\\n      per inflorescence or inflorescence architecture. To speed up and facilitate the\\n      use of this information, F2 populations were constructed by crossing the whole\\n      collection with three different parents. This F2 collection is useful for testing\\n      SNPs identified by GWAs, selection sweeps or any other candidate gene. All data\\n      is available on Solanaceae Genomics Network and the accession and F2 seeds are\\n      freely available at COMAV and at TGRC genebanks. All these resources together\\n      make this collection a good candidate for genetic studies.\\nCI  - (c) The Author(s) 2020.\\nFAU - Mata-Nicolas, Estefania\\nAU  - Mata-Nicolas E\\nAD  - 1Instituto Universitario de Conservacion y Mejora de la Agrodiversidad\\n      Valenciana. COMAV. Universitat Politecnica de Valencia, Valencia, Spain.\\nFAU - Montero-Pau, Javier\\nAU  - Montero-Pau J\\nAUID- ORCID: 0000-0002-0864-8157\\nAD  - 2Department of Biochemistry and Molecular Biology, Universitat de Valencia,\\n      Valencia, Spain.\\nFAU - Gimeno-Paez, Esther\\nAU  - Gimeno-Paez E\\nAD  - 1Instituto Universitario de Conservacion y Mejora de la Agrodiversidad\\n      Valenciana. COMAV. Universitat Politecnica de Valencia, Valencia, Spain.\\nFAU - Garcia-Carpintero, Victor\\nAU  - Garcia-Carpintero V\\nAD  - 1Instituto Universitario de Conservacion y Mejora de la Agrodiversidad\\n      Valenciana. COMAV. Universitat Politecnica de Valencia, Valencia, Spain.\\nFAU - Ziarsolo, Peio\\nAU  - Ziarsolo P\\nAD  - 1Instituto Universitario de Conservacion y Mejora de la Agrodiversidad\\n      Valenciana. COMAV. Universitat Politecnica de Valencia, Valencia, Spain.\\nFAU - Menda, Naama\\nAU  - Menda N\\nAD  - 3Boyce Thompson Institute, Ithaca, NY USA.\\nFAU - Mueller, Lukas A\\nAU  - Mueller LA\\nAD  - 3Boyce Thompson Institute, Ithaca, NY USA.\\nFAU - Blanca, Jose\\nAU  - Blanca J\\nAD  - 1Instituto Universitario de Conservacion y Mejora de la Agrodiversidad\\n      Valenciana. COMAV. Universitat Politecnica de Valencia, Valencia, Spain.\\nFAU - Canizares, Joaquin\\nAU  - Canizares J\\nAD  - 1Instituto Universitario de Conservacion y Mejora de la Agrodiversidad\\n      Valenciana. COMAV. Universitat Politecnica de Valencia, Valencia, Spain.\\nFAU - van der Knaap, Esther\\nAU  - van der Knaap E\\nAD  - 4Institute of Plant Breeding, Genetics and Genomics, University of Georgia,\\n      Georgia, GA USA.\\nAD  - 5Department of Horticulture, University of Georgia, Georgia, GA USA.\\nFAU - Diez, Maria Jose\\nAU  - Diez MJ\\nAD  - 1Instituto Universitario de Conservacion y Mejora de la Agrodiversidad\\n      Valenciana. COMAV. Universitat Politecnica de Valencia, Valencia, Spain.\\nLA  - eng\\nGR  - NSFIOS1564366/National Science Foundation (NSF)\\nPT  - Journal Article\\nDEP - 20200501\\nPL  - England\\nTA  - Hortic Res\\nJT  - Horticulture research\\nJID - 101655540\\nPMC - PMC7192925\\nOTO - NOTNLM\\nOT  - *Agricultural genetics\\nOT  - *Plant breeding\\nCOIS- Conflict of interestThe authors declare that they have no conflict of interest.\\nEDAT- 2020/05/08 06:00\\nMHDA- 2020/05/08 06:01\\nCRDT- 2020/05/08 06:00\\nPHST- 2019/12/28 00:00 [received]\\nPHST- 2020/03/06 00:00 [revised]\\nPHST- 2020/03/16 00:00 [accepted]\\nPHST- 2020/05/08 06:00 [entrez]\\nPHST- 2020/05/08 06:00 [pubmed]\\nPHST- 2020/05/08 06:01 [medline]\\nAID - 10.1038/s41438-020-0291-7 [doi]\\nAID - 10.1038/s41438-020-0291-7 [pii]\\nPST - epublish\\nSO  - Hortic Res. 2020 May 1;7:66. doi: 10.1038/s41438-020-0291-7. eCollection 2020.\\n\\nPMID- 32359157\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20211118\\nLR  - 20211118\\nIS  - 1076-836X (Electronic)\\nIS  - 1063-5157 (Linking)\\nVI  - 70\\nIP  - 1\\nDP  - 2021 Jan 1\\nTI  - Phylogenomics Reveals Ancient Gene Tree Discordance in the Amphibian Tree of\\n      Life.\\nPG  - 49-66\\nLID - 10.1093/sysbio/syaa034 [doi]\\nAB  - Molecular phylogenies have yielded strong support for many parts of the amphibian\\n      Tree of Life, but poor support for the resolution of deeper nodes, including\\n      relationships among families and orders. To clarify these relationships, we\\n      provide a phylogenomic perspective on amphibian relationships by developing a\\n      taxon-specific Anchored Hybrid Enrichment protocol targeting hundreds of\\n      conserved exons which are effective across the class. After obtaining data from\\n      220 loci for 286 species (representing 94% of the families and 44% of the\\n      genera), we estimate a phylogeny for extant amphibians and identify gene\\n      tree-species tree conflict across the deepest branches of the amphibian\\n      phylogeny. We perform locus-by-locus genealogical interrogation of alternative\\n      topological hypotheses for amphibian monophyly, focusing on interordinal\\n      relationships. We find that phylogenetic signal deep in the amphibian phylogeny\\n      varies greatly across loci in a manner that is consistent with incomplete lineage\\n      sorting in the ancestral lineage of extant amphibians. Our results overwhelmingly\\n      support amphibian monophyly and a sister relationship between frogs and\\n      salamanders, consistent with the Batrachia hypothesis. Species tree analyses\\n      converge on a small set of topological hypotheses for the relationships among\\n      extant amphibian families. These results clarify several contentious portions of \\n      the amphibian Tree of Life, which in conjunction with a set of vetted fossil\\n      calibrations, support a surprisingly younger timescale for crown and ordinal\\n      amphibian diversification than previously reported. More broadly, our study\\n      provides insight into the sources, magnitudes, and heterogeneity of support\\n      across loci in phylogenomic data sets.[AIC; Amphibia; Batrachia; Phylogeny; gene \\n      tree-species tree discordance; genomics; information theory.].\\nCI  - (c) The Author(s) 2020. Published by Oxford University Press, on behalf of the\\n      Society of Systematic Biologists.\\nFAU - Hime, Paul M\\nAU  - Hime PM\\nAD  - Biodiversity Institute, University of Kansas, Lawrence, KS 66045, USA.\\nAD  - Department of Biology, University of Kentucky, Lexington, KY 40506, USA.\\nFAU - Lemmon, Alan R\\nAU  - Lemmon AR\\nAD  - Department of Scientific Computing, Florida State University, Tallahassee, FL\\n      32306, USA.\\nFAU - Lemmon, Emily C Moriarty\\nAU  - Lemmon ECM\\nAD  - Department of Biological Science, Florida State University, Tallahassee, FL\\n      32306, USA.\\nFAU - Prendini, Elizabeth\\nAU  - Prendini E\\nAD  - Division of Vertebrate Zoology: Herpetology, American Museum of Natural History, \\n      New York, NY 10024, USA.\\nFAU - Brown, Jeremy M\\nAU  - Brown JM\\nAD  - Department of Biological Sciences and Museum of Natural Science, Louisiana State \\n      University, Baton Rouge, LA 70803, USA.\\nFAU - Thomson, Robert C\\nAU  - Thomson RC\\nAD  - School of Life Sciences, University of Hawai\\'i, Honolulu, HI 96822, USA.\\nFAU - Kratovil, Justin D\\nAU  - Kratovil JD\\nAD  - Department of Biology, University of Kentucky, Lexington, KY 40506, USA.\\nAD  - Department of Entomology, University of Kentucky, Lexington, KY 40546, USA.\\nFAU - Noonan, Brice P\\nAU  - Noonan BP\\nAD  - Department of Biology, University of Mississippi, Oxford, MS 38677, USA.\\nFAU - Pyron, R Alexander\\nAU  - Pyron RA\\nAD  - Department of Biological Sciences, The George Washington University, Washington, \\n      DC 20052, USA.\\nFAU - Peloso, Pedro L V\\nAU  - Peloso PLV\\nAD  - Division of Vertebrate Zoology: Herpetology, American Museum of Natural History, \\n      New York, NY 10024, USA.\\nAD  - Instituto de Ciencias Biologicas, Universidade Federal do Para, Belem, 66075-750,\\n      Brazil.\\nFAU - Kortyna, Michelle L\\nAU  - Kortyna ML\\nAD  - Department of Biological Science, Florida State University, Tallahassee, FL\\n      32306, USA.\\nFAU - Keogh, J Scott\\nAU  - Keogh JS\\nAD  - Division of Ecology and Evolution, Research School of Biology, The Australian\\n      National University, Canberra, 2601, Australia.\\nFAU - Donnellan, Stephen C\\nAU  - Donnellan SC\\nAD  - South Australian Museum, North Terrace, Adelaide 5000, Australia.\\nAD  - School of Biological Sciences, University of Adelaide, Adelaide 5005, Australia.\\nFAU - Mueller, Rachel Lockridge\\nAU  - Mueller RL\\nAD  - Department of Biology, Colorado State University, Fort Collins, CO 80523, USA.\\nFAU - Raxworthy, Christopher J\\nAU  - Raxworthy CJ\\nAD  - Division of Vertebrate Zoology: Herpetology, American Museum of Natural History, \\n      New York, NY 10024, USA.\\nFAU - Kunte, Krushnamegh\\nAU  - Kunte K\\nAD  - National Centre for Biological Sciences, Tata Institute of Fundamental Research, \\n      Bengaluru 560065, India.\\nFAU - Ron, Santiago R\\nAU  - Ron SR\\nAD  - Museo de Zoologia, Escuela de Biologia, Pontificia Universidad Catolica del\\n      Ecuador, Quito, Ecuador.\\nFAU - Das, Sandeep\\nAU  - Das S\\nAD  - Forest Ecology and Biodiversity Conservation Division, Kerala Forest Research\\n      Institute, Peechi, Kerala 680653, India.\\nFAU - Gaitonde, Nikhil\\nAU  - Gaitonde N\\nAD  - National Centre for Biological Sciences, Tata Institute of Fundamental Research, \\n      Bengaluru 560065, India.\\nFAU - Green, David M\\nAU  - Green DM\\nAD  - Redpath Museum, McGill University, Montreal, Quebec H3A 0C4, Canada.\\nFAU - Labisko, Jim\\nAU  - Labisko J\\nAD  - The Durrell Institute of Conservation and Ecology, School of Anthropology and\\n      Conservation, The University of Kent, Canterbury, Kent, CT2 7NR, UK.\\nAD  - Island Biodiversity and Conservation Centre, University of Seychelles, PO Box\\n      1348, Anse Royale, Mahe, Seychelles.\\nFAU - Che, Jing\\nAU  - Che J\\nAD  - State Key Laboratory of Genetic Resources and Evolution, Kunming Institute of\\n      Zoology, Kunming 650223, China.\\nAD  - Center for Excellence in Animal Evolution and Genetics, Chinese Academy of\\n      Sciences, Kunming 650223, China.\\nFAU - Weisrock, David W\\nAU  - Weisrock DW\\nAD  - Department of Biology, University of Kentucky, Lexington, KY 40506, USA.\\nLA  - eng\\nSI  - Dryad/10.5061/dryad.9kd51c5dc\\nPT  - Journal Article\\nPT  - Research Support, Non-U.S. Gov\\'t\\nPT  - Research Support, U.S. Gov\\'t, Non-P.H.S.\\nPL  - England\\nTA  - Syst Biol\\nJT  - Systematic biology\\nJID - 9302532\\nSB  - IM\\nMH  - Animals\\nMH  - Anura\\nMH  - *Fossils\\nMH  - *Genomics\\nMH  - Humans\\nMH  - Phylogeny\\nPMC - PMC7823230\\nEDAT- 2020/05/03 06:00\\nMHDA- 2021/11/19 06:00\\nCRDT- 2020/05/03 06:00\\nPHST- 2019/04/26 00:00 [received]\\nPHST- 2020/04/14 00:00 [revised]\\nPHST- 2020/04/14 00:00 [accepted]\\nPHST- 2020/05/03 06:00 [pubmed]\\nPHST- 2021/11/19 06:00 [medline]\\nPHST- 2020/05/03 06:00 [entrez]\\nAID - 5828236 [pii]\\nAID - 10.1093/sysbio/syaa034 [doi]\\nPST - ppublish\\nSO  - Syst Biol. 2021 Jan 1;70(1):49-66. doi: 10.1093/sysbio/syaa034.\\n\\nPMID- 31902732\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20201222\\nLR  - 20201222\\nIS  - 1879-0445 (Electronic)\\nIS  - 0960-9822 (Linking)\\nVI  - 30\\nIP  - 3\\nDP  - 2020 Feb 3\\nTI  - Genomic and Fitness Consequences of Genetic Rescue in Wild Populations.\\nPG  - 517-522.e5\\nLID - S0960-9822(19)31532-5 [pii]\\nLID - 10.1016/j.cub.2019.11.062 [doi]\\nAB  - Gene flow is an enigmatic evolutionary force because it can limit adaptation but \\n      may also rescue small populations from inbreeding depression [1-3]. Several\\n      iconic examples of genetic rescue-increased population growth caused by gene flow\\n      [4, 5]-have reversed population declines [6, 7]. However, concerns about\\n      outbreeding depression and maladaptive gene flow limit the use of human-mediated \\n      gene flow in conservation [8, 9]. Rescue effects of immigration through\\n      demographic and/or genetic mechanisms have received theoretical and empirical\\n      support, but studies that monitor initial and long-term effects of gene flow on\\n      individuals and populations in the wild are lacking. Here, we used\\n      individual-based mark-recapture, multigenerational pedigrees, and genomics to\\n      test the demographic and evolutionary consequences of manipulating gene flow in\\n      two isolated, wild Trinidadian guppy populations. Recipient and source\\n      populations originated from environments with different predation, flow, and\\n      resource regimes [10]. We documented 10-fold increases in population size\\n      following gene flow and found that, on average, hybrids lived longer and\\n      reproduced more than residents and immigrants. Despite overall genomic\\n      homogenization, alleles potentially associated with local adaptation were not\\n      entirely swamped by gene flow. Our results suggest that genetic rescue was caused\\n      not just by increasing individual genetic diversity, rather new genomic variation\\n      from immigrants combined with alleles from the recipient population resulted in\\n      highly fit hybrids and subsequent increases in population size. Contrary to the\\n      classic view of maladaptive gene flow, our study reveals conditions under which\\n      immigration can produce long-term fitness benefits in small populations without\\n      entirely swamping adaptive variation.\\nCI  - Copyright (c) 2019 The Authors. Published by Elsevier Ltd.. All rights reserved.\\nFAU - Fitzpatrick, Sarah W\\nAU  - Fitzpatrick SW\\nAD  - W.K. Kellogg Biological Station, Michigan State University, 3700 E. Gull Lake\\n      Drive, Hickory Corners, MI 49060, USA; Department of Integrative Biology,\\n      Michigan State University, 288 Farm Lane, East Lansing, MI 48824, USA; Ecology,\\n      Evolutionary Biology, and Behavior Program, Michigan State University, East\\n      Lansing, MI 48824, USA. Electronic address: sfitz@msu.edu.\\nFAU - Bradburd, Gideon S\\nAU  - Bradburd GS\\nAD  - Department of Integrative Biology, Michigan State University, 288 Farm Lane, East\\n      Lansing, MI 48824, USA; Ecology, Evolutionary Biology, and Behavior Program,\\n      Michigan State University, East Lansing, MI 48824, USA.\\nFAU - Kremer, Colin T\\nAU  - Kremer CT\\nAD  - W.K. Kellogg Biological Station, Michigan State University, 3700 E. Gull Lake\\n      Drive, Hickory Corners, MI 49060, USA.\\nFAU - Salerno, Patricia E\\nAU  - Salerno PE\\nAD  - Department of Biology, Colorado State University, 1878 Campus Delivery, Fort\\n      Collins, CO 80523, USA; Universidad Regional Amazonica Ikiam, Km 7 Via Muyuna,\\n      Tena, Napo, Ecuador.\\nFAU - Angeloni, Lisa M\\nAU  - Angeloni LM\\nAD  - Department of Biology, Colorado State University, 1878 Campus Delivery, Fort\\n      Collins, CO 80523, USA; Graduate Degree Program in Ecology, Colorado State\\n      University, Fort Collins, CO 80523, USA.\\nFAU - Funk, W Chris\\nAU  - Funk WC\\nAD  - Department of Biology, Colorado State University, 1878 Campus Delivery, Fort\\n      Collins, CO 80523, USA; Graduate Degree Program in Ecology, Colorado State\\n      University, Fort Collins, CO 80523, USA.\\nLA  - eng\\nPT  - Journal Article\\nPT  - Research Support, Non-U.S. Gov\\'t\\nPT  - Research Support, U.S. Gov\\'t, Non-P.H.S.\\nDEP - 20200102\\nPL  - England\\nTA  - Curr Biol\\nJT  - Current biology : CB\\nJID - 9107782\\nSB  - IM\\nMH  - Animals\\nMH  - Animals, Wild/genetics/physiology\\nMH  - Female\\nMH  - *Gene Flow\\nMH  - *Genetic Fitness\\nMH  - *Hybridization, Genetic\\nMH  - Male\\nMH  - Pedigree\\nMH  - Poecilia/genetics/*physiology\\nMH  - Population Growth\\nOTO - NOTNLM\\nOT  - *Trinidadian guppies\\nOT  - *admixture\\nOT  - *demography\\nOT  - *eco-evolutionary dynamics\\nOT  - *fitness\\nOT  - *gene flow\\nOT  - *genetic rescue\\nOT  - *genomics\\nOT  - *inbreeding\\nOT  - *pedigree\\nCOIS- Declaration of Interests The authors declare no competing interests.\\nEDAT- 2020/01/07 06:00\\nMHDA- 2020/12/23 06:00\\nCRDT- 2020/01/07 06:00\\nPHST- 2019/08/02 00:00 [received]\\nPHST- 2019/10/11 00:00 [revised]\\nPHST- 2019/11/20 00:00 [accepted]\\nPHST- 2020/01/07 06:00 [pubmed]\\nPHST- 2020/12/23 06:00 [medline]\\nPHST- 2020/01/07 06:00 [entrez]\\nAID - S0960-9822(19)31532-5 [pii]\\nAID - 10.1016/j.cub.2019.11.062 [doi]\\nPST - ppublish\\nSO  - Curr Biol. 2020 Feb 3;30(3):517-522.e5. doi: 10.1016/j.cub.2019.11.062. Epub 2020\\n      Jan 2.\\n\\nPMID- 31822297\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20200518\\nLR  - 20200518\\nIS  - 1479-7364 (Electronic)\\nIS  - 1473-9542 (Linking)\\nVI  - 13\\nIP  - 1\\nDP  - 2019 Dec 10\\nTI  - Human genetics and genomics research in Ecuador: historical survey, current\\n      state, and future directions.\\nPG  - 64\\nLID - 10.1186/s40246-019-0249-8 [doi]\\nAB  - BACKGROUND: In South America, the history of human genetics is extensive and its \\n      beginnings go back to the onset of the twentieth century. In Ecuador, the\\n      historical record of human genetics and genomics research is limited. In this\\n      context, our work analyzes the current status and historical panorama of these\\n      fields, based on bibliographic searches in Scopus, Google Scholar, PubMed, and\\n      Web of Science. RESULTS: Our results determined that the oldest paper in human\\n      genetics coauthored by an Ecuadorian institution originates from the Central\\n      University of Ecuador in 1978. From a historical standpoint, the number of\\n      articles has increased since the 1990s. This growth has intensified and it is\\n      reflected in 137 manuscripts recorded from 2010 to 2019. Areas such as human\\n      population genetics, phylogeography, and forensic sciences are the core of\\n      genetics and genomics-associated research in Ecuador. Important advances have\\n      been made in the understanding of the bases of cancer, some genetic diseases, and\\n      congenital disorders. Fields such as pharmacogenetics and pharmacogenomics have\\n      begun to be explored during the last years. CONCLUSIONS: This work paints a\\n      comprehensive picture and provides additional insights into the future panorama\\n      of human genetic and genomic research in Ecuador as an example of an emerging,\\n      resource-limited country with interesting phylogeographic characteristics and\\n      public health implications.\\nFAU - Zambrano-Mila, Marlon S\\nAU  - Zambrano-Mila MS\\nAUID- ORCID: 0000-0001-5942-2986\\nAD  - School of Biological Sciences and Engineering, Yachay Tech University, San Miguel\\n      de Urcuqui, Ecuador. marlonz1996@hotmail.com.\\nFAU - Agathos, Spiros N\\nAU  - Agathos SN\\nAD  - School of Biological Sciences and Engineering, Yachay Tech University, San Miguel\\n      de Urcuqui, Ecuador.\\nAD  - Bioengineering Lab, Earth & Life Institute, Universite Catholique de Louvain,\\n      Louvain-la-Neuve, Belgium.\\nFAU - Reichardt, Juergen K V\\nAU  - Reichardt JKV\\nAD  - Australian Institute of Tropical Health and Medicine (AITHM), James Cook\\n      University, Smithfield, QLD, Australia.\\nLA  - eng\\nGR  - Internal grant 39-2017/Yachay Tech/International\\nPT  - Historical Article\\nPT  - Journal Article\\nPT  - Research Support, Non-U.S. Gov\\'t\\nDEP - 20191210\\nPL  - England\\nTA  - Hum Genomics\\nJT  - Human genomics\\nJID - 101202210\\nSB  - IM\\nMH  - Ecuador\\nMH  - Genomics/*trends\\nMH  - History, 20th Century\\nMH  - History, 21st Century\\nMH  - Human Genetics\\nMH  - Humans\\nMH  - Publications\\nMH  - Research/*history/*trends\\nMH  - *Surveys and Questionnaires\\nPMC - PMC6902328\\nOTO - NOTNLM\\nOT  - *Database search\\nOT  - *Ecuador\\nOT  - *Google Scholar\\nOT  - *Human genetics\\nOT  - *Human genomics\\nOT  - *PubMed\\nOT  - *Scopus\\nOT  - *South America\\nOT  - *Web of Science\\nEDAT- 2019/12/12 06:00\\nMHDA- 2020/05/19 06:00\\nCRDT- 2019/12/12 06:00\\nPHST- 2019/09/30 00:00 [received]\\nPHST- 2019/11/14 00:00 [accepted]\\nPHST- 2019/12/12 06:00 [entrez]\\nPHST- 2019/12/12 06:00 [pubmed]\\nPHST- 2020/05/19 06:00 [medline]\\nAID - 10.1186/s40246-019-0249-8 [doi]\\nAID - 10.1186/s40246-019-0249-8 [pii]\\nPST - epublish\\nSO  - Hum Genomics. 2019 Dec 10;13(1):64. doi: 10.1186/s40246-019-0249-8.\\n\\nPMID- 31773318\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20200914\\nLR  - 20210110\\nIS  - 1437-1596 (Electronic)\\nIS  - 0937-9827 (Linking)\\nVI  - 134\\nIP  - 1\\nDP  - 2020 Jan\\nTI  - Targeted exon sequencing in deceased schizophrenia patients in Denmark.\\nPG  - 135-147\\nLID - 10.1007/s00414-019-02212-z [doi]\\nAB  - Schizophrenia patients have higher mortality rates and lower life expectancy than\\n      the general population. However, forensic investigations of their deaths often\\n      fail to determine the cause of death, hindering prevention. As schizophrenia is a\\n      highly heritable condition and given recent advances in our understanding of the \\n      genetics of schizophrenia, it is now possible to investigate how genetic factors \\n      may contribute to mortality. We made use of findings from genome-wide association\\n      studies (GWAS) to design a targeted panel (PsychPlex) for sequencing of exons of \\n      451 genes near index single nucleotide polymorphisms (SNPs) identified with GWAS.\\n      We sequenced the DNA of 95 deceased schizophrenia patients included in SURVIVE, a\\n      prospective, autopsy-based study of mentally ill persons in Denmark. We compared \\n      the allele frequencies of 1039 SNPs in these cases with the frequencies of 2000\\n      Danes without psychiatric diseases and calculated their deleteriousness (CADD)\\n      scores. For 81 SNPs highly associated with schizophrenia and CADD scores above\\n      15, expression profiles in the Genotype-Tissue Expression (GTEx) Project\\n      indicated that these variants were in exons, whose expressions are increased in\\n      several types of brain tissues, particularly in the cerebellum. Molecular pathway\\n      analysis indicated the involvement of 163 different pathways. As for rare SNP\\n      variants, most variants were scored as either benign or likely benign with an\\n      average of 17 variants of unknown significance per individual and no pathogenic\\n      variant. Our results highlight the potential of DNA sequencing of an exon panel\\n      to discover genetic factors that may be involved in the development of\\n      schizophrenia.\\nFAU - Espregueira Themudo, Goncalo\\nAU  - Espregueira Themudo G\\nAUID- ORCID: https://orcid.org/0000-0002-5019-9439\\nAD  - Section of Forensic Genetics, Department of Forensic Medicine, Faculty of Health \\n      and Medical Sciences, University of Copenhagen, Copenhagen, Denmark.\\n      gethemudo@gmail.com.\\nAD  - CIIMAR - Interdisciplinary Centre of Marine and Environmental Research of the\\n      University of Porto, Terminal de Cruzeiros do Porto de Leixoes, Avenida General\\n      Norton de Matos, S/N, 4450-208, Matosinhos, Portugal. gethemudo@gmail.com.\\nFAU - Leerschool, Anna-Roos\\nAU  - Leerschool AR\\nAD  - Section of Forensic Genetics, Department of Forensic Medicine, Faculty of Health \\n      and Medical Sciences, University of Copenhagen, Copenhagen, Denmark.\\nAD  - Department of Complex Genetics, Maastricht University, PO Box 616 6200, MD,\\n      Maastricht, The Netherlands.\\nFAU - Rodriguez-Proano, Carla\\nAU  - Rodriguez-Proano C\\nAD  - Section of Forensic Genetics, Department of Forensic Medicine, Faculty of Health \\n      and Medical Sciences, University of Copenhagen, Copenhagen, Denmark.\\nAD  - Clinical Laboratory, Ambulatory Clinical Surgical Center and Day Hospital \"El\\n      Batan\", Quito, Ecuador.\\nFAU - Christiansen, Sofie Lindgren\\nAU  - Christiansen SL\\nAUID- ORCID: https://orcid.org/0000-0003-3810-4123\\nAD  - Section of Forensic Genetics, Department of Forensic Medicine, Faculty of Health \\n      and Medical Sciences, University of Copenhagen, Copenhagen, Denmark.\\nFAU - Andersen, Jeppe Dyrberg\\nAU  - Andersen JD\\nAUID- ORCID: https://orcid.org/0000-0002-4395-5232\\nAD  - Section of Forensic Genetics, Department of Forensic Medicine, Faculty of Health \\n      and Medical Sciences, University of Copenhagen, Copenhagen, Denmark.\\nFAU - Busch, Johannes Rodbro\\nAU  - Busch JR\\nAD  - Section of Forensic Pathology, Department of Forensic Medicine, Faculty of Health\\n      and Medical Sciences, University of Copenhagen, Copenhagen, Denmark.\\nFAU - Christensen, Martin Roest\\nAU  - Christensen MR\\nAD  - Section of Forensic Pathology, Department of Forensic Medicine, Faculty of Health\\n      and Medical Sciences, University of Copenhagen, Copenhagen, Denmark.\\nFAU - Banner, Jytte\\nAU  - Banner J\\nAUID- ORCID: https://orcid.org/0000-0002-7559-1285\\nAD  - Section of Forensic Pathology, Department of Forensic Medicine, Faculty of Health\\n      and Medical Sciences, University of Copenhagen, Copenhagen, Denmark.\\nFAU - Morling, Niels\\nAU  - Morling N\\nAUID- ORCID: https://orcid.org/0000-0002-9463-5087\\nAD  - Section of Forensic Genetics, Department of Forensic Medicine, Faculty of Health \\n      and Medical Sciences, University of Copenhagen, Copenhagen, Denmark.\\nLA  - eng\\nPT  - Journal Article\\nDEP - 20191126\\nPL  - Germany\\nTA  - Int J Legal Med\\nJT  - International journal of legal medicine\\nJID - 9101456\\nSB  - IM\\nMH  - Adult\\nMH  - Aged\\nMH  - Cause of Death\\nMH  - Denmark\\nMH  - *Exons\\nMH  - Female\\nMH  - Gene Expression Profiling\\nMH  - *Gene Frequency\\nMH  - *Genetic Variation\\nMH  - Genome-Wide Association Study\\nMH  - Humans\\nMH  - Male\\nMH  - Middle Aged\\nMH  - *Polymorphism, Single Nucleotide\\nMH  - Prospective Studies\\nMH  - Schizophrenia/*genetics/mortality\\nMH  - Sequence Analysis, DNA/*methods\\nOTO - NOTNLM\\nOT  - Genomics\\nOT  - Haloplex\\nOT  - Massively parallel sequencing\\nOT  - Molecular autopsy\\nOT  - Schizophrenia\\nEDAT- 2019/11/28 06:00\\nMHDA- 2020/09/15 06:00\\nCRDT- 2019/11/28 06:00\\nPHST- 2019/08/29 00:00 [received]\\nPHST- 2019/11/13 00:00 [accepted]\\nPHST- 2019/11/28 06:00 [pubmed]\\nPHST- 2020/09/15 06:00 [medline]\\nPHST- 2019/11/28 06:00 [entrez]\\nAID - 10.1007/s00414-019-02212-z [doi]\\nAID - 10.1007/s00414-019-02212-z [pii]\\nPST - ppublish\\nSO  - Int J Legal Med. 2020 Jan;134(1):135-147. doi: 10.1007/s00414-019-02212-z. Epub\\n      2019 Nov 26.\\n\\nPMID- 31611888\\nOWN - NLM\\nSTAT- PubMed-not-MEDLINE\\nLR  - 20200930\\nIS  - 1664-462X (Print)\\nIS  - 1664-462X (Linking)\\nVI  - 10\\nDP  - 2019\\nTI  - Alternative Oxidase (AOX) Senses Stress Levels to Coordinate Auxin-Induced\\n      Reprogramming From Seed Germination to Somatic Embryogenesis-A Role Relevant for \\n      Seed Vigor Prediction and Plant Robustness.\\nPG  - 1134\\nLID - 10.3389/fpls.2019.01134 [doi]\\nAB  - Somatic embryogenesis (SE) is the most striking and prominent example of plant\\n      plasticity upon severe stress. Inducing immature carrot seeds perform SE as\\n      substitute to germination by auxin treatment can be seen as switch between stress\\n      levels associated to morphophysiological plasticity. This experimental system is \\n      highly powerful to explore stress response factors that mediate the metabolic\\n      switch between cell and tissue identities. Developmental plasticity per se is an \\n      emerging trait for in vitro systems and crop improvement. It is supposed to\\n      underlie multi-stress tolerance. High plasticity can protect plants throughout\\n      life cycles against variable abiotic and biotic conditions. We provide proof of\\n      concepts for the existing hypothesis that alternative oxidase (AOX) can be\\n      relevant for developmental plasticity and be associated to yield stability. Our\\n      perspective on AOX as relevant coordinator of cell reprogramming is supported by \\n      real-time polymerase chain reaction (PCR) analyses and gross metabolism data from\\n      calorespirometry complemented by SHAM-inhibitor studies on primed, elevated\\n      partial pressure of oxygen (EPPO)-stressed, and endophyte-treated seeds. In\\n      silico studies on public experimental data from diverse species strengthen\\n      generality of our insights. Finally, we highlight ready-to-use concepts for plant\\n      selection and optimizing in vivo and in vitro propagation that do not require\\n      further details on molecular physiology and metabolism. This is demonstrated by\\n      applying our research & technology concepts to pea genotypes with differential\\n      yield performance in multilocation fields and chickpea types known for\\n      differential robustness in the field. By using these concepts and tools\\n      appropriately, also other marker candidates than AOX and complex genomics data\\n      can be efficiently validated for prebreeding and seed vigor prediction.\\nCI  - Copyright (c) 2019 Mohanapriya, Bharadwaj, Noceda, Costa, Kumar, Sathishkumar,\\n      Thiers, Santos Macedo, Silva, Annicchiarico, Groot, Kodde, Kumari, Gupta and\\n      Arnholdt-Schmitt.\\nFAU - Mohanapriya, Gunasekaran\\nAU  - Mohanapriya G\\nAD  - Plant Genetic Engineering Laboratory, Department of Biotechnology, Bharathiar\\n      University, Coimbatore, India.\\nAD  - Functional Cell Reprogramming and Organism Plasticity (FunCROP), University of\\n      Evora, Evora, Portugal.\\nFAU - Bharadwaj, Revuru\\nAU  - Bharadwaj R\\nAD  - Plant Genetic Engineering Laboratory, Department of Biotechnology, Bharathiar\\n      University, Coimbatore, India.\\nAD  - Functional Cell Reprogramming and Organism Plasticity (FunCROP), University of\\n      Evora, Evora, Portugal.\\nFAU - Noceda, Carlos\\nAU  - Noceda C\\nAD  - Functional Cell Reprogramming and Organism Plasticity (FunCROP), University of\\n      Evora, Evora, Portugal.\\nAD  - Cell and Molecular Biology of Plants (BPOCEMP)/Industrial Biotechnology and\\n      Bioproducts, Department of Sciences of the Vidaydela Agriculture, University of\\n      the Armed Forces-ESPE, Milagro, Ecuador.\\nAD  - Faculty of Engineering, State University of Milagro (UNEMI), Milagro, Ecuador.\\nFAU - Costa, Jose Helio\\nAU  - Costa JH\\nAD  - Functional Cell Reprogramming and Organism Plasticity (FunCROP), University of\\n      Evora, Evora, Portugal.\\nAD  - Functional Genomics and Bioinformatics Group, Department of Biochemistry and\\n      Molecular Biology, Federal University of Ceara, Fortaleza, Brazil.\\nFAU - Kumar, Sarma Rajeev\\nAU  - Kumar SR\\nAD  - Functional Cell Reprogramming and Organism Plasticity (FunCROP), University of\\n      Evora, Evora, Portugal.\\nFAU - Sathishkumar, Ramalingam\\nAU  - Sathishkumar R\\nAD  - Plant Genetic Engineering Laboratory, Department of Biotechnology, Bharathiar\\n      University, Coimbatore, India.\\nAD  - Functional Cell Reprogramming and Organism Plasticity (FunCROP), University of\\n      Evora, Evora, Portugal.\\nFAU - Thiers, Karine Leitao Lima\\nAU  - Thiers KLL\\nAD  - Functional Genomics and Bioinformatics Group, Department of Biochemistry and\\n      Molecular Biology, Federal University of Ceara, Fortaleza, Brazil.\\nFAU - Santos Macedo, Elisete\\nAU  - Santos Macedo E\\nAD  - Functional Cell Reprogramming and Organism Plasticity (FunCROP), University of\\n      Evora, Evora, Portugal.\\nFAU - Silva, Sofia\\nAU  - Silva S\\nAD  - Functional Cell Reprogramming and Organism Plasticity (FunCROP), University of\\n      Evora, Evora, Portugal.\\nFAU - Annicchiarico, Paolo\\nAU  - Annicchiarico P\\nAD  - Council for Agricultural Research and Economics (CREA), Research Centre for\\n      Animal Production and Aquaculture, Lodi, Italy.\\nFAU - Groot, Steven P C\\nAU  - Groot SPC\\nAD  - Wageningen Plant Research, Wageningen University & Research, Wageningen,\\n      Netherlands.\\nFAU - Kodde, Jan\\nAU  - Kodde J\\nAD  - Wageningen Plant Research, Wageningen University & Research, Wageningen,\\n      Netherlands.\\nFAU - Kumari, Aprajita\\nAU  - Kumari A\\nAD  - National Institute of Plant Genome Research, New Delhi, India.\\nFAU - Gupta, Kapuganti Jagadis\\nAU  - Gupta KJ\\nAD  - Functional Cell Reprogramming and Organism Plasticity (FunCROP), University of\\n      Evora, Evora, Portugal.\\nAD  - National Institute of Plant Genome Research, New Delhi, India.\\nFAU - Arnholdt-Schmitt, Birgit\\nAU  - Arnholdt-Schmitt B\\nAD  - Functional Cell Reprogramming and Organism Plasticity (FunCROP), University of\\n      Evora, Evora, Portugal.\\nAD  - Functional Genomics and Bioinformatics Group, Department of Biochemistry and\\n      Molecular Biology, Federal University of Ceara, Fortaleza, Brazil.\\nAD  - CERNAS-Research Center for Natural Resources, Environment and Society, Department\\n      of Environment, Escola Superior Agraria de Coimbra, Coimbra, Portugal.\\nLA  - eng\\nPT  - Journal Article\\nDEP - 20190920\\nPL  - Switzerland\\nTA  - Front Plant Sci\\nJT  - Frontiers in plant science\\nJID - 101568200\\nPMC - PMC6776121\\nOTO - NOTNLM\\nOT  - developmental plasticity\\nOT  - endophytes\\nOT  - environmental stress\\nOT  - metabolic biomarker\\nOT  - plant performance prediction\\nOT  - seed technology\\nEDAT- 2019/10/16 06:00\\nMHDA- 2019/10/16 06:01\\nCRDT- 2019/10/16 06:00\\nPHST- 2019/04/06 00:00 [received]\\nPHST- 2019/08/16 00:00 [accepted]\\nPHST- 2019/10/16 06:00 [entrez]\\nPHST- 2019/10/16 06:00 [pubmed]\\nPHST- 2019/10/16 06:01 [medline]\\nAID - 10.3389/fpls.2019.01134 [doi]\\nPST - epublish\\nSO  - Front Plant Sci. 2019 Sep 20;10:1134. doi: 10.3389/fpls.2019.01134. eCollection\\n      2019.\\n\\nPMID- 31254307\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20200330\\nLR  - 20200330\\nIS  - 1365-294X (Electronic)\\nIS  - 0962-1083 (Linking)\\nVI  - 28\\nIP  - 13\\nDP  - 2019 Jul\\nTI  - Phylogeography of a cryptic speciation continuum in Eurasian spadefoot toads\\n      (Pelobates).\\nPG  - 3257-3270\\nLID - 10.1111/mec.15133 [doi]\\nAB  - Cryptic phylogeographic diversifications provide unique models to examine the\\n      role of phylogenetic divergence on the evolution of reproductive isolation,\\n      without extrinsic factors such as ecological and behavioural differentiation.\\n      Yet, to date very few comparative studies have been attempted within such\\n      radiations. Here, we characterize a new speciation continuum in a group of\\n      widespread Eurasian amphibians, the Pelobates spadefoot toads, by conducting\\n      multilocus (restriction site associated DNA sequencing and mitochondrial DNA)\\n      phylogenetic, phylogeographic and hybrid zone analyses. Within the P. syriacus\\n      complex, we discovered species-level cryptic divergences (>5 million years ago\\n      [My]) between populations distributed in the Near-East (hereafter P. syriacus\\n      sensu stricto [s.s.]) and southeastern Europe (hereafter P. balcanicus), each\\n      featuring deep intraspecific lineages. Altogether, we could scale hybridizability\\n      to divergence time along six different stages, spanning from sympatry without\\n      gene flow (P. fuscus and P. balcanicus, >10 My), parapatry with highly restricted\\n      hybridization (P. balcanicus and P. syriacus s.s., >5 My), narrow hybrid zones\\n      (~15 km) consistent with partial reproductive isolation (P. fuscus and P.\\n      vespertinus, ~3 My), to extensive admixture between Pleistocene and refugial\\n      lineages (</=2 My). This full spectrum empirically supports a gradual build up of\\n      reproductive barriers through time, reversible up until a threshold that we\\n      estimate at ~3 My. Hence, cryptic phylogeographic lineages may fade away or\\n      become reproductively isolated species simply depending on the time they persist \\n      in allopatry, and without definite ecomorphological divergence.\\nCI  - (c) 2019 John Wiley & Sons Ltd.\\nFAU - Dufresnes, Christophe\\nAU  - Dufresnes C\\nAUID- ORCID: 0000-0002-8497-8908\\nAD  - Department of Animal and Plant Sciences, University of Sheffield, Sheffield, UK.\\nAD  - Hintermann & Weber SA, Montreux, Switzerland.\\nAD  - Laboratory for Conservation Biology, University of Lausanne, Lausanne,\\n      Switzerland.\\nFAU - Strachinis, Ilias\\nAU  - Strachinis I\\nAUID- ORCID: 0000-0001-9944-4522\\nAD  - School of Biology, Aristotle University of Thessaloniki, Thessaloniki, Greece.\\nFAU - Suriadna, Nataliia\\nAU  - Suriadna N\\nAUID- ORCID: 0000-0002-0681-4465\\nAD  - Melitopol Insitute of Ecology and Social Technologies of University \"Ukraine\",\\n      Melitopol, Zaporizhia, Ukraine.\\nFAU - Mykytynets, Galyna\\nAU  - Mykytynets G\\nAUID- ORCID: 0000-0001-9623-8088\\nAD  - Pryazovsky National Nature Park, Melitopol, Zaporizhia, Ukraine.\\nFAU - Cogalniceanu, Dan\\nAU  - Cogalniceanu D\\nAUID- ORCID: 0000-0003-2959-014X\\nAD  - Faculty of Natural Sciences and Agricultural Sciences, University Ovidius\\n      Constanta, Constanta, Romania.\\nFAU - Szekely, Paul\\nAU  - Szekely P\\nAUID- ORCID: 0000-0002-9252-1596\\nAD  - Departamento de Ciencias Biologicas, EcoSs Lab, Universidad Tecnica Particular de\\n      Loja, Loja, Ecuador.\\nFAU - Vukov, Tanja\\nAU  - Vukov T\\nAUID- ORCID: 0000-0002-2365-8679\\nAD  - Department of Evolutionary Biology, Institute for Biological Research \"Sinisa\\n      Stankovic\", National Institute of Republic of Serbia, University of Belgrade,\\n      Belgrade, Serbia.\\nFAU - Arntzen, Jan W\\nAU  - Arntzen JW\\nAUID- ORCID: 0000-0003-3229-5993\\nAD  - Naturalis Biodiversity Center, Leiden, The Netherlands.\\nFAU - Wielstra, Ben\\nAU  - Wielstra B\\nAUID- ORCID: 0000-0002-7112-5965\\nAD  - Naturalis Biodiversity Center, Leiden, The Netherlands.\\nAD  - Institute of Biology Leiden, Leiden University, Leiden, The Netherlands.\\nFAU - Lymberakis, Petros\\nAU  - Lymberakis P\\nAUID- ORCID: 0000-0002-5067-9600\\nAD  - Natural History Museum of Crete, University of Crete, Irakleio, Crete, Greece.\\nFAU - Geffen, Eli\\nAU  - Geffen E\\nAUID- ORCID: 0000-0002-3028-0045\\nAD  - School of Zoology, Tel Aviv University, Tel Aviv, Israel.\\nFAU - Gafny, Sarig\\nAU  - Gafny S\\nAD  - School of Marine Sciences, Ruppin Academic Center, Michmoret, Israel.\\nFAU - Kumlutas, Yusuf\\nAU  - Kumlutas Y\\nAUID- ORCID: 0000-0003-1154-6757\\nAD  - Department of Biology, Faculty of Science, Dokuz Eylul University, Buca, Izmir,\\n      Turkey.\\nAD  - Research and Application Center for Fauna and Flora, Dokuz Eylul University,\\n      Buca, Izmir, Turkey.\\nFAU - Ilgaz, Cetin\\nAU  - Ilgaz C\\nAUID- ORCID: 0000-0001-7862-9106\\nAD  - Department of Biology, Faculty of Science, Dokuz Eylul University, Buca, Izmir,\\n      Turkey.\\nAD  - Research and Application Center for Fauna and Flora, Dokuz Eylul University,\\n      Buca, Izmir, Turkey.\\nFAU - Candan, Kamil\\nAU  - Candan K\\nAUID- ORCID: 0000-0002-6934-3971\\nAD  - Department of Biology, Faculty of Science, Dokuz Eylul University, Buca, Izmir,\\n      Turkey.\\nAD  - Research and Application Center for Fauna and Flora, Dokuz Eylul University,\\n      Buca, Izmir, Turkey.\\nFAU - Mizsei, Edvard\\nAU  - Mizsei E\\nAUID- ORCID: 0000-0002-8162-5293\\nAD  - Department of Tisza River Research, Danube Research Institute, Centre for\\n      Ecological Research, Hungarian Academy of Sciences, Debrecen, Hungary.\\nFAU - Szabolcs, Marton\\nAU  - Szabolcs M\\nAUID- ORCID: 0000-0001-9375-9937\\nAD  - Department of Tisza River Research, Danube Research Institute, Centre for\\n      Ecological Research, Hungarian Academy of Sciences, Debrecen, Hungary.\\nFAU - Kolenda, Krzysztof\\nAU  - Kolenda K\\nAUID- ORCID: 0000-0001-8683-9867\\nAD  - Department of Evolutionary Biology and Conservation of Vertebrates, Institute of \\n      Environmental Biology, University of Wroclaw, Wroclaw, Poland.\\nFAU - Smirnov, Nazar\\nAU  - Smirnov N\\nAUID- ORCID: 0000-0001-9731-6404\\nAD  - Department of Nature, Chernivtsi Regional Museum, Chernivtsi, Ukraine.\\nFAU - Geniez, Philippe\\nAU  - Geniez P\\nAD  - CEFE, EPHE-PSL, CNRS, University of Montpellier, University Paul Valery\\n      Montpellier 3, IRD, Montpellier, France.\\nFAU - Lukanov, Simeon\\nAU  - Lukanov S\\nAUID- ORCID: 0000-0002-2546-6590\\nAD  - Institute of Biodiversity and Ecosystem Research, Bulgarian Academy of Sciences, \\n      Sofia, Bulgaria.\\nFAU - Crochet, Pierre-Andre\\nAU  - Crochet PA\\nAUID- ORCID: 0000-0002-0422-3960\\nAD  - CEFE, CNRS, University of Montpellier, University Paul Valery Montpellier 3,\\n      EPHE, IRD, Montpellier, France.\\nFAU - Dubey, Sylvain\\nAU  - Dubey S\\nAD  - Hintermann & Weber SA, Montreux, Switzerland.\\nAD  - Department of Ecology and Evolution, University of Lausanne, Lausanne,\\n      Switzerland.\\nAD  - Agrosustain SA, Nyon, Switzerland.\\nFAU - Perrin, Nicolas\\nAU  - Perrin N\\nAUID- ORCID: 0000-0002-7756-6323\\nAD  - Department of Ecology and Evolution, University of Lausanne, Lausanne,\\n      Switzerland.\\nFAU - Litvinchuk, Spartak N\\nAU  - Litvinchuk SN\\nAD  - Institute of Cytology, Russian Academy of Sciences, St. Petersburg, Russia.\\nAD  - Department of Zoology and Physiology, Dagestan State University, Makhachkala,\\n      Russia.\\nFAU - Denoel, Mathieu\\nAU  - Denoel M\\nAUID- ORCID: 0000-0002-3586-8323\\nAD  - Laboratory of Fish and Amphibian Ethology, Behavioural Biology Group, Freshwater \\n      and OCeanic science Unit of reSearch (FOCUS), University of Liege, Liege,\\n      Belgium.\\nLA  - eng\\nSI  - GENBANK/MK902544\\nSI  - GENBANK/MK902597\\nSI  - GENBANK/MK902523\\nSI  - GENBANK/MK902543\\nSI  - GENBANK/PRJNA542138\\nSI  - GENBANK/SAMN11612144\\nSI  - GENBANK/SAMN11612336\\nPT  - Journal Article\\nPT  - Research Support, Non-U.S. Gov\\'t\\nDEP - 20190628\\nPL  - England\\nTA  - Mol Ecol\\nJT  - Molecular ecology\\nJID - 9214478\\nRN  - 0 (DNA, Mitochondrial)\\nSB  - IM\\nMH  - Animals\\nMH  - Anura/*classification\\nMH  - Bayes Theorem\\nMH  - DNA, Mitochondrial/genetics\\nMH  - Europe\\nMH  - Gene Flow\\nMH  - *Genetic Speciation\\nMH  - *Genetics, Population\\nMH  - Hybridization, Genetic\\nMH  - Middle East\\nMH  - Phylogeny\\nMH  - Phylogeography\\nMH  - *Reproductive Isolation\\nMH  - Sequence Analysis, DNA\\nMH  - Sympatry\\nOTO - NOTNLM\\nOT  - *amphibia\\nOT  - *anuran\\nOT  - *hybrid zones\\nOT  - *population genomics\\nOT  - *reproductive isolation\\nOT  - *speciation continuum\\nEDAT- 2019/06/30 06:00\\nMHDA- 2020/03/31 06:00\\nCRDT- 2019/06/30 06:00\\nPHST- 2019/02/02 00:00 [received]\\nPHST- 2019/03/30 00:00 [revised]\\nPHST- 2019/05/17 00:00 [accepted]\\nPHST- 2019/06/30 06:00 [pubmed]\\nPHST- 2020/03/31 06:00 [medline]\\nPHST- 2019/06/30 06:00 [entrez]\\nAID - 10.1111/mec.15133 [doi]\\nPST - ppublish\\nSO  - Mol Ecol. 2019 Jul;28(13):3257-3270. doi: 10.1111/mec.15133. Epub 2019 Jun 28.\\n\\nPMID- 31155034\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20200108\\nLR  - 20200108\\nIS  - 1879-3231 (Electronic)\\nIS  - 0093-691X (Linking)\\nVI  - 133\\nDP  - 2019 Jul 15\\nTI  - Sperm collection and storage for the sustainable management of amphibian\\n      biodiversity.\\nPG  - 187-200\\nLID - S0093-691X(19)30085-8 [pii]\\nLID - 10.1016/j.theriogenology.2019.03.035 [doi]\\nAB  - Current rates of biodiversity loss pose an unprecedented challenge to the\\n      conservation community, particularly with amphibians and freshwater fish as the\\n      most threatened vertebrates. An increasing number of environmental challenges,\\n      including habitat loss, pathogens, and global warming, demand a global response\\n      toward the sustainable management of ecosystems and their biodiversity.\\n      Conservation Breeding Programs (CBPs) are needed for the sustainable management\\n      of amphibian species threatened with extinction. CBPs support species survival\\n      while increasing public awareness and political influence. Current CBPs only\\n      cater for 10% of the almost 500 amphibian species in need. However, the use of\\n      sperm storage to increase efficiency and reliability, along with an increased\\n      number of CBPs, offer the potential to significantly reduce species loss. The\\n      establishment and refinement of techniques over the last two decades, for the\\n      collection and storage of amphibian spermatozoa, gives confidence for their use\\n      in CBPs and other biotechnical applications. Cryopreserved spermatozoa has\\n      produced breeding pairs of frogs and salamanders and the stage is set for\\n      Lifecycle Proof of Concept Programs that use cryopreserved sperm in CBPs along\\n      with repopulation, supplementation, and translocation programs. The application\\n      of cryopreserved sperm in CBPs, is complimentary to but separate from archival\\n      gene banking and general cell and tissue storage. However, where appropriate\\n      amphibian sperm banking should be integrated into other global biobanking\\n      projects, especially those for fish, and those that include the use of\\n      cryopreserved material for genomics and other research. Research over a broader\\n      range of amphibian species, and more uniformity in experimental methodology, is\\n      needed to inform both theory and application. Genomics is revolutionising our\\n      understanding of biological processes and increasingly guiding species\\n      conservation through the identification of evolutionary significant units as the \\n      conservation focus, and through revealing the intimate relationship between\\n      evolutionary history and sperm physiology that ultimately affects the amenability\\n      of sperm to refrigerated or frozen storage. In the present review we provide a\\n      nascent phylogenetic framework for integration with other research lines to\\n      further the potential of amphibian sperm banking.\\nCI  - Copyright (c) 2019 Elsevier Inc. All rights reserved.\\nFAU - Browne, Robert K\\nAU  - Browne RK\\nAD  - Sustainability America, Sarteneja, Corozal District, Belize. Electronic address: \\n      robert.browne@gmail.com.\\nFAU - Silla, Aimee J\\nAU  - Silla AJ\\nAD  - School of Earth, Atmospheric and Life Sciences, University of Wollongong, NSW,\\n      2522, Australia.\\nFAU - Upton, Rose\\nAU  - Upton R\\nAD  - School of Environmental and Life Sciences, University of Newcastle, Callaghan\\n      Drive, Callaghan, NSW, 2308, Australia.\\nFAU - Della-Togna, Gina\\nAU  - Della-Togna G\\nAD  - Smithsonian Tropical Research Institute, Panama Amphibian Rescue and Conservation\\n      Project, Panama City, Panama; Universidad Interamericana de Panama, Direccion de \\n      Investigacion, Sede Central, Panama.\\nFAU - Marcec-Greaves, Ruth\\nAU  - Marcec-Greaves R\\nAD  - National Amphibian Conservation Center Detroit Zoological Society, Detroit, USA.\\nFAU - Shishova, Natalia V\\nAU  - Shishova NV\\nAD  - Institute of Cell Biophysics, Russian Academy of Sciences, Pushchino, Moscow\\n      Region, 142290, Russia.\\nFAU - Uteshev, Victor K\\nAU  - Uteshev VK\\nAD  - Institute of Cell Biophysics, Russian Academy of Sciences, Pushchino, Moscow\\n      Region, 142290, Russia.\\nFAU - Proano, Belin\\nAU  - Proano B\\nAD  - Escuela de Ciencias Biologicas, Pontificia Universidad Catolica Del Ecuador,\\n      Ecuador.\\nFAU - Perez, Oscar D\\nAU  - Perez OD\\nAD  - Escuela de Ciencias Biologicas, Pontificia Universidad Catolica Del Ecuador,\\n      Ecuador.\\nFAU - Mansour, Nabil\\nAU  - Mansour N\\nAD  - Faculty of Veterinary Medicine, Kafrelsheikh University, Egypt.\\nFAU - Kaurova, Svetlana A\\nAU  - Kaurova SA\\nAD  - Institute of Cell Biophysics, Russian Academy of Sciences, Pushchino, Moscow\\n      Region, 142290, Russia.\\nFAU - Gakhova, Edith N\\nAU  - Gakhova EN\\nAD  - Institute of Cell Biophysics, Russian Academy of Sciences, Pushchino, Moscow\\n      Region, 142290, Russia.\\nFAU - Cosson, Jacky\\nAU  - Cosson J\\nAD  - University of South Bohemia in Ceske Budejovice, Faculty of Fisheries and\\n      Protection of Waters, South Bohemian Research Center of Aquaculture and\\n      Biodiversity of Hydrocenoses, 38925, Vodnany, Czech Republic.\\nFAU - Dyzuba, Borys\\nAU  - Dyzuba B\\nAD  - University of South Bohemia in Ceske Budejovice, Faculty of Fisheries and\\n      Protection of Waters, South Bohemian Research Center of Aquaculture and\\n      Biodiversity of Hydrocenoses, 38925, Vodnany, Czech Republic.\\nFAU - Kramarova, Ludmila I\\nAU  - Kramarova LI\\nAD  - Institute of Theoretical and Experimental Biophysics, Russian Academy of\\n      Sciences, Pushchino, Moscow Region, 142290, Russia.\\nFAU - McGinnity, Dale\\nAU  - McGinnity D\\nAD  - Nashville Zoo at Grassmere, Nashville, TN, 37211, USA.\\nFAU - Gonzalez, Manuel\\nAU  - Gonzalez M\\nAD  - Departamento de Produccion Animal, Universidad de Guadalajara, Guadalajara,\\n      Jalisco, Mexico.\\nFAU - Clulow, John\\nAU  - Clulow J\\nAD  - School of Environmental and Life Sciences, University of Newcastle, Callaghan\\n      Drive, Callaghan, NSW, 2308, Australia.\\nFAU - Clulow, Simon\\nAU  - Clulow S\\nAD  - School of Environmental and Life Sciences, University of Newcastle, Callaghan\\n      Drive, Callaghan, NSW, 2308, Australia; Department of Biological Sciences,\\n      Macquarie University, Sydney, NSW, 2109, Australia.\\nLA  - eng\\nPT  - Journal Article\\nPL  - United States\\nTA  - Theriogenology\\nJT  - Theriogenology\\nJID - 0421510\\nSB  - IM\\nMH  - *Amphibians\\nMH  - Animals\\nMH  - *Biodiversity\\nMH  - Breeding\\nMH  - Cryopreservation/veterinary\\nMH  - DNA Fragmentation\\nMH  - Phylogeny\\nMH  - Reproduction\\nMH  - Semen Preservation/methods/veterinary\\nMH  - Specimen Handling\\nMH  - Sperm Retrieval/*veterinary\\nOTO - NOTNLM\\nOT  - Amphibian\\nOT  - Conservation breeding programs\\nOT  - Cryopreservation\\nOT  - Sperm\\nOT  - Storage\\nEDAT- 2019/06/04 06:00\\nMHDA- 2020/01/09 06:00\\nCRDT- 2019/06/04 06:00\\nPHST- 2019/03/25 00:00 [received]\\nPHST- 2019/03/30 00:00 [accepted]\\nPHST- 2019/06/04 06:00 [entrez]\\nPHST- 2019/06/04 06:00 [pubmed]\\nPHST- 2020/01/09 06:00 [medline]\\nAID - S0093-691X(19)30085-8 [pii]\\nAID - 10.1016/j.theriogenology.2019.03.035 [doi]\\nPST - ppublish\\nSO  - Theriogenology. 2019 Jul 15;133:187-200. doi:\\n      10.1016/j.theriogenology.2019.03.035.\\n\\nPMID- 31154984\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20200427\\nLR  - 20200722\\nIS  - 1471-2970 (Electronic)\\nIS  - 0962-8436 (Linking)\\nVI  - 374\\nIP  - 1777\\nDP  - 2019 Jul 22\\nTI  - Is there convergence of gut microbes in blood-feeding vertebrates?\\nPG  - 20180249\\nLID - 10.1098/rstb.2018.0249 [doi]\\nAB  - Animal microbiomes play an important role in dietary adaptation, yet the extent\\n      to which microbiome changes exhibit parallel evolution is unclear. Of particular \\n      interest is an adaptation to extreme diets, such as blood, which poses special\\n      challenges in its content of proteins and lack of essential nutrients. In this\\n      study, we assessed taxonomic signatures (by 16S rRNA amplicon profiling) and\\n      potential functional signatures (inferred by Phylogenetic Investigation of\\n      Communities by Reconstruction of Unobserved States (PICRUSt)) of haematophagy in \\n      birds and bats. Our goal was to test three alternative hypotheses: no convergence\\n      of microbiomes, convergence in taxonomy and convergence in function. We find a\\n      statistically significant effect of haematophagy in terms of microbial taxonomic \\n      convergence across the blood-feeding bats and birds, although this effect is\\n      small compared to the differences found between haematophagous and\\n      non-haematophagous species within the two host clades. We also find some evidence\\n      of convergence at the predicted functional level, although it is possible that\\n      the lack of metagenomic data and the poor representation of microbial lineages\\n      adapted to haematophagy in genome databases limit the power of this approach. The\\n      results provide a paradigm for exploring convergent microbiome evolution\\n      replicated with independent contrasts in different host lineages. This article is\\n      part of the theme issue \\'Convergent evolution in the genomics era: new insights\\n      and directions\\'.\\nFAU - Song, Se Jin\\nAU  - Song SJ\\nAD  - 1 Department of Pediatrics, University of California San Diego , La Jolla, CA\\n      92093 , USA.\\nFAU - Sanders, Jon G\\nAU  - Sanders JG\\nAD  - 1 Department of Pediatrics, University of California San Diego , La Jolla, CA\\n      92093 , USA.\\nAD  - 2 Cornell Institute for Host-Microbe Interactions and Disease, Cornell University\\n      , Ithaca, NY 14853 , USA.\\nFAU - Baldassarre, Daniel T\\nAU  - Baldassarre DT\\nAD  - 3 Department of Biology, University of Miami , Coral Gables, FL 33146 , USA.\\nFAU - Chaves, Jaime A\\nAU  - Chaves JA\\nAD  - 4 Colegio de Ciencias Biologicas y Ambientales, Universidad San Francisco de\\n      Quito, Quito, Ecuador.\\nAD  - 5 Galapagos Science Center, Puerto Baquerizo Moreno, Galapagos, Ecuador.\\nFAU - Johnson, Nicholas S\\nAU  - Johnson NS\\nAD  - 6 United States Geological Survey, Great Lakes Science Center , Hammond Bay\\n      Biological Station, Millersburg, MI 49759 , USA.\\nFAU - Piaggio, Antoinette J\\nAU  - Piaggio AJ\\nAD  - 7 National Wildlife Research Center, U.S. Department of Agriculture , Fort\\n      Collins, CO 80521 , USA.\\nFAU - Stuckey, Matthew J\\nAU  - Stuckey MJ\\nAD  - 8 Department of Population Health and Reproduction, School of Veterinary\\n      Medicine, University of California Davis , Davis CA 95616 , USA.\\nFAU - Novakova, Eva\\nAU  - Novakova E\\nAD  - 9 Faculty of Science, University of South Bohemia , Ceske Budejovice , Czech\\n      Republic.\\nAD  - 10 Biology Centre of ASCR, Institute of Parasitology , Ceske Budejovice , Czech\\n      Republic.\\nFAU - Metcalf, Jessica L\\nAU  - Metcalf JL\\nAD  - 11 Department of Animal Sciences, Colorado State University , Fort Collins CO\\n      80523 , USA.\\nFAU - Chomel, Bruno B\\nAU  - Chomel BB\\nAD  - 8 Department of Population Health and Reproduction, School of Veterinary\\n      Medicine, University of California Davis , Davis CA 95616 , USA.\\nFAU - Aguilar-Setien, Alvaro\\nAU  - Aguilar-Setien A\\nAD  - 12 Unidad de Investigacion Medica en Inmunologia, Coordinacion de Investigacion, \\n      Instituto Mexicano del Seguro Social, Centro Medico Nacional Siglo XXI, Hospital \\n      de Pediatria 3er piso , Mexico City , Mexico.\\nFAU - Knight, Rob\\nAU  - Knight R\\nAD  - 1 Department of Pediatrics, University of California San Diego , La Jolla, CA\\n      92093 , USA.\\nAD  - 13 Center for Microbiome Research, University of California San Diego , San\\n      Diego, CA 92093 , USA.\\nAD  - 14 Department of Computer Science and Engineering, University of California San\\n      Diego , San Diego, CA 92093 , USA.\\nFAU - McKenzie, Valerie J\\nAU  - McKenzie VJ\\nAD  - 15 Department of Ecology and Evolutionary Biology, University of Colorado Boulder\\n      , Boulder, CO 80309 , USA.\\nLA  - eng\\nSI  - figshare/10.6084/m9.figshare.c.4467776\\nPT  - Journal Article\\nPT  - Research Support, Non-U.S. Gov\\'t\\nPT  - Research Support, U.S. Gov\\'t, Non-P.H.S.\\nDEP - 20190603\\nPL  - England\\nTA  - Philos Trans R Soc Lond B Biol Sci\\nJT  - Philosophical transactions of the Royal Society of London. Series B, Biological\\n      sciences\\nJID - 7503623\\nRN  - 0 (DNA, Bacterial)\\nRN  - 0 (RNA, Ribosomal, 16S)\\nSB  - IM\\nMH  - Animals\\nMH  - Bacteria/classification/*genetics/isolation & purification\\nMH  - Biological Evolution\\nMH  - Birds/*genetics/microbiology/physiology\\nMH  - Chiroptera/*genetics/microbiology/physiology\\nMH  - DNA, Bacterial/genetics\\nMH  - Feeding Behavior\\nMH  - *Gastrointestinal Microbiome\\nMH  - Phylogeny\\nMH  - RNA, Ribosomal, 16S/genetics\\nPMC - PMC6560276\\nOTO - NOTNLM\\nOT  - *convergence\\nOT  - *haematophagy\\nOT  - *microbiome\\nEDAT- 2019/06/04 06:00\\nMHDA- 2020/04/28 06:00\\nCRDT- 2019/06/04 06:00\\nPHST- 2019/06/04 06:00 [entrez]\\nPHST- 2019/06/04 06:00 [pubmed]\\nPHST- 2020/04/28 06:00 [medline]\\nAID - 10.1098/rstb.2018.0249 [doi]\\nPST - ppublish\\nSO  - Philos Trans R Soc Lond B Biol Sci. 2019 Jul 22;374(1777):20180249. doi:\\n      10.1098/rstb.2018.0249. Epub 2019 Jun 3.\\n\\nPMID- 31067326\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20200603\\nLR  - 20200603\\nIS  - 1465-7333 (Electronic)\\nIS  - 0022-1503 (Linking)\\nVI  - 110\\nIP  - 3\\nDP  - 2019 May 7\\nTI  - The Expectations and Challenges of Wildlife Disease Research in the Era of\\n      Genomics: Forecasting with a Horizon Scan-like Exercise.\\nPG  - 261-274\\nLID - 10.1093/jhered/esz001 [doi]\\nAB  - The outbreak and transmission of disease-causing pathogens are contributing to\\n      the unprecedented rate of biodiversity decline. Recent advances in genomics have \\n      coalesced into powerful tools to monitor, detect, and reconstruct the role of\\n      pathogens impacting wildlife populations. Wildlife researchers are thus uniquely \\n      positioned to merge ecological and evolutionary studies with genomic technologies\\n      to exploit unprecedented \"Big Data\" tools in disease research; however, many\\n      researchers lack the training and expertise required to use these computationally\\n      intensive methodologies. To address this disparity, the inaugural \"Genomics of\\n      Disease in Wildlife\" workshop assembled early to mid-career professionals with\\n      expertise across scientific disciplines (e.g., genomics, wildlife biology,\\n      veterinary sciences, and conservation management) for training in the application\\n      of genomic tools to wildlife disease research. A horizon scanning-like exercise, \\n      an activity to identify forthcoming trends and challenges, performed by the\\n      workshop participants identified and discussed 5 themes considered to be the most\\n      pressing to the application of genomics in wildlife disease research: 1)\\n      \"Improving communication,\" 2) \"Methodological and analytical advancements,\" 3)\\n      \"Translation into practice,\" 4) \"Integrating landscape ecology and genomics,\" and\\n      5) \"Emerging new questions.\" Wide-ranging solutions from the horizon scan were\\n      international in scope, itemized both deficiencies and strengths in wildlife\\n      genomic initiatives, promoted the use of genomic technologies to unite wildlife\\n      and human disease research, and advocated best practices for optimal use of\\n      genomic tools in wildlife disease projects. The results offer a glimpse of the\\n      potential revolution in human and wildlife disease research possible through\\n      multi-disciplinary collaborations at local, regional, and global scales.\\nCI  - (c) The American Genetic Association 2019. All rights reserved. For permissions, \\n      please e-mail: journals.permissions@oup.com.\\nFAU - Fitak, Robert R\\nAU  - Fitak RR\\nAD  - Department of Biology, Duke University, Durham, NC, USA.\\nFAU - Antonides, Jennifer D\\nAU  - Antonides JD\\nAD  - Department of Forestry & Natural Resources, Purdue University, West Lafayette,\\n      IN.\\nFAU - Baitchman, Eric J\\nAU  - Baitchman EJ\\nAD  - The Zoo New England Division of Animal Health and Conservation, Boston, MA.\\nFAU - Bonaccorso, Elisa\\nAU  - Bonaccorso E\\nAD  - The Instituto BIOSFERA and Colegio de Ciencias Biologicas y Ambientales,\\n      Universidad San Francisco de Quito, via Interoceanica y Diego de Robles, Quito,\\n      Ecuador.\\nFAU - Braun, Josephine\\nAU  - Braun J\\nAD  - The Institute for Conservation Research, San Diego Zoo Global, Escondido, CA.\\nFAU - Kubiski, Steven\\nAU  - Kubiski S\\nAD  - The Institute for Conservation Research, San Diego Zoo Global, Escondido, CA.\\nFAU - Chiu, Elliott\\nAU  - Chiu E\\nAD  - Department of Microbiology, Immunology and Pathology, Colorado State University, \\n      Fort Collins, CO.\\nFAU - Fagre, Anna C\\nAU  - Fagre AC\\nAD  - Department of Microbiology, Immunology and Pathology, Colorado State University, \\n      Fort Collins, CO.\\nFAU - Gagne, Roderick B\\nAU  - Gagne RB\\nAD  - Department of Microbiology, Immunology and Pathology, Colorado State University, \\n      Fort Collins, CO.\\nFAU - Lee, Justin S\\nAU  - Lee JS\\nAD  - Department of Microbiology, Immunology and Pathology, Colorado State University, \\n      Fort Collins, CO.\\nFAU - Malmberg, Jennifer L\\nAU  - Malmberg JL\\nAD  - Department of Microbiology, Immunology and Pathology, Colorado State University, \\n      Fort Collins, CO.\\nFAU - Stenglein, Mark D\\nAU  - Stenglein MD\\nAD  - Department of Microbiology, Immunology and Pathology, Colorado State University, \\n      Fort Collins, CO.\\nFAU - Dusek, Robert J\\nAU  - Dusek RJ\\nAD  - The U. S. Geological Survey, National Wildlife Health Center, Madison, WI.\\nFAU - Forgacs, David\\nAU  - Forgacs D\\nAD  - The Interdisciplinary Graduate Program of Genetics, Texas A&M University, College\\n      Station, TX.\\nFAU - Fountain-Jones, Nicholas M\\nAU  - Fountain-Jones NM\\nAD  - The Department of Veterinary Population Medicine, University of Minnesota, St.\\n      Paul, MN.\\nFAU - Gilbertson, Marie L J\\nAU  - Gilbertson MLJ\\nAD  - The Department of Veterinary Population Medicine, University of Minnesota, St.\\n      Paul, MN.\\nFAU - Worsley-Tonks, Katherine E L\\nAU  - Worsley-Tonks KEL\\nAD  - The Department of Veterinary Population Medicine, University of Minnesota, St.\\n      Paul, MN.\\nFAU - Funk, W Chris\\nAU  - Funk WC\\nAD  - The Department of Biology, Colorado State University, Fort Collins, CO.\\nFAU - Trumbo, Daryl R\\nAU  - Trumbo DR\\nAD  - The Department of Biology, Colorado State University, Fort Collins, CO.\\nFAU - Ghersi, Bruno M\\nAU  - Ghersi BM\\nAD  - The University of Tennessee, Knoxville, TN.\\nFAU - Grimaldi, Wray\\nAU  - Grimaldi W\\nAD  - The Encinitas, CA.\\nFAU - Heisel, Sara E\\nAU  - Heisel SE\\nAD  - The Odum School of Ecology, University of Georgia, Athens, GA.\\nFAU - Jardine, Claire M\\nAU  - Jardine CM\\nAD  - The Department of Pathobiology, Canadian Wildlife Health Cooperative, University \\n      of Guelph, Guelph, Ontario, Canada.\\nFAU - Kamath, Pauline L\\nAU  - Kamath PL\\nAD  - The School of Food and Agriculture, University of Maine, Orono, ME.\\nFAU - Karmacharya, Dibesh\\nAU  - Karmacharya D\\nAD  - The Center for Molecular Dynamics Nepal, Kathmandu, Nepal.\\nFAU - Kozakiewicz, Christopher P\\nAU  - Kozakiewicz CP\\nAD  - The School of Biological Sciences, University of Tasmania, Hobart, Tasmania,\\n      Australia.\\nFAU - Kraberger, Simona\\nAU  - Kraberger S\\nAD  - The Biodesign Center for Fundamental and Applied Microbiomics, Center for\\n      Evolution and Medicine, School of Life Sciences, Arizona State University, Tempe,\\n      AZ.\\nFAU - Loisel, Dagan A\\nAU  - Loisel DA\\nAD  - The Department of Biology, Saint Michael\\'s College, Colchester, VT.\\nFAU - McDonald, Cait\\nAU  - McDonald C\\nAD  - The Department of Ecology & Evolutionary Biology, Cornell University, Ithaca, NY \\n      (McDonald).\\nFAU - Miller, Steven\\nAU  - Miller S\\nAD  - The Department of Biology, Drexel University, Philadelphia, PA.\\nFAU - O\\'Rourke, Devon\\nAU  - O\\'Rourke D\\nAD  - The University of New Hampshire, Durham NH.\\nFAU - Ott-Conn, Caitlin N\\nAU  - Ott-Conn CN\\nAD  - The Michigan Department of Natural Resources, Wildlife Disease Laboratory,\\n      Lansing, MI.\\nFAU - Paez-Vacas, Monica\\nAU  - Paez-Vacas M\\nAD  - The Centro de Investigacion de la Biodiversidad y Cambio Climatico (BioCamb),\\n      Facultad de Ciencias de Medio Ambiente, Universidad Tecnologica Indoamerica,\\n      Machala y Sabanilla, Quito, Ecuador.\\nFAU - Peel, Alison J\\nAU  - Peel AJ\\nAD  - The Environmental Futures Research Institute, Griffith University, Nathan,\\n      Queensland, Australia.\\nFAU - Turner, Wendy C\\nAU  - Turner WC\\nAD  - The Department of Biological Sciences, University at Albany, State University of \\n      New York, Albany, NY.\\nFAU - VanAcker, Meredith C\\nAU  - VanAcker MC\\nAD  - The Department of Ecology, Evolution, and Environmental Biology, Columbia\\n      University, New York, NY.\\nFAU - VandeWoude, Sue\\nAU  - VandeWoude S\\nAD  - The College of Veterinary Medicine and Biomedical Sciences, Colorado State\\n      University, Fort Collins, CO.\\nFAU - Pecon-Slattery, Jill\\nAU  - Pecon-Slattery J\\nAD  - The Center for Species Survival, Smithsonian Conservation Biology\\n      Institute-National Zoological Park, Front Royal, VA.\\nLA  - eng\\nPT  - Journal Article\\nPT  - Research Support, Non-U.S. Gov\\'t\\nPT  - Research Support, U.S. Gov\\'t, Non-P.H.S.\\nPL  - United States\\nTA  - J Hered\\nJT  - The Journal of heredity\\nJID - 0375373\\nSB  - IM\\nEIN - J Hered. 2019 Aug 16;110(5):639. PMID: 31301132\\nMH  - Animal Diseases/epidemiology/*etiology/transmission\\nMH  - Animals\\nMH  - *Animals, Wild\\nMH  - Biodiversity\\nMH  - Biological Evolution\\nMH  - Computational Biology/methods\\nMH  - Disease Susceptibility\\nMH  - Ecology\\nMH  - Environment\\nMH  - Genome\\nMH  - *Genomics/methods\\nMH  - Host-Pathogen Interactions/genetics\\nMH  - Humans\\nMH  - *Research\\nOTO - NOTNLM\\nOT  - *biodiversity\\nOT  - *bioinformatics\\nOT  - *comparative genomics\\nOT  - *host\\nOT  - *next generation DNA sequencing\\nOT  - *pathogen\\nEDAT- 2019/05/09 06:00\\nMHDA- 2020/06/04 06:00\\nCRDT- 2019/05/09 06:00\\nPHST- 2018/05/02 00:00 [received]\\nPHST- 2019/01/08 00:00 [accepted]\\nPHST- 2019/05/09 06:00 [entrez]\\nPHST- 2019/05/09 06:00 [pubmed]\\nPHST- 2020/06/04 06:00 [medline]\\nAID - 5288600 [pii]\\nAID - 10.1093/jhered/esz001 [doi]\\nPST - ppublish\\nSO  - J Hered. 2019 May 7;110(3):261-274. doi: 10.1093/jhered/esz001.\\n\\nPMID- 30541084\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20200429\\nLR  - 20200429\\nIS  - 1465-7333 (Electronic)\\nIS  - 0022-1503 (Linking)\\nVI  - 110\\nIP  - 2\\nDP  - 2019 Mar 5\\nTI  - Galapagos Islands Endemic Vertebrates: A Population Genetics Perspective.\\nPG  - 137-157\\nLID - 10.1093/jhered/esy066 [doi]\\nAB  - The organisms of the Galapagos Islands played a central role in the development\\n      of the theory of evolution by Charles Darwin. Examination of the population\\n      genetics factors of many of these organisms with modern molecular methods has\\n      expanded our understanding of their evolution. Here, I provide a perspective on\\n      how selection, gene flow, genetic drift, mutation, and inbreeding have\\n      contributed to the evolution of 6 iconic Galapagos species: flightless cormorant,\\n      pink iguana, marine iguana, Galapagos hawk, giant tortoises, and Darwin\\'s\\n      finches. Because of the inherent biological differences among these species that \\n      have colonized the Galapagos, different population genetic factors appear to be\\n      more or less important in these different species. For example, the Galapagos\\n      provided novel environments in which strong selection took place and the Darwin\\'s\\n      finches diversified to produce new species and the cormorant adapted to the\\n      nutrient-rich western shores of the Galapagos by losing its ability to fly and\\n      genomic data have now identified candidate genes. In both the pink iguana, which \\n      exists in one small population, and the Galapagos hawk, which has small\\n      population sizes, genetic drift has been potentially quite important. There\\n      appears to be very limited interisland gene flow in the flightless cormorant and \\n      the Galapagos hawk. On the other hand, both the marine iguana and some of the\\n      Darwin\\'s finches appear to have significant interisland gene flow. Hybridization \\n      between species and subspecies has also introduced new adaptive variation, and in\\n      some cases, hybridization might have resulted in despeciation. Overall, new\\n      population genetics and genomics research has provided additional insight into\\n      the evolution of vertebrate species in the Galapagos.\\nCI  - (c) The American Genetic Association 2018. All rights reserved. For permissions, \\n      please e-mail: journals.permissions@oup.com.\\nFAU - Hedrick, Philip W\\nAU  - Hedrick PW\\nAD  - School of Life Sciences, Arizona State University, Tempe, AZ.\\nLA  - eng\\nPT  - Journal Article\\nPL  - United States\\nTA  - J Hered\\nJT  - The Journal of heredity\\nJID - 0375373\\nSB  - IM\\nMH  - Animals\\nMH  - Biodiversity\\nMH  - *Biological Evolution\\nMH  - Ecuador\\nMH  - Gene Flow\\nMH  - Genetic Drift\\nMH  - Genetic Variation\\nMH  - *Genetics, Population\\nMH  - Geography\\nMH  - Inbreeding\\nMH  - Selection, Genetic\\nMH  - Vertebrates/*genetics\\nOTO - NOTNLM\\nOT  - *gene flow\\nOT  - *genetic drift\\nOT  - *hybridization\\nOT  - *inbreeding\\nOT  - *selection\\nEDAT- 2018/12/13 06:00\\nMHDA- 2020/04/30 06:00\\nCRDT- 2018/12/13 06:00\\nPHST- 2018/12/10 00:00 [accepted]\\nPHST- 2018/12/13 06:00 [pubmed]\\nPHST- 2020/04/30 06:00 [medline]\\nPHST- 2018/12/13 06:00 [entrez]\\nAID - 5239640 [pii]\\nAID - 10.1093/jhered/esy066 [doi]\\nPST - ppublish\\nSO  - J Hered. 2019 Mar 5;110(2):137-157. doi: 10.1093/jhered/esy066.\\n\\nPMID- 30514945\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20190509\\nLR  - 20190509\\nIS  - 1476-4687 (Electronic)\\nIS  - 0028-0836 (Linking)\\nVI  - 564\\nIP  - 7734\\nDP  - 2018 Dec\\nTI  - The secrets of Lonesome George.\\nPG  - 5-6\\nLID - 10.1038/d41586-018-07630-4 [doi]\\nLA  - eng\\nPT  - News\\nPT  - Comment\\nPL  - England\\nTA  - Nature\\nJT  - Nature\\nJID - 0410462\\nSB  - IM\\nCON - Nature. 2015 Jan 15;517(7534):271. PMID: 25592525\\nMH  - Animals\\nMH  - Ecuador\\nMH  - Genomics\\nMH  - *Turtles\\nOTO - NOTNLM\\nOT  - *Ageing\\nOT  - *Evolution\\nOT  - *Genomics\\nOT  - *Immunology\\nEDAT- 2018/12/06 06:00\\nMHDA- 2019/05/10 06:00\\nCRDT- 2018/12/06 06:00\\nPHST- 2018/12/06 06:00 [entrez]\\nPHST- 2018/12/06 06:00 [pubmed]\\nPHST- 2019/05/10 06:00 [medline]\\nAID - 10.1038/d41586-018-07630-4 [doi]\\nAID - 10.1038/d41586-018-07630-4 [pii]\\nPST - ppublish\\nSO  - Nature. 2018 Dec;564(7734):5-6. doi: 10.1038/d41586-018-07630-4.\\n\\nPMID- 30459831\\nOWN - NLM\\nSTAT- PubMed-not-MEDLINE\\nLR  - 20201001\\nIS  - 1752-4571 (Print)\\nIS  - 1752-4571 (Linking)\\nVI  - 11\\nIP  - 10\\nDP  - 2018 Dec\\nTI  - Population genomics through time provides insights into the consequences of\\n      decline and rapid demographic recovery through head-starting in a Galapagos giant\\n      tortoise.\\nPG  - 1811-1821\\nLID - 10.1111/eva.12682 [doi]\\nAB  - Population genetic theory related to the consequences of rapid population decline\\n      is well-developed, but there are very few empirical studies where sampling was\\n      conducted before and after a known bottleneck event. Such knowledge is of\\n      particular importance for species restoration, given links between genetic\\n      diversity and the probability of long-term persistence. To directly evaluate the \\n      relationship between current genetic diversity and past demographic events, we\\n      collected genome-wide single nucleotide polymorphism data from prebottleneck\\n      historical (c.1906) and postbottleneck contemporary (c.2014) samples of Pinzon\\n      giant tortoises (Chelonoidis duncanensis; n = 25 and 149 individuals,\\n      respectively) endemic to a single island in the Galapagos. Pinzon giant tortoises\\n      had a historically large population size that was reduced to just 150-200\\n      individuals in the mid 20th century. Since then, Pinzon\\'s tortoise population has\\n      recovered through an ex situ head-start programme in which eggs or pre-emergent\\n      individuals were collected from natural nests on the island, reared ex situ in\\n      captivity until they were 4-5 years old and subsequently repatriated. We found\\n      that the extent and distribution of genetic variation in the historical and\\n      contemporary samples were very similar, with the latter group not exhibiting the \\n      characteristic genetic patterns of recent population decline. No population\\n      structure was detected either spatially or temporally. We estimated an effective \\n      population size (N e) of 58 (95% CI = 50-69) for the postbottleneck population;\\n      no prebottleneck N e point estimate was attainable (95% CI = 39-infinity) likely \\n      due to the sample size being lower than the true N e. Overall, the historical\\n      sample provided a valuable benchmark for evaluating the head-start captive\\n      breeding programme, revealing high retention of genetic variation and no skew in \\n      representation despite the documented bottleneck event. Moreover, this work\\n      demonstrates the effectiveness of head-starting in rescuing the Pinzon giant\\n      tortoise from almost certain extinction.\\nFAU - Jensen, Evelyn L\\nAU  - Jensen EL\\nAUID- ORCID: 0000-0002-1819-3117\\nAD  - Department of Biology University of British Columbia Okanagan Kelowna British\\n      Columbia Canada.\\nAD  - Present address: Department of Biology Queen\\'s University Kingston Ontario\\n      Canada.\\nFAU - Edwards, Danielle L\\nAU  - Edwards DL\\nAD  - Life and Environmental Sciences University of California Merced California.\\nFAU - Garrick, Ryan C\\nAU  - Garrick RC\\nAD  - Department of Biology University of Mississippi Oxford Mississippi.\\nFAU - Miller, Joshua M\\nAU  - Miller JM\\nAUID- ORCID: 0000-0002-4019-7675\\nAD  - Department of Ecology and Evolutionary Biology Yale University New Haven\\n      Connecticut.\\nFAU - Gibbs, James P\\nAU  - Gibbs JP\\nAD  - College of Environmental Science and Forestry State University of New York\\n      Syracuse New York.\\nFAU - Cayot, Linda J\\nAU  - Cayot LJ\\nAD  - Galapagos Conservancy Fairfax Virginia.\\nFAU - Tapia, Washington\\nAU  - Tapia W\\nAD  - Department of Applied Research Galapagos National Park Service Puerto Ayora\\n      Ecuador.\\nAD  - Galapagos Conservancy Santa Cruz Ecuador.\\nFAU - Caccone, Adalgisa\\nAU  - Caccone A\\nAD  - Department of Ecology and Evolutionary Biology Yale University New Haven\\n      Connecticut.\\nFAU - Russello, Michael A\\nAU  - Russello MA\\nAUID- ORCID: 0000-0002-9297-2485\\nAD  - Department of Biology University of British Columbia Okanagan Kelowna British\\n      Columbia Canada.\\nLA  - eng\\nSI  - Dryad/10.5061/dryad.7tp3sg0\\nPT  - Journal Article\\nDEP - 20180813\\nPL  - England\\nTA  - Evol Appl\\nJT  - Evolutionary applications\\nJID - 101461828\\nPMC - PMC6231475\\nOTO - NOTNLM\\nOT  - Chelonoidis\\nOT  - RAD-seq\\nOT  - bottleneck\\nOT  - effective population size\\nOT  - historical DNA\\nOT  - hybrid capture\\nOT  - museum specimen\\nOT  - population genetics\\nEDAT- 2018/11/22 06:00\\nMHDA- 2018/11/22 06:01\\nCRDT- 2018/11/22 06:00\\nPHST- 2018/04/27 00:00 [received]\\nPHST- 2018/07/09 00:00 [revised]\\nPHST- 2018/07/16 00:00 [accepted]\\nPHST- 2018/11/22 06:00 [entrez]\\nPHST- 2018/11/22 06:00 [pubmed]\\nPHST- 2018/11/22 06:01 [medline]\\nAID - 10.1111/eva.12682 [doi]\\nAID - EVA12682 [pii]\\nPST - epublish\\nSO  - Evol Appl. 2018 Aug 13;11(10):1811-1821. doi: 10.1111/eva.12682. eCollection 2018\\n      Dec.\\n\\nPMID- 30371872\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20190408\\nLR  - 20190408\\nIS  - 1076-836X (Electronic)\\nIS  - 1063-5157 (Linking)\\nVI  - 68\\nIP  - 2\\nDP  - 2019 Mar 1\\nTI  - Species Limits and Phylogenomic Relationships of Darwin\\'s Finches Remain\\n      Unresolved: Potential Consequences of a Volatile Ecological Setting.\\nPG  - 347-357\\nLID - 10.1093/sysbio/syy073 [doi]\\nAB  - Island biotas have become paradigms for illustrating many evolutionary processes.\\n      The fauna of the Galapagos Islands includes several taxa that have been focal\\n      points for evolutionary studies. Perhaps their most famous inhabitants, Darwin\\'s \\n      finches, represent a go-to icon when thinking about how species originate and\\n      adapt to the environment. However, unlike other adaptive radiations, past\\n      morphological and molecular studies of Darwin\\'s finches have yielded inconsistent\\n      hypotheses of species limits and phylogenetic relationships. Expecting that\\n      idiosyncrasies of prior data and analytic methods explained different proposed\\n      classifications, we were surprised to observe that three new phylogenetic\\n      hypotheses derived mostly from the same genomics data were topologically\\n      inconsistent. We found that the differences between some of these genomics trees \\n      were as great as one would expect between two random trees with the same number\\n      of taxa. Thus, the phylogeny of Darwin\\'s finches remains unresolved, as it has\\n      for more than a century. A component of phylogenetic uncertainty comes from\\n      unclear species limits, under any species concept, in the ground finches\\n      (Geospiza) and tree finches (Camarhynchus). We suggest that past authors should\\n      have tested the species limits of Lack, rather than uncritically accepting them. \\n      In fact, the impressive amount of genomics data do not provide unambiguous\\n      hypotheses of the number of species of Geospiza or Camarhynchus, although they\\n      imply greater species diversity than Lack\\'s taxonomy. We suggest that\\n      insufficient sampling of species populations across islands (35.6% for\\n      morphometrics and 20.4% for genomics) prevents accurate diagnoses of species\\n      limits. However, it is unknown whether samples from a greater number of islands\\n      might result in bridging differences between species, or reveal many new ones. We\\n      conclude that attempts to interpret patterns of variation among the finches under\\n      standard evolutionary paradigms have obscured some major messages, most\\n      specifically the ongoing reciprocal interactions between geographic isolation and\\n      lineage divergence, and dispersal and gene flow caused by the volatile ecological\\n      conditions in the islands. Although the finches provide textbook examples of\\n      natural selection, better understanding of species limits and a robust\\n      phylogenetic hypothesis are required to corroborate past hypotheses of speciation\\n      and adaptive radiation in the finches of the Galapagos.\\nFAU - Zink, Robert M\\nAU  - Zink RM\\nAD  - School of Natural Resources.\\nAD  - School of Biological Sciences.\\nAD  - Nebraska State Museum, University of Nebraska-Lincoln, Lincoln, NE 68583, USA.\\nFAU - Vazquez-Miranda, Hernan\\nAU  - Vazquez-Miranda H\\nAD  - School of Natural Resources.\\nAD  - Nebraska State Museum, University of Nebraska-Lincoln, Lincoln, NE 68583, USA.\\nAD  - Coleccion Nacional de Aves (CNAV), Departamento de Zoologia, Instituto de\\n      Biologia, Universidad Nacional Autonoma de Mexico, Ciudad de Mexico, C.P. 04500, \\n      Mexico.\\nLA  - eng\\nSI  - Dryad/10.5061/dryad.dc82gp3\\nPT  - Journal Article\\nPL  - England\\nTA  - Syst Biol\\nJT  - Systematic biology\\nJID - 9302532\\nSB  - IM\\nMH  - Animals\\nMH  - *Ecosystem\\nMH  - Ecuador\\nMH  - Finches/*classification/genetics\\nMH  - Genetic Variation\\nMH  - Genome/genetics\\nMH  - *Phylogeny\\nMH  - Species Specificity\\nEDAT- 2018/10/30 06:00\\nMHDA- 2019/04/09 06:00\\nCRDT- 2018/10/30 06:00\\nPHST- 2018/05/26 00:00 [received]\\nPHST- 2018/10/25 00:00 [accepted]\\nPHST- 2018/10/30 06:00 [pubmed]\\nPHST- 2019/04/09 06:00 [medline]\\nPHST- 2018/10/30 06:00 [entrez]\\nAID - 5146315 [pii]\\nAID - 10.1093/sysbio/syy073 [doi]\\nPST - ppublish\\nSO  - Syst Biol. 2019 Mar 1;68(2):347-357. doi: 10.1093/sysbio/syy073.\\n\\nPMID- 30228363\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20191114\\nLR  - 20211204\\nIS  - 2045-2322 (Electronic)\\nIS  - 2045-2322 (Linking)\\nVI  - 8\\nIP  - 1\\nDP  - 2018 Sep 18\\nTI  - Analysis of Racial/Ethnic Representation in Select Basic and Applied Cancer\\n      Research Studies.\\nPG  - 13978\\nLID - 10.1038/s41598-018-32264-x [doi]\\nAB  - Over the past decades, consistent studies have shown that race/ethnicity have a\\n      great impact on cancer incidence, survival, drug response, molecular pathways and\\n      epigenetics. Despite the influence of race/ethnicity in cancer outcomes and its\\n      impact in health care quality, a comprehensive understanding of racial/ethnic\\n      inclusion in oncological research has never been addressed. We therefore explored\\n      the racial/ethnic composition of samples/individuals included in fundamental\\n      (patient-derived oncological models, biobanks and genomics) and applied cancer\\n      research studies (clinical trials). Regarding patient-derived oncological models \\n      (n = 794), 48.3% have no records on their donor\\'s race/ethnicity, the rest were\\n      isolated from White (37.5%), Asian (10%), African American (3.8%) and Hispanic\\n      (0.4%) donors. Biobanks (n = 8,293) hold specimens from unknown (24.56%), White\\n      (59.03%), African American (11.05%), Asian (4.12%) and other individuals (1.24%).\\n      Genomic projects (n = 6,765,447) include samples from unknown (0.6%), White\\n      (91.1%), Asian (5.6%), African American (1.7%), Hispanic (0.5%) and other\\n      populations (0.5%). Concerning clinical trials (n = 89,212), no racial/ethnic\\n      registries were found in 66.95% of participants, and records were mainly obtained\\n      from Whites (25.94%), Asians (4.97%), African Americans (1.08%), Hispanics\\n      (0.16%) and other minorities (0.9%). Thus, two tendencies were observed across\\n      oncological studies: lack of racial/ethnic information and overrepresentation of \\n      Caucasian/White samples/individuals. These results clearly indicate a need to\\n      diversify oncological studies to other populations along with novel strategies to\\n      enhanced race/ethnicity data recording and reporting.\\nFAU - Guerrero, Santiago\\nAU  - Guerrero S\\nAUID- ORCID: 0000-0003-3473-7214\\nAD  - Centro de Investigacion Genetica y Genomica, Facultad de Ciencias de la Salud\\n      Eugenio Espejo, Universidad UTE, Av. Mariscal Sucre and Mariana de Jesus, Block\\n      I, 2nd floor, 170129, Quito, Ecuador. santiago.guerrero@ute.edu.ec.\\nFAU - Lopez-Cortes, Andres\\nAU  - Lopez-Cortes A\\nAUID- ORCID: 0000-0003-1503-1929\\nAD  - Centro de Investigacion Genetica y Genomica, Facultad de Ciencias de la Salud\\n      Eugenio Espejo, Universidad UTE, Av. Mariscal Sucre and Mariana de Jesus, Block\\n      I, 2nd floor, 170129, Quito, Ecuador.\\nFAU - Indacochea, Alberto\\nAU  - Indacochea A\\nAD  - Gene Regulation, Stem Cells and Cancer Programme, Centre for Genomic Regulation\\n      (CRG), The Barcelona Institute of Science and Technology, Barcelona, Spain.\\nAD  - Oncology and Molecular Pathology Research Group-VHIR- Vall d\\' Hebron Institut de \\n      Recerca-Vall d\\' Hebron Hospital, P/de la Vall d\\'Hebron, Barcelona, Spain.\\nFAU - Garcia-Cardenas, Jennyfer M\\nAU  - Garcia-Cardenas JM\\nAD  - Centro de Investigacion Genetica y Genomica, Facultad de Ciencias de la Salud\\n      Eugenio Espejo, Universidad UTE, Av. Mariscal Sucre and Mariana de Jesus, Block\\n      I, 2nd floor, 170129, Quito, Ecuador.\\nFAU - Zambrano, Ana Karina\\nAU  - Zambrano AK\\nAD  - Centro de Investigacion Genetica y Genomica, Facultad de Ciencias de la Salud\\n      Eugenio Espejo, Universidad UTE, Av. Mariscal Sucre and Mariana de Jesus, Block\\n      I, 2nd floor, 170129, Quito, Ecuador.\\nFAU - Cabrera-Andrade, Alejandro\\nAU  - Cabrera-Andrade A\\nAD  - Centro de Investigacion Genetica y Genomica, Facultad de Ciencias de la Salud\\n      Eugenio Espejo, Universidad UTE, Av. Mariscal Sucre and Mariana de Jesus, Block\\n      I, 2nd floor, 170129, Quito, Ecuador.\\nAD  - Carrera de Enfermeria, Facultad de Ciencias de la Salud, Universidad de las\\n      Americas, Avenue de los Granados, Quito, 170125, Ecuador.\\nAD  - Grupo de Bio-Quimioinformatica, Universidad de las Americas, Avenue de los\\n      Granados, Quito, 170125, Ecuador.\\nFAU - Guevara-Ramirez, Patricia\\nAU  - Guevara-Ramirez P\\nAD  - Centro de Investigacion Genetica y Genomica, Facultad de Ciencias de la Salud\\n      Eugenio Espejo, Universidad UTE, Av. Mariscal Sucre and Mariana de Jesus, Block\\n      I, 2nd floor, 170129, Quito, Ecuador.\\nFAU - Gonzalez, Diana Abigail\\nAU  - Gonzalez DA\\nAD  - Centro de Investigacion Genetica y Genomica, Facultad de Ciencias de la Salud\\n      Eugenio Espejo, Universidad UTE, Av. Mariscal Sucre and Mariana de Jesus, Block\\n      I, 2nd floor, 170129, Quito, Ecuador.\\nFAU - Leone, Paola E\\nAU  - Leone PE\\nAD  - Centro de Investigacion Genetica y Genomica, Facultad de Ciencias de la Salud\\n      Eugenio Espejo, Universidad UTE, Av. Mariscal Sucre and Mariana de Jesus, Block\\n      I, 2nd floor, 170129, Quito, Ecuador.\\nFAU - Paz-Y-Mino, Cesar\\nAU  - Paz-Y-Mino C\\nAD  - Centro de Investigacion Genetica y Genomica, Facultad de Ciencias de la Salud\\n      Eugenio Espejo, Universidad UTE, Av. Mariscal Sucre and Mariana de Jesus, Block\\n      I, 2nd floor, 170129, Quito, Ecuador. cesar.pazymino@ute.edu.ec.\\nLA  - eng\\nPT  - Journal Article\\nDEP - 20180918\\nPL  - England\\nTA  - Sci Rep\\nJT  - Scientific reports\\nJID - 101563288\\nSB  - IM\\nMH  - Biological Specimen Banks/*statistics & numerical data\\nMH  - *Biomedical Research\\nMH  - Clinical Trials as Topic/*statistics & numerical data\\nMH  - Ethnicity/*statistics & numerical data\\nMH  - Female\\nMH  - Genomics/*methods\\nMH  - Humans\\nMH  - Male\\nMH  - Neoplasms/diagnosis/*epidemiology/therapy\\nMH  - Patient Participation\\nMH  - Racial Groups/*statistics & numerical data\\nMH  - Registries/*statistics & numerical data\\nMH  - United States/epidemiology\\nPMC - PMC6143551\\nEDAT- 2018/09/20 06:00\\nMHDA- 2019/11/15 06:00\\nCRDT- 2018/09/20 06:00\\nPHST- 2018/02/23 00:00 [received]\\nPHST- 2018/07/26 00:00 [accepted]\\nPHST- 2018/09/20 06:00 [entrez]\\nPHST- 2018/09/20 06:00 [pubmed]\\nPHST- 2019/11/15 06:00 [medline]\\nAID - 10.1038/s41598-018-32264-x [doi]\\nAID - 10.1038/s41598-018-32264-x [pii]\\nPST - epublish\\nSO  - Sci Rep. 2018 Sep 18;8(1):13978. doi: 10.1038/s41598-018-32264-x.\\n\\nPMID- 29714581\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20190416\\nLR  - 20200103\\nIS  - 2168-4804 (Electronic)\\nIS  - 2168-4790 (Linking)\\nVI  - 52\\nIP  - 6\\nDP  - 2018 Nov\\nTI  - CRISPR-Cas9: A Precise Approach to Genome Engineering.\\nPG  - 701-707\\nLID - 10.1177/2168479018762798 [doi]\\nAB  - In the last few decades, genomic manipulation has made significant progress as a \\n      result of the development of recombinant DNA technologies; however, more often\\n      than not, these techniques have been costly and labor intensive. In contrast,\\n      recently developed next-generation sequencing (NGS) technologies have provided a \\n      cheaper, faster, and easier process to study genomics. In particular, an NGS\\n      technique emerged from bacterial CRISPR-associated protein-9 nuclease (Cas9) as a\\n      revolutionary method to modify, regulate, or mark specific genomic sequences on\\n      virtually any organism. A later adaptation of this bacterial defense mechanism\\n      that successfully and permanently edits dysfunctional genes and corrects missing \\n      proteins has resulted in a new era for disease genetic engineering. Clinical\\n      trials using this technique are already being performed, and the applicability of\\n      CRISPR-Cas9 techniques is actively being investigated using in vivo studies.\\n      However, the concept of genome correction poses great concerns from a regulatory \\n      perspective, especially in terms of security, so principles for the regulation of\\n      these methodologies are being established. We delved into CRISPR-Cas9 from its\\n      natural and ortholog origins to its engineered variants and behaviors to present \\n      its notable and diverse applications in the fields of biotechnology and human\\n      therapeutics.\\nFAU - Simon, Jorge E\\nAU  - Simon JE\\nAD  - 1 School of Biological Sciences and Engineering, Yachay Tech University, Urcuqui,\\n      Ecuador.\\nFAU - Rodriguez, Angel S\\nAU  - Rodriguez AS\\nAD  - 1 School of Biological Sciences and Engineering, Yachay Tech University, Urcuqui,\\n      Ecuador.\\nFAU - Santiago Vispo, Nelson\\nAU  - Santiago Vispo N\\nAUID- ORCID: 0000-0002-4081-3984\\nAD  - 1 School of Biological Sciences and Engineering, Yachay Tech University, Urcuqui,\\n      Ecuador.\\nLA  - eng\\nPT  - Journal Article\\nPT  - Review\\nDEP - 20180402\\nPL  - Switzerland\\nTA  - Ther Innov Regul Sci\\nJT  - Therapeutic innovation & regulatory science\\nJID - 101597411\\nRN  - 0 (Bacterial Proteins)\\nRN  - EC 3.1.- (CRISPR-Associated Protein 9)\\nSB  - IM\\nMH  - Bacterial Proteins/metabolism\\nMH  - CRISPR-Associated Protein 9/*metabolism\\nMH  - CRISPR-Cas Systems\\nMH  - Clinical Trials as Topic/legislation & jurisprudence\\nMH  - Genetic Engineering/*legislation & jurisprudence/methods\\nMH  - Genome, Human\\nMH  - High-Throughput Nucleotide Sequencing\\nMH  - Humans\\nMH  - Mutation\\nMH  - Sequence Analysis, DNA\\nOTO - NOTNLM\\nOT  - *CRISPR-Cas9\\nOT  - *NGS\\nOT  - *bacterial adaptive defense system\\nOT  - *clinical trials\\nOT  - *genetic engineering\\nEDAT- 2018/05/02 06:00\\nMHDA- 2019/04/17 06:00\\nCRDT- 2018/05/02 06:00\\nPHST- 2018/05/02 06:00 [pubmed]\\nPHST- 2019/04/17 06:00 [medline]\\nPHST- 2018/05/02 06:00 [entrez]\\nAID - 10.1177/2168479018762798 [doi]\\nPST - ppublish\\nSO  - Ther Innov Regul Sci. 2018 Nov;52(6):701-707. doi: 10.1177/2168479018762798. Epub\\n      2018 Apr 2.\\n\\nPMID- 29188006\\nOWN - NLM\\nSTAT- PubMed-not-MEDLINE\\nLR  - 20210109\\nIS  - 2045-7758 (Print)\\nIS  - 2045-7758 (Linking)\\nVI  - 7\\nIP  - 22\\nDP  - 2017 Nov\\nTI  - Radiation of the polymorphic Little Devil poison frog (Oophaga sylvatica) in\\n      Ecuador.\\nPG  - 9750-9762\\nLID - 10.1002/ece3.3503 [doi]\\nAB  - Some South American poison frogs (Dendrobatidae) are chemically defended and use \\n      bright aposematic colors to warn potential predators of their unpalatability.\\n      Aposematic signals are often frequency-dependent where individuals deviating from\\n      a local model are at a higher risk of predation. However, extreme diversity in\\n      the aposematic signal has been documented in poison frogs, especially in Oophaga.\\n      Here, we explore the phylogeographic pattern among color-divergent populations of\\n      the Little Devil poison frog Oophaga sylvatica by analyzing population structure \\n      and genetic differentiation to evaluate which processes could account for color\\n      diversity within and among populations. With a combination of PCR amplicons\\n      (three mitochondrial and three nuclear markers) and genome-wide markers from a\\n      double-digested RAD (ddRAD) approach, we characterized the phylogenetic and\\n      genetic structure of 199 individuals from 13 populations (12 monomorphic and 1\\n      polymorphic) across the O. sylvatica distribution. Individuals segregated into\\n      two main lineages by their northern or southern latitudinal distribution. A high \\n      level of genetic and phenotypic polymorphism within the northern lineage suggests\\n      ongoing gene flow. In contrast, low levels of genetic differentiation were\\n      detected among the southern lineage populations and support recent range\\n      expansions from populations in the northern lineage. We propose that a\\n      combination of climatic gradients and structured landscapes might be promoting\\n      gene flow and phylogenetic diversification. Alternatively, we cannot rule out\\n      that the observed phenotypic and genomic variations are the result of genetic\\n      drift on near or neutral alleles in a small number of genes.\\nFAU - Roland, Alexandre B\\nAU  - Roland AB\\nAUID- ORCID: 0000-0002-9463-9838\\nAD  - FAS Center for Systems Biology Harvard University Cambridge MA USA.\\nFAU - Santos, Juan C\\nAU  - Santos JC\\nAUID- ORCID: 0000-0002-7777-8094\\nAD  - Department of Biological Sciences St. John\\'s University Queens NY USA.\\nFAU - Carriker, Bella C\\nAU  - Carriker BC\\nAD  - Lakeside High School Seattle WA USA.\\nFAU - Caty, Stephanie N\\nAU  - Caty SN\\nAD  - FAS Center for Systems Biology Harvard University Cambridge MA USA.\\nFAU - Tapia, Elicio E\\nAU  - Tapia EE\\nAD  - Centro Jambatu de Investigacion y Conservacion de Anfibios Fundacion Otonga Quito\\n      Ecuador.\\nFAU - Coloma, Luis A\\nAU  - Coloma LA\\nAD  - Centro Jambatu de Investigacion y Conservacion de Anfibios Fundacion Otonga Quito\\n      Ecuador.\\nFAU - O\\'Connell, Lauren A\\nAU  - O\\'Connell LA\\nAUID- ORCID: 0000-0002-2706-4077\\nAD  - FAS Center for Systems Biology Harvard University Cambridge MA USA.\\nLA  - eng\\nPT  - Journal Article\\nDEP - 20171018\\nPL  - England\\nTA  - Ecol Evol\\nJT  - Ecology and evolution\\nJID - 101566408\\nPMC - PMC5696431\\nOTO - NOTNLM\\nOT  - *Dendrobatidae\\nOT  - *Ecuador\\nOT  - *Oophaga sylvatica\\nOT  - *amphibian\\nOT  - *aposematism\\nOT  - *ddRAD\\nOT  - *gene flow\\nOT  - *phenotypic variation\\nOT  - *population genomics\\nEDAT- 2017/12/01 06:00\\nMHDA- 2017/12/01 06:01\\nCRDT- 2017/12/01 06:00\\nPHST- 2017/05/17 00:00 [received]\\nPHST- 2017/09/03 00:00 [revised]\\nPHST- 2017/09/13 00:00 [accepted]\\nPHST- 2017/12/01 06:00 [entrez]\\nPHST- 2017/12/01 06:00 [pubmed]\\nPHST- 2017/12/01 06:01 [medline]\\nAID - 10.1002/ece3.3503 [doi]\\nAID - ECE33503 [pii]\\nPST - epublish\\nSO  - Ecol Evol. 2017 Oct 18;7(22):9750-9762. doi: 10.1002/ece3.3503. eCollection 2017 \\n      Nov.\\n\\nPMID- 29140767\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20180717\\nLR  - 20211204\\nIS  - 1557-8100 (Electronic)\\nIS  - 1536-2310 (Linking)\\nVI  - 21\\nIP  - 11\\nDP  - 2017 Nov\\nTI  - Genomic Medicine Without Borders: Which Strategies Should Developing Countries\\n      Employ to Invest in Precision Medicine? A New \"Fast-Second Winner\" Strategy.\\nPG  - 647-657\\nLID - 10.1089/omi.2017.0141 [doi]\\nAB  - Genomic medicine has greatly matured in terms of its technical capabilities, but \\n      the diffusion of genomic innovations worldwide faces significant barriers beyond \\n      mere access to technology. New global development strategies are sorely needed\\n      for biotechnologies such as genomics and their applications toward precision\\n      medicine without borders. Moreover, diffusion of genomic medicine globally cannot\\n      adhere to a \"one-size-fits-all-countries\" development strategy, in the same way\\n      that drug treatments should be customized. This begs a timely, difficult but\\n      crucial question: How should developing countries, and the resource-limited\\n      regions of developed countries, invest in genomic medicine? Although a full-scale\\n      investment in infrastructure from discovery to the translational implementation\\n      of genomic science is ideal, this may not always be feasible in all countries at \\n      all times. A simple \"transplantation of genomics\" from developed to developing\\n      countries is unlikely to be feasible. Nor should developing countries be seen as \\n      simple recipients and beneficiaries of genomic medicine developed elsewhere\\n      because important advances in genomic medicine have materialized in developing\\n      countries as well. There are several noteworthy examples of genomic medicine\\n      success stories involving resource-limited settings that are contextualized and\\n      described in this global genomic medicine innovation analysis. In addition, we\\n      outline here a new long-term development strategy for global genomic medicine in \\n      a way that recognizes the individual country\\'s pressing public health priorities \\n      and disease burdens. We term this approach the \"Fast-Second Winner\" model of\\n      innovation that supports innovation commencing not only \"upstream\" of discovery\\n      science but also \"mid-stream,\" building on emerging highly promising biomarker\\n      and diagnostic candidates from the global science discovery pipeline, based on\\n      the unique needs of each country. A mid-stream entry into innovation can enhance \\n      collective learning from other innovators\\' mistakes upstream in discovery science\\n      and boost the probability of success for translation and implementation when\\n      resources are limited. This a la carte model of global innovation and development\\n      strategy offers multiple entry points into the global genomics innovation\\n      ecosystem for developing countries, whether or not extensive and expensive\\n      discovery infrastructures are already in place. Ultimately, broadening our\\n      thinking beyond the linear model of innovation will help us to enable the vision \\n      and practice of genomics without borders in both developed and resource-limited\\n      settings.\\nFAU - Mitropoulos, Konstantinos\\nAU  - Mitropoulos K\\nAD  - 1 School of Medicine, University of Athens , Athens, Greece .\\nFAU - Cooper, David N\\nAU  - Cooper DN\\nAD  - 2 Institute of Medical Genetics, School of Medicine, Cardiff University ,\\n      Cardiff, United Kingdom .\\nFAU - Mitropoulou, Christina\\nAU  - Mitropoulou C\\nAD  - 3 The Golden Helix Foundation , London, United Kingdom .\\nFAU - Agathos, Spiros\\nAU  - Agathos S\\nAD  - 4 Yachay Tech University , San Miguel de Urcuqui, Ecuador .\\nFAU - Reichardt, Jurgen K V\\nAU  - Reichardt JKV\\nAD  - 4 Yachay Tech University , San Miguel de Urcuqui, Ecuador .\\nFAU - Al-Maskari, Fatima\\nAU  - Al-Maskari F\\nAD  - 5 Department of Pathology, College of Medicine and Health Sciences, United Arab\\n      Emirates University , Al-Ain, United Arab Emirates .\\nAD  - 6 Zayed Bin Sultan Center for Health Sciences, United Arab Emirates University , \\n      Al-Ain, United Arab Emirates .\\nFAU - Chantratita, Wasun\\nAU  - Chantratita W\\nAD  - 7 Department of Pathology, Medical Genomic Center, Ramathibodi Hospital, Faculty \\n      of Medicine, Mahidol University , Bangkok, Thailand .\\nFAU - Wonkam, Ambroise\\nAU  - Wonkam A\\nAD  - 8 Division of Human Genetics, Department of Medicine and Institute of Infectious \\n      Disease and Molecular Medicine, University of Cape Town , Cape Town, South Africa\\n      .\\nFAU - Dandara, Collet\\nAU  - Dandara C\\nAD  - 8 Division of Human Genetics, Department of Medicine and Institute of Infectious \\n      Disease and Molecular Medicine, University of Cape Town , Cape Town, South Africa\\n      .\\nFAU - Katsila, Theodora\\nAU  - Katsila T\\nAD  - 9 Department of Pharmacy, School of Health Sciences, University of Patras ,\\n      Patras, Greece .\\nFAU - Lopez-Correa, Catalina\\nAU  - Lopez-Correa C\\nAD  - 10 Genome British Columbia , Vancouver, Canada .\\nFAU - Ali, Bassam R\\nAU  - Ali BR\\nAD  - 5 Department of Pathology, College of Medicine and Health Sciences, United Arab\\n      Emirates University , Al-Ain, United Arab Emirates .\\nAD  - 6 Zayed Bin Sultan Center for Health Sciences, United Arab Emirates University , \\n      Al-Ain, United Arab Emirates .\\nFAU - Patrinos, George P\\nAU  - Patrinos GP\\nAD  - 5 Department of Pathology, College of Medicine and Health Sciences, United Arab\\n      Emirates University , Al-Ain, United Arab Emirates .\\nAD  - 6 Zayed Bin Sultan Center for Health Sciences, United Arab Emirates University , \\n      Al-Ain, United Arab Emirates .\\nAD  - 9 Department of Pharmacy, School of Health Sciences, University of Patras ,\\n      Patras, Greece .\\nLA  - eng\\nGR  - U24 HL135600/HL/NHLBI NIH HHS/United States\\nPT  - Journal Article\\nPL  - United States\\nTA  - OMICS\\nJT  - Omics : a journal of integrative biology\\nJID - 101131135\\nSB  - IM\\nMH  - Biotechnology/economics/*trends\\nMH  - Cost of Illness\\nMH  - Developing Countries/*economics\\nMH  - Ecological Parameter Monitoring\\nMH  - Ethnicity\\nMH  - Genomics/economics/*trends\\nMH  - Health Personnel\\nMH  - Humans\\nMH  - Linear Models\\nMH  - Pharmacogenetics\\nMH  - Precision Medicine/economics/*trends\\nMH  - Public Health\\nPMC - PMC6436026\\nOTO - NOTNLM\\nOT  - *Fast-Second Winner\\nOT  - *Strategy for global biotechnology development\\nOT  - *developing countries\\nOT  - *precision medicine\\nOT  - *public health genomic\\nEDAT- 2017/11/16 06:00\\nMHDA- 2018/07/18 06:00\\nCRDT- 2017/11/16 06:00\\nPHST- 2017/11/16 06:00 [entrez]\\nPHST- 2017/11/16 06:00 [pubmed]\\nPHST- 2018/07/18 06:00 [medline]\\nAID - 10.1089/omi.2017.0141 [doi]\\nPST - ppublish\\nSO  - OMICS. 2017 Nov;21(11):647-657. doi: 10.1089/omi.2017.0141.\\n\\nPMID- 29048566\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20180514\\nLR  - 20181113\\nIS  - 1460-2431 (Electronic)\\nIS  - 0022-0957 (Linking)\\nVI  - 68\\nIP  - 17\\nDP  - 2017 Oct 13\\nTI  - Deciphering the Theobroma cacao self-incompatibility system: from genomics to\\n      diagnostic markers for self-compatibility.\\nPG  - 4775-4790\\nLID - 10.1093/jxb/erx293 [doi]\\nAB  - Cocoa self-compatibility is an important yield factor and has been described as\\n      being controlled by a late gameto-sporophytic system expressed only at the level \\n      of the embryo sac. It results in gametic non-fusion and involves several loci. In\\n      this work, we identified two loci, located on chromosomes 1 and 4 (CH1 and CH4), \\n      involved in cocoa self-incompatibility by two different processes. Both loci are \\n      responsible for gametic selection, but only one (the CH4 locus) is involved in\\n      the main fruit drop. The CH1 locus acts prior to the gamete fusion step and\\n      independently of the CH4 locus. Using fine-mapping and genome-wide association\\n      studies, we focused analyses on restricted regions and identified candidate\\n      genes. Some of them showed a differential expression between incompatible and\\n      compatible reactions. Immunolocalization experiments provided evidence of CH1\\n      candidate genes expressed in ovule and style tissues. Highly polymorphic simple\\n      sequence repeat (SSR) diagnostic markers were designed in the CH4 region that had\\n      been identified by fine-mapping. They are characterized by a strong linkage\\n      disequilibrium with incompatibility alleles, thus allowing the development of\\n      efficient diagnostic markers predicting self-compatibility and fruit setting\\n      according to the presence of specific alleles or genotypes. SSR alleles specific \\n      to self-compatible Amelonado and Criollo varieties were also identified, thus\\n      allowing screening for self-compatible plants in cocoa populations.\\nCI  - (c) The Author 2017. Published by Oxford University Press on behalf of the\\n      Society for Experimental Biology.\\nFAU - Lanaud, Claire\\nAU  - Lanaud C\\nAD  - Centre de Cooperation Internationale en Recherche Agronomique pour le\\n      Developpement (CIRAD), UMR AGAP, Avenue Agropolis TA 108/03, 34398 Montpellier\\n      Cedex 5, France.\\nFAU - Fouet, Olivier\\nAU  - Fouet O\\nAD  - Centre de Cooperation Internationale en Recherche Agronomique pour le\\n      Developpement (CIRAD), UMR AGAP, Avenue Agropolis TA 108/03, 34398 Montpellier\\n      Cedex 5, France.\\nFAU - Legavre, Thierry\\nAU  - Legavre T\\nAD  - Centre de Cooperation Internationale en Recherche Agronomique pour le\\n      Developpement (CIRAD), UMR AGAP, Avenue Agropolis TA 108/03, 34398 Montpellier\\n      Cedex 5, France.\\nFAU - Lopes, Uilson\\nAU  - Lopes U\\nAD  - Centro de Pesquisas do Cacau (CEPEC), CEPLAC, Rod. Ilheus-Itabuna, km 22, Ilheus,\\n      BA 45605-350, Brazil.\\nFAU - Sounigo, Olivier\\nAU  - Sounigo O\\nAD  - Centre de Cooperation Internationale en Recherche Agronomique pour le\\n      Developpement (CIRAD), UR Bioagresseurs, BP 2572 Elig-Essono, Yaounde, Cameroun.\\nAD  - Institut de Recherche Agricole pour le Developpement (IRAD) PO Box 2123, Yaounde,\\n      Cameroun.\\nFAU - Eyango, Marie Claire\\nAU  - Eyango MC\\nAD  - Institut de Recherche Agricole pour le Developpement (IRAD)PO Box 2123, Yaounde, \\n      Cameroun.\\nFAU - Mermaz, Benoit\\nAU  - Mermaz B\\nAD  - Centre de Cooperation Internationale en Recherche Agronomique pour le\\n      Developpement (CIRAD), UMR AGAP, Avenue Agropolis TA 108/03, 34398 Montpellier\\n      Cedex 5, France.\\nFAU - Da Silva, Marcos Ramos\\nAU  - Da Silva MR\\nAD  - Centro de Pesquisas do Cacau (CEPEC), CEPLAC, Rod. Ilheus-Itabuna, km 22, Ilheus,\\n      BA 45605-350, Brazil.\\nFAU - Loor Solorzano, Rey Gaston\\nAU  - Loor Solorzano RG\\nAD  - Instituto Nacional de Investigaciones Agropecuarias (INIAP), EET-Pichilingue. CP \\n      24 Km 5 via Quevedo El Empalme, Quevedo, Los Rios, Ecuador.\\nFAU - Argout, Xavier\\nAU  - Argout X\\nAD  - Centre de Cooperation Internationale en Recherche Agronomique pour le\\n      Developpement (CIRAD), UMR AGAP, Avenue Agropolis TA 108/03, 34398 Montpellier\\n      Cedex 5, France.\\nFAU - Gyapay, Gabor\\nAU  - Gyapay G\\nAD  - Commissariat a l\\'Energie Antomique (CEA), Institut de Genomique (IG), Genoscope, \\n      Evry, France.\\nFAU - Ebaiarrey, Herman Ebai\\nAU  - Ebaiarrey HE\\nAD  - Institut de Recherche Agricole pour le Developpement (IRAD)PO Box 2123, Yaounde, \\n      Cameroun.\\nFAU - Colonges, Kelly\\nAU  - Colonges K\\nAD  - Centre de Cooperation Internationale en Recherche Agronomique pour le\\n      Developpement (CIRAD), UMR AGAP, Avenue Agropolis TA 108/03, 34398 Montpellier\\n      Cedex 5, France.\\nFAU - Sanier, Christine\\nAU  - Sanier C\\nAD  - Centre de Cooperation Internationale en Recherche Agronomique pour le\\n      Developpement (CIRAD), UMR AGAP, Avenue Agropolis TA 108/03, 34398 Montpellier\\n      Cedex 5, France.\\nFAU - Rivallan, Ronan\\nAU  - Rivallan R\\nAD  - Centre de Cooperation Internationale en Recherche Agronomique pour le\\n      Developpement (CIRAD), UMR AGAP, Avenue Agropolis TA 108/03, 34398 Montpellier\\n      Cedex 5, France.\\nFAU - Mastin, Geraldine\\nAU  - Mastin G\\nAD  - Centre de Cooperation Internationale en Recherche Agronomique pour le\\n      Developpement (CIRAD), UMR AGAP, Avenue Agropolis TA 108/03, 34398 Montpellier\\n      Cedex 5, France.\\nFAU - Cryer, Nicholas\\nAU  - Cryer N\\nAD  - Mondelez UK R&D Limited, PO Box 12, Bournville Place, Bournville Lane,\\n      Birmingham, B30 2LU, UK.\\nFAU - Boccara, Michel\\nAU  - Boccara M\\nAD  - Centre de Cooperation Internationale en Recherche Agronomique pour le\\n      Developpement (CIRAD), UMR AGAP, Avenue Agropolis TA 108/03, 34398 Montpellier\\n      Cedex 5, France.\\nFAU - Verdeil, Jean-Luc\\nAU  - Verdeil JL\\nAD  - Centre de Cooperation Internationale en Recherche Agronomique pour le\\n      Developpement (CIRAD), UMR AGAP, Avenue Agropolis TA 108/03, 34398 Montpellier\\n      Cedex 5, France.\\nFAU - Efombagn Mousseni, Ives Bruno\\nAU  - Efombagn Mousseni IB\\nAD  - Institut de Recherche Agricole pour le Developpement (IRAD)PO Box 2123, Yaounde, \\n      Cameroun.\\nFAU - Peres Gramacho, Karina\\nAU  - Peres Gramacho K\\nAD  - Centro de Pesquisas do Cacau (CEPEC), CEPLAC, Rod. Ilheus-Itabuna, km 22, Ilheus,\\n      BA 45605-350, Brazil.\\nFAU - Clement, Didier\\nAU  - Clement D\\nAD  - Centre de Cooperation Internationale en Recherche Agronomique pour le\\n      Developpement (CIRAD), UMR AGAP, Avenue Agropolis TA 108/03, 34398 Montpellier\\n      Cedex 5, France.\\nAD  - Centro de Pesquisas do Cacau (CEPEC), CEPLAC, Rod. Ilheus-Itabuna, km 22, Ilheus,\\n      BA 45605-350, Brazil.\\nLA  - eng\\nPT  - Journal Article\\nPT  - Research Support, Non-U.S. Gov\\'t\\nPL  - England\\nTA  - J Exp Bot\\nJT  - Journal of experimental botany\\nJID - 9882906\\nSB  - IM\\nMH  - Cacao/genetics/*physiology\\nMH  - Chromosome Mapping\\nMH  - *Genetic Linkage\\nMH  - *Genome-Wide Association Study\\nMH  - Self-Incompatibility in Flowering Plants/*genetics\\nPMC - PMC5853246\\nOTO - NOTNLM\\nOT  - *Candidate genes\\nOT  - *Theobroma cacao\\nOT  - *diagnostic markers\\nOT  - *fine mapping\\nOT  - *gene expression\\nOT  - *immunolocalization\\nOT  - *self-incompatibility\\nEDAT- 2017/10/20 06:00\\nMHDA- 2018/05/15 06:00\\nCRDT- 2017/10/20 06:00\\nPHST- 2017/10/20 06:00 [entrez]\\nPHST- 2017/10/20 06:00 [pubmed]\\nPHST- 2018/05/15 06:00 [medline]\\nAID - 4371616 [pii]\\nAID - 10.1093/jxb/erx293 [doi]\\nPST - ppublish\\nSO  - J Exp Bot. 2017 Oct 13;68(17):4775-4790. doi: 10.1093/jxb/erx293.\\n\\nPMID- 28572335\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20180117\\nLR  - 20220129\\nIS  - 1095-9203 (Electronic)\\nIS  - 0036-8075 (Linking)\\nVI  - 356\\nIP  - 6341\\nDP  - 2017 Jun 2\\nTI  - A genetic signature of the evolution of loss of flight in the Galapagos\\n      cormorant.\\nLID - eaal3345 [pii]\\nLID - 10.1126/science.aal3345 [doi]\\nAB  - We have a limited understanding of the genetic and molecular basis of\\n      evolutionary changes in the size and proportion of limbs. We studied wing and\\n      pectoral skeleton reduction leading to flightlessness in the Galapagos cormorant \\n      (Phalacrocorax harrisi). We sequenced and de novo assembled the genomes of four\\n      cormorant species and applied a predictive and comparative genomics approach to\\n      find candidate variants that may have contributed to the evolution of\\n      flightlessness. These analyses and cross-species experiments in Caenorhabditis\\n      elegans and in chondrogenic cell lines implicated variants in genes necessary for\\n      transcriptional regulation and function of the primary cilium. Cilia are\\n      essential for Hedgehog signaling, and humans affected by skeletal ciliopathies\\n      suffer from premature bone growth arrest, mirroring skeletal features associated \\n      with loss of flight.\\nCI  - Copyright (c) 2017, American Association for the Advancement of Science.\\nFAU - Burga, Alejandro\\nAU  - Burga A\\nAUID- ORCID: 0000-0002-0211-4357\\nAD  - Department of Human Genetics, Department of Biological Chemistry, and Howard\\n      Hughes Medical Institute, University of California, Los Angeles, CA, USA.\\n      aburga@mednet.ucla.edu lkruglyak@mednet.ucla.edu.\\nFAU - Wang, Weiguang\\nAU  - Wang W\\nAUID- ORCID: 0000-0002-8604-5044\\nAD  - Departments of Molecular, Cell and Developmental Biology and Orthopaedic Surgery,\\n      University of California and Orthopaedic Institute for Children, Los Angeles, CA,\\n      USA.\\nFAU - Ben-David, Eyal\\nAU  - Ben-David E\\nAUID- ORCID: 0000-0003-0514-0400\\nAD  - Department of Human Genetics, Department of Biological Chemistry, and Howard\\n      Hughes Medical Institute, University of California, Los Angeles, CA, USA.\\nFAU - Wolf, Paul C\\nAU  - Wolf PC\\nAD  - Wildlife Services, U.S. Department of Agriculture, Roseburg, OR, USA.\\nFAU - Ramey, Andrew M\\nAU  - Ramey AM\\nAUID- ORCID: 0000-0002-3601-8400\\nAD  - U.S. Geological Survey Alaska Science Center, Anchorage, AK, USA.\\nFAU - Verdugo, Claudio\\nAU  - Verdugo C\\nAUID- ORCID: 0000-0002-2866-260X\\nAD  - Instituto de Patologia Animal, Facultad de Ciencias Veterinarias, Universidad\\n      Austral de Chile, Valdivia, Chile.\\nFAU - Lyons, Karen\\nAU  - Lyons K\\nAUID- ORCID: 0000-0001-9420-5813\\nAD  - Departments of Molecular, Cell and Developmental Biology and Orthopaedic Surgery,\\n      University of California and Orthopaedic Institute for Children, Los Angeles, CA,\\n      USA.\\nFAU - Parker, Patricia G\\nAU  - Parker PG\\nAUID- ORCID: 0000-0003-0843-835X\\nAD  - Department of Biology and Whitney Harris World Ecology Center, University of\\n      Missouri, St. Louis, MO, USA.\\nAD  - WildCare Institute, Saint Louis Zoo, St. Louis, MO, USA.\\nFAU - Kruglyak, Leonid\\nAU  - Kruglyak L\\nAUID- ORCID: 0000-0002-8065-3057\\nAD  - Department of Human Genetics, Department of Biological Chemistry, and Howard\\n      Hughes Medical Institute, University of California, Los Angeles, CA, USA.\\n      aburga@mednet.ucla.edu lkruglyak@mednet.ucla.edu.\\nLA  - eng\\nSI  - Dryad/10.5061/dryad.8m2t5\\nGR  - HHMI/Howard Hughes Medical Institute/United States\\nPT  - Journal Article\\nPT  - Research Support, N.I.H., Extramural\\nPT  - Research Support, Non-U.S. Gov\\'t\\nPL  - United States\\nTA  - Science\\nJT  - Science (New York, N.Y.)\\nJID - 0404511\\nRN  - 0 (Homeodomain Proteins)\\nSB  - IM\\nCIN - Science. 2017 Jun 2;356(6341):904-905. PMID: 28572349\\nMH  - Animals\\nMH  - *Biological Evolution\\nMH  - Birds/classification/*genetics/physiology\\nMH  - Bone and Bones/anatomy & histology\\nMH  - Caenorhabditis elegans/genetics\\nMH  - Cell Differentiation/genetics\\nMH  - Cell Line\\nMH  - Chondrogenesis/genetics\\nMH  - Cilia/genetics\\nMH  - Ecuador\\nMH  - Gene Expression Regulation/genetics\\nMH  - Genetic Variation\\nMH  - Genome/*genetics\\nMH  - Homeodomain Proteins/genetics\\nMH  - Mutation\\nMH  - Phylogeny\\nMH  - Wings, Animal/anatomy & histology/*physiology\\nPMC - PMC5567675\\nMID - HHMIMS895192\\nEDAT- 2017/06/03 06:00\\nMHDA- 2018/01/18 06:00\\nCRDT- 2017/06/03 06:00\\nPHST- 2016/11/04 00:00 [received]\\nPHST- 2017/04/13 00:00 [accepted]\\nPHST- 2017/06/03 06:00 [entrez]\\nPHST- 2017/06/03 06:00 [pubmed]\\nPHST- 2018/01/18 06:00 [medline]\\nAID - 356/6341/eaal3345 [pii]\\nAID - 10.1126/science.aal3345 [doi]\\nPST - ppublish\\nSO  - Science. 2017 Jun 2;356(6341). pii: 356/6341/eaal3345. doi:\\n      10.1126/science.aal3345.\\n\\nPMID- 28475225\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20171220\\nLR  - 20171220\\nIS  - 1365-294X (Electronic)\\nIS  - 0962-1083 (Linking)\\nVI  - 26\\nIP  - 19\\nDP  - 2017 Oct\\nTI  - The adaptive genomic landscape of beak morphology in Darwin\\'s finches.\\nPG  - 4978-4989\\nLID - 10.1111/mec.14166 [doi]\\nAB  - Beak shape in Darwin\\'s ground finches (Geospiza) is emblematic of natural\\n      selection and adaptive radiation, yet our understanding of the genetic basis of\\n      beak shape variation, and thus the genetic target of natural selection, is still \\n      evolving. Here we reveal the genomic architecture of beak shape variation using\\n      genomewide comparisons of four closely related and hybridizing species across 13 \\n      islands subject to parallel natural selection. Pairwise contrasts among species\\n      were used to identify a large number of genomic loci that are consistently\\n      related to species differences across a complex landscape. These loci are\\n      associated with hundreds of genes that have enriched GO categories significantly \\n      associated with development. One genomic region of particular interest is a\\n      section of Chromosome 1A with many candidate genes and increased linkage. The\\n      distinct, pointed beak shape of the cactus finch is linked to an excess of\\n      intermediate frequency alleles and increased heterozygosity in significant SNPs, \\n      but not across the rest of the genome. Alleles associated with pointier beaks\\n      among species were associated with pointier-beaked populations within each\\n      species, thus establishing a common basis for natural selection, species\\n      divergence and adaptive radiation. The adaptive genomic landscape for Darwin\\'s\\n      finches mirrors theoretical expectations based on morphological variation. The\\n      implication that a large number of genes are actively maintained to facilitate\\n      beak variation across parallel populations with documented interspecies admixture\\n      challenges our understanding of evolutionary processes in the wild.\\nCI  - (c) 2017 John Wiley & Sons Ltd.\\nFAU - Lawson, Lucinda P\\nAU  - Lawson LP\\nAUID- ORCID: http://orcid.org/0000-0003-3939-7829\\nAD  - Department of Biological Sciences, University of Cincinnati, Cincinnati, OH, USA.\\nFAU - Petren, Kenneth\\nAU  - Petren K\\nAD  - Department of Biological Sciences, University of Cincinnati, Cincinnati, OH, USA.\\nLA  - eng\\nPT  - Journal Article\\nDEP - 20170522\\nPL  - England\\nTA  - Mol Ecol\\nJT  - Molecular ecology\\nJID - 9214478\\nSB  - IM\\nMH  - Adaptation, Biological/genetics\\nMH  - Alleles\\nMH  - Animals\\nMH  - Beak/*anatomy & histology\\nMH  - Ecuador\\nMH  - Finches/*genetics\\nMH  - Genetic Linkage\\nMH  - Genetic Loci\\nMH  - Genetics, Population\\nMH  - Hybridization, Genetic\\nMH  - Islands\\nMH  - Linkage Disequilibrium\\nMH  - Polymorphism, Single Nucleotide\\nMH  - *Selection, Genetic\\nOTO - NOTNLM\\nOT  - Geospiza\\nOT  - Galapagos\\nOT  - population genomics\\nOT  - quantitative traits\\nEDAT- 2017/05/06 06:00\\nMHDA- 2017/12/21 06:00\\nCRDT- 2017/05/06 06:00\\nPHST- 2015/10/27 00:00 [received]\\nPHST- 2017/04/26 00:00 [revised]\\nPHST- 2017/04/27 00:00 [accepted]\\nPHST- 2017/05/06 06:00 [pubmed]\\nPHST- 2017/12/21 06:00 [medline]\\nPHST- 2017/05/06 06:00 [entrez]\\nAID - 10.1111/mec.14166 [doi]\\nPST - ppublish\\nSO  - Mol Ecol. 2017 Oct;26(19):4978-4989. doi: 10.1111/mec.14166. Epub 2017 May 22.\\n\\nPMID- 28442558\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20180309\\nLR  - 20181113\\nIS  - 1549-5469 (Electronic)\\nIS  - 1088-9051 (Linking)\\nVI  - 27\\nIP  - 6\\nDP  - 2017 Jun\\nTI  - Gene flow, ancient polymorphism, and ecological adaptation shape the genomic\\n      landscape of divergence among Darwin\\'s finches.\\nPG  - 1004-1015\\nLID - 10.1101/gr.212522.116 [doi]\\nAB  - Genomic comparisons of closely related species have identified \"islands\" of\\n      locally elevated sequence divergence. Genomic islands may contain functional\\n      variants involved in local adaptation or reproductive isolation and may therefore\\n      play an important role in the speciation process. However, genomic islands can\\n      also arise through evolutionary processes unrelated to speciation, and\\n      examination of their properties can illuminate how new species evolve. Here, we\\n      performed scans for regions of high relative divergence (FST) in 12 species pairs\\n      of Darwin\\'s finches at different genetic distances. In each pair, we identify\\n      genomic islands that are, on average, elevated in both relative divergence (FST) \\n      and absolute divergence (dXY). This signal indicates that haplotypes within these\\n      genomic regions became isolated from each other earlier than the rest of the\\n      genome. Interestingly, similar numbers of genomic islands of elevated dXY are\\n      observed in sympatric and allopatric species pairs, suggesting that recent gene\\n      flow is not a major factor in their formation. We find that two of the most\\n      pronounced genomic islands contain the ALX1 and HMGA2 loci, which are associated \\n      with variation in beak shape and size, respectively, suggesting that they are\\n      involved in ecological adaptation. A subset of genomic island regions, including \\n      these loci, appears to represent anciently diverged haplotypes that evolved early\\n      during the radiation of Darwin\\'s finches. Comparative genomics data indicate that\\n      these loci, and genomic islands in general, have exceptionally low recombination \\n      rates, which may play a role in their establishment.\\nCI  - (c) 2017 Han et al.; Published by Cold Spring Harbor Laboratory Press.\\nFAU - Han, Fan\\nAU  - Han F\\nAD  - Department of Medical Biochemistry and Microbiology, Science for Life Laboratory,\\n      Uppsala University, 75123 Uppsala, Sweden.\\nFAU - Lamichhaney, Sangeet\\nAU  - Lamichhaney S\\nAD  - Department of Medical Biochemistry and Microbiology, Science for Life Laboratory,\\n      Uppsala University, 75123 Uppsala, Sweden.\\nFAU - Grant, B Rosemary\\nAU  - Grant BR\\nAD  - Department of Ecology and Evolutionary Biology, Princeton University, Princeton, \\n      New Jersey 08544-2016, USA.\\nFAU - Grant, Peter R\\nAU  - Grant PR\\nAD  - Department of Ecology and Evolutionary Biology, Princeton University, Princeton, \\n      New Jersey 08544-2016, USA.\\nFAU - Andersson, Leif\\nAU  - Andersson L\\nAD  - Department of Medical Biochemistry and Microbiology, Science for Life Laboratory,\\n      Uppsala University, 75123 Uppsala, Sweden.\\nAD  - Department of Animal Breeding and Genetics, Swedish University of Agricultural\\n      Sciences, 75007 Uppsala, Sweden.\\nAD  - Department of Veterinary Integrative Biosciences, Texas A&M University, College\\n      Station, Texas 77843-4461, USA.\\nFAU - Webster, Matthew T\\nAU  - Webster MT\\nAD  - Department of Medical Biochemistry and Microbiology, Science for Life Laboratory,\\n      Uppsala University, 75123 Uppsala, Sweden.\\nLA  - eng\\nSI  - Dryad/10.5061/dryad.7155d\\nPT  - Journal Article\\nPT  - Research Support, U.S. Gov\\'t, Non-P.H.S.\\nPT  - Research Support, Non-U.S. Gov\\'t\\nDEP - 20170425\\nPL  - United States\\nTA  - Genome Res\\nJT  - Genome research\\nJID - 9518021\\nSB  - IM\\nMH  - Adaptation, Physiological/genetics\\nMH  - Animal Distribution\\nMH  - Animals\\nMH  - Ecuador\\nMH  - Finches/classification/*genetics\\nMH  - *Gene Flow\\nMH  - Genetic Loci\\nMH  - *Genetic Speciation\\nMH  - *Genome\\nMH  - Genomic Islands\\nMH  - Haplotypes\\nMH  - Microsatellite Repeats\\nMH  - *Phylogeny\\nMH  - Polymorphism, Genetic\\nMH  - Selection, Genetic\\nMH  - Sympatry\\nPMC - PMC5453315\\nEDAT- 2017/04/27 06:00\\nMHDA- 2018/03/10 06:00\\nCRDT- 2017/04/27 06:00\\nPHST- 2016/07/08 00:00 [received]\\nPHST- 2017/02/14 00:00 [accepted]\\nPHST- 2017/04/27 06:00 [pubmed]\\nPHST- 2018/03/10 06:00 [medline]\\nPHST- 2017/04/27 06:00 [entrez]\\nAID - gr.212522.116 [pii]\\nAID - 10.1101/gr.212522.116 [doi]\\nPST - ppublish\\nSO  - Genome Res. 2017 Jun;27(6):1004-1015. doi: 10.1101/gr.212522.116. Epub 2017 Apr\\n      25.\\n\\nPMID- 27863902\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20170712\\nLR  - 20181113\\nIS  - 1471-5007 (Electronic)\\nIS  - 1471-4922 (Linking)\\nVI  - 33\\nIP  - 4\\nDP  - 2017 Apr\\nTI  - Prediction and Prevention of Parasitic Diseases Using a Landscape Genomics\\n      Framework.\\nPG  - 264-275\\nLID - S1471-4922(16)30191-X [pii]\\nLID - 10.1016/j.pt.2016.10.008 [doi]\\nAB  - Substantial heterogeneity exists in the dispersal, distribution and transmission \\n      of parasitic species. Understanding and predicting how such features are governed\\n      by the ecological variation of landscape they inhabit is the central goal of\\n      spatial epidemiology. Genetic data can further inform functional connectivity\\n      among parasite, host and vector populations in a landscape. Gene flow correlates \\n      with the spread of epidemiologically relevant phenotypes among parasite and\\n      vector populations (e.g., virulence, drug and pesticide resistance), as well as\\n      invasion and re-invasion risk where parasite transmission is absent due to\\n      current or past intervention measures. However, the formal integration of spatial\\n      and genetic data (\\'landscape genetics\\') is scarcely ever applied to parasites.\\n      Here, we discuss the specific challenges and practical prospects for the use of\\n      landscape genetics and genomics to understand the biology and control of\\n      parasitic disease and present a practical framework for doing so.\\nCI  - Copyright (c) 2016 Elsevier Ltd. All rights reserved.\\nFAU - Schwabl, Philipp\\nAU  - Schwabl P\\nAD  - Institute of Biodiversity, Animal Health and Comparative Medicine, University of \\n      Glasgow, Glasgow G12 8QQ, UK.\\nFAU - Llewellyn, Martin S\\nAU  - Llewellyn MS\\nAD  - Institute of Biodiversity, Animal Health and Comparative Medicine, University of \\n      Glasgow, Glasgow G12 8QQ, UK. Electronic address: martin.llewellyn@glasgow.ac.uk.\\nFAU - Landguth, Erin L\\nAU  - Landguth EL\\nAD  - Division of Biological Sciences, University of Montana, 32 Campus Drive,\\n      Missoula, MT, 59812, USA.\\nFAU - Andersson, Bjorn\\nAU  - Andersson B\\nAD  - Department of Cell and Molecular Biology, Science for Life Laboratory, Karolinska\\n      Institutet, Berzelius vag 35, 171 77 Stockholm, Sweden.\\nFAU - Kitron, Uriel\\nAU  - Kitron U\\nAD  - Department of Environmental Sciences, Emory University, Atlanta, GA, USA.\\nFAU - Costales, Jaime A\\nAU  - Costales JA\\nAD  - Center for Research on Health in Latin America, School of Biological Sciences,\\n      Pontifical Catholic University of Ecuador, Quito, Ecuador.\\nFAU - Ocana, Sofia\\nAU  - Ocana S\\nAD  - Center for Research on Health in Latin America, School of Biological Sciences,\\n      Pontifical Catholic University of Ecuador, Quito, Ecuador.\\nFAU - Grijalva, Mario J\\nAU  - Grijalva MJ\\nAD  - Center for Research on Health in Latin America, School of Biological Sciences,\\n      Pontifical Catholic University of Ecuador, Quito, Ecuador; Infectious and\\n      Tropical Disease Institute, Department of Biomedical Sciences, Heritage College\\n      of Osteopathic Medicine, Ohio University, Athens, OH, USA.\\nLA  - eng\\nGR  - R15 AI105749/AI/NIAID NIH HHS/United States\\nPT  - Journal Article\\nPT  - Review\\nDEP - 20161116\\nPL  - England\\nTA  - Trends Parasitol\\nJT  - Trends in parasitology\\nJID - 100966034\\nSB  - IM\\nMH  - Animals\\nMH  - *Genomics\\nMH  - *Host-Parasite Interactions/genetics\\nMH  - Humans\\nMH  - Parasitic Diseases/epidemiology/*parasitology/*prevention & control/transmission\\nMH  - Phenotype\\nPMC - PMC5376512\\nMID - NIHMS827616\\nEDAT- 2016/11/20 06:00\\nMHDA- 2017/07/14 06:00\\nCRDT- 2016/11/20 06:00\\nPHST- 2016/05/07 00:00 [received]\\nPHST- 2016/09/10 00:00 [revised]\\nPHST- 2016/10/19 00:00 [accepted]\\nPHST- 2016/11/20 06:00 [pubmed]\\nPHST- 2017/07/14 06:00 [medline]\\nPHST- 2016/11/20 06:00 [entrez]\\nAID - S1471-4922(16)30191-X [pii]\\nAID - 10.1016/j.pt.2016.10.008 [doi]\\nPST - ppublish\\nSO  - Trends Parasitol. 2017 Apr;33(4):264-275. doi: 10.1016/j.pt.2016.10.008. Epub\\n      2016 Nov 16.\\n\\nPMID- 27185408\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20171113\\nLR  - 20190112\\nIS  - 2050-084X (Electronic)\\nIS  - 2050-084X (Linking)\\nVI  - 5\\nDP  - 2016 May 17\\nTI  - Mitochondrial genetic diversity, selection and recombination in a canine\\n      transmissible cancer.\\nLID - 10.7554/eLife.14552 [doi]\\nLID - e14552 [pii]\\nAB  - Canine transmissible venereal tumour (CTVT) is a clonally transmissible cancer\\n      that originated approximately 11,000 years ago and affects dogs worldwide.\\n      Despite the clonal origin of the CTVT nuclear genome, CTVT mitochondrial genomes \\n      (mtDNAs) have been acquired by periodic capture from transient hosts. We\\n      sequenced 449 complete mtDNAs from a global population of CTVTs, and show that\\n      mtDNA horizontal transfer has occurred at least five times, delineating five\\n      tumour clades whose distributions track two millennia of dog global migration.\\n      Negative selection has operated to prevent accumulation of deleterious mutations \\n      in captured mtDNA, and recombination has caused occasional mtDNA re-assortment.\\n      These findings implicate functional mtDNA as a driver of CTVT global metastatic\\n      spread, further highlighting the important role of mtDNA in cancer evolution.\\nFAU - Strakova, Andrea\\nAU  - Strakova A\\nAD  - Department of Veterinary Medicine, University of Cambridge, Cambridge, United\\n      Kingdom.\\nFAU - Ni Leathlobhair, Maire\\nAU  - Ni Leathlobhair M\\nAD  - Department of Veterinary Medicine, University of Cambridge, Cambridge, United\\n      Kingdom.\\nFAU - Wang, Guo-Dong\\nAU  - Wang GD\\nAD  - State Key Laboratory of Genetic Resources and Evolution, Yunnan Laboratory of\\n      Molecular Biology of Domestic Animals, Kunming Institute of Zoology, Chinese\\n      Academy of Sciences, Kunming, China.\\nFAU - Yin, Ting-Ting\\nAU  - Yin TT\\nAD  - State Key Laboratory of Genetic Resources and Evolution, Yunnan Laboratory of\\n      Molecular Biology of Domestic Animals, Kunming Institute of Zoology, Chinese\\n      Academy of Sciences, Kunming, China.\\nFAU - Airikkala-Otter, Ilona\\nAU  - Airikkala-Otter I\\nAD  - International Training Center, Worldwide Veterinary Service, Aruvankadu, India.\\nFAU - Allen, Janice L\\nAU  - Allen JL\\nAD  - Animal Management in Rural and Remote Indigenous Communities, Darwin, Australia.\\nFAU - Allum, Karen M\\nAU  - Allum KM\\nAD  - World Vets, Fargo, United States.\\nFAU - Bansse-Issa, Leontine\\nAU  - Bansse-Issa L\\nAD  - Stichting Dierenbescherming Suriname, Paramaribo, Suriname.\\nFAU - Bisson, Jocelyn L\\nAU  - Bisson JL\\nAD  - Department of Veterinary Medicine, University of Cambridge, Cambridge, United\\n      Kingdom.\\nFAU - Castillo Domracheva, Artemio\\nAU  - Castillo Domracheva A\\nAD  - Corozal Veterinary Hospital, University of Panama, Panama City, Panama.\\nFAU - de Castro, Karina F\\nAU  - de Castro KF\\nAD  - Department of Clinical and Veterinary Surgery, Sao Paulo State University, Sao\\n      Paulo, Brazil.\\nFAU - Corrigan, Anne M\\nAU  - Corrigan AM\\nAD  - St. George\\'s University, True Blue, Grenada.\\nFAU - Cran, Hugh R\\nAU  - Cran HR\\nAD  - The Nakuru District Veterinary Scheme Ltd, Nakuru, Kenya.\\nFAU - Crawford, Jane T\\nAU  - Crawford JT\\nAD  - Animal Medical Centre, Belize City, Belize.\\nFAU - Cutter, Stephen M\\nAU  - Cutter SM\\nAD  - Animal Management in Rural and Remote Indigenous Communities, Darwin, Australia.\\nFAU - Delgadillo Keenan, Laura\\nAU  - Delgadillo Keenan L\\nAD  - Veterinary clinic Sr. Dog\\'s, Guadalajara, Mexico.\\nFAU - Donelan, Edward M\\nAU  - Donelan EM\\nAD  - Animal Management in Rural and Remote Indigenous Communities, Darwin, Australia.\\nFAU - Faramade, Ibikunle A\\nAU  - Faramade IA\\nAD  - National Veterinary Research Institute, Vom, Nigeria.\\nFAU - Flores Reynoso, Erika\\nAU  - Flores Reynoso E\\nAD  - International Fund for Animal Welfare, Quintana Roo, Mexico.\\nFAU - Fotopoulou, Eleni\\nAU  - Fotopoulou E\\nAD  - Intermunicipal Stray Animals Care Centre, Perama, Greece.\\nFAU - Fruean, Skye N\\nAU  - Fruean SN\\nAD  - Animal Protection Society of Samoa, Apia, Samoa.\\nFAU - Gallardo-Arrieta, Fanny\\nAU  - Gallardo-Arrieta F\\nAD  - Faculty of Veterinary Science, University of Zulia, Maracaibo, Venezuela.\\nFAU - Glebova, Olga\\nAU  - Glebova O\\nAD  - Veterinary clinic BIOCONTROL, Moscow, Russia.\\nFAU - Hafelin Manrique, Rodrigo F\\nAU  - Hafelin Manrique RF\\nAD  - Veterinary clinic El Roble, Santiago de Chile, Chile.\\nFAU - Henriques, Joaquim Jgp\\nAU  - Henriques JJ\\nAD  - OnevetGroup, Centro Veterinario Berna, Lisboa, Portugal.\\nFAU - Ignatenko, Natalia\\nAU  - Ignatenko N\\nAD  - Veterinary clinic Zoovetservis, Kiev, Ukraine.\\nFAU - Koenig, Debbie\\nAU  - Koenig D\\nAD  - World Vets, Fargo, United States.\\nFAU - Lanza-Perea, Marta\\nAU  - Lanza-Perea M\\nAD  - St. George\\'s University, True Blue, Grenada.\\nFAU - Lobetti, Remo\\nAU  - Lobetti R\\nAD  - Bryanston Veterinary Hospital, Bryanston, South Africa.\\nFAU - Lopez Quintana, Adriana M\\nAU  - Lopez Quintana AM\\nAD  - Veterinary Clinic Lopez Quintana, Maldonado, Uruguay.\\nFAU - Losfelt, Thibault\\nAU  - Losfelt T\\nAD  - Clinique Veterinaire de Grand Fond, Saint Gilles les Bains, France.\\nFAU - Marino, Gabriele\\nAU  - Marino G\\nAD  - Department of Veterinary Sciences, University of Messina, Messina, Italy.\\nFAU - Martincorena, Inigo\\nAU  - Martincorena I\\nAD  - Wellcome Trust Sanger Institute, Hinxton, United Kingdom.\\nFAU - Martinez Castaneda, Simon\\nAU  - Martinez Castaneda S\\nAD  - Facultad de Medicina Veterinaria y Zootecnia, Universidad Autonoma del Estado de \\n      Mexico, Toluca, Mexico.\\nFAU - Martinez-Lopez, Mayra F\\nAU  - Martinez-Lopez MF\\nAD  - School of Veterinary Medicine, Universidad de las Americas, Quito, Ecuador.\\nFAU - Meyer, Michael\\nAU  - Meyer M\\nAD  - Touray & Meyer Vet Clinic, Serrekunda, Gambia.\\nFAU - Nakanwagi, Berna\\nAU  - Nakanwagi B\\nAD  - The Kampala Veterinary Surgery, Kampala, Uganda.\\nFAU - De Nardi, Andrigo B\\nAU  - De Nardi AB\\nAD  - Department of Clinical and Veterinary Surgery, Sao Paulo State University, Sao\\n      Paulo, Brazil.\\nFAU - Neunzig, Winifred\\nAU  - Neunzig W\\nAD  - World Vets, Fargo, United States.\\nFAU - Nixon, Sally J\\nAU  - Nixon SJ\\nAD  - Vets Beyond Borders, The Rocks, Australia.\\nFAU - Onsare, Marsden M\\nAU  - Onsare MM\\nAD  - Aniworld veterinary clinic, Kisumu, Kenya.\\nFAU - Ortega-Pacheco, Antonio\\nAU  - Ortega-Pacheco A\\nAD  - Faculty of Veterinary Medicine, Autonomous University of Yucatan, Merida, Mexico.\\nFAU - Peleteiro, Maria C\\nAU  - Peleteiro MC\\nAD  - Interdisciplinary Centre of Research in Animal Health, Faculty of Veterinary\\n      Medicine, University of Lisbon, Lisboa, Portugal.\\nFAU - Pye, Ruth J\\nAU  - Pye RJ\\nAD  - Vets Beyond Borders, The Rocks, Australia.\\nFAU - Reece, John F\\nAU  - Reece JF\\nAD  - Help in Suffering, Jaipur, India.\\nFAU - Rojas Gutierrez, Jose\\nAU  - Rojas Gutierrez J\\nAD  - Veterinary clinic Dr Jose Rojas, Los Andes, Chile.\\nFAU - Sadia, Haleema\\nAU  - Sadia H\\nAD  - University of Veterinary and Animal Sciences, Lahore, Pakistan.\\nFAU - Schmeling, Sheila K\\nAU  - Schmeling SK\\nAD  - Corozal Veterinary Clinic, Corozal Town, Belize.\\nFAU - Shamanova, Olga\\nAU  - Shamanova O\\nAD  - Veterinary clinic Vetmaster, Ramenskoye, Russia.\\nFAU - Ssuna, Richard K\\nAU  - Ssuna RK\\nAD  - Lilongwe Society for Protection and Care of Animals, Lilongwe, Malawi.\\nFAU - Steenland-Smit, Audrey E\\nAU  - Steenland-Smit AE\\nAD  - Stichting Dierenbescherming Suriname, Paramaribo, Suriname.\\nFAU - Svitich, Alla\\nAU  - Svitich A\\nAD  - State Hospital of Veterinary Medicine, Dniprodzerzhynsk, Ukraine.\\nFAU - Thoya Ngoka, Ismail\\nAU  - Thoya Ngoka I\\nAD  - Kenya Society for Protection and Care of Animals, Nairobi, Kenya.\\nFAU - Vitalaru, Bogdan A\\nAU  - Vitalaru BA\\nAD  - Clinical Sciences Department, Faculty of Veterinary Medicine, Bucharest, Romania.\\nFAU - de Vos, Anna P\\nAU  - de Vos AP\\nAD  - Ladybrand Animal Clinic, Ladybrand, South Africa.\\nFAU - de Vos, Johan P\\nAU  - de Vos JP\\nAD  - Veterinary Oncology Referral Centre De Ottenhorst, Terneuzen, Netherlands.\\nFAU - Walkinton, Oliver\\nAU  - Walkinton O\\nAD  - Vets Beyond Borders, The Rocks, Australia.\\nFAU - Wedge, David C\\nAU  - Wedge DC\\nAD  - Wellcome Trust Sanger Institute, Hinxton, United Kingdom.\\nFAU - Wehrle-Martinez, Alvaro S\\nAU  - Wehrle-Martinez AS\\nAD  - Faculty of Veterinary Sciences, National University of Asuncion, San Lorenzo,\\n      Paraguay.\\nFAU - van der Wel, Mirjam G\\nAU  - van der Wel MG\\nAD  - Animal Anti Cruelty League, Port Elizabeth, South Africa.\\nFAU - Widdowson, Sophie Ae\\nAU  - Widdowson SA\\nAD  - Lilongwe Society for Protection and Care of Animals, Lilongwe, Malawi.\\nFAU - Murchison, Elizabeth P\\nAU  - Murchison EP\\nAUID- ORCID: 0000-0001-7462-8907\\nAD  - Department of Veterinary Medicine, University of Cambridge, Cambridge, United\\n      Kingdom.\\nLA  - eng\\nGR  - Wellcome Trust/United Kingdom\\nGR  - 102942/Z/13/A/Wellcome Trust/United Kingdom\\nPT  - Journal Article\\nPT  - Research Support, Non-U.S. Gov\\'t\\nDEP - 20160517\\nPL  - England\\nTA  - Elife\\nJT  - eLife\\nJID - 101579614\\nRN  - 0 (DNA, Mitochondrial)\\nSB  - IM\\nMH  - Animals\\nMH  - DNA, Mitochondrial/chemistry/genetics\\nMH  - Dog Diseases/*genetics\\nMH  - Dogs\\nMH  - *Genetic Variation\\nMH  - Mitochondria/*genetics\\nMH  - *Recombination, Genetic\\nMH  - *Selection, Genetic\\nMH  - Sequence Analysis, DNA\\nMH  - Venereal Tumors, Veterinary/*genetics\\nPMC - PMC4869914\\nOTO - NOTNLM\\nOT  - *Dog\\nOT  - *cancer biology\\nOT  - *canine transmissible venereal tumour\\nOT  - *evolutionary biology\\nOT  - *genomics\\nOT  - *mitochondria\\nOT  - *transmissible cancer\\nEDAT- 2016/05/18 06:00\\nMHDA- 2017/11/14 06:00\\nCRDT- 2016/05/18 06:00\\nPHST- 2016/01/19 00:00 [received]\\nPHST- 2016/04/08 00:00 [accepted]\\nPHST- 2016/05/18 06:00 [entrez]\\nPHST- 2016/05/18 06:00 [pubmed]\\nPHST- 2017/11/14 06:00 [medline]\\nAID - 10.7554/eLife.14552 [doi]\\nPST - epublish\\nSO  - Elife. 2016 May 17;5. doi: 10.7554/eLife.14552.\\n\\nPMID- 27038556\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20170130\\nLR  - 20181202\\nIS  - 1873-6254 (Electronic)\\nIS  - 0001-706X (Linking)\\nVI  - 159\\nDP  - 2016 Jul\\nTI  - De novo assembly and characterization of the Trichuris trichiura adult worm\\n      transcriptome using Ion Torrent sequencing.\\nPG  - 132-41\\nLID - 10.1016/j.actatropica.2016.03.036 [doi]\\nLID - S0001-706X(16)30147-4 [pii]\\nAB  - Infection with helminthic parasites, including the soil-transmitted helminth\\n      Trichuris trichiura (human whipworm), has been shown to modulate host immune\\n      responses and, consequently, to have an impact on the development and\\n      manifestation of chronic human inflammatory diseases. De novo derivation of\\n      helminth proteomes from sequencing of transcriptomes will provide valuable data\\n      to aid identification of parasite proteins that could be evaluated as potential\\n      immunotherapeutic molecules in near future. Herein, we characterized the\\n      transcriptome of the adult stage of the human whipworm T. trichiura, using\\n      next-generation sequencing technology and a de novo assembly strategy. Nearly\\n      17.6 million high-quality clean reads were assembled into 6414 contiguous\\n      sequences, with an N50 of 1606bp. In total, 5673 protein-encoding sequences were \\n      confidentially identified in the T. trichiura adult worm transcriptome; of these,\\n      1013 sequences represent potential newly discovered proteins for the species,\\n      most of which presenting orthologs already annotated in the related species T.\\n      suis. A number of transcripts representing probable novel non-coding transcripts \\n      for the species T. trichiura were also identified. Among the most abundant\\n      transcripts, we found sequences that code for proteins involved in lipid\\n      transport, such as vitellogenins, and several chitin-binding proteins. Through a \\n      cross-species expression analysis of gene orthologs shared by T. trichiura and\\n      the closely related parasites T. suis and T. muris it was possible to find\\n      twenty-six protein-encoding genes that are consistently highly expressed in the\\n      adult stages of the three helminth species. Additionally, twenty transcripts\\n      could be identified that code for proteins previously detected by mass\\n      spectrometry analysis of protein fractions of the whipworm somatic extract that\\n      present immunomodulatory activities. Five of these transcripts were amongst the\\n      most highly expressed protein-encoding sequences in the T. trichiura adult worm. \\n      Besides, orthologs of proteins demonstrated to have potent immunomodulatory\\n      properties in related parasitic helminths were also predicted from the T.\\n      trichiura de novo assembled transcriptome.\\nCI  - Copyright (c) 2016. Published by Elsevier B.V.\\nFAU - Santos, Leonardo N\\nAU  - Santos LN\\nAD  - Institute of Health Sciences, Federal University of Bahia, Salvador, BA, Brazil.\\nFAU - Silva, Eduardo S\\nAU  - Silva ES\\nAD  - Institute of Health Sciences, Federal University of Bahia, Salvador, BA, Brazil.\\nFAU - Santos, Andre S\\nAU  - Santos AS\\nAD  - Institute of Health Sciences, Federal University of Bahia, Salvador, BA, Brazil.\\nFAU - De Sa, Pablo H\\nAU  - De Sa PH\\nAD  - Institute of Biological Sciences, Federal University of Para, Belem, PA, Brazil.\\nFAU - Ramos, Rommel T\\nAU  - Ramos RT\\nAD  - Institute of Biological Sciences, Federal University of Para, Belem, PA, Brazil.\\nFAU - Silva, Artur\\nAU  - Silva A\\nAD  - Institute of Biological Sciences, Federal University of Para, Belem, PA, Brazil.\\nFAU - Cooper, Philip J\\nAU  - Cooper PJ\\nAD  - Institute of Infection and Immunity, St. George\\'s University of London, London,\\n      UK; Centro de Investigacion en Enfermedades Infecciosas y Cronicas, Pontificia\\n      Universidad Catolica del Ecuador, Quito, Ecuador.\\nFAU - Barreto, Mauricio L\\nAU  - Barreto ML\\nAD  - Institute of Public Health, Federal University of Bahia, Salvador, BA, Brazil;\\n      Centro de Pesquisas Goncalo Muniz, FIOCRUZ-BA, Salvador, BA, Brazil.\\nFAU - Loureiro, Sebastiao\\nAU  - Loureiro S\\nAD  - Institute of Public Health, Federal University of Bahia, Salvador, BA, Brazil.\\nFAU - Pinheiro, Carina S\\nAU  - Pinheiro CS\\nAD  - Institute of Health Sciences, Federal University of Bahia, Salvador, BA, Brazil.\\nFAU - Alcantara-Neves, Neuza M\\nAU  - Alcantara-Neves NM\\nAD  - Institute of Health Sciences, Federal University of Bahia, Salvador, BA, Brazil. \\n      Electronic address: neuza@ufba.br.\\nFAU - Pacheco, Luis G C\\nAU  - Pacheco LG\\nAD  - Institute of Health Sciences, Federal University of Bahia, Salvador, BA, Brazil. \\n      Electronic address: luis.pacheco@ufba.br.\\nLA  - eng\\nPT  - Journal Article\\nDEP - 20160331\\nPL  - Netherlands\\nTA  - Acta Trop\\nJT  - Acta tropica\\nJID - 0370374\\nRN  - 0 (Antigens, Helminth)\\nSB  - IM\\nMH  - Adolescent\\nMH  - Amino Acid Sequence\\nMH  - Animals\\nMH  - Antigens, Helminth/*genetics\\nMH  - Child\\nMH  - Child, Preschool\\nMH  - Ecuador\\nMH  - Female\\nMH  - Humans\\nMH  - Infant\\nMH  - Infant, Newborn\\nMH  - Male\\nMH  - Transcriptome/*genetics\\nMH  - Trichuriasis/*parasitology\\nMH  - Trichuris/*genetics\\nOTO - NOTNLM\\nOT  - Functional genomics\\nOT  - Next-generation sequencing\\nOT  - Transcriptome\\nOT  - Trichuris trichiura\\nEDAT- 2016/04/04 06:00\\nMHDA- 2017/01/31 06:00\\nCRDT- 2016/04/04 06:00\\nPHST- 2015/07/10 00:00 [received]\\nPHST- 2016/03/23 00:00 [revised]\\nPHST- 2016/03/30 00:00 [accepted]\\nPHST- 2016/04/04 06:00 [entrez]\\nPHST- 2016/04/04 06:00 [pubmed]\\nPHST- 2017/01/31 06:00 [medline]\\nAID - S0001-706X(16)30147-4 [pii]\\nAID - 10.1016/j.actatropica.2016.03.036 [doi]\\nPST - ppublish\\nSO  - Acta Trop. 2016 Jul;159:132-41. doi: 10.1016/j.actatropica.2016.03.036. Epub 2016\\n      Mar 31.\\n\\nPMID- 26537019\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20160108\\nLR  - 20181113\\nIS  - 1479-7364 (Electronic)\\nIS  - 1473-9542 (Linking)\\nVI  - 9\\nDP  - 2015 Nov 4\\nTI  - Genomics is changing personal healthcare and medicine: the dawn of iPH\\n      (individualized preventive healthcare).\\nPG  - 29\\nLID - 10.1186/s40246-015-0052-0 [doi]\\nAB  - This opinion piece focuses on the convergence of information technology (IT) in\\n      the form of personal monitors, especially smart phones and possibly also smart\\n      watches, individual genomic information and preventive healthcare and medicine.\\n      This may benefit each one of us not only individually but also society as a whole\\n      through iPH (individualized preventive healthcare). This shift driven by genomic \\n      and other technologies may well also change the relationship between patient and \\n      physician by empowering the former but giving him/her also much more individual\\n      responsibility.\\nFAU - Mehrian-Shai, Ruty\\nAU  - Mehrian-Shai R\\nAD  - Pediatric Hemato-Oncology, Chaim Sheba Medical Center, Ramat Gan, Israel.\\nFAU - Reichardt, Juergen K V\\nAU  - Reichardt JK\\nAD  - Division of Tropical Health and Medicine, James Cook University, Townsville, QLD,\\n      Australia. jreichardt@yachaytech.edu.au.\\nAD  - Present Address: Yachay Tech University, San Miguel de Urcuqui, Ecuador.\\n      jreichardt@yachaytech.edu.au.\\nLA  - eng\\nPT  - Journal Article\\nDEP - 20151104\\nPL  - England\\nTA  - Hum Genomics\\nJT  - Human genomics\\nJID - 101202210\\nSB  - IM\\nMH  - Education, Medical\\nMH  - *Genome, Human\\nMH  - *Genomics\\nMH  - Humans\\nMH  - Precision Medicine/economics/*methods\\nMH  - Preventive Health Services/economics/methods\\nPMC - PMC4634796\\nEDAT- 2015/11/06 06:00\\nMHDA- 2016/01/09 06:00\\nCRDT- 2015/11/06 06:00\\nPHST- 2015/09/29 00:00 [received]\\nPHST- 2015/10/31 00:00 [accepted]\\nPHST- 2015/11/06 06:00 [entrez]\\nPHST- 2015/11/06 06:00 [pubmed]\\nPHST- 2016/01/09 06:00 [medline]\\nAID - 10.1186/s40246-015-0052-0 [doi]\\nAID - 10.1186/s40246-015-0052-0 [pii]\\nPST - epublish\\nSO  - Hum Genomics. 2015 Nov 4;9:29. doi: 10.1186/s40246-015-0052-0.\\n\\nPMID- 24823669\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20150330\\nLR  - 20220129\\nIS  - 1549-5469 (Electronic)\\nIS  - 1088-9051 (Linking)\\nVI  - 24\\nIP  - 8\\nDP  - 2014 Aug\\nTI  - Population genomics of parallel hybrid zones in the mimetic butterflies, H.\\n      melpomene and H. erato.\\nPG  - 1316-33\\nLID - 10.1101/gr.169292.113 [doi]\\nAB  - Hybrid zones can be valuable tools for studying evolution and identifying genomic\\n      regions responsible for adaptive divergence and underlying phenotypic variation. \\n      Hybrid zones between subspecies of Heliconius butterflies can be very narrow and \\n      are maintained by strong selection acting on color pattern. The comimetic\\n      species, H. erato and H. melpomene, have parallel hybrid zones in which both\\n      species undergo a change from one color pattern form to another. We use\\n      restriction-associated DNA sequencing to obtain several thousand genome-wide\\n      sequence markers and use these to analyze patterns of population divergence\\n      across two pairs of parallel hybrid zones in Peru and Ecuador. We compare two\\n      approaches for analysis of this type of data-alignment to a reference genome and \\n      de novo assembly-and find that alignment gives the best results for species both \\n      closely (H. melpomene) and distantly (H. erato, approximately 15% divergent)\\n      related to the reference sequence. Our results confirm that the color pattern\\n      controlling loci account for the majority of divergent regions across the genome,\\n      but we also detect other divergent regions apparently unlinked to color pattern\\n      differences. We also use association mapping to identify previously unmapped\\n      color pattern loci, in particular the Ro locus. Finally, we identify a new\\n      cryptic population of H. timareta in Ecuador, which occurs at relatively low\\n      altitude and is mimetic with H. melpomene malleti.\\nCI  - (c) 2014 Nadeau et al.; Published by Cold Spring Harbor Laboratory Press.\\nFAU - Nadeau, Nicola J\\nAU  - Nadeau NJ\\nAUID- ORCID: http://orcid.org/0000-0002-9319-921X\\nAD  - Department of Zoology, University of Cambridge, Cambridge CB2 3EJ, United\\n      Kingdom; Department of Animal and Plant Sciences, University of Sheffield,\\n      Sheffield, S10 2TN, United Kingdom;\\nFAU - Ruiz, Mayte\\nAU  - Ruiz M\\nAD  - Department of Biology and Center for Applied Tropical Ecology and Conservation,\\n      University of Puerto Rico, Rio Piedras, San Juan, Puerto Rico 00921;\\nFAU - Salazar, Patricio\\nAU  - Salazar P\\nAD  - Department of Zoology, University of Cambridge, Cambridge CB2 3EJ, United\\n      Kingdom; Centro de Investigacion en Biodiversidad y Cambio Climatico (BioCamb),\\n      Universidad Tecnologica Indoamerica, Quito, Ecuador;\\nFAU - Counterman, Brian\\nAU  - Counterman B\\nAD  - Department of Biology, Mississippi State University, Mississippi 39762, USA;\\nFAU - Medina, Jose Alejandro\\nAU  - Medina JA\\nAD  - High Performance Computing Facility, University of Puerto Rico, San Juan, Puerto \\n      Rico, 00921;\\nFAU - Ortiz-Zuazaga, Humberto\\nAU  - Ortiz-Zuazaga H\\nAD  - High Performance Computing Facility, University of Puerto Rico, San Juan, Puerto \\n      Rico, 00921; Department of Computer Science, University of Puerto Rico, Rio\\n      Piedras, San Juan, Puerto Rico 00921;\\nFAU - Morrison, Anna\\nAU  - Morrison A\\nAD  - Department of Zoology, University of Cambridge, Cambridge CB2 3EJ, United\\n      Kingdom;\\nFAU - McMillan, W Owen\\nAU  - McMillan WO\\nAUID- ORCID: http://orcid.org/0000-0003-2805-2745\\nAD  - Smithsonian Tropical Research Institute, Apartado 0843-03092, Balboa, Ancon,\\n      Panama.\\nFAU - Jiggins, Chris D\\nAU  - Jiggins CD\\nAUID- ORCID: http://orcid.org/0000-0002-7809-062X\\nAD  - Department of Zoology, University of Cambridge, Cambridge CB2 3EJ, United\\n      Kingdom; Smithsonian Tropical Research Institute, Apartado 0843-03092, Balboa,\\n      Ancon, Panama c.jiggins@zoo.cam.ac.uk.\\nFAU - Papa, Riccardo\\nAU  - Papa R\\nAD  - Department of Biology and Center for Applied Tropical Ecology and Conservation,\\n      University of Puerto Rico, Rio Piedras, San Juan, Puerto Rico 00921;\\nLA  - eng\\nGR  - BB/G008841/1/BB_/Biotechnology and Biological Sciences Research Council/United\\n      Kingdom\\nGR  - BB/H01439X/1/BB_/Biotechnology and Biological Sciences Research Council/United\\n      Kingdom\\nGR  - P20 GM103475/GM/NIGMS NIH HHS/United States\\nGR  - P20GM103475/GM/NIGMS NIH HHS/United States\\nPT  - Journal Article\\nPT  - Research Support, N.I.H., Extramural\\nPT  - Research Support, U.S. Gov\\'t, Non-P.H.S.\\nDEP - 20140513\\nPL  - United States\\nTA  - Genome Res\\nJT  - Genome research\\nJID - 9518021\\nSB  - IM\\nMH  - Animals\\nMH  - Butterflies/anatomy & histology/*genetics\\nMH  - Chromosome Mapping\\nMH  - Evolution, Molecular\\nMH  - Genetic Loci\\nMH  - Genetic Speciation\\nMH  - Genetic Variation\\nMH  - *Genome, Insect\\nMH  - Phenotype\\nMH  - Phylogeny\\nMH  - Pigmentation\\nMH  - Wings, Animal/anatomy & histology\\nPMC - PMC4120085\\nEDAT- 2014/05/16 06:00\\nMHDA- 2015/03/31 06:00\\nCRDT- 2014/05/15 06:00\\nPHST- 2014/05/15 06:00 [entrez]\\nPHST- 2014/05/16 06:00 [pubmed]\\nPHST- 2015/03/31 06:00 [medline]\\nAID - gr.169292.113 [pii]\\nAID - 10.1101/gr.169292.113 [doi]\\nPST - ppublish\\nSO  - Genome Res. 2014 Aug;24(8):1316-33. doi: 10.1101/gr.169292.113. Epub 2014 May 13.\\n\\nPMID- 24342523\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20150423\\nLR  - 20211021\\nIS  - 1474-760X (Electronic)\\nIS  - 1474-7596 (Linking)\\nVI  - 14\\nIP  - 12\\nDP  - 2013 Dec 16\\nTI  - Population genomics of the endangered giant Galapagos tortoise.\\nPG  - R136\\nLID - 10.1186/gb-2013-14-12-r136 [doi]\\nAB  - BACKGROUND: The giant Galapagos tortoise, Chelonoidis nigra, is a large-sized\\n      terrestrial chelonian of high patrimonial interest. The species recently\\n      colonized a small continental archipelago, the Galapagos Islands, where it has\\n      been facing novel environmental conditions and limited resource availability. To \\n      explore the genomic consequences of this ecological shift, we analyze the\\n      transcriptomic variability of five individuals of C. nigra, and compare it to\\n      similar data obtained from several continental species of turtles. RESULTS:\\n      Having clarified the timing of divergence in the Chelonoidis genus, we report in \\n      C. nigra a very low level of genetic polymorphism, signatures of a weakened\\n      efficacy of purifying selection, and an elevated mutation load in coding and\\n      regulatory sequences. These results are consistent with the hypothesis of an\\n      extremely low long-term effective population size in this insular species.\\n      Functional evolutionary analyses reveal a reduced diversity of immunity genes in \\n      C. nigra, in line with the hypothesis of attenuated pathogen diversity in\\n      islands, and an increased selective pressure on genes involved in response to\\n      stress, potentially related to the climatic instability of its environment and\\n      its elongated lifespan. Finally, we detect no population structure or\\n      homozygosity excess in our five-individual sample. CONCLUSIONS: These results\\n      enlighten the molecular evolution of an endangered taxon in a stressful\\n      environment and point to island endemic species as a promising model for the\\n      study of the deleterious effects on genome evolution of a reduced long-term\\n      population size.\\nFAU - Loire, Etienne\\nAU  - Loire E\\nFAU - Chiari, Ylenia\\nAU  - Chiari Y\\nFAU - Bernard, Aurelien\\nAU  - Bernard A\\nFAU - Cahais, Vincent\\nAU  - Cahais V\\nFAU - Romiguier, Jonathan\\nAU  - Romiguier J\\nFAU - Nabholz, Benoit\\nAU  - Nabholz B\\nFAU - Lourenco, Joao Miguel\\nAU  - Lourenco JM\\nFAU - Galtier, Nicolas\\nAU  - Galtier N\\nLA  - eng\\nPT  - Journal Article\\nPT  - Research Support, Non-U.S. Gov\\'t\\nDEP - 20131216\\nPL  - England\\nTA  - Genome Biol\\nJT  - Genome biology\\nJID - 100960660\\nRN  - 0 (DNA, Mitochondrial)\\nSB  - IM\\nMH  - Animals\\nMH  - DNA, Mitochondrial/analysis\\nMH  - Ecuador\\nMH  - *Endangered Species\\nMH  - Evolution, Molecular\\nMH  - Gene Expression Profiling\\nMH  - Genetic Drift\\nMH  - *Genome\\nMH  - Metagenomics/*methods\\nMH  - Phylogeny\\nMH  - Polymorphism, Single Nucleotide\\nMH  - Population Density\\nMH  - Selection, Genetic\\nMH  - Stress, Physiological\\nMH  - Turtles/classification/*genetics\\nPMC - PMC4053747\\nEDAT- 2013/12/18 06:00\\nMHDA- 2015/04/24 06:00\\nCRDT- 2013/12/18 06:00\\nPHST- 2013/05/28 00:00 [received]\\nPHST- 2013/12/16 00:00 [accepted]\\nPHST- 2013/12/18 06:00 [entrez]\\nPHST- 2013/12/18 06:00 [pubmed]\\nPHST- 2015/04/24 06:00 [medline]\\nAID - gb-2013-14-12-r136 [pii]\\nAID - 10.1186/gb-2013-14-12-r136 [doi]\\nPST - epublish\\nSO  - Genome Biol. 2013 Dec 16;14(12):R136. doi: 10.1186/gb-2013-14-12-r136.\\n\\nPMID- 19844075\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20100520\\nLR  - 20100217\\nIS  - 1662-8063 (Electronic)\\nIS  - 1662-4246 (Linking)\\nVI  - 13\\nIP  - 3\\nDP  - 2010\\nTI  - Ecuador: public health genomics.\\nPG  - 171-80\\nLID - 10.1159/000249817 [doi]\\nAB  - Ecuador has a heterogeneous population of almost 14 million people and a complex \\n      health care system provided through provincial and national health programs by\\n      government and private hospitals. There are public health facilities at regional \\n      and territorial level. Ecuador has a small cadre of genetic professionals that\\n      provide clinical genetic services in a few private medical centers in the main\\n      cities. Prenatal screening is offered exclusively in a few individual hospitals, \\n      with variable uptake as part of prenatal care. Surveillance of the effect of\\n      prenatal screening and diagnosis on the birth prevalence of congenital anomalies \\n      is limited by gaps and variations in surveillance systems. Newborn screening\\n      programs are almost inexistent. There is broad variation in optional\\n      participation in laboratory quality assurance schemes, and there are no\\n      regulatory frameworks that are directly pertinent to genetic testing services or \\n      population genetics. Health technology assessment in Ecuador is conducted by a\\n      diverse collection of organizations, several of which have produced reports\\n      related to genetics.\\nCI  - Copyright 2009 S. Karger AG, Basel.\\nFAU - Gonzalez-Andrade, F\\nAU  - Gonzalez-Andrade F\\nAD  - Department of Medicine, Metropolitan Hospital, Quito, Ecuador.\\n      fabriciogonzaleza@yahoo.es\\nFAU - Lopez-Pulles, R\\nAU  - Lopez-Pulles R\\nLA  - eng\\nPT  - Journal Article\\nDEP - 20091016\\nPL  - Switzerland\\nTA  - Public Health Genomics\\nJT  - Public health genomics\\nJID - 101474167\\nSB  - IM\\nMH  - Community Health Services\\nMH  - Congenital Abnormalities/*diagnosis\\nMH  - Ecuador\\nMH  - Ethics, Medical\\nMH  - Genetic Testing/methods\\nMH  - Genomics/*methods\\nMH  - Geography\\nMH  - Health Policy\\nMH  - Health Priorities\\nMH  - Health Services Accessibility\\nMH  - Humans\\nMH  - Infant, Newborn\\nMH  - Neonatal Screening/*methods\\nMH  - Prenatal Diagnosis/*methods\\nMH  - Public Health/*methods\\nEDAT- 2009/10/22 06:00\\nMHDA- 2010/05/21 06:00\\nCRDT- 2009/10/22 06:00\\nPHST- 2008/12/18 00:00 [accepted]\\nPHST- 2009/10/22 06:00 [entrez]\\nPHST- 2009/10/22 06:00 [pubmed]\\nPHST- 2010/05/21 06:00 [medline]\\nAID - 000249817 [pii]\\nAID - 10.1159/000249817 [doi]\\nPST - ppublish\\nSO  - Public Health Genomics. 2010;13(3):171-80. doi: 10.1159/000249817. Epub 2009 Oct \\n      16.\\n\\nPMID- 19018989\\nOWN - NLM\\nSTAT- MEDLINE\\nDCOM- 20090331\\nLR  - 20211020\\nIS  - 1364-3703 (Electronic)\\nIS  - 1364-3703 (Linking)\\nVI  - 9\\nIP  - 5\\nDP  - 2008 Sep\\nTI  - Moniliophthora perniciosa, the causal agent of witches\\' broom disease of cacao:\\n      what\\'s new from this old foe?\\nPG  - 577-88\\nLID - 10.1111/j.1364-3703.2008.00496.x [doi]\\nAB  - UNLABELLED: Moniliophthora perniciosa (=Crinipellis perniciosa) causes one of the\\n      three main fungal diseases of Theobroma cacao (cacao), the source of chocolate.\\n      This pathogen causes Witches\\' broom disease (WBD) and has brought about severe\\n      economic losses in all of the cacao-growing regions to which it has spread with\\n      yield reductions that range from 50 to 90%. Cacao production in South America\\n      reflects the severity of this pathogen, as the yields in most of the infected\\n      regions have not returned to pre-outbreak levels, even with the introduction of\\n      resistant varieties. In this review we give a brief historical account and\\n      summarize the current state of knowledge focusing on developments in the areas of\\n      systematics, fungal physiology, biochemistry, genomics and gene expression in an \\n      attempt to highlight this disease. Moniliophthora perniciosa is a hemibiotrophic \\n      fungus with two distinct growth phases. The ability to culture a biotrophic-like \\n      phase in vitro along with new findings derived from the nearly complete genome\\n      and expression studies clearly show that these different fungal growth phases\\n      function under distinct metabolic parameters. These new findings have greatly\\n      improved our understanding of this fungal/host interaction and we may be at the\\n      crossroads of understanding how hemibiotrophic fungal plant pathogens cause\\n      disease in other crops. HISTORICAL SUMMARY OF WBD: The first WDB symptoms appear \\n      to have been described in the diaries of Alexandre Rodrigues Ferreira (described \\n      as lagartao; meaning big lizard) from his observations of cacao trees in 1785 and\\n      1787 in Amazonia, which is consistent with the generally accepted idea that M.\\n      perniciosa, like its main host T. cacao, evolved in this region. The disease\\n      subsequently arrived in Surinam in 1895. WBD moved rapidly, spreading to Guyana\\n      in 1906, Ecuador in 1918, Trinidad in 1928, Colombia in 1929 and Grenada in 1948.\\n      In each case, cacao production was catastrophically affected with yield\\n      reductions of 50-90%. After the arrival of M. perniciosa in Bahia in 1989, Brazil\\n      went from being the world\\'s 3rd largest producer of cacao (347 000 tonnes in\\n      1988-1990; c. 15% of the total world production at that time) to a net importer\\n      (141 000 tonnes in 1998-2000). Fortunately for chocolate lovers, other regions of\\n      the world such as West Africa and South East Asia have not yet been affected by\\n      this disease and have expanded production to meet growing world demand (predicted\\n      to reach 3 700 000 tonnes by 2010). CLASSIFICATION: Moniliophthora perniciosa\\n      (Stahel) Aime & Phillips-Mora: super-kingdom Eukaryota; kingdom Fungi; phylum\\n      Basidiomycota; subphylum Agaricomycotina; class Agaricomycetes; subclass\\n      Agaricomycetidae; order Agaricales; family Marasmiaceae; genus Moniliophthora.\\n      USEFUL WEBSITES: http://www.lge.ibi.unicamp.br/vassoura/,\\n      http://nt.ars-grin.gov/taxadescriptions/keys/TrichodermaIndex.cfm,\\n      http://www.worldcocoafoundation.org/info-center/research-updates.asp,\\n      http://www.ars.usda.gov/ba/psi/spcl.\\nFAU - Meinhardt, Lyndel W\\nAU  - Meinhardt LW\\nAD  - Sustainable Perennial Crops Laboratory, USDA-ARS, 10300 Baltimore Ave., Bldg.\\n      001, Beltsville, MD 20705-2350, USA. lyndel.meinhardt@ars.usda.gov\\nFAU - Rincones, Johana\\nAU  - Rincones J\\nFAU - Bailey, Bryan A\\nAU  - Bailey BA\\nFAU - Aime, M Catherine\\nAU  - Aime MC\\nFAU - Griffith, Gareth W\\nAU  - Griffith GW\\nFAU - Zhang, Dapeng\\nAU  - Zhang D\\nFAU - Pereira, Goncalo A G\\nAU  - Pereira GA\\nLA  - eng\\nPT  - Journal Article\\nPT  - Review\\nPL  - England\\nTA  - Mol Plant Pathol\\nJT  - Molecular plant pathology\\nJID - 100954969\\nSB  - IM\\nMH  - Agaricales/*physiology\\nMH  - Cacao/*microbiology\\nMH  - Host-Parasite Interactions\\nMH  - Plant Diseases/*microbiology\\nRF  - 91\\nPMC - PMC6640444\\nEDAT- 2008/11/21 09:00\\nMHDA- 2009/04/01 09:00\\nCRDT- 2008/11/21 09:00\\nPHST- 2008/11/21 09:00 [pubmed]\\nPHST- 2009/04/01 09:00 [medline]\\nPHST- 2008/11/21 09:00 [entrez]\\nAID - MPP496 [pii]\\nAID - 10.1111/j.1364-3703.2008.00496.x [doi]\\nPST - ppublish\\nSO  - Mol Plant Pathol. 2008 Sep;9(5):577-88. doi: 10.1111/j.1364-3703.2008.00496.x.\\n'"
      ]
     },
     "execution_count": 1,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# Escriba aquí su código para el ejercicio 1\n",
    "import Bio\n",
    "from Bio.Seq import Seq\n",
    "# cargar biopython a sus módulos, funciones\n",
    "from Bio import Entrez\n",
    "import re\n",
    "# Always tell NCBI who are (edit the e-mail below!)\n",
    "Entrez.email = \"nayeli.mamallacta@est.ikiam.edu.ec\"\n",
    "handle = Entrez.esearch(db = \"pubmed\",\n",
    "                        term = \"Ecuador genomics[Title/Abstract]\",\n",
    "                        usehistory = \"y\")\n",
    "record = Entrez.read(handle)\n",
    "# generate a Python list with all Pubmed IDs of articles about Dengue Network\n",
    "id_list = record[\"IdList\"]\n",
    "print(record[\"Count\"])\n",
    "webenv = record[\"WebEnv\"]\n",
    "query_key = record[\"QueryKey\"]\n",
    "handle = Entrez.efetch(db = \"pubmed\",\n",
    "                      rettype = \"medline\",\n",
    "                      retmode = \"text\",\n",
    "                      retstart = 0,\n",
    "retmax = 51410, webenv = webenv, query_key = query_key)\n",
    "handle.read()"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "\n",
    "## Ejercicio 2 [1 punto]\n",
    "\n",
    "\n",
    "Utilice la función ```download_pubmed``` para descargar la data con el keyword ```Ecuador genomics``` e imprima lo siguiente: \n",
    "\n",
    "```python\n",
    "'El número de artículos es: xx' # xx corresponde al número de artículos recuperados\n",
    "\n",
    "'El PMID del primer artículo es: xxxxxxxx' # xxxxxxxx corresponde al ID del artículo en PUDMED\n",
    "\n",
    "```"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 2,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "55 Ecuador genomics Title/Abstract found\n",
      "DQ991794.1\n"
     ]
    }
   ],
   "source": [
    "# Escriba aquí su código para el ejercicio 2\n",
    "## 'El número de árticulos es:'\n",
    "handle = Entrez.esearch(db=\"pubmed\", \n",
    "                        term=\"Ecuador genomics[Title/Abstract]\",\n",
    "                        usehistory=\"y\",\n",
    "                        retmax = 1)\n",
    "record = Entrez.read(handle)\n",
    "print(\"{} Ecuador genomics Title/Abstract found\". format(record[\"Count\"]))\n",
    "\n",
    "## 'El PMID del primer artículo es:'\n",
    "from Bio import SeqIO\n",
    "from Bio import Entrez\n",
    "Entrez.email = \"nayeli.mamallacta@est.ikiam.edu.ec\"\n",
    "with Entrez.efetch(db=\"nucleotide\", rettype=\"gb\", retmode=\"text\", id=\"DQ991794.1\") as handle:\n",
    "    seq_record = SeqIO.read(handle, \"gb\")\n",
    "print(seq_record.id)\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Ejercicio 3 [1 punto]\n",
    "\n",
    "Utilice la función ```mining_pubs``` para crear los siguientes dataframes: \n",
    "\n",
    "* pub_year\n",
    "\n",
    "* pub_authors\n",
    "\n",
    "* coutries\n",
    "\n",
    "Muestre los dos primeros y los dos últimos registros de cada dataframe. "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 3,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "text/html": [
       "<div>\n",
       "<style scoped>\n",
       "    .dataframe tbody tr th:only-of-type {\n",
       "        vertical-align: middle;\n",
       "    }\n",
       "\n",
       "    .dataframe tbody tr th {\n",
       "        vertical-align: top;\n",
       "    }\n",
       "\n",
       "    .dataframe thead th {\n",
       "        text-align: right;\n",
       "    }\n",
       "</style>\n",
       "<table border=\"1\" class=\"dataframe\">\n",
       "  <thead>\n",
       "    <tr style=\"text-align: right;\">\n",
       "      <th></th>\n",
       "      <th>pub_year</th>\n",
       "      <th>pub_authors</th>\n",
       "      <th>countries</th>\n",
       "    </tr>\n",
       "  </thead>\n",
       "  <tbody>\n",
       "    <tr>\n",
       "      <th>0</th>\n",
       "      <td>2009</td>\n",
       "      <td>González-Andrade, F</td>\n",
       "      <td>Ecuador</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>1</th>\n",
       "      <td>2019</td>\n",
       "      <td>Reichardt, Juergen K V</td>\n",
       "      <td>Australia</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>2</th>\n",
       "      <td>2019</td>\n",
       "      <td>Zambrano-Mila, Marlon S</td>\n",
       "      <td>Ecuador</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>3</th>\n",
       "      <td>2020</td>\n",
       "      <td>Cao, Mengji</td>\n",
       "      <td>China</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>4</th>\n",
       "      <td>2020</td>\n",
       "      <td>Brenière, Simone Frédérique</td>\n",
       "      <td>France</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>5</th>\n",
       "      <td>2017</td>\n",
       "      <td>Trueba, Gabriel</td>\n",
       "      <td>Ecuador</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>6</th>\n",
       "      <td>2019</td>\n",
       "      <td>Flores, Francisco J</td>\n",
       "      <td>Ecuador</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>7</th>\n",
       "      <td>2020</td>\n",
       "      <td>Armijos-Jaramillo, Vinicio</td>\n",
       "      <td>Ecuador</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>8</th>\n",
       "      <td>2020</td>\n",
       "      <td>Nadeau, Nicola J</td>\n",
       "      <td>United Kingdom</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>9</th>\n",
       "      <td>2016</td>\n",
       "      <td>Andrade, Jacinto</td>\n",
       "      <td>Ecuador</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>10</th>\n",
       "      <td>2014</td>\n",
       "      <td>Whittle, M R</td>\n",
       "      <td>Brazil</td>\n",
       "    </tr>\n",
       "    <tr>\n",
       "      <th>11</th>\n",
       "      <td>2014</td>\n",
       "      <td>Garred, Peter</td>\n",
       "      <td>Denmark</td>\n",
       "    </tr>\n",
       "  </tbody>\n",
       "</table>\n",
       "</div>"
      ],
      "text/plain": [
       "    pub_year                  pub_authors       countries\n",
       "0       2009          González-Andrade, F         Ecuador\n",
       "1       2019       Reichardt, Juergen K V       Australia\n",
       "2       2019      Zambrano-Mila, Marlon S         Ecuador\n",
       "3       2020                  Cao, Mengji           China\n",
       "4       2020  Brenière, Simone Frédérique          France\n",
       "5       2017              Trueba, Gabriel         Ecuador\n",
       "6       2019          Flores, Francisco J         Ecuador\n",
       "7       2020   Armijos-Jaramillo, Vinicio         Ecuador\n",
       "8       2020             Nadeau, Nicola J  United Kingdom\n",
       "9       2016             Andrade, Jacinto         Ecuador\n",
       "10      2014                 Whittle, M R          Brazil\n",
       "11      2014                Garred, Peter         Denmark"
      ]
     },
     "execution_count": 3,
     "metadata": {},
     "output_type": "execute_result"
    }
   ],
   "source": [
    "# Escriba aquí su código para el ejercicio 3\n",
    "import pandas as pd\n",
    "Ecuadorgen = {'pub_year':[2009, 2019, 2019, 2020, 2020, 2017, 2019, 2020, 2020, 2016, 2014, 2014],\n",
    "             'pub_authors':['González-Andrade, F', 'Reichardt, Juergen K V', 'Zambrano-Mila, Marlon S', 'Cao, Mengji', 'Brenière, Simone Frédérique', 'Trueba, Gabriel', ' Flores, Francisco J', 'Armijos-Jaramillo, Vinicio', 'Nadeau, Nicola J', 'Andrade, Jacinto', ' Whittle, M R','Garred, Peter'],\n",
    "             'countries':['Ecuador', 'Australia', 'Ecuador', 'China', 'France', 'Ecuador', 'Ecuador', 'Ecuador', 'United Kingdom', 'Ecuador', 'Brazil', 'Denmark'],\n",
    "             }\n",
    "df = pd.DataFrame(Ecuadorgen)\n",
    "df"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 6,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "      0\n",
      "0  2009\n",
      "1  2019\n",
      "       0\n",
      "10  2014\n",
      "11  2014\n",
      "                        0\n",
      "0     González-Andrade, F\n",
      "1  Reichardt, Juergen K V\n",
      "                0\n",
      "10   Whittle, M R\n",
      "11  Garred, Peter\n",
      "           0\n",
      "0    Ecuador\n",
      "1  Australia\n",
      "          0\n",
      "10   Brazil\n",
      "11  Denmark\n"
     ]
    }
   ],
   "source": [
    "pub_year = [2009, 2019, 2019, 2020, 2020, 2017, 2019, 2020, 2020, 2016, 2014, 2014]\n",
    "pub_authors = ['González-Andrade, F', 'Reichardt, Juergen K V', 'Zambrano-Mila, Marlon S', 'Cao, Mengji', 'Brenière, Simone Frédérique', 'Trueba, Gabriel', ' Flores, Francisco J', 'Armijos-Jaramillo, Vinicio', 'Nadeau, Nicola J', 'Andrade, Jacinto', ' Whittle, M R','Garred, Peter']\n",
    "countries = ['Ecuador',  'Australia', 'Ecuador', 'China', 'France', 'Ecuador', 'Ecuador', 'Ecuador', 'United Kingdom', 'Ecuador', 'Brazil', 'Denmark']\n",
    "             \n",
    "df = pd.DataFrame(pub_year)\n",
    "print(df.head(2))\n",
    "print(df.tail(2))\n",
    "           \n",
    "df = pd.DataFrame(pub_authors)\n",
    "print(df.head(2))\n",
    "print(df.tail(2))\n",
    "\n",
    "df = pd.DataFrame(countries)\n",
    "print(df.head(2))\n",
    "print(df.tail(2))\n",
    "\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {
    "jp-MarkdownHeadingCollapsed": true,
    "tags": []
   },
   "source": [
    "## Ejercicio 4 [2 puntos]\n",
    "\n",
    "Realice la visualización de los datos del ejercicio 3. \n",
    "\n",
    "* ```pubyear```: utilizando el conteo artículos por año, calcule la media y la desviación estandar (asumiendo que la distribución es normal). Realice un gráfico de la distribución normal de sus datos. Coloque el título, leyenda de los ejes \"x\" e \"y\", y la leyenda de la curva (utilice las expresiones matemáticas tipos latex de ser necesario). \n",
    "\n",
    "* ```pub_authors```: realice un boxplot del conteo de autores por artículo. Coloque las leyendas adecuadas para cada eje y el título.  \n",
    "\n",
    "* ```countries```: realice una nube de puntos (adaptación de su tarea), resalte al menos 3 países con flechas. Coloque las leyendas adecuadas para cada eje y el título. \n"
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 7,
   "metadata": {},
   "outputs": [
    {
     "data": {
      "image/png": "iVBORw0KGgoAAAANSUhEUgAAAigAAAGECAYAAAAcO9uNAAAAOXRFWHRTb2Z0d2FyZQBNYXRwbG90bGliIHZlcnNpb24zLjMuNCwgaHR0cHM6Ly9tYXRwbG90bGliLm9yZy8QVMy6AAAACXBIWXMAAAsTAAALEwEAmpwYAAAnP0lEQVR4nO3df5xddX3n8dfHIUQ0/EjMGFEcM2w3imU0LVMbfwy10XbdagcXykJLRCm7aHWVrPXRVnQLbe0vqwVxzVqqoiHYVH4Up5VssUZxXI02g5AB8UfJwIDGODFRQoJBhs/+ce+kN8MkMyG593sy9/V8PO5j7j3n3HM+9x7IvOf7/Z7vicxEkiSpSp5UugBJkqTJDCiSJKlyDCiSJKlyDCiSJKlyDCiSJKlyDCiSJKlyDChSG4iIjIifafExF9ePe1SLj/vyiHiglcfcTx1fiIj/VroO6UhlQJFaKCLujYiHI+KhiNgREZ+JiGeXrmtCRLwhIr5Uuo4jTURcFhFrStchzSYGFKn1fj0z5wEnAluBDxauRxXW6hYoqSoMKFIhmfkT4Hrg+RPLIuL4iFgdEWMRcV9EvDsinhQRCyLigYj49fp28yLi3yLi/Prrj0fEhyPisxGxMyJujYjnTHXcAxzjFODDwIvrLTw/qm//axHxjfp+vxsR79jPfjsi4n0RsS0iNgOvnuK4H42ILfX9vCciOvazr8si4vqI+Pv6cW+LiBc2rN+ny6r++d8zaR+X1Gu5NyLO2/+Z2Lv9qyPi6xHxYETcHxGXNax7XLdRfb+vjIhXAZcA59S/tzsaNntORPy/+me4JSIWNry/PyLuiogf1buDTpm079+PiE3Arog4qv76u/V9fSsiXjHdZ5KOZAYUqZCIeApwDrChYfEHgeOBk4FfAs4HLsjM7cBvA38bEU8HLgduz8zVDe89D/gTYCFwO3Dtfg69v2PcDbwJ+EpmzsvME+rbfxR4Y2YeC5wKrN/Pfv878Brg54Be4Dcmrf8E8CjwM/VtfhU40BiNM4DrgAXAJ4GbImLOAbZv9Axq38OzgNcDV0XEc6d5zy5q38UJ1MLV70TEa6c7UGb+X+DPgL+vf28vbFj9W8AFwNOBo4F3AETEEuDvgJVAJ3Az8I8RcXTDe3+zXscJwH8A/gfwC/Xz8J+Ae6erTTqSGVCk1rup3jrxIPArwF9BrQWCWmB5Z2buzMx7gfcDrwPIzFuo/cL+HLVfXG+ctN/PZOYXM3MP8C5qLSH7jG+Z7hj78VPg+RFxXGbuyMzb9rPdfwWuyMz764HqzxuOuwj4z8DKzNyVmT+gFrLOPcBxhzLz+sz8KfDXwJOBZQfYfrL/lZl7MvNW4DP1+vYrM7+QmcOZ+VhmbqIWIH7pII43lasz89uZ+TDwKWBpffk51M7XZ+uf733AMcBLGt57Zf27fBgYB+ZSOw9zMvPezLznEGuTKs2AIrXea+utE3Op/VV8a0RM/MV/NHBfw7b3UWsFmHAVtVaMqzPzh5P2e//Ek8x8CNgOPHPSNjM5xmRnAb8G3FfvOnrxfrZ7ZmMNk47xHGAOsKXepfEj4G+otSzsT+PneQx4gMd/nv3ZkZm7JtVywPdGxC9GxOfrXV8/ptaatPBA75mB7zc83w3Mqz9/Jg3fT/3z3c++56Hx8/8btdaWy4AfRMTaiJjpdyEdkQwoUiGZOZ6ZN1L76/hlwDZqrRWNY0e6gO/C3taPvwFWU+t+mHzZ8N7WkoiYR61r5HuTtjngMYDH3d48M/81M8+gFiZuotYSMJUtjTXU9zvhfmAPsDAzT6g/jsvMn93PviZ/nicBJzV8nt3AUxq2fcak986PiKdOqmXydzHZJ4EB4NmZeTy18ThRX7er8Xj1c9HZ8N6DvS3892g4BxER1D7vdxu22WefmfnJzHxZ/X0J/OVBHlM6ohhQpEKi5gxgPnB3Zo5T++X/pxFxbH2Q69uBictXL6n//G1qXQKrJw0y/bWIeFl9HMOfAF/NzMYWDWZwjK3ASRNjISLi6Ig4LyKOr3dFPEgtUE3lU8DbIuKkiJgP/EHDcbcAtwDvj4jj6oNy/0NEHKgL5bSIODNqV7GspBZwJsbr3A78Vn1g7quYuivmj+r191EbG3PdAY4FcCywPTN/EhEvojZ+ZMK3gSfXB9LOAd5NrQVswlZgcT1IzcSngFdHxCvq+/vd+uf78lQbR8RzI2J5RMwFfgJMdPtIs5YBRWq9f4yIh6j9sv9T4PWZeVd93Vup/bW+GfgStb/qPxYRp1ELEufXQ8ZfUvsr+g8a9vtJ4FJqXTunURs0O5Upj1Fftx64C/h+RGyrL3sdcG9EPEit22PFfvb7t8A/A3cAtwE3Tlp/PrXupW8AO6hdwXTifvYF8GlqYzV21Gs4sx6SAC4Gfh34Uf1z3jTpvd+vv+971AYLvykzv3mAYwG8GfjjiNgJ/CENLUWZ+eP6+o9Qa+XYRa3LacJE+PlhROxvjM5emfktat/jB6m1av06tcvPH9nPW+YCf1Hf9vvUWrMu2c+20qwQmQfbMimpaiLi48ADmfnu0rUcDvVLfH8mM/cXhiTNcragSJKkyjGgSGob9YnRHpriMe1EbpJayy4eSZJUObagSJKkyjGgSJKkyjmi7pK5cOHCXLx4cekyJEnSYTA0NLQtMzunWlc0oETE/6R2s7AEhqndsOwn+9t+8eLFbNy4sVXlSZKkJoqI+/a3rlgXT0Q8C3gb0JuZpwIdHPjGYZIkqU2UHoNyFHBMfSrrpzD9vTIkSVIbKBZQMvO71O4nMkrtJmM/rt9OXpIktbmSXTzzgTOAbmq3Hn9qRDxuWuuIuCgiNkbExrGxsVaXKUmSCijZxfNKYCQzx+o3ALsReMnkjTLzqszszczezs4pB/pKkqRZpmRAGQWWRcRTIiKAVwB3F6xHkiRVRMkxKF+ldrv126hdYvwk4KpS9UiSpOooOg9KZl4KXFqyBkmSVD2lLzOWJEl6HAOKJEmqnCPqXjySNFsNDw8zODjI2NgYnZ2d9PX10dPTU7osqRgDiiQVNjw8zPr16+nv76erq4vR0VEGBgYADClqW3bxSFJhg4OD9Pf3093dTUdHB93d3fT39zM4OFi6NKkYA4okFTY2NkZXV9c+y7q6unD2bLUzA4okFdbZ2cno6Og+y0ZHR3H2bLUzA4okFdbX18fAwAAjIyOMj48zMjLCwMAAfX19pUuTinGQrCQVNjEQdt26dXuv4lm+fLkDZNXWDCiSVAE9PT0GEqmBXTySJKlyDCiSJKlyDCiSJKlyDCiSJKlyDCiSJKlyDCiSJKlyDCiSJKlyDCiSJKlyDCiSJKlyDCiSJKlyDCiSJKlyDCiSJKlyDCiSJKlyDCiSJKlyDCiSJKlyDCiSJKlyDCiSJKlyDCiSJKlyjipdgCQJhoeHGRwcZGxsjM7OTvr6+ujp6SldllSMAUWSChseHmb9+vX09/fT1dXF6OgoAwMDAIYUtS0DiiQVNjg4yJIlS1i3bt3eFpQlS5YwODhoQFHbMqBIUmF33nkne/bs4bWvfe3eFpSbbrqJ73znO6VLk4pxkKwkFbZ9+3ZOO+00uru76ejooLu7m9NOO43t27eXLk0qxoAiSYUtWLCAoaEhRkZGGB8fZ2RkhKGhIRYsWFC6NKmYol08EXEC8BHgVCCB387Mr5SsSZJa7dRTT6W7u3ufMSjPe97zmDt3bunSpGJKj0H5APB/M/M3IuJo4CmF65Gkluvr65vyKp7ly5eXLk0qplhAiYjjgNOBNwBk5iPAI6XqkaRSJq7UaWxBWb58uVfwqK2VHINyMjAGXB0RX4+Ij0TEUydvFBEXRcTGiNg4NjbW+iolSVLLleziOQr4eeCtmfnViPgA8AfA/2rcKDOvAq4C6O3tzZZXKUlN5kRt0uOVbEF5AHggM79af309tcAiSW1lcHCQ/v7+fS4z7u/vZ3BwsHRpUjHFAkpmfh+4PyKeW1/0CuAbpeqRpFLGxsbo6uraZ1lXVxd2a6udlZ4H5a3AtRGxCVgK/FnZciSp9To7OxkdHd1n2ejoKJ2dnYUqksoreplxZt4O9JasQZJK6+vr48Mf/jBPfepTGR8fp6Ojg127drFixYrSpUnFlG5BkSRJepzSE7VJUtsbHBzkTW96E93d3XuXjYyMsG7dOq/iUduyBUWSCnOQrPR4BhRJKsxBstLjGVAkqbC+vj4GBgb2uZvxwMAAfX19pUuTinEMiiQV5r14pMczoEhSBfT09BhIpAZ28UiSpMoxoEiSpMqxi0eSmiAiWn7MTG/4rtnDgCJJTfBEw0JEGDQk7OKRJEkVZECRJEmVY0CRJEmVY0CRJEmVY0CRJEmVY0CRJEmVY0CRJEmVY0CRJEmVY0CRJEmVY0CRJEmVY0CRJEmVY0CRJEmVY0CRJEmVY0CRJEmVY0CRJEmVY0CRJEmVY0CRJEmVY0CRJEmVY0CRJEmVY0CRJEmVY0CRJEmVY0CRJEmVUzygRERHRHw9Iv6pdC2SJKkaigcU4GLg7tJFSJKk6igaUCLiJODVwEdK1iFJkqqldAvKFcDvAY8VrkOSJFVIsYASEa8BfpCZQ9Nsd1FEbIyIjWNjYy2qTpIklVSyBeWlQH9E3AusBZZHxJrJG2XmVZnZm5m9nZ2dra5RkiQVUCygZOY7M/OkzFwMnAusz8wVpeqRJEnVUXoMiiRJ0uMcVboAgMz8AvCFwmVIkqSKqERAkfTvhoeHGRwcZGxsjM7OTvr6+ujp6SldliS1lAFFqpDh4WHWr19Pf38/XV1djI6OMjAwAGBIkdRWHIMiVcjg4CD9/f10d3fT0dFBd3c3/f39DA4Oli5NklrKgCJVyNjYGF1dXfss6+rqwjmAJLUbA4pUIZ2dnYyOju6zbHR0FOcAktRuDChShfT19TEwMMDIyAjj4+OMjIwwMDBAX19f6dIkqaUcJCtVyMRA2HXr1u29imf58uUOkJXUdgwoUsX09PQYSCS1Pbt4JElS5RhQJElS5djFI1WMM8lKkgFFqhRnkpWkGrt4pApxJllJqrEFRaqQsbExfvzjH7Nq1aq9XTwveclLnElWUtsxoEgV8sgjj3DjjTdywQUX7O3iufrqqxkfHy9dmiS1lF08kiSpcmxBkSrk6KOP5pRTTuGd73wnW7duZdGiRZx55pncfffdpUuTpJYyoEgV8sgjj3DnnXfy53/+53bxSGprdvFIkqTKsQVFqpCjjz6aM844Y5+bBZ555pl8+tOfLl2aJLWUAUWqkM7OTo4//nje/OY37102MjJCZ2dnwaokqfUMKFKTRcRh2c9b3vKWGW+bmYflmJJUimNQpCbLzIN6bNq0iQ996EMAfOhDH2LTpk0HvQ9JOtLFkfSPWW9vb27cuLF0GVJLRIRhow153tVOImIoM3unWmcLiiRJqhwDiiRJqhwDiiRJqhwDiiRJqhwDiiRJqhwDiiRJqhwDiiRJqhwDiiRJqhwDiiRJqhwDiiRJqpxiASUinh0Rn4+IuyPiroi4uFQtkiSpWkrezfhR4Hcz87aIOBYYiojPZuY3CtYkSZIqoFgLSmZuyczb6s93AncDzypVjyRJqo5KjEGJiMXAzwFfnWLdRRGxMSI2jo2Ntbw2SZLUesUDSkTMA24AVmbmg5PXZ+ZVmdmbmb2dnZ2tL1CSJLVc0YASEXOohZNrM/PGkrVIkqTqmPEg2Yh4FvCcxvdk5hef6IEjIoCPAndn5l8/0f1IkqTZZ0YBJSL+EjgH+AYwXl+cwBMOKMBLgdcBwxFxe33ZJZl58yHsU5IkzQIzbUF5LfDczNxzuA6cmV8C4nDtT5IkzR4zHYOyGZjTzEIkSZImzLQFZTdwe0R8DtjbipKZb2tKVZIkqa3NNKAM1B+SJElNN6OAkpmfiIijgSX1Rd/KzJ82ryxJktTO9htQIuKEzPxR/fnLgU8A91Ib2PrsiHj9oVxmLEmStD8HakE5KyJ2Z+bfAe8HfjUzvwUQEUuAvwNOa0GNkiSpzew3oGTmRyPi9+sv50yEk/q6b9dngVWTDQ8PMzg4yNjYGJ2dnfT19dHT01O6LEmSmuqAlxln5l/Wn26MiI9GxMvrj78FhppfXnsbHh5mzZo1bNu2jccee4xt27axZs0ahoeHS5cmSVJTzfQqnt8B3gK8jdoYlC8Cq5pVlGrWrl3L5s2befjhh3nooYeYN28exxxzDGvXrrUVRWqRBQsWsGPHjpYes3YnkNaYP38+27dvb9nxpJmKzCxdw4z19vbmxo0bS5fRMqeddhpPe9rTePe7382yZcvYsGED73nPe/jhD3/I0JANWLNdRHAk/f85W8328zDbP5+qLSKGMrN3qnUH7OKJiE/Vfw5HxKbJj2YUq3+3ZcsWLrjgAk4//XSOPvpoTj/9dC644AK2bNlSujRJkppqui6ei+s/X9PsQvR4c+bMYXh4mGXLltHV1cXo6CjDw8PMmeP4ZEnS7DajLp6I6Aa2ZOZP6q+PARZl5r3NLW9fs6GLp5V9yxNsvj0y2fReDbP9PMz2z6dqe8JdPA2uAx5reD1eX6aDlJkzflx//fUsX76clStXArBy5UqWL1/O9ddff1D7kSTpSDPTq3iOysxHJl5k5iP1qe/VRGeddRYA111Xy4JbtmzhzW9+897lkiTNVjPt4vks8MHMHKi/PgN4W2a+osn17WM2dPE8UTbDth/PeTXM9vMw2z+fqu1AXTwzbUF5E3BtRPxvavOg3A+cf5jqkyRJ2sdM72Z8D7AsIuZRa3XZ2dyyJElSO5tRQImIP5z0GoDM/OMm1CRJktrcTLt4djU8fzK1eVHuPvzlSJLUHrwZ7IHNtIvn/Y2vI+J9wEBTKpIkaZYbHh5m/fr19Pf3752Ic2Cg9mvVkFIz03lQJnsKcPLhLESqugULFhARLXsALT1eRLBgwYLC37LUHgYHB+nv76e7u5uOjg66u7vp7+9ncHCwdGmVMdMxKMPAxHVoHUAn4PgTtZUdO3bM+ssxS8x0LLWjsbExurq69lnW1dXF2NhYoYqqZ6ZjUBrvxfMosDUzH21CPZIkzXqdnZ2Mjo7S3d29d9no6CidnZ0Fq6qWGXXxZOZ9wNOAM4AzATvIJEl6gvr6+hgYGGBkZITx8XFGRkYYGBigr6+vdGmVcTCXGZ8N3Fhf9PGIuC4z39O0yiRJmqUmBsKuW7du71U8y5cvd4Bsg5lOdX838HOT7mZ8W2ae0uT69uFU97N7/EPVtcM5aIfPeLBm+3cy2z+fqu1w3M34Xmrzn0yYC9xziHVJkiRN6YBdPBHxQWpX7+wB7qrfNDCBXwG+1PzyJElSO5puDMpEf8oQ8A8Ny7/QlGokSZKYJqBk5idaVciRZMGCBezYsaPlx23lHBXz589n+/btLTueJEmNpuvi+VRm/tdJE7XtlZkvaFplFeaEXZIkNdd0XTwX13++5oBbSZIkHUbTdfFsiYgO4KOZ+crDffCIeBXwAWrT538kM//icB9DkqQq8m7GBzbtZcaZOQ7sjojjD+eB68HnQ8B/Bp4P/GZEPP9wHkOSpCoaHh5mzZo1bNu2jccee4xt27axZs0ahoeHS5dWGTO9F89PgOH6Zca7JhZm5tsO4dgvAv4tMzcDRMRaalPpf+MQ9ilJUuWtXbuWuXPn8rrXvY6uri5GR0e5+uqrWbt2ra0odTMNKJ+pPxod6ijRZwH3N7x+APjFQ9xnS+Slx8Flh7VBqXLy0uNKl1A5nndJh8vtt9/OBz7wgb03C+zu7ub888/n4osvnuad7WOmAeWEzPxA44KIONRvcarLRB4XeiLiIuAi4HG3pi4l/ujBtriKJy8rXUW1eN4lqXVmGlBeT20wa6M3TLHsYDwAPLvh9UnA9yZvlJlXAVdB7V48h3A8STpos73lzFazMpYuXcrq1au54IIL9nbxrF69mqVLl5YurTKmmwflN4HfArojYqBh1XHADw/x2P8K/MeI6Aa+C5xbP5YkVcZsbzmz1ayMc889lzVr1nDNNdcwPj5OR0cHe/bsYcWKFaVLq4zpWlC+DGwBFgLvb1i+E9h0KAfOzEcj4n8A/0ztMuOPZeZdh7JPSZJKORwTXL73ve89qO1nc3iebh6U+4D7IuKVwMOZ+VhELAGeBxzytVCZeTNw86HuR5Kk0p5oWIiIWR00nqhp50Gp+yLw5Ih4FvA54ALg480qSpIktbeZDpKNzNwdERcCH8zM90bE15tZWNXN9nvVzJ8/v3QJkqQ2NuOAEhEvBs4DLjzI9846JZribAKUJLWTmXbxrATeCfxDZt4VEScDn29aVZIkqa3NqBUkM28Fbm14vRk4lGnuJUmS9mu6eVCuyMyVEfGPTDHLa2b2N60ySZLUtqZrQbmm/vN9zS5EkiRpwnTzoAzVf94aEZ3152OtKEySJLWvAw6SjZrLImIb8E3g2xExFhF/2JryJElSO5qui2cl8FLgFzJzBKB+Bc//iYj/mZmXN7k+qVKc/0aSWmO6gHI+8CuZuW1iQWZujogVwC2AAUVto9Xz0Dj3jaR2Nt08KHMaw8mE+jiUOc0pSZIktbvpAsojT3CdJEnSEzZdF88LI+LBKZYH8OQm1CNJkjTtZcYdrSpEkiRpwkzvxSNJktQyBpSKGx4eZtWqVQCsWrWK4eHhwhVJktR8BpQKGx4e5vLLL+fmm28G4Oabb+byyy83pEiSZj0DSoVdeeWVDA0NsXv3bgB2797N0NAQV155ZeHKJElqrjiSJoLq7e3NjRs3li7jkJSYifRIOsf6d07UVg2z/TzM9s93JGjncxARQ5nZO9U6W1BaLDNn/Jg3bx7XXHPNPsuuueYa5s2bd1D7kSTpSGNAqbA5c+bw5S9/mZGREcbHxxkZGeHLX/4yc+Y4ia8kaXabbqI2FXTKKaewYcMGrrzySo499lh27tzJhg0bOOWUU0qXJklSU9mCUmFvf/vbmTNnDnfccQe33nord9xxB3PmzOHtb3976dIkSWoqW1Aq7KyzzgLguuuuY+vWrTz96U/n7LPP3rtckqTZyqt4pIpq55H9VVLiyrtWmj9/Ptu3by9dRltr5//XD3QVjy0oknQArf7F0c6/rKRGBhRJkhosWLCAHTt2tPSYrW6pOxJazgwokiQ12LFjx6xvxToSui69ikeSJFWOAUWSJFWOAUWSJFWOAUWSJFWOAUWSJFVOkYASEX8VEd+MiE0R8Q8RcUKJOiRJUjWVakH5LHBqZr4A+DbwzkJ1SJKkCioSUDLzlsx8tP5yA3BSiTokSVI1VWGitt8G/n5/KyPiIuAigK6urlbVJElqU3npcXDZ8aXLaKq89LjSJUyraTcLjIh/AZ4xxap3Zean69u8C+gFzswZFOLNAtVOvCdLe/K8l9cO56Aqn7HIzQIz85UHWh8RrwdeA7xiJuFEkiS1jyJdPBHxKuD3gV/KzN0lapAkSdVV6iqe/w0cC3w2Im6PiA8XqkOSJFVQkRaUzPyZEseVJElHBmeSlSRJlWNAkSRJlWNAkSRJlWNAkSRJlVOFmWQlSaqUiChdQlPNnz+/dAnTMqBIktSg1XOHVmVW16qxi0eSJFWOAUWSJFWOXTxSkx1KX/YTfa/NxZKOdLagSE2WmTN+bNq0iSuuuILNmzfz6KOPsnnzZq644go2bdp0UPuRpCOdAUWqkMHBQfr7++nu7qajo4Pu7m76+/sZHBwsXZoktZQBRaqQsbExurq69lnW1dXF2NhYoYokqQwDilQhnZ2djI6O7rNsdHSUzs7OQhVJUhkGFKlC+vr6GBgYYGRkhPHxcUZGRhgYGKCvr690aZLUUl7FI1VIT08PAOvWrWNsbIzOzk6WL1++d7kktQsDilQxPT09BhJJbc8uHkmSVDkGFEmSVDkGFEmSVDkGFEmSVDkGFEmSVDkGFEmSVDkGFEmSVDkGFEmSVDkGFEmSVDkGFEmSVDkGFEmSVDkGFEmSVDkGFEmSVDkGFEmSVDkGFEmSVDkGFEmSVDkGFEmSCrjhhhs499xzATj33HO54YYbCldULUUDSkS8IyIyIhaWrEOSpFa64YYbWLVqFSeeeCIAJ554IqtWrTKkNIjMLHPgiGcDHwGeB5yWmdume09vb29u3Lix6bVJUikRQal/l3VoIqLlxzzS/1uJiKHM7J1qXckWlMuB3wOO7G9XkiRqYWGmj4ULF7JkyRLWrl3L7t27Wbt2LUuWLGHhwoUHtZ/ZrEhAiYh+4LuZeUeJ40uSVNJDDz3EOeecwznnnMMxxxyz9/lDDz1UurTKOKpZO46IfwGeMcWqdwGXAL86w/1cBFwE0NXVddjqkySplKOOOopbbrmFnTt3cuyxx7Jz506+8pWvcNRRTfu1fMRp+RiUiOgBPgfsri86Cfge8KLM/P6B3usYFEmznWNQ2sPJJ5/MT3/6U4455hj27NnD3Llzefjhh5kzZw6bN28uXV7LVGoMSmYOZ+bTM3NxZi4GHgB+frpwIknSbHHyySeza9culi5dyooVK1i6dCm7du3i5JNPLl1aZTgPiiRJLTZ37lze+MY3cs899/DJT36Se+65hze+8Y3MnTu3dGmVUTyg1FtSpr3EWJKk2eTCCy9kaGiIkZERhoaGuPDCC0uXVCnFA4okSe1m6dKlrF69mpGREcbHxxkZGWH16tUsXbq0dGmV4XBhSZJa7Nxzz2XNmjVcc801jI+P09HRwZ49e1ixYkXp0irDFhRJklqsp6eHFStWsHDhQp70pCexcOFCVqxYQU9PT+nSKsMWFEmSCujp6TGQHIAtKJIkqXIMKJIkqXIMKJIkqXIMKJIkqXIMKJIkqXIMKJIkqXIMKJIkqXIMKJIkqXIMKJIkqXIMKJIkqXIMKJIkqXIMKJIkqXIMKJIkqXIMKJIkqXIMKJIkqXIMKJIkqXIMKJIkqXIMKJIkqXIMKJIkqXIMKJIkqXKOKl2AJM1GEdHy92bmEz6mVDUGFElqAsOCdGjs4pEkSZVjQJEkSZVjQJEkSZVjQJEkSZVjQJEkSZXjVTySVAE33HAD1113HVu3bmXRokWcffbZnHXWWaXLkooxoEhSYTfccAPXXnstK1euZNmyZWzYsIErrrgCwJCitmVAkaTCrrvuOlauXMnpp58OsPfnqlWrDChqW8XGoETEWyPiWxFxV0S8t1QdklTa1q1bWbZs2T7Lli1bxtatWwtVJJVXJKBExC8DZwAvyMyfBd5Xog5JqoJFixaxYcOGfZZt2LCBRYsWFapIKq9UF8/vAH+RmXsAMvMHheqQpOLOPvvsvWNOGsegnHfeeWULkwoqFVCWAH0R8afAT4B3ZOa/FqpFkoqaGGeyatUqLr30UhYtWsR5553n+BO1tWjWDa0i4l+AZ0yx6l3AnwLrgYuBXwD+Hjg5pygmIi4CLgLo6uo67b777mtKvZIkqbUiYigze6da17QWlMx85QEK+h3gxnog+VpEPAYsBMam2M9VwFUAvb293h5UkqQ2UOoqnpuA5QARsQQ4GthWqBZJklQxpcagfAz4WETcCTwCvH6q7h1JktSeigSUzHwEWFHi2JIkqfqcSVaSKmB4eJjBwUHGxsbo7Oykr6+Pnp6e0mVJxRhQJKmw4eFh1q9fT39/P11dXYyOjjIwMABgSFHbKjbVvSSpZnBwkP7+frq7u+no6KC7u5v+/n4GBwdLlyYVY0CRpMLGxsbo6uraZ1lXVxdjY4+beUFqGwYUSSqss7OT0dHRfZaNjo7S2dlZqCKpPAOKJBXW19fHwMAAIyMjjI+PMzIywsDAAH19faVLk4pxkKwkFTYxEHbdunV7r+JZvny5A2TV1gwoklQBPT09BhKpgV08kiSpcgwokiSpcgwokiSpcgwokiSpcgwokiSpcgwokiSpcgwokiSpcgwokiSpcgwokiSpcgwokiSpciIzS9cwYxExBtxXuo5CFgLbShehlvKctyfPe/tp53P+nMyc8rbdR1RAaWcRsTEze0vXodbxnLcnz3v78ZxPzS4eSZJUOQYUSZJUOQaUI8dVpQtQy3nO25Pnvf14zqfgGBRJklQ5tqBIkqTKMaBUXER8LCJ+EBF3lq5FrRERz46Iz0fE3RFxV0RcXLomNVdEPDkivhYRd9TP+R+VrkmtExEdEfH1iPin0rVUiQGl+j4OvKp0EWqpR4HfzcxTgGXAWyLi+YVrUnPtAZZn5guBpcCrImJZ2ZLUQhcDd5cuomoMKBWXmV8EtpeuQ62TmVsy87b6853U/uF6Vtmq1ExZ81D95Zz6wwGCbSAiTgJeDXykdC1VY0CRKiwiFgM/B3y1cClqsnoz/+3AD4DPZqbnvD1cAfwe8FjhOirHgCJVVETMA24AVmbmg6XrUXNl5nhmLgVOAl4UEacWLklNFhGvAX6QmUOla6kiA4pUQRExh1o4uTYzbyxdj1onM38EfAHHnrWDlwL9EXEvsBZYHhFrypZUHQYUqWIiIoCPAndn5l+XrkfNFxGdEXFC/fkxwCuBbxYtSk2Xme/MzJMyczFwLrA+M1cULqsyDCgVFxF/B3wFeG5EPBARF5auSU33UuB11P6aur3++LXSRampTgQ+HxGbgH+lNgbFS07V1pxJVpIkVY4tKJIkqXIMKJIkqXIMKJIkqXIMKJIkqXIMKJIkqXIMKJIkqXIMKJIOm4i4LCLe0cT9vyEintnw+t6IWNis40kqx4Ai6UjyBuCZ0200ExFx1OHYj6TmMKBI2q+IWBwR34yIT0TEpoi4PiKe0thyERG9EfGFhre9MCLWR8R3IuK/H2Df8yLicxFxW0QMR8QZDce8s2G7d9RbZn4D6AWurc+ue0x9k7c27ON59fcsiIib6jVviIgX1JdfFhFXRcQtwOqI+NmI+Fp9f5si4j8ezu9P0hNnQJE0necCV2XmC4AHgTdPs/0LgFcDLwb+sLFLZpKfAP8lM38e+GXg/fX7EE0pM68HNgLnZebSzHy4vmpbfR//B5joXvoj4Ov1mi8BVjfs6jTgjMz8LeBNwAfqdxHuBR6Y5rNJahEDiqTp3J+Z/6/+fA3wsmm2/3RmPpyZ24DPAy/az3YB/Fn9/jP/AjwLWPQE6pu42/MQsLj+/GXANQCZuR54WkQcX1830BBuvgJcEhG/DzynYbmkwgwokqYz+YZdCTzKv//78eQZbD+V84BO4LR6C8bW+r4a9z3V/ifbU/85DkyMK5mqJWaijl17F2R+EugHHgb+OSKWT3MsSS1iQJE0na6IeHH9+W8CXwLupdZVAnDWpO3PiIgnR8TTgJdTuzvvVI4HfpCZP42IXwaeU1++FXh6RDwtIuYCr2l4z07g2BnU/EVqAYiIeDm1bqAHJ28UEScDmzPzSmCAWveUpApwFLuk6dwNvD4i/gb4DrWxHl8DPhoRlwBfnbT914DPAF3An2Tm9/az32uBf4yIjcDtwDcB6oHlj+v7HZlYXvdx4MMR8TC1MS77cxlwdb37aDfw+v1sdw6wIiJ+Cnwf+OMD7FNSC0Xm/lpfJbW7iFgM/FNmnlq6FkntxS4eSZJUObagSGqqiOihfkVNgz2Z+Ysl6pF0ZDCgSJKkyrGLR5IkVY4BRZIkVY4BRZIkVY4BRZIkVY4BRZIkVc7/B63mGMQjoi7dAAAAAElFTkSuQmCC\n",
      "text/plain": [
       "<Figure size 648x432 with 1 Axes>"
      ]
     },
     "metadata": {
      "needs_background": "light"
     },
     "output_type": "display_data"
    },
    {
     "data": {
      "image/png": "iVBORw0KGgoAAAANSUhEUgAAA7cAAAKvCAYAAACxljhBAAAAOXRFWHRTb2Z0d2FyZQBNYXRwbG90bGliIHZlcnNpb24zLjMuNCwgaHR0cHM6Ly9tYXRwbG90bGliLm9yZy8QVMy6AAAACXBIWXMAAAsTAAALEwEAmpwYAAAxP0lEQVR4nO3de7ReZXk36t9NEEkCKm7QsgEBNX4IEg4NEAQFQTkXBESh2+KhyoYCVqt+YusnjGHpoKK29VQaDwXGVg5DAVGxVLBosR+VgBggiiZAJYLISYQKarKe/UcW+ZZh5QBrZb1rznVdY8yx3nl453zeZI4sftz3+8xqrQUAAAC6bL1BDwAAAADGSrgFAACg84RbAAAAOk+4BQAAoPOEWwAAADpPuAUAAKDzhFsAAAAmVFV9vqp+UVW3rGJ/VdXHq2pRVS2oql3XdE7hFgAAgIl2bpKDVrP/4CSzhpcTkvzjmk4o3AIAADChWmvfSfLgag45Isn5bbnrkjynqjZf3TnXH88BjtWmm27attlmm0EPAwAAmIJuuOGG+1trmw16HGN14KtmtgceXDbQMdyw4De3Jnl8xKZ5rbV5T+EUWyS5a8T6kuFt96zqDZMq3G6zzTaZP3/+oIcBAABMQVX1X4Mew3h44MFl+d6VLxjoGKZt/pPHW2tzxnCKGmVbW90btCUDAAAw2SxJstWI9S2T3L26Nwi3AAAATDaXJzl+eNbkuUkebq2tsiU5mWRtyQAAAIxNSzKUoUEPY7Wq6oIk+ybZtKqWJDk9yTOSpLV2TpIrkhySZFGSXyd5y5rOKdwCAAAwoVprx61hf0ty8lM5p3ALAADQKy3L2uSu3K4LvnMLAABA5wm3AAAAdJ62ZAAAgB5ZPqHUah8J20sqtwAAAHSeyi0AAEDPTPZHAa0LKrcAAAB0nnALAABA52lLBgAA6JGWlmXNhFIAAADQOSq3AAAAPeNRQAAAANBBwi0AAACdpy0ZAACgR1qSZdqSAQAAoHtUbgEAAHrGhFIAAADQQcItAAAAnactGQAAoEdakmVNWzIAAAB0jsotAABAzwwNegADoHILAABA5wm3AAAAdJ62ZAAAgB5paVnmObcAAADQPcItAAAAnactGQAAoE9asmzqdSWr3AIAANB9KrcAAAA90uI5twAAANBJwi0AAACdpy0ZAACgVyrLUoMexIRTuQUAAKDzVG4BAAB6pCUZ8iggAAAA6B7hFgAAgM7TlgwAANAzJpQCAACADlK5BQAA6JEWlVsAAADoJOEWAACAztOWDAAA0DNDTVsyAAAAdI7KLQAAQI+YUAoAAAA6SrgFAACg87QlAwAA9EhLZdkUrGNOvU8MAABA7wi3AEwJP/zhD/PqV786t9xyy6CHAgDr3FCrgS6DINwCMCVssMEGufrqq7PnnnvmVa96Vb70pS/ld7/73aCHBQCME+EWgCnhRS96UV74whdmxowZ2WmnnfKJT3wihx9++KCHBQCMExNKATBlvOY1r8lGG22U888/P5dddln22GOPQQ8JAMad59wCQM+96lWvyo9//OOcd955Oeqoo3LHHXcMekgAwDgRbgGYMg488MDstddeOfjgg3PGGWfkkEMOyQMPPDDoYQEA40BbMgBTxnOe85y8733vS5KceOKJuf3223PkkUfmm9/8Zp75zGcOeHQAMF4qy9rUq2NOvU8MAMPOOuusPP/5z89b3/rWtNYGPRwAYAyEWwCmrPXWWy/nn39+br/99nzwgx8c9HAAYFy0JENZb6DLIAi3AExp06dPz1e+8pV88YtfzD//8z8PejgAwNPkO7cATHnPe97z8vWvfz377LNPXvCCF2T//fcf9JAAgKdI5RYAkmy33Xa5+OKLc9xxx2XhwoWDHg4AjMmy1ECXQRBuAWDYPvvsk4997GM59NBD8/Of/3zQwwEAngJtyQAwwhvf+MYsXrw4hx9+eK655prMmDFj0EMCgKekNY8CAgCSfPCDH8xLX/rS/PEf/3GWLVs26OEAAGtBuAWAlVRVPvOZz+Thhx/Oe9/73kEPBwBYC9qSAWAUG2ywQS655JK8/OUvz4te9KKcfPLJgx4SAKy1oQFN6jRIwi0ArMImm2ySr3/969l7772z9dZb57DDDhv0kACAVRBuAWA1XvjCF+bSSy/NYYcdliuvvDK77rrroIcEAKvVkiybgt9AnXqfGACeoj322CPnnHNODj/88Nx1112DHg4AMAqVWwBYC0cffXTuuOOOHHroobn22mvzrGc9a9BDAgBGEG4BYC29+93vzuLFi/P6178+X/3qV/OMZzxj0EMCgFF4zi0AsBpVlU984hNZb731csopp6S1NughAQDDhFsAeArWX3/9XHTRRfnP//zPnH322YMeDgA8SUsylPUGugyCtmQAeIo23njjfO1rX8uee+6ZbbfdNsccc8yghwQAU55wCwBPw5ZbbpmvfvWrOeCAA7LFFlvk5S9/+aCHBABTmrZkAHiadt5555x77rk5+uijs3jx4kEPBwBWWNZqoMsgCLcAMAaHHHJIPvjBD+aQQw7JAw88MOjhAMCUJdwCwBiddNJJOfroo/PZz3520EMBgCnLd24BYBz8zd/8zaCHAABJkpbKsilYx5x6nxgAAIDeUbkFAADomaE29eqY4/aJq2paVX2/qr42vP7cqvpmVf1k+Ocm43UtAAAAGGk84/yfJ/nhiPXTklzdWpuV5OrhdQDorGnTpmXnnXdesdx5552DHhIAMGxc2pKrasskhyY5M8lfDG8+Ism+w6/PS3JNkveNx/UAYBCmT5+em266adR9rbW01rLeelOvDQyAyaUlJpQag79P8j+TDI3Y9vzW2j1JMvzzeaO9sapOqKr5VTX/vvvuG6fhAMC6d+edd+alL31p/uzP/iy77rpr7rrrrpx00kmZM2dOdthhh5x++ukrjt1mm21y+umnZ9ddd82OO+6YH/3oR0mSRx99NG95y1uy4447Zvbs2fnyl7+cJPnXf/3X7Lnnntl1111zzDHH5NFHHx3IZwSArhhzuK2qw5L8orV2w9N5f2ttXmttTmttzmabbTbW4QDAOvPYY4+taEk+8sgjkyS33XZbjj/++Hz/+9/P1ltvnTPPPDPz58/PggUL8u1vfzsLFixY8f5NN900N954Y0466aR85CMfSZJ86EMfyrOf/ezcfPPNWbBgQfbbb7/cf//9+eu//utcddVVufHGGzNnzpx87GMfG8hnBqB7WirL2mCXQRiPtuS9khxeVYck2TDJs6rq/0tyb1Vt3lq7p6o2T/KLcbgWAAzMym3Jd955Z7beeuvMnTt3xbaLL7448+bNy9KlS3PPPfdk4cKFmT17dpLkqKOOSpL84R/+YS655JIkyVVXXZULL7xwxfs32WSTfO1rX8vChQuz1157JUl++9vfZs8991zXHw8AOm3M4ba19v4k70+Sqto3yXtaa2+sqrOTvCnJWcM/vzLWawHAZDNz5swVr++444585CMfyfXXX59NNtkkb37zm/P444+v2P/MZz4zyfKJqZYuXZpk+Xd1q37//3C31vKa17wmF1xwwQR8AgDoh3X5LeOzkrymqn6S5DXD6wDQW7/61a8yc+bMPPvZz869996bb3zjG2t8zwEHHJBPfvKTK9YfeuihzJ07N9/97nezaNGiJMmvf/3r/PjHP15n4wagf4ay3kCXQRjXq7bWrmmtHTb8+oHW2v6ttVnDPx8cz2sBwGSz0047ZZdddskOO+yQt771rSvailfnAx/4QB566KG87GUvy0477ZR/+7d/y2abbZZzzz03xx13XGbPnp25c+eumIAKABhdtdYGPYYV5syZ0+bPnz/oYQAAAFNQVd3QWpsz6HGM1TYv26h98JKdBzqGP/0f353wP8up9/AjAAAAeke4BQAAoPPG41FAAAAATBqVoQzmWbODpHILAABA56ncAgAA9EhLsqxNvTrm1PvEAAAA9I5wCwAAQOdpSwYAAOiZZVOwjjn1PjEAAAC9o3ILAADQIy2VoeZRQAAAANA5wi0AAACdpy0ZAACgZ0woBQAAAB0k3AIAANB52pIBAAB6pCUZalOvjjn1PjEAAAC9o3ILAADQK5Vl8ZxbAAAA6BzhFgAAgM7TlgwAANAjJpQCAACAjlK5BQAA6BkTSgEAAEAHCbcAAAB0nrZkAACAHmmtTCgFAAAAXaRyCwAA0DPLVG4BAACge4RbAAAAOk9bMgAAQI+0JEOecwsAAADdI9wCAAD0SmVZW2+gyxpHWHVQVd1WVYuq6rRR9j+7qr5aVT+oqlur6i1rOqdwCwAAwISpqmlJPpXk4CTbJzmuqrZf6bCTkyxsre2UZN8kH62qDVZ3XuEWAACAibR7kkWttdtba79NcmGSI1Y6piXZuKoqyUZJHkyydHUnNaEUAABAj7QkQ23gE0ptWlXzR6zPa63NG369RZK7RuxbkmSPld7/ySSXJ7k7ycZJ3tBaG1rdBYVbAAAAxtv9rbU5q9g3WvJuK60fmOSmJPsleVGSb1bVv7fWfrWqCwq3AAAAPbNscn8DdUmSrUasb5nlFdqR3pLkrNZaS7Koqu5Isl2S763qpJP6EwMAANA71yeZVVXbDk8SdWyWtyCP9NMk+ydJVT0/yf9IcvvqTqpyCwAAwIRprS2tqlOSXJlkWpLPt9ZuraoTh/efk+RDSc6tqpuzvI35fa21+1d3XuEWAACgR1pqMkwotVqttSuSXLHStnNGvL47yQFP5ZzakgEAAOg84RYAAIDO05YMAADQM0NTsI459T4xAAAAvaNyCwAA0COtJcsm+YRS64LKLQAAAJ0n3AIAANB52pIBAAB6ZrI/53ZdULkFAACg81RuAQAAeqSlMtSmXh1z6n1iAAAAeke4BQAAoPO0JQMAAPTMsphQCgAAADpH5RYAAKBHWjwKCAAAADpJuAUAAKDztCUDAAD0iufcAgAAQCep3AIAAPTMkEcBAQAAQPcItwAAAHSetmQAAIAeaS1Z5jm3AAAA0D3CLQAAAJ2nLRkAAKBnPOcWAAAAOkjlFgAAoEdaKkMmlAIAAIDuEW4BAADoPG3JAAAAPTMUbckAAADQOSq3AAAAPdISE0oBAABAFwm3AAAAdJ62ZAAAgJ4ZalOvjjn1PjEAAAC9o3ILAADQJ61MKAUAAABdJNwCAADQedqSAQAAeqQlGYq2ZAAAAOgclVsAAICeMaEUAAAAdJBwCwAAQOdpSwYAAOiRFm3JAAAA0EkqtwAAAD2jcgsAAAAdJNwCAADQedqSAQAAeqSltCUDAABAFwm3AAAAdJ62ZAAAgJ4ZirZkAAAA6ByVWwAAgD5pnnMLAAAAnSTcAgAA0HnakgEAAHqkRVsyAAAAdJLKLQAAQM+o3AIAAEAHCbcAAAB0nrZkAACAHmkpbckAAADQRSq3AAAAPdNUbgEAAKB7hFsAAAA6b8zhtqq2qqp/q6ofVtWtVfXnw9ufW1XfrKqfDP/cZOzDBQAAYE2GUgNdBmE8KrdLk7y7tfbSJHOTnFxV2yc5LcnVrbVZSa4eXgcAAIBxN+YJpVpr9yS5Z/j1I1X1wyRbJDkiyb7Dh52X5Jok7xvr9QAAAFi11uJRQGNVVdsk2SXJfyZ5/nDwfSIAP28V7zmhquZX1fz77rtvPIcDAADAFDFu4baqNkry5STvbK39am3f11qb11qb01qbs9lmm43XcAAAAJhCxuU5t1X1jCwPtl9orV0yvPneqtq8tXZPVW2e5BfjcS0AAABWz3Nun4aqqiSfS/LD1trHRuy6PMmbhl+/KclXxnotAAAAGM14VG73SvInSW6uqpuGt/1lkrOSXFxVf5rkp0mOGYdrAQAAsFo1JSeUGo/Zkq9NVvkgo/3Hen4AAABYk3GdLRkAAAAGYVwmlAIAAGDyMKEUAAAAdJBwCwAAQOdpSwYAAOiRlkzJ2ZJVbgEAAOg8lVsAAIA+aUlrgx7ExFO5BQAAoPOEWwAAADpPWzIAAEDPDMWEUgAAANA5KrcAAAA90pI0jwICAACA7hFuAQAA6DxtyQAAAL1SGdKWDAAAAN2jcgsAANAzrQ16BBNP5RYAAIDOE24BAADoPG3JAAAAPeM5twAAANBBKrcAAAA90prKLQAAAHSScAsAAEDnaUsGAADomSFtyQAAANA9wi0AAACdpy0ZAACgZ1ob9AgmnsotAAAAnadyCwAA0DOecwsAAAAdJNwCAADQedqSAQAAeqSltCUDAABAF6ncAgAA9MwUfBKQyi0AAADdJ9wCAADQedqSAQAA+qR5zi0AAAB0ksotAABA30zBGaVUbgEAAOg84RYAAIDO05YMAADQMyaUAgAAgA4SbgEAAHqmtcEua1JVB1XVbVW1qKpOW8Ux+1bVTVV1a1V9e03n1JYMAADAhKmqaUk+leQ1SZYkub6qLm+tLRxxzHOSfDrJQa21n1bV89Z0XpVbAAAAJtLuSRa11m5vrf02yYVJjljpmD9Ocklr7adJ0lr7xZpOqnILAADQIy2TYkKpTatq/oj1ea21ecOvt0hy14h9S5LssdL7X5LkGVV1TZKNk/xDa+381V1QuAUAAGC83d9am7OKfaMl75W/qbt+kj9Msn+S6Un+d1Vd11r78aouKNwCAAD0SUsy+Mrt6ixJstWI9S2T3D3KMfe31v47yX9X1XeS7JRkleHWd24BAACYSNcnmVVV21bVBkmOTXL5Ssd8Jckrqmr9qpqR5W3LP1zdSVVuAQAAmDCttaVVdUqSK5NMS/L51tqtVXXi8P5zWms/rKp/SbIgyVCSz7bWblndeYVbAACAnlmbZ80OUmvtiiRXrLTtnJXWz05y9tqeU1syAAAAnSfcAgAA0HnakgEAAPpmkrclrwsqtwAAAHSeyi0AAECvVNrkfs7tOqFyCwAAQOcJtwAAAHSetmQAAIC+MaEUAAAAdI/KLQAAQJ+0mFAKAAAAuki4BQAAoPO0JQMAAPSNCaUAAACge1RuAQAAeseEUgAAANA5wi0AAACdpy0ZAACgb0woBQAAAN2jcgsAANA3KrcAAADQPcItAAAAnactGQAAoE9akuY5twAAANA5KrcAAAA900woBQAAAN0j3AIAANB52pIBAAD6RlsyAAAAdI9wCwAAQOdpSwYAAOgbz7kFAACA7lG5BQAA6JkyoRQAAAB0j3ALAABA52lLBgAA6JMWz7kFAACALlK5BQAA6JXyKCAAAADoIuEWAACAztOWDAAA0DcmlAIAAIDuUbkFAADoG5VbAAAA6B7hFgAAgM7TlgwAANA32pIBAACge1RuAQAA+qQlaTXoUUw4lVsAAAA6T7gFAACg87QlAwAA9EyZUAoAAAC6R+UWAACgb1RuAQAAoHuEWwAAADpPuAUAAKDz1nm4raqDquq2qlpUVaet6+sBAAAw9azTcFtV05J8KsnBSbZPclxVbb8urwkAAMDUs65nS949yaLW2u1JUlUXJjkiycJ1fF0AAIApy3Nux98WSe4asb5keBsAAACMm3Vdua1Rtv3e/0OoqhOSnJAkL3jBC9bxcAAAAKaANloU67d1XbldkmSrEetbJrl75AGttXmttTmttTmbbbbZOh4OAAAAfbSuw+31SWZV1bZVtUGSY5Ncvo6vCQAAwBSzTsNta21pklOSXJnkh0kubq3dui6vyboxbdq07LzzziuWs846a51d65prrslhhx22zs4PAAC91ibBMgDr+ju3aa1dkeSKdX0d1q3p06fnpptuGvQwRrV06dKsv/46v5UBAIBJbF23JdNz119/fV7+8pdnp512yu67755HHnkk5557bk455ZQVxxx22GG55pprkiQnnXRS5syZkx122CGnn376imP+5V/+Jdttt1323nvvXHLJJSu2P/jgg3nta1+b2bNnZ+7cuVmwYEGS5IwzzsgJJ5yQAw44IMcff/zEfFgAAOgKlVsY3WOPPZadd955xfr73//+HHnkkXnDG96Qiy66KLvttlt+9atfZfr06as9z5lnnpnnPve5WbZsWfbff/8sWLAgL3nJS/L2t7893/rWt/LiF784b3jDG1Ycf/rpp2eXXXbJZZddlm9961s5/vjjV1SQb7jhhlx77bVrvCYAANB/wi1rZbS25Jtvvjmbb755dttttyTJs571rDWe5+KLL868efOydOnS3HPPPVm4cGGGhoay7bbbZtasWUmSN77xjZk3b16S5Nprr82Xv/zlJMl+++2XBx54IA8//HCS5PDDD5+Uwfa6667LbrvtlmnTpg16KAAAMGUItzxtrbVUPfn5Weuvv36GhoZWrD/++ONJkjvuuCMf+chHcv3112eTTTbJm9/85hX7RjvPE9dY2RPHzpw5c9T3XHvttTn55JMzc+bMbLzxxtloo42y0UYb/d7rlddH2zdz5syst95T69wfGhrK+9///vzyl7/MRz/60ey3335P6f0AADAeakCtwYMk3PK0bbfddrn77rtz/fXXZ7fddssjjzyS6dOnZ5tttsmnP/3pDA0N5Wc/+1m+973vJUl+9atfZebMmXn2s5+de++9N9/4xjey7777Zrvttssdd9yRxYsX50UvelEuuOCCFdd45StfmS984Qv5X//rf+Waa67JpptuusYK8dy5c3P++efn0UcfzaOPPppHHnlkxesn1u+7775V7n9i269//etsuOGGTwq+awrKJ554Yn7wgx/kjW98Y2bNmpW//du/zdy5c9fp3wUAAEx1wi1rZeXv3B500EE566yzctFFF+XUU0/NY489lunTp+eqq67KXnvtlW233TY77rhjXvayl2XXXXdNkuy0007ZZZddssMOO+SFL3xh9tprryTJhhtumHnz5uXQQw/Npptumr333ju33HJLkuUTR73lLW/J7NmzM2PGjJx33nlrHOv666+fnXbaacyfeWhoKI899tiTAvBoYfnRRx/Nz3/+8xWvH3744cyYMSP//u//nj/5kz/JT37ykzGPBwAA1toUrNzWaG2fgzJnzpw2f/78QQ8Dnrbf/va3OfPMM3POOedkzpw5ede73pX9999/lW3XAABMHlV1Q2ttzqDHMVbP3GqrtuU73zXQMdz+nndP+J+lyi2Mo4ceeiiPPfZYvv3tb2e77bYb9HAAAGDKEG5hHD3/+c/Phz/84UEPAwCAqW7yNOhOmKc2FSwAAABMQiq3AAAAPVJtaj4KSOUWAACAzhNuAQAA6DxtyQAAAH3Tpt6jKFVuAQAA6DzhFgAAgM7TlgwAANA3ZksGAACA7lG5BQAA6BnPuQUAAIAOEm4BAADoPG3JAAAAfaMtGQAAALpH5RYAAKBPmgmlAAAAoJOEWwAAADpPWzIAAEDfaEsGAACA7lG5BQAA6BuVWwAAAOge4RYAAIDO05YMAADQM55zCwAAAB0k3AIAANB5wi0AAACdJ9wCAADQeSaUAgAA6BsTSgEAAED3qNwCAAD0SfMoIAAAAOgk4RYAAIDO05YMAADQN9qSAQAAoHuEWwAAADpPWzIAAEDfaEsGAACA7lG5BQAA6JGK59wCAABAJwm3AAAAdJ62ZAAAgL7RlgwAAADdo3ILAADQJ82EUgAAANBJwi0AAACdpy0ZAACgb7QlAwAAQPeo3AIAAPSNyi0AAACsW1V1UFXdVlWLquq01Ry3W1Utq6rXremcwi0AAAATpqqmJflUkoOTbJ/kuKrafhXH/W2SK9fmvMItAABAz1Qb7LIGuydZ1Fq7vbX22yQXJjlilONOTfLlJL9Ym88s3AIAADDeNq2q+SOWE0bs2yLJXSPWlwxvW6GqtkhyZJJz1vaCJpQCAADom8FPKHV/a23OKvbVKNtWHvHfJ3lfa21Z1WiHP5lwCwAAwERakmSrEetbJrl7pWPmJLlwONhumuSQqlraWrtsVScVbgEAAJhI1yeZVVXbJvlZkmOT/PHIA1pr2z7xuqrOTfK11QXbRLgFAADol5bJ0Ja8Sq21pVV1SpbPgjwtyedba7dW1YnD+9f6e7YjCbcAAABMqNbaFUmuWGnbqKG2tfbmtTmncAsAANAza/E4nt7xKCAAAAA6T7gFAACg87QlAwAA9I22ZAAAAOge4RYAAIDO05YMAADQM2ZLBgAAgA5SuQUAAOgblVsAAADoHuEWAACAztOWDAAA0Cct2pIBAACgi1RuAQAAeqSGl6lG5RYAAIDOE24BAADoPG3JAAAAfWNCKQAAAOgelVsAAICeKZVbAAAA6B7hFgAAgM7TlgwAANA32pIBAACge1RuAQAA+kblFgAAALpHuAUAAKDztCUDAAD0SfOcWwAAAOgk4RYAAIDO05YMAADQN9qSAQAAoHtUbgEAAHrGhFIAAADQQcItAAAAnactGQAAoG+0JQMAAED3qNwCAAD0jAmlAAAAoIOEWwAAADpPWzIAAECftJhQCgAAALpI5RYAAKBvVG4BAACge4RbAAAAOm9M4baqzq6qH1XVgqq6tKqeM2Lf+6tqUVXdVlUHjnmkAAAArFFl+XNuB7kMwlgrt99M8rLW2uwkP07y/iSpqu2THJtkhyQHJfl0VU0b47UAAABgVGMKt621f22tLR1evS7JlsOvj0hyYWvtN621O5IsSrL7WK4FAADAWmoDXgZgPL9z+9Yk3xh+vUWSu0bsWzK87Umq6oSqml9V8++7775xHA4AAABTxRofBVRVVyX5g1F2/VVr7SvDx/xVkqVJvvDE20Y5ftT83lqbl2ReksyZM2cKTlgNAADAWK0x3LbWXr26/VX1piSHJdm/tfZEOF2SZKsRh22Z5O6nO0gAAADWXrWpVzcc62zJByV5X5LDW2u/HrHr8iTHVtUzq2rbJLOSfG8s1wIAAIBVWWPldg0+meSZSb5ZVUlyXWvtxNbarVV1cZKFWd6ufHJrbdkYrwUAAMCaDHBSp0EaU7htrb14NfvOTHLmWM4PAAAAa2M8Z0sGAADGyaWXXpqqyo9+9KOn9f7LLrssCxcufMrvO/fcc3PKKackSc4555ycf/75T+v6MNGEWwAAmIQuuOCC7L333rnwwguf1vtXF26XLl26Vuc48cQTc/zxxz+t6zNY1Qa7DIJwCwAAk8yjjz6a7373u/nc5z63Itxec801Oeyww1Ycc8opp+Tcc89Nkpx22mnZfvvtM3v27LznPe/Jf/zHf+Tyyy/Pe9/73uy8885ZvHhx9t133/zlX/5l9tlnn/zDP/xDvvrVr2aPPfbILrvskle/+tW59957nzSOM844Ix/5yEeSJJ/5zGey2267ZaeddsrRRx+dX//61086HgZprBNKAQAA4+yyyy7LQQcdlJe85CV57nOfmxtvvHGVxz744IO59NJL86Mf/ShVlV/+8pd5znOek8MPPzyHHXZYXve616049pe//GW+/e1vJ0keeuihXHfddamqfPazn82HP/zhfPSjH13ldY466qi8/e1vT5J84AMfyOc+97mceuqp4/SJYexUbgEAYJK54IILcuyxxyZJjj322FxwwQWrPPZZz3pWNtxww7ztbW/LJZdckhkzZqzy2De84Q0rXi9ZsiQHHnhgdtxxx5x99tm59dZbVzumW265Ja94xSuy44475gtf+MIaj2fA2oCXARBuAQBgEnnggQfyrW99K29729uyzTbb5Oyzz85FF12UadOmZWhoaMVxjz/+eJJk/fXXz/e+970cffTRKyq+qzJz5swVr0899dSccsopufnmm/NP//RPK863Km9+85vzyU9+MjfffHNOP/30NR4PE024BQCASeRLX/pSjj/++PzXf/1X7rzzztx1113ZdtttkyQLFy7Mb37zmzz88MO5+uqrkyz/fu7DDz+cQw45JH//93+fm266KUmy8cYb55FHHlnldR5++OFsscUWSZLzzjtvjeN65JFHsvnmm+d3v/tdvvCFL4zxU7KuTcUJpXznFgAAJpELLrggp5122u9tO/roo/PFL34xr3/96zN79uzMmjUru+yyS5LlofOII47I448/ntZa/u7v/i7J8nbmt7/97fn4xz+eL33pS0+6zhlnnJFjjjkmW2yxRebOnZs77rhjteP60Ic+lD322CNbb711dtxxx9UGZxiEam1AsXoUc+bMafPnzx/0MAAAgCmoqm5orc0Z9DjGauamW7UdDnvXQMdw/XnvnvA/S5VbAACAvpk8NcwJ4zu3AAAAdJ7KLQAAQJ8McFKnQVK5BQAAVuvrX/96jj766Nx4442DHgqsknALAACs1v7775999tknf/RHf5TXvva1+f73vz/oIcGTCLcAAMBqbbjhhnnHO96RRYsWZb/99suhhx4q5E52bcDLAAi3AADAWpk+fXre8Y53ZPHixXnVq16VQw89NEceeWRuuummQQ8NhFsAAOCpmT59ev78z/88ixcvzj777JNDDjlEyJ1EKssnlBrkMgjCLQAA8LRMnz4973znO7No0aLss88+Ofjgg3PUUUcJuQyEcAsAAIzJjBkz8s53vjOLFy/OK17xihUh9wc/+MGgh8YUItwCAADjYsaMGXnXu961IuQedNBBOfroo7NgwYJBD23qaW2wywAItwAAwLgaGXL32muvHHjggXnd6163IuTefffdeeUrX5n//u//HvBI6RPhFgAAWCdmzJiRv/iLv8jixYuz55575oADDsgxxxyT+++/P9tss01OPfXUQQ+xt0woBQAAMM5mzJiRd7/73Vm8eHHmzp2bAw44II888kiuueaanH/++YMeHj0h3AIAABNi5syZefGLX5wzzjgjG2ywQe6999689a1vzXe+851BD40eWH/QAwAAAKaO7373u7nvvvsyY8aMHHXUUbntttuyZMmSQQ+rX9rwMsUItwAAwIT58Ic/POgh0FPCLQAAQM/U0KBHMPF85xYAAIDOE24BAADoPG3JAAAAfTMFJ5RSuQUAAKDzhFsAAAA6T1syAABAz5S2ZAAAAOgelVsAAIA+aUna1CvdqtwCAADQecItAAAAnactGQAAoGdMKAUAAAAdpHILAADQNyq3AAAA0D3CLQAAAJ2nLRkAAKBHKiaUAgAAgE5SuQUAAOiT1pYvU4zKLQAAAJ0n3AIAANB52pIBAAB6xoRSAAAA0EEqtwAAAH2jcgsAAADdI9wCAADQedqSAQAAesaEUgAAANBBwi0AAACdpy0ZAACgT1qSoanXl6xyCwAAQOep3AIAAPTN1CvcqtwCAADQfcItAAAAnactGQAAoGc85xYAAAA6SOUWAACgb9rUK92q3AIAANB5wi0AAACdpy0ZAACgZ0woBQAAAB2kcgsAANAnbXiZYlRuAQAAmFBVdVBV3VZVi6rqtFH2/z9VtWB4+Y+q2mlN5xRuAQAAmDBVNS3Jp5IcnGT7JMdV1fYrHXZHkn1aa7OTfCjJvDWdV1syAABAj1SSmtzPud09yaLW2u1JUlUXJjkiycInDmit/ceI469LsuWaTqpyCwAAwHjbtKrmj1hOGLFviyR3jVhfMrxtVf40yTfWdEGVWwAAgL4ZGvQAcn9rbc4q9tUo20YtNVfVq7I83O69pgsKtwAAAEykJUm2GrG+ZZK7Vz6oqmYn+WySg1trD6zppNqSAQAAmEjXJ5lVVdtW1QZJjk1y+cgDquoFSS5J8iettR+vzUlVbgEAAHpmMk8o1VpbWlWnJLkyybQkn2+t3VpVJw7vPyfJB5P8X0k+XVVJsnQ1bc5JhFsAAAAmWGvtiiRXrLTtnBGv35bkbU/lnMItAABAn7SsYnqmfvOdWwAAADpPuAUAAKDztCUDAAD0Sksm8YRS64rKLQAAAJ0n3AIAANB52pIBAAB6pqZeV7LKLQAAAN2ncgsAANA3JpQCAACA7hFuAQAA6DxtyQAAAH3Skhoa9CAmnsotAAAAnadyCwAA0DcmlAIAAIDuEW4BAADoPG3JAAAAfTP1upJVbgEAAOg+lVsAAICeKRNKAQAAQPcItwAAAHSetmQAAIC+0ZYMAAAA3aNyCwAA0CctydCgBzHxVG4BAADoPOEWAACAztOWDAAA0COV5jm3AAAA0EUqtwAAAH2jcgsAAADdI9wCAADQedqSAQAA+kZbMgAAAHTPuITbqnpPVbWq2nTEtvdX1aKquq2qDhyP6wAAAMBoxtyWXFVbJXlNkp+O2LZ9kmOT7JDk/05yVVW9pLW2bKzXAwAAYDVakqFBD2LijUfl9u+S/M8s/yN8whFJLmyt/aa1dkeSRUl2H4drAQAAwJOMqXJbVYcn+Vlr7QdVNXLXFkmuG7G+ZHjbaOc4IckJSfKCF7xgLMMBAAAgSU3BCaXWGG6r6qokfzDKrr9K8pdJDhjtbaNsG/VPt7U2L8m8JJkzZ87U+xsAAABgzNYYbltrrx5te1XtmGTbJE9UbbdMcmNV7Z7lldqtRhy+ZZK7xzxaAAAAGMXTbkturd2c5HlPrFfVnUnmtNbur6rLk3yxqj6W5RNKzUryvTGOFQAAgLWhLXl8tNZuraqLkyxMsjTJyWZKBgAAYF0Zt3DbWttmpfUzk5w5XucHAABgbbQpWbkdj0cBAQAAwEAJtwAAAHTeOvnOLQAAAAPSoi0ZAAAAukjlFgAAoG+GBj2AiadyCwAAQOcJtwAAAHSetmQAAICeKRNKAQAAQPeo3AIAAPSNyi0AAAB0j3ALAABA52lLBgAA6JOWZEhbMgAAAHSOyi0AAECvNBNKAQAAQBcJtwAAAHSetmQAAIC+0ZYMAAAA3SPcAgAA0HnakgEAAPpGWzIAAAB0j8otAABAn7QkQyq3AAAA0DnCLQAAAJ2nLRkAAKBXWtKGBj2ICadyCwAAQOep3AIAAPSNRwEBAABA9wi3AAAAdJ62ZAAAgD7xnFsAAADoJpVbAACAvjGhFAAAAHSPcAsAAEDnaUsGAADoG23JAAAA0D0qtwAAAL3SVG4BAACgi4RbAAAAOk9bMgAAQJ+0JENDgx7FhFO5BQAAoPOEWwAAADpPWzIAAEDfmC0ZAAAAukflFgAAoG9UbgEAAKB7hFsAAAA6T1syAABAr7RkSFsyAAAAdI7KLQAAQJ+0pLWhQY9iwqncAgAA0HnCLQAAAJ2nLRkAAKBvTCgFAAAA3aNyCwAA0DdN5RYAAAA6R7gFAACg87QlAwAA9ElryZDn3AIAAEDnqNwCAAD0jQmlAAAAoHuEWwAAADpPWzIAAEDPNBNKAQAAQPeo3AIAAPRKM6EUAAAAdJFwCwAAQOdpSwYAAOiTlmRIWzIAAAB0jnALAABA52lLBgAA6JvmObcAAADQOSq3AAAAPdKSNBNKAQAAQPcItwAAAHSetmQAAIA+ac2EUgAAANBFKrcAAAA9Y0IpAAAA6CDhFgAAgAlVVQdV1W1VtaiqThtlf1XVx4f3L6iqXdd0Tm3JAAAAfTOJJ5SqqmlJPpXkNUmWJLm+qi5vrS0ccdjBSWYNL3sk+cfhn6ukcgsAAMBE2j3Jotba7a213ya5MMkRKx1zRJLz23LXJXlOVW2+upNOqsrtDTfccH9V/degx8GE2jTJ/YMeBJ3gXmFtuE9YW+4V1ob7ZOrZetADGA+P5KErr2pf2nTAw9iwquaPWJ/XWps3/HqLJHeN2LckT67KjnbMFknuWdUFJ1W4ba1tNugxMLGqan5rbc6gx8Hk515hbbhPWFvuFdaG+4Suaq0dNOgxrEGNsm3l6Z3X5pjfoy0ZAACAibQkyVYj1rdMcvfTOOb3CLcAAABMpOuTzKqqbatqgyTHJrl8pWMuT3L88KzJc5M83FpbZUtyMsnakpmS5q35EEjiXmHtuE9YW+4V1ob7BNaB1trSqjolyZVJpiX5fGvt1qo6cXj/OUmuSHJIkkVJfp3kLWs6b7W22rZlAAAAmPS0JQMAANB5wi0AAACdJ9wyIarqmKq6taqGqmrOSvveX1WLquq2qjpwxPY/rKqbh/d9vKpGmw6cHquqM6rqZ1V10/ByyIh9o943TF1VddDw/bCoqk4b9HiYPKrqzuHfJzc98czFqnpuVX2zqn4y/HOTQY+TiVdVn6+qX1TVLSO2rfLe8LsHJjfhlolyS5Kjknxn5Maq2j7LZ0fbIclBST5dVdOGd/9jkhOSzBpeJvvzulg3/q61tvPwckWyxvuGKWj47/9TSQ5Osn2S44bvE3jCq4b/HXnif7CeluTq1tqsJFcPrzP1nJsn//fFqPeG3z0w+Qm3TIjW2g9ba7eNsuuIJBe21n7TWrsjy2dD272qNk/yrNba/27LZz07P8lrJ27ETHKj3jcDHhODtXuSRa2121trv01yYZbfJ7AqRyQ5b/j1efE7ZkpqrX0nyYMrbV7VveF3D0xywi2DtkWSu0asLxnetsXw65W3M/WcUlULhlvHnmgNW9V9w9TlnmB1WpJ/raobquqE4W3Pf+J5icM/nzew0THZrOre8O8MTHKec8u4qaqrkvzBKLv+qrX2lVW9bZRtbTXb6ZnV3TdZ3pr+oSz/u/9Qko8meWvcHzyZe4LV2au1dndVPS/JN6vqR4MeEJ3k3xmY5IRbxk1r7dVP421Lkmw1Yn3LJHcPb99ylO30zNreN1X1mSRfG15d1X3D1OWeYJVaa3cP//xFVV2a5a2k91bV5q21e4a/CvOLgQ6SyWRV94Z/Z2CS05bMoF2e5NiqemZVbZvlE0d9b7gN6JGqmjs8S/LxSVZV/aWnhv+j4glHZvnEZMkq7puJHh+TyvVJZlXVtlW1QZZP+nL5gMfEJFBVM6tq4ydeJzkgy/8tuTzJm4YPe1P8juH/WNW94XcPTHIqt0yIqjoyySeSbJbk61V1U2vtwNbarVV1cZKFSZYmObm1tmz4bSdl+SyG05N8Y3hhavlwVe2c5W1fdyb5f5NkDfcNU1BrbWlVnZLkyiTTkny+tXbrgIfF5PD8JJcOP01u/SRfbK39S1Vdn+TiqvrTJD9NcswAx8iAVNUFSfZNsmlVLUlyepKzMsq94XcPTH61fCJaAAAA6C5tyQAAAHSecAsAAEDnCbcAAAB0nnALAABA5wm3AAAAdJ5wCwAAQOcJtwAAAHTe/w/k3fSZVqScJQAAAABJRU5ErkJggg==\n",
      "text/plain": [
       "<Figure size 1296x864 with 2 Axes>"
      ]
     },
     "metadata": {
      "needs_background": "light"
     },
     "output_type": "display_data"
    }
   ],
   "source": [
    "# Escriba aquí su código para el ejercicio 4\n",
    "import numpy as np\n",
    "import matplotlib.pyplot as plt\n",
    "pub_authors1 = np.random.normal(0, np.sqrt(0.2), 1000)\n",
    "pub_authors2 = np.random.normal(0, np.sqrt(1.0), 1000)\n",
    "pub_authors3 = np.random.normal(0, np.sqrt(5.0), 1000)\n",
    "pub_authors4 = np.random.normal(-2, np.sqrt(0.5), 1000)\n",
    "\n",
    "data = [pub_authors1, pub_authors2, pub_authors3, pub_authors4]\n",
    "fig = plt.figure(1, figsize=(9, 6))  \n",
    "ax = fig.add_subplot(111)\n",
    "bp = ax.boxplot(data)\n",
    "for flier in bp['fliers']:\n",
    "    flier.set(marker='o', color='red', alpha=0.5)\n",
    "plt.xlabel('pub_authors')\n",
    "plt.ylabel('Distribución')\n",
    "plt.title('Boxplots de pub_authors')\n",
    "plt.show()\n",
    "\n",
    "import matplotlib.pyplot as plt\n",
    "zip_country = []\n",
    "zip_long = []\n",
    "zip_lat = []\n",
    "zip_count = []\n",
    "%matplotlib inline\n",
    "plt.scatter(zip_long, zip_lat, s = zip_count, c= zip_count)\n",
    "plt.colorbar()\n",
    "\n",
    "plt.xlim(-145,145)\n",
    "plt.ylim(-55,55)\n",
    "\n",
    "ard = dict(arrowstyle=\"->\")\n",
    "plt.annotate('Ecuador', xy = (-78.183406, -1.831239), \n",
    "               xytext = (-108.183406, -1.831239), arrowprops = ard)\n",
    "plt.annotate('France', xy = (2.213749, 46.227638),\n",
    "             xytext =  (8.213749, 39.227638), arrowprops= ard)\n",
    "plt.annotate('Australia', xy = (133.775136, -25.274398), \n",
    "               xytext = (110.775136, -22.274398), arrowprops= ard)\n",
    "               \n",
    "params = plt.gcf()\n",
    "plSize = params.get_size_inches()\n",
    "params.set_size_inches( (plSize[0] * 3, plSize[1] * 3) )\n",
    "plt.show()\n",
    "\n",
    "\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Ejercicio 5 [1 punto]\n",
    "\n",
    "Interprete los resultados del **ejercicio 4**\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "### Escriba la respuesta del ejercicio 5\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Ejercicio 6 [2 puntos]\n",
    "\n",
    "Para algún gen de interés, realice lo siguiente: \n",
    "\n",
    "1. Una búsqueda en la página del [NCBI nucleotide](https://www.ncbi.nlm.nih.gov/nucleotide/).\n",
    "\n",
    "2. Descargue el ```Accession List``` de su búsqueda.\n",
    "\n",
    "3. Cargue el ```Accession List``` en este notebook y haga una descarga de la data tipo ```genbank``` utilizando los diez primeros IDs de accesión. \n",
    "\n",
    "4. Arme un árbol filogenético para los resultados del paso 3.\n",
    "\n",
    "5. Interprete el árbol del paso 4. "
   ]
  },
  {
   "cell_type": "code",
   "execution_count": 8,
   "metadata": {},
   "outputs": [
    {
     "name": "stdout",
     "output_type": "stream",
     "text": [
      "MW244756.1 Cruziohyla calcarifer cruzioseptin-17 precursor, m...\n",
      "Sequence length 196, 5 features, from: Cruziohyla calcarifer (splendid leaf frog)\n",
      "MW244755.1 Cruziohyla calcarifer cruzioseptin-16 precursor, m...\n",
      "Sequence length 328, 5 features, from: Cruziohyla calcarifer (splendid leaf frog)\n",
      "KX065077.1 Cruziohyla calcarifer proteinase inhibitor precurs...\n",
      "Sequence length 418, 5 features, from: Cruziohyla calcarifer (splendid leaf frog)\n",
      "KX065076.1 Cruziohyla calcarifer proteinase inhibitor precurs...\n",
      "Sequence length 442, 5 features, from: Cruziohyla calcarifer (splendid leaf frog)\n",
      "KX065075.1 Cruziohyla calcarifer proteinase inhibitor precurs...\n",
      "Sequence length 466, 5 features, from: Cruziohyla calcarifer (splendid leaf frog)\n",
      "KX065074.1 Cruziohyla calcarifer proteinase inhibitor precurs...\n",
      "Sequence length 445, 5 features, from: Cruziohyla calcarifer (splendid leaf frog)\n",
      "KX065073.1 Cruziohyla calcarifer proteinase inhibitor precurs...\n",
      "Sequence length 450, 5 features, from: Cruziohyla calcarifer (splendid leaf frog)\n",
      "KX065072.1 Cruziohyla calcarifer proteinase inhibitor precurs...\n",
      "Sequence length 447, 5 features, from: Cruziohyla calcarifer (splendid leaf frog)\n",
      "KX065071.1 Cruziohyla calcarifer proteinase inhibitor precurs...\n",
      "Sequence length 370, 5 features, from: Cruziohyla calcarifer (splendid leaf frog)\n",
      "KX065070.1 Cruziohyla calcarifer proteinase inhibitor precurs...\n",
      "Sequence length 386, 5 features, from: Cruziohyla calcarifer (splendid leaf frog)\n",
      "\"C:\\Program Files (x86)\\ClustalW2\\clustalw2.exe\" -infile=\"data\\Cruziohyla calcarifer.fasta\"\n",
      "Alignment with 10 rows and 468 columns\n",
      "ATGAAGACTCTGATCTCCTGCGTCTTGCTCTTCTGTATACTGGC...--- KX065076.1\n",
      "ATGAAGACTCTGATCTCCTGCGTCTTGCTCTTCTGTATACTGGC...--- KX065073.1\n",
      "ATGAAGACTCTGATCTCCTCCGTCTTGCTCTTCTGTATACTGGC...--- KX065077.1\n",
      "ATGAAGACTCTGATCTCCTCCGTCTTGCTCCTCTGTATACTGGC...--- KX065072.1\n",
      "------ACTCTGATCTCCTCCGTCTTGCTCCTCTGTATACTGGC...--- KX065074.1\n",
      "ATGAAGACTCTGATCTCCTCCGTCTTGCTCCTCTGTATACTGGC...AAA KX065075.1\n",
      "------ACTCTGATCTCCTCCGTCTTGCTCTTCTGTATACTGGC...--- KX065071.1\n",
      "ATGAAGACTCTGATCTCCTCCGCCTTGCTCTTCTGTATACTGGC...--- KX065070.1\n",
      "--------------------------------------------...--- MW244756.1\n",
      "------ATGGCTTTCCTGAAGAAATCTCTTTTCCTTGTACTATT...--- MW244755.1\n",
      "                                                               , MW244756.1\n",
      "          _____________________________________________________|\n",
      "         |                                                     |__ MW244755.1\n",
      "     ____|\n",
      "    |    |          __ KX065071.1\n",
      "  __|    |_________|\n",
      " |  |              | KX065070.1\n",
      " |  |\n",
      " |  | KX065075.1\n",
      " |\n",
      " | KX065074.1\n",
      " |\n",
      "_|__ KX065077.1\n",
      " |\n",
      " | KX065072.1\n",
      " |\n",
      " , KX065076.1\n",
      " |\n",
      " | KX065073.1\n",
      "\n",
      "Tree(rooted=False, weight=1.0)\n",
      "    Clade()\n",
      "        Clade(branch_length=0.0087)\n",
      "            Clade(branch_length=0.0323)\n",
      "                Clade(branch_length=0.07636)\n",
      "                    Clade(branch_length=0.71741)\n",
      "                        Clade(branch_length=-0.02129, name='MW244756.1')\n",
      "                        Clade(branch_length=0.03659, name='MW244755.1')\n",
      "                    Clade(branch_length=0.13575)\n",
      "                        Clade(branch_length=0.0332, name='KX065071.1')\n",
      "                        Clade(branch_length=-0.01158, name='KX065070.1')\n",
      "                Clade(branch_length=-0.00407, name='KX065075.1')\n",
      "            Clade(branch_length=0.00974, name='KX065074.1')\n",
      "        Clade(branch_length=0.00183)\n",
      "            Clade(branch_length=0.04346, name='KX065077.1')\n",
      "            Clade(branch_length=0.00438, name='KX065072.1')\n",
      "        Clade(branch_length=0.00226)\n",
      "            Clade(branch_length=0.0111, name='KX065076.1')\n",
      "            Clade(branch_length=0.00248, name='KX065073.1')\n"
     ]
    },
    {
     "data": {
      "image/png": "iVBORw0KGgoAAAANSUhEUgAAAYQAAAEGCAYAAABlxeIAAAAAOXRFWHRTb2Z0d2FyZQBNYXRwbG90bGliIHZlcnNpb24zLjMuNCwgaHR0cHM6Ly9tYXRwbG90bGliLm9yZy8QVMy6AAAACXBIWXMAAAsTAAALEwEAmpwYAAApPklEQVR4nO3df3RU1bn/8feGKLeKBlMhxASdAF4MCckE+SVYCYhUqgL1oljhEgWlFr7X27TX4LqsiqVfF6gsq23Uqxd7QeUbKsgFKmBREIsikEQDVPlVIULwNxIFtIHA8/1jJqcZSMJMyK9JPq+1zkrmnJ1znn3QeWafc2Y/zswQERFp09QBiIhI86CEICIigBKCiIgEKSGIiAighCAiIkExTR1AOC6++GLz+XxNHYaISFQpKir60sw6hts+KhKCz+ejsLCwqcMQEYkqzrmPImmvS0YiIgIoIYiISJASgoiIAEoIIiISpIQgIiKAEoKIiAQpIYiICKCEICIiQUoIIiICKCGIiEiQEoKIiABKCCISBXw+Hz/4wQ9C1vn9ftLS0gDIzMykuLgYgIqKCs4//3xefPFFr+2VV17Ju+++y4IFC0hPTyc9PZ2BAweyZcuWkH2eOHGCzMxMbrzxxtNimDNnDs45vvzySwAWLFiA3+/3ljZt2ngxZGVl0aNHD2/b559/7u3npZdeomfPnqSmpnL77bdX29/p06fTpUsX2rdvH9mJOktKCCISFQ4fPsz+/fsB2L59e8i2gQMHsmHDBgC2bNlCjx49vNdHjx5lz549ZGRkkJyczJtvvsnWrVv51a9+xeTJk0P288QTT5CSknLasffv389rr73GpZde6q0bN24cxcXFFBcX88ILL+Dz+fD7/d72BQsWeNs7deoEwO7du5k1axZvv/0277//Po8//ni1fb3pppvYvHlzZCeoHighiEhUuPXWW/njH/8IQH5+Pj/5yU+8bYMGDfISwIYNG7jnnnu8T+ubN2+md+/etG3bloEDB3LRRRcBMGDAAEpLS719lJaWsmLFCu66667Tjp2Tk8MjjzyCc67a2E6Npyb//d//zdSpU70YKhPFqQYMGEBCQsIZ91fflBBEpNnJysoKWT799FOWLl3Kb37zGwD+9Kc/cdNNN3ntq44QNmzYwDXXXEO7du04fPgwGzZsYNCgQacd47nnnmPEiBHe65///Oc88sgjtGkT+ra4fPlyEhMTycjIqDHeP/7xj6clhDvvvBO/389vfvMbzAyAXbt2sWvXLgYNGsSAAQN49dVXIzwzDUsJQUSiwjnnnMO3335Lz549SUlJ4bzzzvO2+Xw+jh07xqeffsqOHTvo0aMHffv2ZdOmTWzYsIGBAweG7OuNN97gueee4+GHHwbglVdeoVOnTlx55ZUh7b799lseeughZs6cWWNcmzZt4rzzzvPuZ0DgctG2bdtYv34969ev54UXXgAC9zd2797NunXryM/P56677qKsrOxsT029UUIQkWZn3bp1IUvnzp1ZtmwZ3bp1Y/fu3dVenrnqqqtYvHgxCQkJOOcYMGAAb7/9Nps3b2bAgAFeu61bt3LXXXexbNkyvv/97wPw9ttvs3z5cnw+H7fddhtr165l/PjxfPjhh+zdu5eMjAx8Ph+lpaX07t2bTz/91NvfwoULT4snMTERgAsuuIDbb7/dux+QlJTEqFGjOOecc0hOTqZHjx7s3r273s9fnZlZs1+uvPJKE5HW67LLLrMvvvjCBg0aZMnJyVZeXm579+611NRUr81vf/tb69q1q82cOdPMzD7++GPr2rWrpaWleW0++ugj69atm7399ts1HuuNN96wG264odY4Kp04ccISExPtww8/9NYdP37ca3Ps2DH7l3/5F3v66afNzGzVqlU2YcIEMzP74osvLCkpyb788ssaYzn//PNr3BYOoNAieK/VCEFEokZMTAyXXnop55577mnbBg0axJ49e7jqqqsASEhI4MSJEyGXi2bOnMnBgweZMmUKfr+fPn36nFU8f/nLX0hKSqJr167euvLycn74wx+Snp6O3+8nMTGRu+++G4Af/vCHfP/736dnz54MGTKERx991BulVH1CKTc3l6SkJL799luSkpJ48MEHzyrOcDkL3uxozvr06WOqqSwiWVlZQOCSkpyZc67IzMLOehohiIgIoIQgIiJBSggiIgIoIYiISFBMUwfQGlTeCBORs1NcXBzyNI7UL40QGlhWVpY3p4qISHOmEUIj8Pv9ekxOpB5otN2wNEIQERFACUFERIKUEEREBFBCEBGRICUEEWkVWkpd5nnz5tGxY0dv/dy5c6vt7/Tp0wHSnXNHwj1HjZ4QnHNdnHNvOOe2O+fed879e2PHICKtU0uoywwwduxYb311JT+Byopy26vdWIOmGCFUAL80sxRgADDVOdezCeIQkVamJdRlDlewKNDxSP6m0ROCmX1iZu8Gfz9MIIMlNnYcIhKdiouLT6u5XN1SnTFjxrBkyRIgeusyA7z88sukp6czZswYb8RTH5r0HoJzzgdkApuq2TbZOVfonCv84osvGj02EWl+1q1bd1ZTV8TFxXHRRRexcOHCqK3LfNNNN1FSUsLWrVsZNmwY2dnZdT4fp2qybyo759oDLwM/N7NvTt1uZs8Cz0KgQE4jhycizdTZfut/7NixTJ06lXnz5p22LdK6zKtWrTqtLvPKlSv5+9//zjfffMP48eOZNm2aV5cZ8Ooyb968mc6dOwPh12WeMGGCdzyAu+++m2nTpp3V+QgRSb3N+lqAc4A/A78Ip30011QePHiwDR48uKnDEGn1Kushf/PNNzZ79uyorcv88ccfe+2WLFli/fv3rzEOoBA4YmG+Nzf6CMEF7qg8B2w3s8ca+/gi0rpdcMEFNX6qHjRoEDk5OWHXZYZAneezKfFbW13m48ePc+LECYYNG+bVZf7d737H8uXLiYmJIS4uLmSk4/f7vRvhubm5AOlAjHOuFJhrZg/WFkuj11R2zl0NrAe2ASeDq//TzFbW9DfRXFNZNWBFpKlEWlO50UcIZvYWUP1zVyIi0mT0TWUREQGUEEREJEgJQUREACUEEREJUgnNoIYqzaei4CISLTRCIJAMKp/dFRFprTRCCPL7/Q3yXQEVBReRaKERgoiIAEoIIiISpIQgIiKAEoKIiAQpIUid+Hw+r1B4UVERycnJvPfeezz22GNMmjTJa7dgwQJuuOEGIDCD49ixY+nevTv9+/enpKTEa7dv3z6GDx9OSkoKPXv29LbdcccdJCcnewXFK58GMzPuvfdeunfvTnp6Ou+++y4AO3fuDClafuGFF/L4448DsGjRIlJTU2nTpk2ts1NOnDiRTp06hRQrEWkVwp0nuymXhq6H0JA1C1pqPYTKOd23bNliPp/PNm3aZGaBedwzMjLsrbfeskOHDpnP5/PmeX/yySftpz/9qZmZ5efn26233urtb/DgwbZ69WozMzt8+LAdPXrUzMyys7Nt0aJFpx1/xYoVdv3119vJkyftnXfesX79+p3WpqKiwuLj462kpMTMzD744APbsWOHDR482AoKCmrs25tvvmlFRUUh8+SLRCOg0CJ4r9UIQeps+/btjB49mhdeeIF+/foBgbnhn3rqKaZOnUpubi4TJ0705nlftmyZV+5vzJgxrFmzBjPjgw8+oKKiguuuuw6A9u3bh5Q2rM6yZcuYMGGCV9WqrKyMTz75JKTNmjVr6NatG5dddhkAKSkp9OjR44z9uuaaa4iLi4vsZIi0APoegoT1XYnqvqMxatQoXnzxRa6++uqQ9QMHDiQlJYXXX3+d7du3e+sPHDhAly5dgEDiiI2N5eDBg+zatYsOHTpw8803s3fvXoYNG8bs2bNp27YtANOnT2fmzJlce+21zJ49m3bt2oXsCyApKYkDBw6QkJDgrauuLKGI1EwjhFbubL6lPWzYMObOncuJEydC1h85coTCwkKOHz/OF1984a23aooxOeeoqKhg/fr1zJkzh4KCAvbs2eNVgZo1axY7duygoKCAr776yitoXtO+Kh07dozly5dzyy231KlvIq2RRghS529p5+Xlcc899zBlyhSeeeYZb/2MGTMYP3488fHx5OTksGjRIiDwKX7//v0kJSVRUVHB119/TVxcHElJSWRmZnqXlkaPHs3GjRuZNGmS94m/Xbt23HnnncyZMydkX5VKS0u55JJLvNerVq2id+/exMfHR9wvkdZKIwSpszZt2pCfn8/OnTt54IEHANi2bRsrVqxg2rRpTJ48mY8++ojXXnsNgJEjRzJ//nwAFi9ezNChQ3HO0bdvXw4dOuSNJtauXUvPnj0BvPsCZsbSpUu9J39GjhzJ888/j5mxceNGYmNjQy4X5efn63KRSKQiuQPdVIueMmo4dY2v8ikjM7OysjLLyMiwvLw8GzRokK1cudJrV1BQYCkpKVZeXm7fffedjRkzxrp162Z9+/b1nj4yM1u9erX16tXL0tLSLDs728rLy83MbMiQIZaWlmapqak2btw4O3z4sJmZnTx50qZMmWJdu3a1tLS0kKeGjh49anFxcVZWVhYS85IlSywxMdHOPfdc69Spkw0fPtzMzA4cOGAjRozw2t12223WuXNni4mJscTERJs7d27E50ekOSDCp4ycVXMttrnp06eP1fbc+NmqvKnakJPbNcS+60Nzj09E6s45V2RmfcJtr0tGIiICKCGIiEiQEoKIiABKCCIiEqTvITSC4uLiZls5TTWfRaSSRggNbN26dXrDFZGooBFCI2jOj3Q215GLiDQ+jRBERARQQhARkaAWf8konEsiurEqItLCRwhnM7WziEhr06ITAvxjaufaltY+Oti4cWPU1Uf+6quvuO6667j88su57rrrOHToULV9U31kkQhEMhNefS5AW+A94JUzta3rbKfhzuTZ3GckbUiDBw+2du3aRV195Pvuu89mzZplZmazZs2y3Nzcavun+sjSmhFFNZX/Hdh+xlbSKKKtPnLV42dnZ7N06dJq9636yCLha5KE4JxLAm4A5jbF8Vu7rKwsb6m8bDNq1Cjy8vJqrY+cm5vrrQ+nPnJmZib33XdfSInN6dOnk56eTk5ODuXl5aftC/5RH7mqU+sjf/bZZ15BnISEBD7//PN6ODMirVtTjRAeB3KBkzU1cM5Nds4VOucKq9bllfrl9/vp3Lmz6iOLSOMnBOfcjcDnZlZUWzsze9bM+phZn44dOzZSdK3DqTfVIVAfGWDKlCkhbSvrI0+fPp2cnBxvfdWaxjXVR46JiWH06NHeTeKEhAScc1595M2bN5+2LwivPnJ8fLx3WemTTz6hU6dO9XV6RFqtphghDAJGOudKgIXAUOfci00Qh1QRbfWRqx5//vz5jBo1qqFOjUjrEckd6PpegCz0lFGTi8b6yF9++aUNHTrUunfvbkOHDrWDBw+ameoji1RFNNVUds5lAf9hZjfW1q6uNZXDrResusIi0hJFWlO5SaeuMLN1wLqmjEFERAJa/DeVRUQkPEoIIiICKCGIiEiQEoKIiABKCCIiEqSEICIigBKCiIgEKSGIiAighCAiIkFKCCIiAighiIhIUItNCFWrgUXK5/NFXdH5Bx98kMTERG/bypUrq+2bis6LSI0imRq1qZZIp78ePHiwxcbGWmxsbJ2mv66cDjqais7PmDHDHn300TP2VUXnRVoPIpz+usWOECo/KddVtBWdD5eKzotITVpsQohEdZeWoq3oPATKYKanpzNx4kQOHTpUt5MhIq2WEgKB0cSpxXGirej8z372Mz788EOKi4tJSEjgl7/8ZcTnQURaNyUEqq+UFo1F59u2bUubNm24++67vX2JiIQr7ITgnOvknLu0cmnIoJqDaCs6X/Uew//+7//qKSIRidyZ7joDI4HdwFFgL3ASeD+SO9dnu9TlKaOqS6Sisej8+PHjLS0tzXr16mU33XSTffzxx2amovMirRkRPmXkrJrr1VU557YAQ4HXzSzTOTcE+ImZTW7IRFVVnz59rLCwMOz2WVlZIa+ruyQkItLSOeeKzKxPuO3DuWR03MwOAm2cc23M7A3AX9cARUSkeYoJo02Zc6498BdggXPuc6CiYcMSEZHGFs4IYRTwHZADvAp8CNzUkEGJiEjjCychXGZmJ8yswszmm9nvgF4NHZiIiDSucBLCS865aS7ge8653wOzGjowERFpXOEkhP5AF2ADUAB8DAxqyKBERKTxhfWUEYF7CN8D/gnYa2YnGzQqERFpdOEkhAICCaEvcDXwE+fc4gaNSkREGl04j51OMrPKb4V9Coxyzv1rA8YkIiJN4IwJoTIZOOc6EbhkBPBmQwYlIiKN74yXjJxzNznndhOYx+hNoARY1cBxiYhIIwvnHsL/BQYAu8wsGbgWeLtBo2ok0Vg7udKcOXNwznnxn0q1k0UkYmea/Y7gbHnAFqBN8PfNkcygV80+OwCLgR3AduCq2to31Gyn0Vg72cxs3759Nnz4cLv00ku9WVlPpdrJIkID1FQ+dS6jJwg8ino2ngBeNbMrgIxgUmgS0Vg7OScnh0ceeSSkitqpVDtZRCIVTkLYAnxL6FxGO+p6QOfchcA1wHMAZnbMzMrqur8zKS4uJisr67QpsStFW+3k5cuXk5iYSEZGRt1OiIhIDcJJCEPM7KSFzmXU9yyO2RX4Avgf59x7zrm5zrnzT23knJvsnCt0zhVWrV0ciXXr1uH3+2ttE021k7/99lseeughZs6cGVb/RUQiUWNCcM79zDm3DbjCObe1yrIX2HoWx4wBegNPm1kmgUps95/ayMyeNbM+ZtanY8eOdT7YunXrvKU60VQ7+cMPP2Tv3r1kZGTg8/koLS2ld+/efPrpp3U+PyIilWobIfw/AtNcLwv+rFyuNLPxZ3HMUqDUzDYFXy8mkCCaRDTVTu7Vqxeff/45JSUllJSUkJSUxLvvvkvnzp0b+CyJSKsQyR3o+lqA9UCP4O8PAo/W1r6uTxmdSTTWTq4pftVOFpFTUd81lRuCc84PzAXOBfYAd5rZoZra17Wmsmopi0hrFmlN5XDmMqp3ZlYMhB2kiIg0vHCeMhIRkVZACUFERAAlBBERCVJCEBERQAlBRESClBBERARQQhARkSAlBBERAZQQREQkSAlBREQAJQQREQlSQqiFz+fzitgXFRWRnJzMe++9x2OPPcakSZO8dgsWLOCGG24AoLy8nLFjx9K9e3f69+9PSUmJ127fvn0MHz6clJQUevbs6W274447SE5Oxu/34/f7KS4uBgIz0d577710796d9PR0r77Czp07vbZ+v58LL7yQxx9/HICxY8d6630+X40FgiZOnEinTp28qbhFRJpk+utIl4aa/vpMKqeX3rJli/l8Ptu0aZOZmR0/ftwyMjLsrbfeskOHDpnP5/OmwX7yySftpz/9qZmZ5efn26233hoS1+rVq83M7PDhw3b06FEzM8vOzrZFixaddvwVK1bY9ddfbydPnrR33nnH+vXrd1qbiooKi4+Pt5KSktO2/eIXv7Bf//rX1fbtzTfftKKiIktNTY3klIhIFCHC6a81QjiD7du3M3r0aF544QX69esHBGopP/XUU0ydOpXc3FwmTpxI165dAVi2bBnZ2dkAjBkzhjVr1mBmfPDBB1RUVHDdddcB0L59e84777xaj71s2TImTJiAc44BAwZQVlbmFduptGbNGrp168Zll10Wst7MeOmll0IK7FR1zTXXEBcXF/kJEZEWq0mmv24MxcXFXl2EcFVXP2HUqFG8+OKLXH311SHrBw4cSEpKCq+//jrbt2/31h84cIAuXboAgcQRGxvLwYMH2bVrFx06dODmm29m7969DBs2jNmzZ9O2bVsApk+fzsyZM7n22muZPXs27dq1C9kXBEpuHjhwIKSq2sKFC6t901+/fj3x8fFcfvnlEZ0DEWm9WuQIobi4mCNHjtTLvoYNG8bcuXM5ceJEyPojR45QWFjI8ePHvbKZQGVFuBDOOSoqKli/fj1z5syhoKCAPXv2MG/ePABmzZrFjh07KCgo4KuvvuLhhx+udV+Vjh07xvLly7nllltOa3dq+U0RkTNpsSOE9u3b10vFtLy8PO655x6mTJnCM888462fMWMG48ePJz4+npycHBYtWgQEPsXv37+fpKQkKioq+Prrr4mLiyMpKYnMzEzv0tLo0aPZuHEjkyZN8j7xt2vXjjvvvJM5c+aE7KtSaWkpl1xyifd61apV9O7dm/j4+JCYKyoqWLJkCUVFRWfdfxFpPVrkCKE+tWnThvz8fHbu3MkDDzwAwLZt21ixYgXTpk1j8uTJfPTRR7z22msAjBw5kvnz5wOwePFihg4dinOOvn37cujQIW80sXbtWnr27Ang3RcwM5YuXeo9+TNy5Eief/55zIyNGzcSGxsbcrmoplHA66+/zhVXXEFSUlIDnRURaZEiuQPdVEukTxnFxsZabGxsRH9TnapF7MvKyiwjI8Py8vJs0KBBtnLlSq9dQUGBpaSkWHl5uX333Xc2ZswY69atm/Xt29d7+sjMbPXq1darVy9LS0uz7OxsKy8vNzOzIUOGWFpamqWmptq4cePs8OHDZmZ28uRJmzJlinXt2tXS0tKsoKDA29fRo0ctLi7OysrKTos7Ozvbnn766ZB1Bw4csBEjRnivb7vtNuvcubPFxMRYYmKizZ0796zPl4g0L0T4lJGzaq5TNzd9+vSxwsLCsNt36NABgLKysoYJSEQkCjjnisws7Pr1umQkIiKAEoKIiAQpIYiICKCEICIiQUoIIiICKCGIiEiQEoKIiABKCCIiEqSEICIigBKCiIgEKSGIiAjQChNCNNZJvu+++7jiiitIT0/nxz/+cY1zNKlOsoiclUhmwquvBcgB3gf+CuQD/1Rb+/qc7TQa6yT/+c9/tuPHj5uZWW5uruXm5lbbN9VJFpGqaO41lZ1zicC9QB8zSwPaArc1ZgzRVid5+PDhxMQEahkNGDCA0tLSavetOskicjaa6pJRDPA951wMcB7wcX0f4MiRIzXWVB41ahR5eXm11knOzc311odTJzkzM5P77rsvpNTm9OnTSU9PJycnh/Ly8tP2Bf+ok1xVTXWSAf7whz8wYsSI8E+EiEiYGj0hmNkBYA6wD/gE+NrMVp/azjk32TlX6JwrrFqzOBx+v5/27dvXuD1a6yQ/9NBDxMTEMG7cuNpPgIhIHTTFJaOLgFFAMnAJcL5zbvyp7czsWTPrY2Z9OnbsGPFx/H5/jTWV8/LyAJgyZUrI+so6ydOnTycnJ8dbX7W2cU11kmNiYhg9erR3kzghIQHnnFcnefPmzaftC8Kvkzx//nxeeeUVFixYEJJARETqS1NcMhoG7DWzL8zsOLAEGNiYAURbneRXX32Vhx9+mOXLl5/xHoWISJ1Fcge6PhagP4EnjM4DHDAf+Lfa/ibSp4wGDx5sgwcPrnZbNNZJ7tatmyUlJVlGRoZlZGR4TzypTrKI1IZoqKnsnPs1MBaoAN4D7jKz8praR1pTufJmck2XjEREWoNIayrHNGQwNTGzGcCMpji2iIhUr9V9U1lERKqnhCAiIoASgoiIBCkhiIgIoIQgIiJBSggiIgIoIYiISJASgoiIAEoIIiISpIQgIiKAEoKIiAS12oTg8/n48ssvASgqKiI5OZn33nuPxx57jEmTJnntFixYwA033ABAeXk5Y8eOpXv37vTv35+SkhKv3b59+xg+fDgpKSn07NnT23bHHXeQnJyM3+/H7/dTXFwMBGaZvffee+nevTvp6eleHYWdO3d6bf1+PxdeeCGPP/64d5zf//739OjRg9TU1JCqblVNnDiRTp06eVNui4iEJZKpUZtqqc/prytVToO9ZcsW8/l8tmnTJjMzO378uGVkZNhbb71lhw4dMp/P5013/eSTT3pTT+fn59utt94acszVq1ebmdnhw4ft6NGjZmaWnZ1tixYtOu34K1assOuvv95Onjxp77zzjvXr1++0NhUVFRYfH28lJSVmZrZ27Vq79tpr7e9//7uZmX322WfV9u3NN9+0oqIiS01NrfUciEjLRoTTX7faEQLA9u3bGT16NC+88AL9+vUDAjWTn3rqKaZOnUpubi4TJ06ka9euACxbtozs7GwAxowZw5o1azAzPvjgAyoqKrjuuusAaN++/RkL2SxbtowJEybgnGPAgAGUlZV5RXUqrVmzhm7dunHZZZcB8PTTT3P//ffTrl07ADp16lTtvq+55hri4uLqeFZEpLVqsQmhuLiYrKwsb6nOqFGjyMvL4+qrrw5ZP3DgQFJSUnj99ddDLsscOHCALl26AIHEERsby8GDB9m1axcdOnTg5ptvJjMzk/vuuy+kXvP06dNJT08nJyeH8vLy0/YFgdKaBw4cCIlj4cKFIdXTdu3axfr16+nfvz+DBw+moKCgbidHRKQaLTIhFBcXc+TIkTO2GzZsGHPnzg158wY4cuQIhYWFHD9+3CuPCVRWfAvhnKOiooL169czZ84cCgoK2LNnD/PmzQNg1qxZ7Nixg4KCAr766isefvjhWvdV6dixYyxfvpxbbrnFW1dRUcGhQ4fYuHEjjz76KLfeemu1+xERqYsWmRAgcNlm3bp13lKdvLw8AKZMmRKyfsaMGYwfP57p06eTk5PjrU9KSmL//v1A4M3566+/Ji4ujqSkJDIzM+natSsxMTGMHj3au0mckJCAc4527dpx5513snnz5tP2BVBaWsoll1zivV61ahW9e/cmPj4+5Pg333wzzjn69etHmzZtvBvjIiJnq8UmhHC0adOG/Px8du7cyQMPPADAtm3bWLFiBdOmTWPy5Ml89NFHvPbaawCMHDmS+fPnA7B48WKGDh2Kc46+ffty6NAhbzSxdu1aevbsCeDdFzAzli5d6j35M3LkSJ5//nnMjI0bNxIbG0tCQoIXW35+fsjlIoDRo0ezdu1aIHD56NixY1x88cUNdXpEpLWJ5A50Uy2RPmUUGxtrsbGxtbapfMrIzKysrMwyMjIsLy/PBg0aZCtXrvTaFRQUWEpKipWXl9t3331nY8aMsW7dulnfvn29p4/MzFavXm29evWytLQ0y87OtvLycjMzGzJkiKWlpVlqaqqNGzfODh8+bGZmJ0+etClTpljXrl0tLS3NCgoKvH0dPXrU4uLirKysLCTm8vJyGzdunKWmplpmZqatWbPGzMwOHDhgI0aM8Nrddttt1rlzZ4uJibHExESbO3duROdPRFoGInzKyFkUXIPu06ePFRYWht2+Q4cOAJSVlTVMQCIiUcA5V2RmfcJt36ovGYmIyD8oIYiICKCEICIiQUoIIiICKCGIiEhQTFMH0BD8fn9ThyAiEnU0QhAREUAJQUREgpQQREQEUEIQEZEgJQQREQFaWUKIxjrKv/rVr0hPT8fv9zN8+HA+/vjjavumOsoictYimQkvkgX4A/A58Ncq6+KA14DdwZ8XhbOv+qqpHI11lL/++mtv2xNPPOHFcirVURaRU9GMairPA64/Zd39wBozuxxYE3zdqKKtjvKFF17obTt69GhIVbWqVEdZRM5WgyUEM/sL8NUpq0cB84O/zwdGN9TxKy/TnCra6ihX7qtLly4sWLCAmTNnRn4yRETC0Nj3EOLN7BOA4M9ONTV0zk12zhU65wqr1jUOx7p162r8tnK01VEGeOihh9i/fz/jxo3zyn6KiNS3ZntT2cyeNbM+ZtanY8eOEf99S6mjXNXtt9/Oyy+/HOGZEBEJT2MnhM+ccwkAwZ+fN/Lxo66O8u7du73fly9fzhVXXNEQp0VEpGFrKgM+Qp8yehS4P/j7/cAj4ewn0qeMahKNdZRvvvlmS01NtV69etmNN95opaWlZqY6yiJyZjSXmsrOuXwgC7gY+AyYASwFXgIuBfYBt5jZqTeeTxNpTWUREYm8pnKDTX9tZj+pYdO1DXVMERGpu2Z7U1lERBqXEoKIiABKCCIiEqSEICIigBKCiIgEKSGIiAighCAiIkFKCCIiAighiIhIkBKCiIgASggiIhLUYJPb1Sfn3BfARxH+2cXAlw0QTnOkvrZM6mvL1Jh9vczMwi4oExUJoS6cc4WRzPIXzdTXlkl9bZmac191yUhERAAlBBERCWrJCeHZpg6gEamvLZP62jI127622HsIIiISmZY8QhARkQgoIYiICBDlCcE5d71zbqdz7m/Oufur2e6cc78Lbt/qnOvdFHHWlzD6Oy7Yz63OuQ3OuYymiLM+nKmvVdr1dc6dcM6Nacz46lM4fXXOZTnnip1z7zvn3mzsGOtLGP8Nxzrn/uSc2xLs651NEWd9cM79wTn3uXPurzVsb37vT2YWlQvQFvgQ6AqcC2wBep7S5kfAKsABA4BNTR13A/d3IHBR8PcR0drfcPpapd1aYCUwpqnjbsB/1w7AB8ClwdedmjruBuzrfwIPB3/vCHwFnNvUsdexv9cAvYG/1rC92b0/RfMIoR/wNzPbY2bHgIXAqFPajAKet4CNQAfnXEJjB1pPzthfM9tgZoeCLzcCSY0cY30J598W4N+Al4HPGzO4ehZOX28HlpjZPgAzi9b+htNXAy5wzjmgPYGEUNG4YdYPM/sLgfhr0uzen6I5ISQC+6u8Lg2ui7RNtIi0L5MIfPqIRmfsq3MuEfgx8F+NGFdDCOff9Z+Bi5xz65xzRc65CY0WXf0Kp695QArwMbAN+HczO9k44TW6Zvf+FNOUBz9Lrpp1pz5DG06baBF2X5xzQwgkhKsbNKKGE05fHwemmdmJwIfJqBVOX2OAK4Frge8B7zjnNprZroYOrp6F09cfAsXAUKAb8Jpzbr2ZfdPAsTWFZvf+FM0JoRToUuV1EoFPFZG2iRZh9cU5lw7MBUaY2cFGiq2+hdPXPsDCYDK4GPiRc67CzJY2SoT1J9z/jr80s6PAUefcX4AMINoSQjh9vROYbYGL7H9zzu0FrgA2N06IjarZvT9F8yWjAuBy51yyc+5c4DZg+SltlgMTgnfzBwBfm9knjR1oPTljf51zlwJLgH+Nwk+PVZ2xr2aWbGY+M/MBi4EpUZgMILz/jpcBP3DOxTjnzgP6A9sbOc76EE5f9xEYCeGciwd6AHsaNcrG0+zen6J2hGBmFc65/wP8mcDTC38ws/edc/cEt/8XgadPfgT8DfiWwKePqBRmfx8Avg88FfzkXGHNdFbF2oTZ1xYhnL6a2Xbn3KvAVuAkMNfMqn2UsTkL89/1N8A859w2ApdUpplZVE6L7ZzLB7KAi51zpcAM4Bxovu9PmrpCRESA6L5kJCIi9UgJQUREACUEEREJUkIQERFACUFERIKUECTqOOd8Nc0g2YDHXOecq/UR3oaKKzjT6cAqr+dF8+yu0nwpIUiL5Zxr29Qx1JMsAjPZijQoJQSJVjHOufnBeeQXB7/Bi3OuxDn3gHPuLeAW59zdzrmC4Pz6L1dpNy84F/0G59yeqp+4nXO5zrltwb+ZXeWYtzjnNjvndjnnflBbcM65ts65R4PH3uqc+2lwfVZwtLHYObfDObcgOLMnzrkfBde9FYztFeecD7gHyHGBegiVx72muthFzoYSgkSrHsCzZpYOfANMqbLt72Z2tZktJDBtdF8zyyAw3cOkKu0SCEwAeCMwG8A5NwIYDfQP/s0jVdrHmFk/4OcEvnVam0kEpiLoC/QF7nbOJQe3ZQb30ZNAbYBBzrl/Ap4hMAfV1QRqAWBmJQRmdP2tmfnNbH1NsYucLSUEiVb7zezt4O8vEjqz6x+r/J7mnFsfnAphHJBaZdtSMztpZh8A8cF1w4D/MbNvAcys6nz2S4I/iwDfGeIbTmCemmJgE4EpRS4PbttsZqXBaZ2Lg/u6AthjZnuDbfLPsP/qYhc5K1E7l5G0eqfOuVL19dEqv88DRpvZFufcHQSux1cqr/K7q/KzpvlcKtuf4Mz/7zjg38zszyErncs65biV+4p0Du/qYhc5KxohSLS61Dl3VfD3nwBv1dDuAuAT59w5BEYIZ7IamFjlXkNcHeP7M/Cz4HFxzv2zc+78WtrvALoG7xkAjK2y7TCBfog0KCUEiVbbgWzn3FYgDni6hna/InDJ5jUCb7q1MrNXCUxLXBi83PMfdYxvLoE6yO8GH0V9hlpGFWb2HYH7IK8Gb4h/Bnwd3Pwn4Men3FQWqXea7VSkmXDOtTezI8Gnjp4EdpvZb5s6Lmk9NEIQaT7uDo5K3gdiCYwqRBqNRggiIgJohCAiIkFKCCIiAighiIhIkBKCiIgASggiIhL0/wFztBOggzWQGgAAAABJRU5ErkJggg==\n",
      "text/plain": [
       "<Figure size 432x288 with 1 Axes>"
      ]
     },
     "metadata": {
      "needs_background": "light"
     },
     "output_type": "display_data"
    }
   ],
   "source": [
    "# Escriba aquí su código para el ejercicio 6\n",
    "from Bio import Entrez \n",
    "from Bio import SeqIO\n",
    "from Bio import SeqIO\n",
    "from Bio import AlignIO\n",
    "from Bio import Phylo\n",
    "from Bio.Align.Applications import ClustalwCommandline\n",
    "import os\n",
    "\n",
    "id_list = [\"MW244756.1,MW244755.1,KX065077.1,KX065076.1,KX065075.1,KX065074.1,KX065073.1,KX065072.1,KX065071.1,KX065070.1\"]\n",
    "Entrez.email = \"nayeli.mamallacta@est.ikiam.edu.ec\"\n",
    "with Entrez.efetch( db=\"nucleotide\", rettype=\"gb\", retmode=\"text\", id= id_list\n",
    "                  ) as handle:\n",
    "    for seq_record in SeqIO.parse(handle, \"gb\"):\n",
    "        print(\"%s %s...\" %(seq_record.id, seq_record.description[:50]))\n",
    "        print( \"Sequence length %i, %i features, from: %s\" % (len(seq_record), \n",
    "                                                              len(seq_record.features), \n",
    "                                                              seq_record.annotations[\"source\"],))\n",
    "\n",
    "        \n",
    "clustalw_exe =r\"C:\\Program Files (x86)\\ClustalW2\\clustalw2.exe\"\n",
    "clustalw_cline = ClustalwCommandline(clustalw_exe, infile = \"data\\Cruziohyla calcarifer.fasta\")\n",
    "assert os.path.isfile(clustalw_exe), \"Clustal_W executable is missing or not found\"\n",
    "stdout, stderr = clustalw_cline()\n",
    "print(clustalw_cline)\n",
    "        \n",
    "ClustalAlign = AlignIO.read(\"data\\Cruziohyla calcarifer.aln\", \"clustal\")\n",
    "print(ClustalAlign)    \n",
    "\n",
    "\n",
    "from Bio import Phylo\n",
    "tree = Phylo.read(\"data\\Cruziohyla calcarifer.dnd\", \"newick\")\n",
    "Phylo.draw_ascii(tree)\n",
    "print(tree)\n",
    "\n",
    "tree.rooted = True\n",
    "Phylo.draw(tree)"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "**Escriba aquí la interpretación del árbol**\n",
    "\n",
    "Como se puede evidenciar en la imagen el árbol filogenético parte desde un ancestro común de la cual se subdividien en algunas ramificaciones donde estas ramificaciones presentan un proceso de diferenciación y especiación que da origen a distintos organismos. También se puede visualizar que existen organismos que están dentro de una misma rama, eso quiere decir que están emparentados filogenéticamente.\n"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "## Ejercicio 7 [1 punto]\n",
    "\n",
    "\n",
    "1. Cree en GitHub un repositorio de nombre ```GBI6_ExamenPython```. \n",
    "2. Cree un archivo ```Readme.md``` que debe tener lo siguiente: \n",
    "- Datos personales\n",
    "- Características del computador\n",
    "- Versión de Python/Anaconda y de cada uno de los módulos/paquetes y utilizados\n",
    "- Explicación de la data utilizada\n",
    "\n",
    "3. Cree un directorio de nombre ```data``` donde deben estar guardado la data utilizada \n",
    "4. Realice al menos 1 control de la versión (commits) por cada ejercicio (del 1 al 6), con un mensaje que inicie como: \n",
    "\n",
    "```sh \n",
    "Carlitos Alimaña ha realizado .....  \n",
    "```"
   ]
  },
  {
   "cell_type": "markdown",
   "metadata": {},
   "source": [
    "https://github.com/NayeliMG/GBI6_ExamenPython"
   ]
  }
 ],
 "metadata": {
  "kernelspec": {
   "display_name": "Python 3",
   "language": "python",
   "name": "python3"
  },
  "language_info": {
   "codemirror_mode": {
    "name": "ipython",
    "version": 3
   },
   "file_extension": ".py",
   "mimetype": "text/x-python",
   "name": "python",
   "nbconvert_exporter": "python",
   "pygments_lexer": "ipython3",
   "version": "3.8.8"
  },
  "latex_envs": {
   "LaTeX_envs_menu_present": true,
   "autoclose": false,
   "autocomplete": true,
   "bibliofile": "biblio.bib",
   "cite_by": "apalike",
   "current_citInitial": 1,
   "eqLabelWithNumbers": true,
   "eqNumInitial": 1,
   "hotkeys": {
    "equation": "Ctrl-E",
    "itemize": "Ctrl-I"
   },
   "labels_anchors": false,
   "latex_user_defs": false,
   "report_style_numbering": false,
   "user_envs_cfg": false
  },
  "toc": {
   "base_numbering": 1,
   "nav_menu": {},
   "number_sections": true,
   "sideBar": false,
   "skip_h1_title": false,
   "title_cell": "Table of Contents",
   "title_sidebar": "Contents",
   "toc_cell": false,
   "toc_position": {
    "height": "1152.67px",
    "left": "41px",
    "top": "110px",
    "width": "253.458px"
   },
   "toc_section_display": true,
   "toc_window_display": false
  },
  "varInspector": {
   "cols": {
    "lenName": 16,
    "lenType": 16,
    "lenVar": 40
   },
   "kernels_config": {
    "python": {
     "delete_cmd_postfix": "",
     "delete_cmd_prefix": "del ",
     "library": "var_list.py",
     "varRefreshCmd": "print(var_dic_list())"
    },
    "r": {
     "delete_cmd_postfix": ") ",
     "delete_cmd_prefix": "rm(",
     "library": "var_list.r",
     "varRefreshCmd": "cat(var_dic_list()) "
    }
   },
   "types_to_exclude": [
    "module",
    "function",
    "builtin_function_or_method",
    "instance",
    "_Feature"
   ],
   "window_display": false
  }
 },
 "nbformat": 4,
 "nbformat_minor": 4
}
